<!doctype html>
<html
  itemscope
  class=""
  lang="en-us"
  itemtype="http://schema.org/WebPage">
  <head>
    
    <meta charset="utf-8" />


<meta
  name="viewport"
  content="width=device-width, initial-scale=1, maximum-scale=5" />


<meta name="theme-name" content="hugoplate" />















<link rel="shortcut icon" href="/images/fpg_logo_hu8359c659081f4f0de026e43505d3fe2b_71406_96x0_resize_lanczos_3.png" type="image/x-icon">
<link rel="icon" href="/images/fpg_logo_hu8359c659081f4f0de026e43505d3fe2b_71406_96x0_resize_lanczos_3.png" type="image/x-icon">
<link rel="icon" type="image/png" sizes="48x48" href="/images/fpg_logo_hu8359c659081f4f0de026e43505d3fe2b_71406_48x0_resize_lanczos_3.png">
<link rel="icon" type="image/png" sizes="96x96" href="/images/fpg_logo_hu8359c659081f4f0de026e43505d3fe2b_71406_96x0_resize_lanczos_3.png">
<link rel="apple-touch-icon" sizes="144x144" href="/images/fpg_logo_hu8359c659081f4f0de026e43505d3fe2b_71406_144x0_resize_lanczos_3.png">








<link rel="manifest" href="/manifest.webmanifest" />
<meta
  name="msapplication-TileColor"
  content="#ddd" />
<meta
  name="theme-color"
  content="#ffffff" />





























  


<title>AT4G19440</title>















  <meta
    name="keywords"
    content="Boilerplate, Hugo" />




<meta
  name="description"
  content="AT4G19440 ; MTSF4 ; Arabidopsis thaliana" />



  <meta name="author" content="zeon.studio" />






  
  












  

  
  
  


  
  


  





<meta property="og:title" content="AT4G19440" />
<meta property="og:description" content="AT4G19440 ; MTSF4 ; Arabidopsis thaliana" />
<meta property="og:type" content="website" />
<meta property="og:url" content="/genes/arabidopsis_thaliana/at4g19440/" />


<meta name="twitter:title" content="AT4G19440" />
<meta name="twitter:description" content="AT4G19440 ; MTSF4 ; Arabidopsis thaliana" />





















<script>
  let indexURL = "/searchindex.json";
  let includeSectionsInSearch = ["genes","pages","sections","submit"];
  let no_results_for = "No results for";
  let empty_search_results_placeholder = "Type something to search..";
</script>




















    
    
<meta http-equiv="x-dns-prefetch-control" content="on" />
<link rel="preconnect" href="https://use.fontawesome.com" crossorigin />
<link rel="preconnect" href="//cdnjs.cloudflare.com" />
<link rel="preconnect" href="//www.googletagmanager.com" />
<link rel="preconnect" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="https://use.fontawesome.com" />
<link rel="dns-prefetch" href="//ajax.googleapis.com" />
<link rel="dns-prefetch" href="//cdnjs.cloudflare.com" />
<link rel="dns-prefetch" href="//www.googletagmanager.com" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//connect.facebook.net" />
<link rel="dns-prefetch" href="//platform.linkedin.com" />
<link rel="dns-prefetch" href="//platform.twitter.com" />




<link rel="preconnect" href="https://fonts.googleapis.com" />
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
<script>
  (function () {
    const googleFont = document.createElement("link");
    googleFont.href = "https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Montserrat:wght@500;700&display=swap";
    googleFont.type = "text/css";
    googleFont.rel = "stylesheet";
    document.head.appendChild(googleFont);
  })();
</script>





  
    
      
    
  

  
    
      
    
  

  
    
      
    
  

  
    
      
    
  

  
    
      
    
  











  
  



<link
  href="/css/style.min.ad82ade437df81f5c5a48af63c5d6c5da442a19f83f854c3b4168220f83e4f5f.css"
  integrity="sha256-rYKt5DffgfXFpIr2PF1sXaRCoZ+D+FTDtBaCIPg+T18="
  rel="stylesheet" />


<link
  defer
  async
  rel="stylesheet"
  href="/css/style-lazy.min.901e5cbd6a6883c52a645fd5c91bb3fec340173bc306529fe6cb027c818eb6bf.css"
  integrity="sha256-kB5cvWpog8UqZF/VyRuz/sNAFzvDBlKf5ssCfIGOtr8="
  media="print"
  onload="this.media='all'; this.onload=null;" />

  </head>

  <body style="padding-top: 2px; padding-bottom: 2px;">
    
    
      



      

    
  
    
    <header
  class="header sticky top-0 z-30">
  <nav class="navbar container">
    
    <div class="order-0">
      
      <a class="navbar-brand block" href="/">
        






















  
  funPlantGenes


      </a>
    </div>
    
    <input id="nav-toggle" type="checkbox" class="hidden" />
    <label
      for="nav-toggle"
      class="order-3 cursor-pointer flex items-center lg:hidden text-dark dark:text-white lg:order-1">
      <svg id="show-button" class="h-6 fill-current block" viewBox="0 0 20 20">
        <title>Menu Open</title>
        <path d="M0 3h20v2H0V3z m0 6h20v2H0V9z m0 6h20v2H0V0z"></path>
      </svg>
      <svg id="hide-button" class="h-6 fill-current hidden" viewBox="0 0 20 20">
        <title>Menu Close</title>
        <polygon
          points="11 9 22 9 22 11 11 11 11 22 9 22 9 11 -2 11 -2 9 9 9 9 -2 11 -2"
          transform="rotate(45 10 10)"></polygon>
      </svg>
    </label>
    

    
    <ul
      id="nav-menu"
      class="navbar-nav order-3 hidden lg:flex w-full pb-6 lg:order-1 lg:w-auto lg:space-x-2 lg:pb-0 xl:space-x-8">
      
      
        
        
        
        
          <li class="nav-item">
            <a
              class="nav-link "
              
              href="/species/"
              >SPECIES</a
            >
          </li>
        
      
        
        
        
        
          <li class="nav-item">
            <a
              class="nav-link "
              
              href="/genes/"
              >GENES</a
            >
          </li>
        
      
        
        
        
        
          <li class="nav-item">
            <a
              class="nav-link "
              
              href="/tags/"
              >FUNCTION</a
            >
          </li>
        
      
        
        
        
        
          <li class="nav-item nav-dropdown group relative">
            <span
              class="nav-link 
                
                
                
              
                
                
                
               inline-flex items-center">
              HELP
              <svg class="h-4 w-4 fill-current" viewBox="0 0 20 20">
                <path
                  d="M9.293 12.95l.707.707L15.657 8l-1.414-1.414L10 10.828 5.757 6.586 4.343 8z" />
              </svg>
            </span>
            <ul
              class="nav-dropdown-list lg:group-hover:visible lg:group-hover:opacity-100">
              
                
                
                <li class="nav-dropdown-item">
                  <a
                    class="nav-dropdown-link "
                    
                    href="/link/">
                    LINKS
                  </a>
                </li>
              
                
                
                <li class="nav-dropdown-item">
                  <a
                    class="nav-dropdown-link "
                    
                    href="/submit/">
                    SUBMIT
                  </a>
                </li>
              
            </ul>
          </li>
        
      
      
        <li class="mt-4 inline-block lg:hidden">
          <a
            class="btn btn-outline-primary btn-sm"
            href="/submit">
            Submit
          </a>
        </li>
      
    </ul>

    
     

  <div class="order-1 ml-auto flex items-center md:order-2 lg:ml-0" style="margin-right: 20px;">

    <input
  id="search-modal-input"
  type="text"
  placeholder="Search"
  autocomplete="on"
  aria-label="Search"
  aria-describedby="Search"
  placeholder="Search Post ..."
  style="width: 120px; /* 设置宽度为800像素 */
         padding: 5px;
         border: 2px solid #ccc;
         border-radius: 20px;
         box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
         outline: none;
         transition: border-color 0.3s ease-in-out;
         margin-right: 20px; /* 添加右边距 */"/>
    <button
        id="search-button"
        aria-label="search"
        class="border-border text-dark hover:text-primary dark:border-darkmode-border mr-5 inline-block border-r pr-5 text-xl dark:text-white dark:hover:text-darkmode-primary"
        data-target="search-modal"
        onclick="search()">
        <i class="fa-solid fa-search"></i>
    </button>

   
   </div>

    <div class="order-1 ml-auto flex items-center md:order-2 lg:ml-0" style="margin-right: 20px;"></div>
          









          


  <div class="theme-switcher mr-5">
    <input id="theme-switcher" data-theme-switcher type="checkbox" />
    <label for="theme-switcher">
      <span class="sr-only">theme switcher</span>
      <span>
        
        <svg
          class="absolute left-[4px] top-[4px] z-10 opacity-100 dark:opacity-0"
          viewBox="0 0 56 56"
          fill="#fff"
          height="16"
          width="16">
          <path
            d="M30 4.6c0-1-.9-2-2-2a2 2 0 0 0-2 2v5c0 1 .9 2 2 2s2-1 2-2Zm9.6 9a2 2 0 0 0 0 2.8c.8.8 2 .8 2.9 0L46 13a2 2 0 0 0 0-2.9 2 2 0 0 0-3 0Zm-26 2.8c.7.8 2 .8 2.8 0 .8-.7.8-2 0-2.9L13 10c-.7-.7-2-.8-2.9 0-.7.8-.7 2.1 0 3ZM28 16a12 12 0 0 0-12 12 12 12 0 0 0 12 12 12 12 0 0 0 12-12 12 12 0 0 0-12-12Zm23.3 14c1.1 0 2-.9 2-2s-.9-2-2-2h-4.9a2 2 0 0 0-2 2c0 1.1 1 2 2 2ZM4.7 26a2 2 0 0 0-2 2c0 1.1.9 2 2 2h4.9c1 0 2-.9 2-2s-1-2-2-2Zm37.8 13.6a2 2 0 0 0-3 0 2 2 0 0 0 0 2.9l3.6 3.5a2 2 0 0 0 2.9 0c.8-.8.8-2.1 0-3ZM10 43.1a2 2 0 0 0 0 2.9c.8.7 2.1.8 3 0l3.4-3.5c.8-.8.8-2.1 0-2.9-.8-.8-2-.8-2.9 0Zm20 3.4c0-1.1-.9-2-2-2a2 2 0 0 0-2 2v4.9c0 1 .9 2 2 2s2-1 2-2Z" />
        </svg>
        
        <svg
          class="absolute left-[4px] top-[4px] z-10 opacity-0 dark:opacity-100"
          viewBox="0 0 24 24"
          fill="none"
          height="16"
          width="16">
          <path
            fill="#000"
            fill-rule="evenodd"
            clip-rule="evenodd"
            d="M8.2 2.2c1-.4 2 .6 1.6 1.5-1 3-.4 6.4 1.8 8.7a8.4 8.4 0 0 0 8.7 1.8c1-.3 2 .5 1.5 1.5v.1a10.3 10.3 0 0 1-9.4 6.2A10.3 10.3 0 0 1 3.2 6.7c1-2 2.9-3.5 4.9-4.4Z" />
        </svg>
      </span>
    </label>
  </div>

  
  <script>
    var darkMode = false;

    
    if (window.matchMedia("(prefers-color-scheme: dark)").matches){darkMode = true}
    

    if (localStorage.getItem("theme") === "dark"){darkMode = true}
    else if (localStorage.getItem("theme") === "light"){darkMode = false}
    if (darkMode){document.documentElement.classList.toggle("dark")}
    var themeSwitch = document.querySelectorAll("[data-theme-switcher]");

    document.addEventListener("DOMContentLoaded", () => {
      [].forEach.call(themeSwitch, function (ts) {
        ts.checked = darkMode ? true : false;
        ts.addEventListener("click", () => {
          document.documentElement.classList.toggle("dark");
          localStorage.setItem(
            "theme",
            document.documentElement.classList.contains("dark") ? "dark" : "light"
          );
        });
      });
    });
  </script>


    </div>

  </nav>
       
</header>

    




  
    <div
      class="search-modal "
      aria-hidden="true"
      style="--color-primary: #121212">
      <div data-target="close-search-modal" class="search-modal-overlay"></div>
      <div
        class="search-wrapper"
        #data-image="true"
        data-description="true"
        data-tags="true"
        data-categories="true">
        <div class="search-wrapper-header">
          <label for="search-modal-input" style="margin-top:-1px">
            <span class="sr-only">search icon</span>
          </label>
        </div>   
        <div class="search-wrapper-body">
          <div class="search-result" data-search-result></div>
          <span class="search-result-empty">
            Type something to search..
          </span>
        </div>
        <div class="search-wrapper-footer">
          <span>
            <kbd>
              <svg
                xmlns="http://www.w3.org/2000/svg"
                width="14"
                height="14"
                fill="currentColor"
                viewBox="0 0 16 16">
                <path
                  d="M3.204 11h9.592L8 5.519 3.204 11zm-.753-.659 4.796-5.48a1 1 0 0 1 1.506 0l4.796 5.48c.566.647.106 1.659-.753 1.659H3.204a1 1 0 0 1-.753-1.659z" />
              </svg>
            </kbd>
            <kbd>
              <svg
                xmlns="http://www.w3.org/2000/svg"
                width="14"
                height="14"
                fill="currentColor"
                style="margin-top:1px;"
                viewBox="0 0 16 16">
                <path
                  d="M3.204 5h9.592L8 10.481 3.204 5zm-.753.659 4.796 5.48a1 1 0 0 0 1.506 0l4.796-5.48c.566-.647.106-1.659-.753-1.659H3.204a1 1 0 0 0-.753 1.659z" />
              </svg>
            </kbd>
            to navigate
          </span>
          <span>
            <kbd>
              <svg
                xmlns="http://www.w3.org/2000/svg"
                width="12"
                height="12"
                fill="currentColor"
                style="display:inline-block;"
                viewBox="0 0 16 16">
                <path
                  fill-rule="evenodd"
                  d="M14.5 1.5a.5.5 0 0 1 .5.5v4.8a2.5 2.5 0 0 1-2.5 2.5H2.707l3.347 3.346a.5.5 0 0 1-.708.708l-4.2-4.2a.5.5 0 0 1 0-.708l4-4a.5.5 0 1 1 .708.708L2.707 8.3H12.5A1.5 1.5 0 0 0 14 6.8V2a.5.5 0 0 1 .5-.5z" />
              </svg>
            </kbd>
            to select
          </span>
          <span class="search-result-info"></span>
          <span data-target="close-search-modal">
            <kbd>ESC</kbd> to close
          </span>
        </div>
      </div>
    </div>
  


  
    <main>
      
  <section class="section-sm" style="padding-top: 15px; padding-bottom: 15px;">
    <div class="container">
      <div class="row justify-center">
        <div class="lg:col-11">
          <div class="content">
            <h2 id="gene-details">Gene Details:</h2>
<ul>
<li><strong>Gene ID:</strong> <a href="https://www.arabidopsis.org/locus?name=AT4G19440"




 target="_blank"
 


>AT4G19440</a>
</li>
<li><strong>Gene Symbol:</strong> <u>MTSF4</u></li>
<li><strong>Gene Name:</strong> MITOCHONDRIAL TRANSCRIPT STABILITY FACTOR 4</li>
<li><strong>Description:</strong>   Tetratricopeptide repeat (TPR)-like superfamily protein;(source:Araport11)</li>
<li><strong>TAIR Accession:</strong> locus:2140371</li>
<li><strong>Genome:</strong> <a href="https://www.arabidopsis.org/download/list?dir=Genes%2FAraport11_genome_release"




 target="_blank"
 


>Araport11_genome_release</a>
</li>
<li><strong>Species:</strong> <em>Arabidopsis thaliana</em></li>
</ul>
<h2 id="transcripts">Transcripts:</h2>
<ul>
<li><a href="https://www.arabidopsis.org/gene?name=AT4G19440.1"




 target="_blank"
 


>AT4G19440.1</a>
    <a href="https://www.arabidopsis.org/gene?name=AT4G19440.2"




 target="_blank"
 


>AT4G19440.2</a>
    <a href="https://www.arabidopsis.org/gene?name=AT4G19440.3"




 target="_blank"
 


>AT4G19440.3</a>
    <a href="https://www.arabidopsis.org/gene?name=AT4G19440.4"




 target="_blank"
 


>AT4G19440.4</a>
</li>
</ul>
<h2 id="plant-ontology-annotations">Plant Ontology Annotations:</h2>
<ul>
<li><a href="https://browser.planteome.org/amigo/term/PO:0009005"




 target="_blank"
 


>PO:0009005</a>
 &mdash; root &mdash; raíz (Spanish, exact), radices (exact, plural), radix (exact), 根 (Japanese, exact), aerial root (narrow), climbing root (narrow)</li>
</ul>
<h2 id="gene-ontology">Gene Ontology:</h2>
<ul>
<li><a href="https://amigo.geneontology.org/amigo/term/GO:0009507"




 target="_blank"
 


>GO:0009507</a>
 &mdash; located in &mdash; chloroplast</li>
<li><a href="https://amigo.geneontology.org/amigo/term/NONE"




 target="_blank"
 


>NONE</a>
 &mdash; involved in &mdash; mitochondrial RNA 3-end processing	GO:0000965	29420	P	other metabolic processes	IMP	biochemical/chemical analysis		Publication:501801625|PMID:36468791  	TAIR	2023-01-26
AT4G19440	gene:6532553040	AT4G19440.4	located in	chloroplast	GO:0009507	175	C	chloroplast	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19440	gene:6532553034	AT4G19440.3	located in	chloroplast	GO:0009507	175	C	chloroplast	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19440	gene:2140370	AT4G19440.1	located in	chloroplast	GO:0009507	175	C	chloroplast	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19450	locus:2140386	AT4G19450	acts upstream of or within	organic cyclic compound biosynthetic process	GO:1901362	43684	P	biosynthetic process	IEA	traceable computational prediction	AGI_LocusCode:AT1G21970|AGI_LocusCode:AT1G77450|AGI_LocusCode:AT3G19290|AGI_LocusCode:AT4G27410|AGI_LocusCode:AT4G34000|AGI_LocusCode:AT5G03150|AGI_LocusCode:AT5G67450|AGI_LocusCode:AT5G62380	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19450	locus:2140386	AT4G19450	acts upstream of or within	carboxylic acid biosynthetic process	GO:0046394	13251	P	biosynthetic process	IEA	traceable computational prediction	AGI_LocusCode:AT1G21970	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19450	locus:2140386	AT4G19450	acts upstream of or within	heterocycle biosynthetic process	GO:0018130	8928	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G21970|AGI_LocusCode:AT1G77450|AGI_LocusCode:AT3G19290|AGI_LocusCode:AT4G27410|AGI_LocusCode:AT4G34000|AGI_LocusCode:AT5G03150|AGI_LocusCode:AT5G67450|AGI_LocusCode:AT5G62380	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19450	locus:2140386	AT4G19450	is active in	membrane	GO:0016020	453	C	other membranes	IBA	none	PANTHER:PTN000491237|UniProtKB:Q582K8|UniProtKB:Q582K9|TAIR:locus:2056103|SGD:S000002212	Communication:501741973		2023-04-13
AT4G19450	locus:2140386	AT4G19450	acts upstream of or within	heterocycle biosynthetic process	GO:0018130	8928	P	biosynthetic process	IEA	traceable computational prediction	AGI_LocusCode:AT1G21970|AGI_LocusCode:AT1G77450|AGI_LocusCode:AT3G19290|AGI_LocusCode:AT4G27410|AGI_LocusCode:AT4G34000|AGI_LocusCode:AT5G03150|AGI_LocusCode:AT5G67450|AGI_LocusCode:AT5G62380	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19450	locus:2140386	AT4G19450	acts upstream of or within	cellular nitrogen compound biosynthetic process	GO:0044271	19681	P	biosynthetic process	IEA	traceable computational prediction	AGI_LocusCode:AT1G21970|AGI_LocusCode:AT1G77450|AGI_LocusCode:AT3G19290|AGI_LocusCode:AT4G27410|AGI_LocusCode:AT4G34000|AGI_LocusCode:AT5G03150|AGI_LocusCode:AT5G67450|AGI_LocusCode:AT5G62380	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19450	gene:6532553248	AT4G19450.2	located in	mitochondrion	GO:0005739	486	C	mitochondrion	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19450	locus:2140386	AT4G19450	acts upstream of or within	organic cyclic compound biosynthetic process	GO:1901362	43684	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G21970|AGI_LocusCode:AT1G77450|AGI_LocusCode:AT3G19290|AGI_LocusCode:AT4G27410|AGI_LocusCode:AT4G34000|AGI_LocusCode:AT5G03150|AGI_LocusCode:AT5G67450|AGI_LocusCode:AT5G62380	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19450	locus:2140386	AT4G19450	acts upstream of or within	carboxylic acid biosynthetic process	GO:0046394	13251	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G21970	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19450	locus:2140386	AT4G19450	acts upstream of or within	carboxylic acid biosynthetic process	GO:0046394	13251	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G21970	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19450	locus:2140386	AT4G19450	acts upstream of or within	response to light stimulus	GO:0009416	6176	P	response to light stimulus	IEA	traceable computational prediction	AGI_LocusCode:AT1G21970|AGI_LocusCode:AT4G01120|AGI_LocusCode:AT5G47640	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19450	locus:2140386	AT4G19450	acts upstream of or within	cellular nitrogen compound biosynthetic process	GO:0044271	19681	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G21970|AGI_LocusCode:AT1G77450|AGI_LocusCode:AT3G19290|AGI_LocusCode:AT4G27410|AGI_LocusCode:AT4G34000|AGI_LocusCode:AT5G03150|AGI_LocusCode:AT5G67450|AGI_LocusCode:AT5G62380	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19450	locus:2140386	AT4G19450	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	Unknown molecular function		Communication:1345790	TAIR	2009-05-08
AT4G19450	locus:2140386	AT4G19450	acts upstream of or within	cellular nitrogen compound biosynthetic process	GO:0044271	19681	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G21970|AGI_LocusCode:AT1G77450|AGI_LocusCode:AT3G19290|AGI_LocusCode:AT4G27410|AGI_LocusCode:AT4G34000|AGI_LocusCode:AT5G03150|AGI_LocusCode:AT5G67450|AGI_LocusCode:AT5G62380	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19450	locus:2140386	AT4G19450	acts upstream of or within	aromatic compound biosynthetic process	GO:0019438	10128	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G21970|AGI_LocusCode:AT1G77450|AGI_LocusCode:AT3G19290|AGI_LocusCode:AT4G27410|AGI_LocusCode:AT4G34000|AGI_LocusCode:AT5G03150|AGI_LocusCode:AT5G67450|AGI_LocusCode:AT5G62380	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19450	locus:2140386	AT4G19450	acts upstream of or within	heterocycle biosynthetic process	GO:0018130	8928	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G21970|AGI_LocusCode:AT1G77450|AGI_LocusCode:AT3G19290|AGI_LocusCode:AT4G27410|AGI_LocusCode:AT4G34000|AGI_LocusCode:AT5G03150|AGI_LocusCode:AT5G67450|AGI_LocusCode:AT5G62380	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19450	locus:2140386	AT4G19450	acts upstream of or within	response to lipid	GO:0033993	28865	P	response to chemical	IEA	traceable computational prediction	AGI_LocusCode:AT1G21970|AGI_LocusCode:AT2G46270|AGI_LocusCode:AT3G19290|AGI_LocusCode:AT4G27410|AGI_LocusCode:AT4G34000|AGI_LocusCode:AT5G62380	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19450	locus:2140386	AT4G19450	acts upstream of or within	aromatic compound biosynthetic process	GO:0019438	10128	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G21970|AGI_LocusCode:AT1G77450|AGI_LocusCode:AT3G19290|AGI_LocusCode:AT4G27410|AGI_LocusCode:AT4G34000|AGI_LocusCode:AT5G03150|AGI_LocusCode:AT5G67450|AGI_LocusCode:AT5G62380	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19450	locus:2140386	AT4G19450	acts upstream of or within	aromatic compound biosynthetic process	GO:0019438	10128	P	biosynthetic process	IEA	traceable computational prediction	AGI_LocusCode:AT1G21970|AGI_LocusCode:AT1G77450|AGI_LocusCode:AT3G19290|AGI_LocusCode:AT4G27410|AGI_LocusCode:AT4G34000|AGI_LocusCode:AT5G03150|AGI_LocusCode:AT5G67450|AGI_LocusCode:AT5G62380	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19460	locus:2122920	AT4G19460	acts upstream of or within	defense response to fungus	GO:0050832	18013	P	response to external stimulus	IEA	traceable computational prediction	AGI_LocusCode:AT1G80840|AGI_LocusCode:AT2G38470	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19460	gene:3439984	AT4G19460.1	located in	chloroplast	GO:0009507	175	C	chloroplast	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19460	locus:2122920	AT4G19460	acts upstream of or within	defense response to fungus	GO:0050832	18013	P	response to biotic stimulus	IEA	traceable computational prediction	AGI_LocusCode:AT1G80840|AGI_LocusCode:AT2G38470	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19460	locus:2122920	AT4G19460	enables	glycosyltransferase activity	GO:0016757	14916	F	transferase activity	ISS	Recognized domains		Communication:1674999	TAIR	2004-04-21
AT4G19460	locus:2122920	AT4G19460	acts upstream of or within	defense response to fungus	GO:0050832	18013	P	response to stress	IEA	traceable computational prediction	AGI_LocusCode:AT1G80840|AGI_LocusCode:AT2G38470	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19460	locus:2122920	AT4G19460	acts upstream of or within	root development	GO:0048364	18902	P	anatomical structure development	IEA	traceable computational prediction	AGI_LocusCode:AT3G16857|AGI_LocusCode:AT4G31920	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19460	locus:2122920	AT4G19460	acts upstream of or within	root development	GO:0048364	18902	P	multicellular organism development	IEA	traceable computational prediction	AGI_LocusCode:AT3G16857|AGI_LocusCode:AT4G31920	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19470	gene:2122924	AT4G19470.1	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19480	locus:2122935	AT4G19480	involved in	biological_process	GO:0008150	5239	P	unknown biological processes	ND	Unknown biological process	NONE	Communication:1345790	TAIR	2022-02-01
AT4G19480	locus:2122935	AT4G19480	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	none	NONE	Communication:1345790	TIGR	2003-10-04
AT4G19480	gene:2122934	AT4G19480.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19485	locus:6532567266	AT4G19485	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	Unknown molecular function		Communication:1345790	TAIR	2019-09-24
AT4G19485	locus:6532567266	AT4G19485	involved in	biological_process	GO:0008150	5239	P	unknown biological processes	ND	Unknown biological process	NONE	Communication:1345790	TAIR	2022-02-01
AT4G19485	locus:6532567266	AT4G19485	is active in	cellular_component	GO:0005575	163	C	unknown cellular components	ND	Unknown cellular component	NONE	Communication:1345790	TAIR	2022-02-01
AT4G19490	locus:2122945	AT4G19490	part of	GARP complex	GO:0000938	15012	C	other intracellular components	IDA	none		Publication:501742163|PMID:21398432  		2021-04-01
AT4G19490	locus:2122945	AT4G19490	part of	GARP complex	GO:0000938	15012	C	Golgi apparatus	IDA	none		Publication:501742163|PMID:21398432  		2021-04-01
AT4G19490	locus:2122945	AT4G19490	located in	Golgi membrane	GO:0000139	54	C	Golgi apparatus	IDA	none		Publication:501742163|PMID:21398432  		2020-04-11
AT4G19490	locus:2122945	AT4G19490	located in	Golgi membrane	GO:0000139	54	C	other intracellular components	IDA	none		Publication:501742163|PMID:21398432  		2020-04-11
AT4G19490	gene:1005714184	AT4G19490.2	located in	Golgi apparatus	GO:0005794	48	C	Golgi apparatus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19490	gene:2122944	AT4G19490.1	located in	Golgi apparatus	GO:0005794	48	C	Golgi apparatus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19490	locus:2122945	AT4G19490	enables	syntaxin binding	GO:0019905	9745	F	protein binding	IBA	none	PANTHER:PTN000311465|WB:WBGene00006934	Communication:501741973		2021-04-02
AT4G19490	locus:2122945	AT4G19490	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:Q0WQF4	Publication:501742163|PMID:21398432  		2021-04-01
AT4G19490	locus:2122945	AT4G19490	part of	GARP complex	GO:0000938	15012	C	cytoplasm	IBA	none	PANTHER:PTN000311465|MGI:MGI:2178798|WB:WBGene00006934|SGD:S000002434|FB:FBgn0011232|TAIR:locus:2122945|UniProtKB:Q9P1Q0	Communication:501741973		2023-06-07
AT4G19490	locus:2122945	AT4G19490	part of	GARP complex	GO:0000938	15012	C	other intracellular components	IBA	none	PANTHER:PTN000311465|MGI:MGI:2178798|WB:WBGene00006934|SGD:S000002434|FB:FBgn0011232|TAIR:locus:2122945|UniProtKB:Q9P1Q0	Communication:501741973		2023-06-07
AT4G19490	locus:2122945	AT4G19490	located in	Golgi membrane	GO:0000139	54	C	other membranes	IDA	none		Publication:501742163|PMID:21398432  		2020-04-11
AT4G19490	locus:2122945	AT4G19490	part of	GARP complex	GO:0000938	15012	C	Golgi apparatus	IBA	none	PANTHER:PTN000311465|MGI:MGI:2178798|WB:WBGene00006934|SGD:S000002434|FB:FBgn0011232|TAIR:locus:2122945|UniProtKB:Q9P1Q0	Communication:501741973		2023-06-07
AT4G19490	locus:2122945	AT4G19490	involved in	retrograde transport, endosome to Golgi	GO:0042147	10282	P	other cellular processes	IBA	none	PANTHER:PTN000311465|SGD:S000002434|UniProtKB:Q9P1Q0	Communication:501741973		2022-11-23
AT4G19490	locus:2122945	AT4G19490	part of	GARP complex	GO:0000938	15012	C	cytoplasm	IDA	none		Publication:501742163|PMID:21398432  		2021-04-01
AT4G19490	locus:2122945	AT4G19490	involved in	retrograde transport, endosome to Golgi	GO:0042147	10282	P	transport	IBA	none	PANTHER:PTN000311465|SGD:S000002434|UniProtKB:Q9P1Q0	Communication:501741973		2022-11-23
AT4G19490	locus:2122945	AT4G19490	involved in	Golgi to vacuole transport	GO:0006896	4808	P	other cellular processes	IBA	none	PANTHER:PTN000311465|SGD:S000002434	Communication:501741973		2021-03-31
AT4G19490	locus:2122945	AT4G19490	involved in	Golgi to vacuole transport	GO:0006896	4808	P	transport	IBA	none	PANTHER:PTN000311465|SGD:S000002434	Communication:501741973		2021-03-31
AT4G19500	locus:2122955	AT4G19500	acts upstream of or within	positive regulation of macromolecule metabolic process	GO:0010604	29564	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G56170|AGI_LocusCode:AT3G19290|AGI_LocusCode:AT4G34000	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19500	locus:2122955	AT4G19500	involved in	signal transduction	GO:0007165	7243	P	signal transduction	IEA	none	InterPro:IPR000157	AnalysisReference:501756966		2023-05-23
AT4G19500	locus:2122955	AT4G19500	acts upstream of or within	vegetative to reproductive phase transition of meristem	GO:0010228	19673	P	reproduction	IEA	traceable computational prediction	AGI_LocusCode:AT5G47640	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19500	locus:2122955	AT4G19500	acts upstream of or within	cellular catabolic process	GO:0044248	19683	P	catabolic process	IEA	traceable computational prediction	AGI_LocusCode:AT3G19290	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19500	gene:6530297453	AT4G19500.2	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19500	locus:2122955	AT4G19500	enables	NAD(P)+ nucleosidase activity	GO:0050135	16367	F	hydrolase activity	IEA	none	EC:3.2.2.6	AnalysisReference:501756967		2023-05-23
AT4G19500	locus:2122955	AT4G19500	acts upstream of or within	cellular catabolic process	GO:0044248	19683	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT3G19290	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19500	locus:2122955	AT4G19500	acts upstream of or within	defense response to other organism	GO:0098542	46569	P	response to external stimulus	IEA	traceable computational prediction	AGI_LocusCode:AT5G67300	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19500	locus:2122955	AT4G19500	acts upstream of or within	regulation of developmental process	GO:0050793	17954	P	other biological processes	IEA	traceable computational prediction	AGI_LocusCode:AT3G19290	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19500	locus:2122955	AT4G19500	acts upstream of or within	flower development	GO:0009908	11347	P	flower development	IEA	traceable computational prediction	AGI_LocusCode:AT5G16560	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19500	locus:2122955	AT4G19500	acts upstream of or within	positive regulation of nitrogen compound metabolic process	GO:0051173	19381	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G56170|AGI_LocusCode:AT3G19290|AGI_LocusCode:AT4G34000	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19500	locus:2122955	AT4G19500	involved in	defense response	GO:0006952	5542	P	response to stress	IEA	none	InterPro:IPR044974	AnalysisReference:501756966		2023-05-23
AT4G19500	locus:2122955	AT4G19500	enables	ADP binding	GO:0043531	22291	F	nucleotide binding	IEA	none	InterPro:IPR002182	AnalysisReference:501756966		2023-05-23
AT4G19500	locus:2122955	AT4G19500	enables	NAD+ nucleotidase, cyclic ADP-ribose generating	GO:0061809	53245	F	hydrolase activity	IEA	none	EC:3.2.2.6	AnalysisReference:501756967		2023-05-23
AT4G19500	locus:2122955	AT4G19500	acts upstream of or within	defense response to other organism	GO:0098542	46569	P	response to stress	IEA	traceable computational prediction	AGI_LocusCode:AT5G67300	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19500	locus:2122955	AT4G19500	acts upstream of or within	vegetative to reproductive phase transition of meristem	GO:0010228	19673	P	post-embryonic development	IEA	traceable computational prediction	AGI_LocusCode:AT5G47640	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19500	locus:2122955	AT4G19500	acts upstream of or within	defense response to other organism	GO:0098542	46569	P	response to biotic stimulus	IEA	traceable computational prediction	AGI_LocusCode:AT5G67300	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19500	locus:2122955	AT4G19500	acts upstream of or within	vegetative to reproductive phase transition of meristem	GO:0010228	19673	P	anatomical structure development	IEA	traceable computational prediction	AGI_LocusCode:AT5G47640	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19500	locus:2122955	AT4G19500	enables	ADP binding	GO:0043531	22291	F	other binding	IEA	none	InterPro:IPR002182	AnalysisReference:501756966		2023-05-23
AT4G19500	locus:2122955	AT4G19500	acts upstream of or within	regulation of gene expression	GO:0010468	28541	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G56170|AGI_LocusCode:AT3G19290|AGI_LocusCode:AT4G34000|AGI_LocusCode:AT5G67300	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19500	locus:2122955	AT4G19500	acts upstream of or within	cellular catabolic process	GO:0044248	19683	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT3G19290	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19500	locus:2122955	AT4G19500	acts upstream of or within	vegetative to reproductive phase transition of meristem	GO:0010228	19673	P	multicellular organism development	IEA	traceable computational prediction	AGI_LocusCode:AT5G47640	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19510	locus:2122965	AT4G19510	acts upstream of or within	cellular response to oxygen-containing compound	GO:1901701	44644	P	response to chemical	IEA	traceable computational prediction	AGI_LocusCode:AT2G22430	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19510	gene:6532548490	AT4G19510.10	located in	chloroplast	GO:0009507	175	C	chloroplast	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19510	locus:2122965	AT4G19510	acts upstream of or within	response to lipid	GO:0033993	28865	P	response to chemical	IEA	traceable computational prediction	AGI_LocusCode:AT2G22430|AGI_LocusCode:AT5G65310	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19510	locus:2122965	AT4G19510	enables	NAD(P)+ nucleosidase activity	GO:0050135	16367	F	hydrolase activity	IEA	none	EC:3.2.2.6	AnalysisReference:501756967		2023-05-23
AT4G19510	locus:2122965	AT4G19510	enables	ADP binding	GO:0043531	22291	F	nucleotide binding	IEA	none	InterPro:IPR002182	AnalysisReference:501756966		2023-05-23
AT4G19510	gene:6532557116	AT4G19510.9	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19510	locus:2122965	AT4G19510	acts upstream of or within	cellular response to organic substance	GO:0071310	33835	P	response to chemical	IEA	traceable computational prediction	AGI_LocusCode:AT2G22430	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19510	gene:1005714183	AT4G19510.2	located in	chloroplast	GO:0009507	175	C	chloroplast	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19510	locus:2122965	AT4G19510	acts upstream of or within	signal transduction	GO:0007165	7243	P	signal transduction	IEA	traceable computational prediction	AGI_LocusCode:AT2G22430	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19510	locus:2122965	AT4G19510	enables	NAD+ nucleotidase, cyclic ADP-ribose generating	GO:0061809	53245	F	hydrolase activity	IEA	none	EC:3.2.2.6	AnalysisReference:501756967		2023-05-23
AT4G19510	locus:2122965	AT4G19510	acts upstream of or within	cellular response to organic substance	GO:0071310	33835	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT2G22430	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19510	gene:6532550568	AT4G19510.3	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19510	locus:2122965	AT4G19510	involved in	signal transduction	GO:0007165	7243	P	signal transduction	IEA	none	InterPro:IPR000157	AnalysisReference:501756966		2023-05-23
AT4G19510	locus:2122965	AT4G19510	acts upstream of or within	response to inorganic substance	GO:0010035	14767	P	response to chemical	IEA	traceable computational prediction	AGI_LocusCode:AT2G22430|AGI_LocusCode:AT5G47640	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19510	locus:2122965	AT4G19510	enables	ADP binding	GO:0043531	22291	F	other binding	IEA	none	InterPro:IPR002182	AnalysisReference:501756966		2023-05-23
AT4G19510	locus:2122965	AT4G19510	acts upstream of or within	cellular response to oxygen-containing compound	GO:1901701	44644	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT2G22430	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19520	gene:6532545513	AT4G19520.3	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19520	locus:2122980	AT4G19520	involved in	defense response	GO:0006952	5542	P	response to stress	IEA	none	InterPro:IPR044974	AnalysisReference:501756966		2023-05-23
AT4G19520	gene:2122979	AT4G19520.1	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19520	locus:2122980	AT4G19520	involved in	signal transduction	GO:0007165	7243	P	signal transduction	IEA	none	InterPro:IPR000157	AnalysisReference:501756966		2023-05-23
AT4G19520	gene:6532546846	AT4G19520.2	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19520	locus:2122980	AT4G19520	acts upstream of or within	cellular response to hypoxia	GO:0071456	33992	P	response to abiotic stimulus	IEP	Transcript levels (e.g. RT-PCR)	NONE	Publication:501742169|PMID:21398256  	TAIR	2011-05-30
AT4G19520	locus:2122980	AT4G19520	acts upstream of or within	cellular response to hypoxia	GO:0071456	33992	P	response to chemical	IEP	Transcript levels (e.g. RT-PCR)	NONE	Publication:501742169|PMID:21398256  	TAIR	2011-05-30
AT4G19520	locus:2122980	AT4G19520	enables	NAD+ nucleotidase, cyclic ADP-ribose generating	GO:0061809	53245	F	hydrolase activity	IEA	none	EC:3.2.2.6	AnalysisReference:501756967		2023-05-23
AT4G19520	gene:6532545514	AT4G19520.4	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19520	locus:2122980	AT4G19520	acts upstream of or within	cellular response to hypoxia	GO:0071456	33992	P	response to stress	IEP	Transcript levels (e.g. RT-PCR)	NONE	Publication:501742169|PMID:21398256  	TAIR	2011-05-30
AT4G19520	locus:2122980	AT4G19520	enables	NAD(P)+ nucleosidase activity	GO:0050135	16367	F	hydrolase activity	IEA	none	EC:3.2.2.6	AnalysisReference:501756967		2023-05-23
AT4G19520	locus:2122980	AT4G19520	enables	ADP binding	GO:0043531	22291	F	nucleotide binding	IEA	none	InterPro:IPR002182	AnalysisReference:501756966		2023-05-23
AT4G19520	locus:2122980	AT4G19520	enables	ADP binding	GO:0043531	22291	F	other binding	IEA	none	InterPro:IPR002182	AnalysisReference:501756966		2023-05-23
AT4G19520	locus:2122980	AT4G19520	acts upstream of or within	cellular response to hypoxia	GO:0071456	33992	P	other cellular processes	IEP	Transcript levels (e.g. RT-PCR)	NONE	Publication:501742169|PMID:21398256  	TAIR	2011-05-30
AT4G19520	gene:6532545515	AT4G19520.5	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19530	locus:2122985	AT4G19530	acts upstream of or within	innate immune response-activating signaling pathway	GO:0002758	24358	P	response to stress	IDA	protein expression in heterologous system	NONE	Publication:501729591|PMID:19132868  	TAIR	2020-12-18
AT4G19530	locus:2122985	AT4G19530	enables	ADP binding	GO:0043531	22291	F	nucleotide binding	IEA	none	InterPro:IPR002182	AnalysisReference:501756966		2023-05-23
AT4G19530	locus:2122985	AT4G19530	acts upstream of or within	innate immune response-activating signaling pathway	GO:0002758	24358	P	response to biotic stimulus	IDA	protein expression in heterologous system	NONE	Publication:501729591|PMID:19132868  	TAIR	2020-12-18
AT4G19530	locus:2122985	AT4G19530	enables	ADP binding	GO:0043531	22291	F	other binding	IEA	none	InterPro:IPR002182	AnalysisReference:501756966		2023-05-23
AT4G19530	locus:2122985	AT4G19530	acts upstream of or within	innate immune response-activating signaling pathway	GO:0002758	24358	P	signal transduction	IDA	protein expression in heterologous system	NONE	Publication:501729591|PMID:19132868  	TAIR	2020-12-18
AT4G19530	locus:2122985	AT4G19530	acts upstream of or within	defense response	GO:0006952	5542	P	response to stress	IEP	RNA-seq evidence	NONE	Publication:501786735|PMID:31568832  	TAIR	2019-10-04
AT4G19530	gene:2122984	AT4G19530.1	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19530	locus:2122985	AT4G19530	enables	NAD(P)+ nucleosidase activity	GO:0050135	16367	F	hydrolase activity	IEA	none	EC:3.2.2.6	AnalysisReference:501756967		2023-05-23
AT4G19530	locus:2122985	AT4G19530	acts upstream of or within	innate immune response-activating signaling pathway	GO:0002758	24358	P	response to external stimulus	IDA	protein expression in heterologous system	NONE	Publication:501729591|PMID:19132868  	TAIR	2020-12-18
AT4G19530	locus:2122985	AT4G19530	enables	NAD+ nucleotidase, cyclic ADP-ribose generating	GO:0061809	53245	F	hydrolase activity	IEA	none	EC:3.2.2.6	AnalysisReference:501756967		2023-05-23
AT4G19530	gene:6532559558	AT4G19530.2	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19540	locus:2122995	AT4G19540	enables	4 iron, 4 sulfur cluster binding	GO:0051539	21612	F	other binding	IBA	none	PANTHER:PTN002452332|UniProtKB:Q8TB37	Communication:501741973		2021-04-02
AT4G19540	locus:2122995	AT4G19540	involved in	mitochondrial respiratory chain complex I assembly	GO:0032981	26712	P	cellular component organization	IBA	none	PANTHER:PTN000586411|TAIR:locus:2122995|UniProtKB:Q8TB37	Communication:501741973		2023-06-07
AT4G19540	locus:2122995	AT4G19540	is active in	mitochondrion	GO:0005739	486	C	mitochondrion	IBA	none	PANTHER:PTN000586411|TAIR:locus:2122995|UniProtKB:Q8TB37	Communication:501741973		2023-04-13
AT4G19540	locus:2122995	AT4G19540	involved in	mitochondrial respiratory chain complex I assembly	GO:0032981	26712	P	cellular component organization	IMP	none		Publication:501757160|PMID:24179128  		2022-02-08
AT4G19540	locus:2122995	AT4G19540	located in	mitochondrion	GO:0005739	486	C	mitochondrion	IDA	none		Publication:501732766|PMID:18957412  	TAIR	2011-10-07
AT4G19540	locus:2122995	AT4G19540	involved in	iron-sulfur cluster assembly	GO:0016226	6105	P	other metabolic processes	IBA	none	PANTHER:PTN002452332|TAIR:locus:2087148	Communication:501741973		2021-03-31
AT4G19540	locus:2122995	AT4G19540	located in	mitochondrial matrix	GO:0005759	479	C	mitochondrion	IDA	none		Publication:501757160|PMID:24179128  		2022-02-08
AT4G19540	locus:2122995	AT4G19540	involved in	iron-sulfur cluster assembly	GO:0016226	6105	P	cellular component organization	IBA	none	PANTHER:PTN002452332|TAIR:locus:2087148	Communication:501741973		2021-03-31
AT4G19540	gene:2122994	AT4G19540.1	located in	chloroplast	GO:0009507	175	C	chloroplast	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19540	locus:2122995	AT4G19540	located in	mitochondrial matrix	GO:0005759	479	C	other intracellular components	IDA	none		Publication:501757160|PMID:24179128  		2022-02-08
AT4G19540	locus:2122995	AT4G19540	acts upstream of or within positive effect	mitochondrial translation	GO:0032543	27970	P	other metabolic processes	IMP	none		Publication:501757160|PMID:24179128  		2022-02-08
AT4G19540	locus:2122995	AT4G19540	acts upstream of or within positive effect	mitochondrial translation	GO:0032543	27970	P	translation	IMP	none		Publication:501757160|PMID:24179128  		2022-02-08
AT4G19540	locus:2122995	AT4G19540	involved in	iron-sulfur cluster assembly	GO:0016226	6105	P	other cellular processes	IBA	none	PANTHER:PTN002452332|TAIR:locus:2087148	Communication:501741973		2021-03-31
AT4G19550	locus:2122930	AT4G19550	acts upstream of or within	RNA metabolic process	GO:0016070	4899	P	nucleobase-containing compound metabolic process	IEA	traceable computational prediction	AGI_LocusCode:AT1G56170|AGI_LocusCode:AT3G19290|AGI_LocusCode:AT4G34000	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19550	locus:2122930	AT4G19550	acts upstream of or within	small molecule metabolic process	GO:0044281	34292	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT3G19290|AGI_LocusCode:AT4G01120	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19550	gene:2122929	AT4G19550.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19550	locus:2122930	AT4G19550	acts upstream of or within	RNA metabolic process	GO:0016070	4899	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G56170|AGI_LocusCode:AT3G19290|AGI_LocusCode:AT4G34000	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19550	locus:2122930	AT4G19550	acts upstream of or within	macromolecule modification	GO:0043412	21458	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT3G11100	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19560	locus:2122940	AT4G19560	involved in	positive regulation of DNA-templated transcription, elongation	GO:0032786	25789	P	other cellular processes	IBA	none	PANTHER:PTN000002653|UniProtKB:O60563|SGD:S000003543|UniProtKB:O75909|UniProtKB:O60583|FB:FBgn0025455	Communication:501741973		2023-06-07
AT4G19560	locus:2122940	AT4G19560	part of	cyclin/CDK positive transcription elongation factor complex	GO:0008024	709	C	nucleus	IBA	none	PANTHER:PTN000002653|UniProtKB:O60563|PomBase:SPBC530.13|SGD:S000003543|PomBase:SPBC32F12.06|FB:FBgn0025455|UniProtKB:O60583	Communication:501741973		2023-06-07
AT4G19560	gene:2122939	AT4G19560.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19560	gene:6532561847	AT4G19560.2	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19560	locus:2122940	AT4G19560	involved in	regulation of transcription by RNA polymerase II	GO:0006357	7463	P	biosynthetic process	IBA	none	PANTHER:PTN000002458|UniProtKB:O60583|FB:FBgn0004597|UniProtKB:O75909|PomBase:SPBC530.13|UniProtKB:O60563|FB:FBgn0025455|SGD:S000004970	Communication:501741973		2023-06-07
AT4G19560	locus:2122940	AT4G19560	part of	cyclin/CDK positive transcription elongation factor complex	GO:0008024	709	C	nucleoplasm	IBA	none	PANTHER:PTN000002653|UniProtKB:O60563|PomBase:SPBC530.13|SGD:S000003543|PomBase:SPBC32F12.06|FB:FBgn0025455|UniProtKB:O60583	Communication:501741973		2023-06-07
AT4G19560	locus:2122940	AT4G19560	involved in	regulation of transcription by RNA polymerase II	GO:0006357	7463	P	nucleobase-containing compound metabolic process	IEA	none	InterPro:IPR043198	AnalysisReference:501756966		2023-06-11
AT4G19560	locus:2122940	AT4G19560	is active in	nucleus	GO:0005634	537	C	nucleus	IBA	none	PANTHER:PTN000002458|TAIR:locus:2143671|FB:FBgn0004597|MGI:MGI:1921539|MGI:MGI:1328363|PomBase:SPBC530.13|UniProtKB:Q5B399|RGD:620864|MGI:MGI:1913921|WB:WBGene00000508|UniProtKB:Q96S94|FB:FBgn0022936|SGD:S000004970|MGI:MGI:1858199|FB:FBgn0025455|WB:WBGene00009650|UniProtKB:Q9UK58|SGD:S000006229|SGD:S000003543|PomBase:SPBC32F12.06|UniProtKB:P51946|PomBase:SPBC12D12.06|UniProtKB:O75909|UniProtKB:O60583|UniProtKB:O60563|PomBase:SPBP16F5.02	Communication:501741973		2023-06-07
AT4G19560	locus:2122940	AT4G19560	involved in	regulation of transcription by RNA polymerase II	GO:0006357	7463	P	biosynthetic process	IEA	none	InterPro:IPR043198	AnalysisReference:501756966		2023-06-11
AT4G19560	locus:2122940	AT4G19560	involved in	positive regulation of DNA-templated transcription, elongation	GO:0032786	25789	P	other metabolic processes	IBA	none	PANTHER:PTN000002653|UniProtKB:O60563|SGD:S000003543|UniProtKB:O75909|UniProtKB:O60583|FB:FBgn0025455	Communication:501741973		2023-06-07
AT4G19560	locus:2122940	AT4G19560	involved in	regulation of transcription by RNA polymerase II	GO:0006357	7463	P	other cellular processes	IBA	none	PANTHER:PTN000002458|UniProtKB:O60583|FB:FBgn0004597|UniProtKB:O75909|PomBase:SPBC530.13|UniProtKB:O60563|FB:FBgn0025455|SGD:S000004970	Communication:501741973		2023-06-07
AT4G19560	locus:2122940	AT4G19560	enables	cyclin-dependent protein serine/threonine kinase regulator activity	GO:0016538	2025	F	enzyme regulator activity	IEA	none	InterPro:IPR043198	AnalysisReference:501756966		2023-06-11
AT4G19560	locus:2122940	AT4G19560	enables	cyclin-dependent protein serine/threonine kinase activator activity	GO:0061575	46533	F	enzyme regulator activity	IBA	none	PANTHER:PTN000002653|PomBase:SPBC32F12.06|WB:WBGene00000508|PomBase:SPBC530.13|WB:WBGene00009650|UniProtKB:O60563	Communication:501741973		2023-06-07
AT4G19560	locus:2122940	AT4G19560	involved in	positive regulation of DNA-templated transcription, elongation	GO:0032786	25789	P	nucleobase-containing compound metabolic process	IBA	none	PANTHER:PTN000002653|UniProtKB:O60563|SGD:S000003543|UniProtKB:O75909|UniProtKB:O60583|FB:FBgn0025455	Communication:501741973		2023-06-07
AT4G19560	locus:2122940	AT4G19560	involved in	regulation of transcription by RNA polymerase II	GO:0006357	7463	P	other metabolic processes	IBA	none	PANTHER:PTN000002458|UniProtKB:O60583|FB:FBgn0004597|UniProtKB:O75909|PomBase:SPBC530.13|UniProtKB:O60563|FB:FBgn0025455|SGD:S000004970	Communication:501741973		2023-06-07
AT4G19560	locus:2122940	AT4G19560	involved in	regulation of transcription by RNA polymerase II	GO:0006357	7463	P	other metabolic processes	IEA	none	InterPro:IPR043198	AnalysisReference:501756966		2023-06-11
AT4G19560	locus:2122940	AT4G19560	involved in	regulation of transcription by RNA polymerase II	GO:0006357	7463	P	nucleobase-containing compound metabolic process	IBA	none	PANTHER:PTN000002458|UniProtKB:O60583|FB:FBgn0004597|UniProtKB:O75909|PomBase:SPBC530.13|UniProtKB:O60563|FB:FBgn0025455|SGD:S000004970	Communication:501741973		2023-06-07
AT4G19560	locus:2122940	AT4G19560	involved in	regulation of transcription by RNA polymerase II	GO:0006357	7463	P	other cellular processes	IEA	none	InterPro:IPR043198	AnalysisReference:501756966		2023-06-11
AT4G19560	locus:2122940	AT4G19560	involved in	positive regulation of DNA-templated transcription, elongation	GO:0032786	25789	P	biosynthetic process	IBA	none	PANTHER:PTN000002653|UniProtKB:O60563|SGD:S000003543|UniProtKB:O75909|UniProtKB:O60583|FB:FBgn0025455	Communication:501741973		2023-06-07
AT4G19570	locus:2122950	AT4G19570	involved in	biological_process	GO:0008150	5239	P	unknown biological processes	ND	Unknown biological process	NONE	Communication:1345790	TAIR	2022-02-01
AT4G19570	locus:2122950	AT4G19570	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	Unknown molecular function		Communication:1345790	TAIR	2021-10-28
AT4G19570	gene:2122949	AT4G19570.1	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19580	locus:2122960	AT4G19580	involved in	biological_process	GO:0008150	5239	P	unknown biological processes	ND	Unknown biological process	NONE	Communication:1345790	TAIR	2022-02-01
AT4G19580	locus:2122960	AT4G19580	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	Unknown molecular function		Communication:1345790	TAIR	2021-10-28
AT4G19580	gene:2122959	AT4G19580.1	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19590	locus:2122970	AT4G19590	acts upstream of or within	negative regulation of macromolecule metabolic process	GO:0010605	29565	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT5G67450	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19590	locus:2122970	AT4G19590	acts upstream of or within	regulation of nucleobase-containing compound metabolic process	GO:0019219	10748	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT5G03150|AGI_LocusCode:AT5G67450	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19590	locus:2122970	AT4G19590	acts upstream of or within	regulation of nucleobase-containing compound metabolic process	GO:0019219	10748	P	nucleobase-containing compound metabolic process	IEA	traceable computational prediction	AGI_LocusCode:AT5G03150|AGI_LocusCode:AT5G67450	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19590	locus:2122970	AT4G19590	acts upstream of or within	RNA metabolic process	GO:0016070	4899	P	nucleobase-containing compound metabolic process	IEA	traceable computational prediction	AGI_LocusCode:AT5G03150|AGI_LocusCode:AT5G67450	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19590	locus:2122970	AT4G19590	acts upstream of or within	regulation of gene expression	GO:0010468	28541	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT5G03150|AGI_LocusCode:AT5G67450	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19590	gene:6532550131	AT4G19590.3	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19590	locus:2122970	AT4G19590	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	Unknown molecular function		Communication:1345790	TAIR	2021-10-28
AT4G19590	locus:2122970	AT4G19590	acts upstream of or within	RNA metabolic process	GO:0016070	4899	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT5G03150|AGI_LocusCode:AT5G67450	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19590	locus:2122970	AT4G19590	acts upstream of or within	regulation of nucleobase-containing compound metabolic process	GO:0019219	10748	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT5G03150|AGI_LocusCode:AT5G67450	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19590	gene:2122969	AT4G19590.1	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19590	gene:6532561681	AT4G19590.2	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19600	locus:2122975	AT4G19600	involved in	regulation of transcription by RNA polymerase II	GO:0006357	7463	P	biosynthetic process	IBA	none	PANTHER:PTN000002458|UniProtKB:O60583|FB:FBgn0004597|UniProtKB:O75909|PomBase:SPBC530.13|UniProtKB:O60563|FB:FBgn0025455|SGD:S000004970	Communication:501741973		2023-06-07
AT4G19600	locus:2122975	AT4G19600	involved in	regulation of transcription by RNA polymerase II	GO:0006357	7463	P	nucleobase-containing compound metabolic process	IBA	none	PANTHER:PTN000002458|UniProtKB:O60583|FB:FBgn0004597|UniProtKB:O75909|PomBase:SPBC530.13|UniProtKB:O60563|FB:FBgn0025455|SGD:S000004970	Communication:501741973		2023-06-07
AT4G19600	locus:2122975	AT4G19600	involved in	positive regulation of DNA-templated transcription, elongation	GO:0032786	25789	P	biosynthetic process	IBA	none	PANTHER:PTN000002653|UniProtKB:O60563|SGD:S000003543|UniProtKB:O75909|UniProtKB:O60583|FB:FBgn0025455	Communication:501741973		2023-06-07
AT4G19600	gene:2122974	AT4G19600.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19600	locus:2122975	AT4G19600	acts upstream of or within	leaf development	GO:0048366	18907	P	multicellular organism development	IGI	double mutant analysis	Tair:gene:2153357	Publication:501721292|PMID:17468259  	TAIR	2007-11-13
AT4G19600	locus:2122975	AT4G19600	involved in	regulation of transcription by RNA polymerase II	GO:0006357	7463	P	other cellular processes	IBA	none	PANTHER:PTN000002458|UniProtKB:O60583|FB:FBgn0004597|UniProtKB:O75909|PomBase:SPBC530.13|UniProtKB:O60563|FB:FBgn0025455|SGD:S000004970	Communication:501741973		2023-06-07
AT4G19600	locus:2122975	AT4G19600	enables	cyclin-dependent protein serine/threonine kinase activator activity	GO:0061575	46533	F	enzyme regulator activity	IBA	none	PANTHER:PTN000002653|PomBase:SPBC32F12.06|WB:WBGene00000508|PomBase:SPBC530.13|WB:WBGene00009650|UniProtKB:O60563	Communication:501741973		2023-06-07
AT4G19600	locus:2122975	AT4G19600	acts upstream of or within	trichome morphogenesis	GO:0010090	14838	P	other cellular processes	IMP	RNAi experiments	NONE	Publication:501721292|PMID:17468259  	TAIR	2007-11-13
AT4G19600	locus:2122975	AT4G19600	acts upstream of or within	response to virus	GO:0009615	7143	P	response to biotic stimulus	IEP	Transcript levels (e.g. Northerns)	NONE	Publication:501721292|PMID:17468259  	TAIR	2007-11-13
AT4G19600	locus:2122975	AT4G19600	involved in	regulation of transcription by RNA polymerase II	GO:0006357	7463	P	other metabolic processes	IBA	none	PANTHER:PTN000002458|UniProtKB:O60583|FB:FBgn0004597|UniProtKB:O75909|PomBase:SPBC530.13|UniProtKB:O60563|FB:FBgn0025455|SGD:S000004970	Communication:501741973		2023-06-07
AT4G19600	locus:2122975	AT4G19600	part of	cyclin/CDK positive transcription elongation factor complex	GO:0008024	709	C	nucleus	IBA	none	PANTHER:PTN000002653|UniProtKB:O60563|PomBase:SPBC530.13|SGD:S000003543|PomBase:SPBC32F12.06|FB:FBgn0025455|UniProtKB:O60583	Communication:501741973		2023-06-07
AT4G19600	locus:2122975	AT4G19600	acts upstream of or within	leaf development	GO:0048366	18907	P	anatomical structure development	IGI	double mutant analysis	Tair:gene:2153357	Publication:501721292|PMID:17468259  	TAIR	2007-11-13
AT4G19600	locus:2122975	AT4G19600	involved in	positive regulation of DNA-templated transcription, elongation	GO:0032786	25789	P	other cellular processes	IBA	none	PANTHER:PTN000002653|UniProtKB:O60563|SGD:S000003543|UniProtKB:O75909|UniProtKB:O60583|FB:FBgn0025455	Communication:501741973		2023-06-07
AT4G19600	locus:2122975	AT4G19600	part of	cyclin/CDK positive transcription elongation factor complex	GO:0008024	709	C	nucleoplasm	IBA	none	PANTHER:PTN000002653|UniProtKB:O60563|PomBase:SPBC530.13|SGD:S000003543|PomBase:SPBC32F12.06|FB:FBgn0025455|UniProtKB:O60583	Communication:501741973		2023-06-07
AT4G19600	locus:2122975	AT4G19600	acts upstream of or within	trichome morphogenesis	GO:0010090	14838	P	cell differentiation	IMP	RNAi experiments	NONE	Publication:501721292|PMID:17468259  	TAIR	2007-11-13
AT4G19600	locus:2122975	AT4G19600	is active in	nucleus	GO:0005634	537	C	nucleus	IBA	none	PANTHER:PTN000002458|TAIR:locus:2143671|FB:FBgn0004597|MGI:MGI:1921539|MGI:MGI:1328363|PomBase:SPBC530.13|UniProtKB:Q5B399|RGD:620864|MGI:MGI:1913921|WB:WBGene00000508|UniProtKB:Q96S94|FB:FBgn0022936|SGD:S000004970|MGI:MGI:1858199|FB:FBgn0025455|WB:WBGene00009650|UniProtKB:Q9UK58|SGD:S000006229|SGD:S000003543|PomBase:SPBC32F12.06|UniProtKB:P51946|PomBase:SPBC12D12.06|UniProtKB:O75909|UniProtKB:O60583|UniProtKB:O60563|PomBase:SPBP16F5.02	Communication:501741973		2023-06-07
AT4G19600	locus:2122975	AT4G19600	involved in	positive regulation of DNA-templated transcription, elongation	GO:0032786	25789	P	other metabolic processes	IBA	none	PANTHER:PTN000002653|UniProtKB:O60563|SGD:S000003543|UniProtKB:O75909|UniProtKB:O60583|FB:FBgn0025455	Communication:501741973		2023-06-07
AT4G19600	locus:2122975	AT4G19600	acts upstream of or within	trichome morphogenesis	GO:0010090	14838	P	anatomical structure development	IMP	RNAi experiments	NONE	Publication:501721292|PMID:17468259  	TAIR	2007-11-13
AT4G19600	locus:2122975	AT4G19600	involved in	positive regulation of DNA-templated transcription, elongation	GO:0032786	25789	P	nucleobase-containing compound metabolic process	IBA	none	PANTHER:PTN000002653|UniProtKB:O60563|SGD:S000003543|UniProtKB:O75909|UniProtKB:O60583|FB:FBgn0025455	Communication:501741973		2023-06-07
AT4G19600	locus:2122975	AT4G19600	acts upstream of or within	response to virus	GO:0009615	7143	P	response to external stimulus	IEP	Transcript levels (e.g. Northerns)	NONE	Publication:501721292|PMID:17468259  	TAIR	2007-11-13
AT4G19610	locus:2122990	AT4G19610	is active in	nucleolus	GO:0005730	527	C	nucleolus	IBA	none	PANTHER:PTN001924395|MGI:MGI:1921361|UniProtKB:Q9Y4C8	Communication:501741973		2023-06-07
AT4G19610	locus:2122990	AT4G19610	involved in	mRNA splicing, via spliceosome	GO:0000398	17088	P	nucleobase-containing compound metabolic process	IBA	none	PANTHER:PTN000567445|UniProtKB:Q8WXF0|TAIR:locus:2035651|UniProtKB:O75494|TAIR:locus:2079502|UniProtKB:Q6K9C3|FB:FBgn0260944|UniProtKB:Q07955|UniProtKB:Q13247|TAIR:locus:2086804|RGD:1587490	Communication:501741973		2023-06-07
AT4G19610	gene:2122989	AT4G19610.1	located in	nucleolus	GO:0005730	527	C	nucleolus	HDA	none	NONE	Publication:501714224|PMID:15496452  	kvanwijk	2021-05-10
AT4G19610	locus:2122990	AT4G19610	involved in	mRNA splicing, via spliceosome	GO:0000398	17088	P	other metabolic processes	IBA	none	PANTHER:PTN000567445|UniProtKB:Q8WXF0|TAIR:locus:2035651|UniProtKB:O75494|TAIR:locus:2079502|UniProtKB:Q6K9C3|FB:FBgn0260944|UniProtKB:Q07955|UniProtKB:Q13247|TAIR:locus:2086804|RGD:1587490	Communication:501741973		2023-06-07
AT4G19610	gene:2122989	AT4G19610.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19610	locus:2122990	AT4G19610	is active in	nuclear speck	GO:0016607	524	C	nucleoplasm	IBA	none	PANTHER:PTN000567445|UniProtKB:Q96PK6|MGI:MGI:98284|UniProtKB:Q07955|UniProtKB:Q08170|TAIR:locus:2144945|MGI:MGI:98287|TAIR:locus:2049766|TAIR:locus:2024770|TAIR:locus:2086804|RGD:1359422|UniProtKB:Q13247|TAIR:locus:2199837|UniProtKB:P84103|TAIR:locus:2035651|UniProtKB:Q01130|TAIR:locus:2099931|TAIR:locus:2079502|TAIR:locus:2131864|UniProtKB:Q9BRL6|TAIR:locus:2124799|MGI:MGI:98283|TAIR:locus:2173338|FB:FBgn0265298|UniProtKB:O75494|UniProtKB:Q9BWF3|MGI:MGI:98285|UniProtKB:Q586R7	Communication:501741973		2023-06-07
AT4G19610	locus:2122990	AT4G19610	enables	RNA binding	GO:0003723	1217	F	RNA binding	IBA	none	PANTHER:PTN000567445|TAIR:locus:2049766|FB:FBgn0011640|FB:FBgn0028554|TAIR:locus:2114643|UniProtKB:Q07955|FB:FBgn0260944|TAIR:locus:2083936|MGI:MGI:98284|FB:FBgn0011305|ZFIN:ZDB-GENE-030131-3019|TAIR:locus:2173088|TAIR:locus:2046638|MGI:MGI:1100865|UniProtKB:Q57VT4|RGD:1309233|UniProtKB:P84103|TAIR:locus:2199837|TAIR:locus:2086804|TAIR:locus:2024770|UniProtKB:Q13247|RGD:3664|TAIR:locus:2124799|FB:FBgn0030479|TAIR:locus:2079502|FB:FBgn0283477|UniProtKB:Q9BWF3|UniProtKB:Q586R7|FB:FBgn0265298|UniProtKB:O75494|TAIR:locus:2173338|MGI:MGI:98283	Communication:501741973		2023-06-07
AT4G19620	locus:2123000	AT4G19620	involved in	biological_process	GO:0008150	5239	P	unknown biological processes	ND	Unknown biological process	NONE	Communication:1345790	TAIR	2022-02-01
AT4G19620	gene:2122999	AT4G19620.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19620	locus:2123000	AT4G19620	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	none	NONE	Communication:1345790	TIGR	2003-10-04
AT4G19630	locus:2123005	AT4G19630	involved in	regulation of transcription by RNA polymerase II	GO:0006357	7463	P	nucleobase-containing compound metabolic process	IBA	none	PANTHER:PTN008492349|SGD:S000001249|SGD:S000003041|MGI:MGI:3045337|UniProtKB:Q9ULV5|UniProtKB:Q03933|CGD:CAL0000178027|SGD:S000005666|UniProtKB:A0A024DAP6|WB:WBGene00002004|UniProtKB:P38529|CGD:CAL0000175059|UniProtKB:Q00613|PomBase:SPAC8C9.14|CGD:CAL0000195882|FB:FBgn0001222|UniProtKB:A0A024DB95|PomBase:SPAC2E12.02|UniProtKB:P38531	Communication:501741973		2023-06-07
AT4G19630	locus:2123005	AT4G19630	enables	DNA-binding transcription factor activity	GO:0003700	4449	F	DNA-binding transcription factor activity	IBA	none	PANTHER:PTN008492349|SGD:S000001249|SGD:S000003041|MGI:MGI:1347058|UniProtKB:Q9ULV5|UniProtKB:Q03933|TAIR:locus:2057371|SGD:S000005666|WB:WBGene00002004|UniProtKB:P38529|CGD:CAL0000195882|PomBase:SPAC8C9.14|UniProtKB:Q00613|UniProtKB:Q5KMX8|TAIR:locus:2149050|PomBase:SPAC2E12.02|FB:FBgn0001222|UniProtKB:P38531	Communication:501741973		2023-06-07
AT4G19630	locus:2123005	AT4G19630	involved in	cellular response to heat	GO:0034605	29752	P	response to abiotic stimulus	IBA	none	PANTHER:PTN000797409|TAIR:locus:2057371	Communication:501741973		2022-11-23
AT4G19630	locus:2123005	AT4G19630	is active in	nucleus	GO:0005634	537	C	nucleus	IBA	none	PANTHER:PTN008492349|PomBase:SPAC8C9.14|TAIR:locus:2144603|UniProtKB:Q9UBD0|UniProtKB:K7LH36|PomBase:SPAC2E12.02|UniProtKB:Q43457|SGD:S000005666|TAIR:locus:2057371|WB:WBGene00002004|TAIR:locus:2117139|UniProtKB:I1LGH5|UniProtKB:Q03933|UniProtKB:P38530|SGD:S000003041|UniProtKB:I1MPZ9|UniProtKB:Q00613|CGD:CAL0000195882|UniProtKB:P38531|TAIR:locus:2075447|UniProtKB:Q942D6|FB:FBgn0001222|TAIR:locus:2149050|CGD:CAL0000178027|TAIR:locus:2204750|UniProtKB:P38529|UniProtKB:Q9ULV5|MGI:MGI:96239|MGI:MGI:96238|UniProtKB:Q96LI6|SGD:S000001249|MGI:MGI:3045337|TAIR:locus:2005495	Communication:501741973		2023-06-07
AT4G19630	locus:2123005	AT4G19630	acts upstream of or within	regulation of DNA-templated transcription	GO:0006355	7461	P	other cellular processes	ISS	none		Publication:1345963|PMID:11118137  		2021-04-01
AT4G19630	locus:2123005	AT4G19630	involved in	regulation of transcription by RNA polymerase II	GO:0006357	7463	P	other cellular processes	IBA	none	PANTHER:PTN008492349|SGD:S000001249|SGD:S000003041|MGI:MGI:3045337|UniProtKB:Q9ULV5|UniProtKB:Q03933|CGD:CAL0000178027|SGD:S000005666|UniProtKB:A0A024DAP6|WB:WBGene00002004|UniProtKB:P38529|CGD:CAL0000175059|UniProtKB:Q00613|PomBase:SPAC8C9.14|CGD:CAL0000195882|FB:FBgn0001222|UniProtKB:A0A024DB95|PomBase:SPAC2E12.02|UniProtKB:P38531	Communication:501741973		2023-06-07
AT4G19630	locus:2123005	AT4G19630	enables	RNA polymerase II cis-regulatory region sequence-specific DNA binding	GO:0000978	35769	F	nucleic acid binding	IBA	none	PANTHER:PTN008492349|MGI:MGI:96239|MGI:MGI:96238|UniProtKB:Q9ULV5|UniProtKB:Q00613|PomBase:SPAC8C9.14|UniProtKB:Q03933|PomBase:SPAC2E12.02|UniProtKB:P38531|SGD:S000005666|UniProtKB:P38529	Communication:501741973		2023-06-07
AT4G19630	gene:2123004	AT4G19630.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19630	locus:2123005	AT4G19630	involved in	regulation of transcription by RNA polymerase II	GO:0006357	7463	P	other metabolic processes	IBA	none	PANTHER:PTN008492349|SGD:S000001249|SGD:S000003041|MGI:MGI:3045337|UniProtKB:Q9ULV5|UniProtKB:Q03933|CGD:CAL0000178027|SGD:S000005666|UniProtKB:A0A024DAP6|WB:WBGene00002004|UniProtKB:P38529|CGD:CAL0000175059|UniProtKB:Q00613|PomBase:SPAC8C9.14|CGD:CAL0000195882|FB:FBgn0001222|UniProtKB:A0A024DB95|PomBase:SPAC2E12.02|UniProtKB:P38531	Communication:501741973		2023-06-07
AT4G19630	locus:2123005	AT4G19630	acts upstream of or within	regulation of DNA-templated transcription	GO:0006355	7461	P	biosynthetic process	ISS	none		Publication:1345963|PMID:11118137  		2021-04-01
AT4G19630	locus:2123005	AT4G19630	enables	RNA polymerase II cis-regulatory region sequence-specific DNA binding	GO:0000978	35769	F	DNA binding	IBA	none	PANTHER:PTN008492349|MGI:MGI:96239|MGI:MGI:96238|UniProtKB:Q9ULV5|UniProtKB:Q00613|PomBase:SPAC8C9.14|UniProtKB:Q03933|PomBase:SPAC2E12.02|UniProtKB:P38531|SGD:S000005666|UniProtKB:P38529	Communication:501741973		2023-06-07
AT4G19630	locus:2123005	AT4G19630	involved in	cellular response to heat	GO:0034605	29752	P	other cellular processes	IBA	none	PANTHER:PTN000797409|TAIR:locus:2057371	Communication:501741973		2022-11-23
AT4G19630	locus:2123005	AT4G19630	enables	DNA-binding transcription factor activity	GO:0003700	4449	F	DNA-binding transcription factor activity	ISS	Recognized domains		Publication:1345963|PMID:11118137  	TAIR	2003-06-06
AT4G19630	locus:2123005	AT4G19630	involved in	regulation of transcription by RNA polymerase II	GO:0006357	7463	P	biosynthetic process	IBA	none	PANTHER:PTN008492349|SGD:S000001249|SGD:S000003041|MGI:MGI:3045337|UniProtKB:Q9ULV5|UniProtKB:Q03933|CGD:CAL0000178027|SGD:S000005666|UniProtKB:A0A024DAP6|WB:WBGene00002004|UniProtKB:P38529|CGD:CAL0000175059|UniProtKB:Q00613|PomBase:SPAC8C9.14|CGD:CAL0000195882|FB:FBgn0001222|UniProtKB:A0A024DB95|PomBase:SPAC2E12.02|UniProtKB:P38531	Communication:501741973		2023-06-07
AT4G19630	locus:2123005	AT4G19630	acts upstream of or within	regulation of DNA-templated transcription	GO:0006355	7461	P	other metabolic processes	ISS	none		Publication:1345963|PMID:11118137  		2021-04-01
AT4G19630	locus:2123005	AT4G19630	acts upstream of or within	regulation of DNA-templated transcription	GO:0006355	7461	P	nucleobase-containing compound metabolic process	ISS	none		Publication:1345963|PMID:11118137  		2021-04-01
AT4G19630	locus:2123005	AT4G19630	involved in	cellular response to heat	GO:0034605	29752	P	response to stress	IBA	none	PANTHER:PTN000797409|TAIR:locus:2057371	Communication:501741973		2022-11-23
AT4G19640	locus:2123010	AT4G19640	enables	GTPase activity	GO:0003924	1041	F	hydrolase activity	IBA	none	PANTHER:PTN001292747|UniProtKB:Q13637|UniProtKB:P61106|SGD:S000001722|SGD:S000001856|UniProtKB:Q9ULC3|FB:FBgn0015797|SGD:S000004252|MGI:MGI:105072|UniProtKB:Q9UL25|WB:WBGene00004268|UniProtKB:Q9UL26|TAIR:locus:2041036|UniProtKB:P20339|MGI:MGI:2443036|SGD:S000005615|PomBase:SPAC4C5.02c|UniProtKB:Q9H0T7|UniProtKB:P62820|SGD:S000005037|UniProtKB:Q9H0N0|UniProtKB:P20340|UniProtKB:Q86YS6|TAIR:locus:2082672|FB:FBgn0285937|TAIR:locus:505006713|PomBase:SPBC1703.10|MGI:MGI:105926|UniProtKB:P51148|UniProtKB:P61020	Communication:501741973		2023-06-07
AT4G19640	locus:2123010	AT4G19640	enables	GTP binding	GO:0005525	1035	F	other binding	IEA	none	InterPro:IPR001806|InterPro:IPR005225	AnalysisReference:501756966		2023-06-11
AT4G19640	locus:2123010	AT4G19640	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:Q9SKN1	Publication:501760216|PMID:24824487  		2021-04-01
AT4G19640	locus:2123010	AT4G19640	located in	early endosome	GO:0005769	260	C	endosome	IDA	none		Publication:1346367|PMID:11532937  		2016-08-01
AT4G19640	locus:2123010	AT4G19640	located in	endoplasmic reticulum	GO:0005783	268	C	endoplasmic reticulum	HDA	none	NONE	Publication:501750648|PMID:22923678  	dupree	2021-04-07
AT4G19640	locus:2123010	AT4G19640	involved in	late endosome to vacuole transport	GO:0045324	11761	P	transport	IMP	none		Publication:501760216|PMID:24824487  		2021-03-31
AT4G19640	locus:2123010	AT4G19640	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:Q9LT31	Publication:501723826|PMID:18055610  		2021-04-01
AT4G19640	locus:2123010	AT4G19640	involved in	intracellular protein transport	GO:0006886	6091	P	other cellular processes	IMP	none	UniProtKB:Q7X659	Publication:501760216|PMID:24824487  		2021-03-31
AT4G19640	locus:2123010	AT4G19640	is active in	endomembrane system	GO:0012505	9932	C	other cellular components	IBA	none	PANTHER:PTN001292747|RGD:1306489|MGI:MGI:105926|UniProtKB:P61020|UniProtKB:P18066|UniProtKB:A0A5K1K8H7|UniProtKB:C8V3Z5|MGI:MGI:99833|UniProtKB:O14966|CGD:CAL0000201225|FB:FBgn0015372|UniProtKB:Q9NX57|WB:WBGene00004270|UniProtKB:Q9NP72|UniProtKB:P20340|FB:FBgn0039966|UniProtKB:Q9H0N0|UniProtKB:Q53S08|FB:FBgn0015794|dictyBase:DDB_G0271984|SGD:S000005615|MGI:MGI:894313|FB:FBgn0015791|RGD:620881|SGD:S000005248|TAIR:locus:2081486|FB:FBgn0014010|UniProtKB:P51147|UniProtKB:Q0ILQ6|WB:WBGene00004268|TAIR:locus:2140045|MGI:MGI:104640|UniProtKB:Q9UL25|UniProtKB:Q9H0U4|UniProtKB:P57729|UniProtKB:Q8IHR8|SGD:S000004252|UniProtKB:Q13636|UniProtKB:O96193|CGD:CAL0000181874|UniProtKB:P51148|MGI:MGI:97842|UniProtKB:D6XLG8|FB:FBgn0285937|MGI:MGI:2385107|ZFIN:ZDB-GENE-031118-30|TAIR:locus:2082672|RGD:1307095|RGD:620892|UniProtKB:Q86YS6|TAIR:locus:2124092|UniProtKB:Q9NRW1|SGD:S000005037|RGD:628598|RGD:620936|UniProtKB:P20339|TAIR:locus:2042321|dictyBase:DDB_G0282537|CGD:CAL0000194609|UniProtKB:Q586J5|RGD:619736|UniProtKB:Q5B3G5|TAIR:locus:2041036|UniProtKB:Q9UL26|WB:WBGene00004269|WB:WBGene00004276|FB:FBgn0015793|SGD:S000001856|UniProtKB:Q8I3W9|FB:FBgn0015797|UniProtKB:Q5B6I8|TAIR:locus:2123010|UniProtKB:P61106|SGD:S000001722|UniProtKB:Q13637	Communication:501741973		2023-06-07
AT4G19640	locus:2123010	AT4G19640	located in	intracellular organelle	GO:0043229	19387	C	other intracellular components	IDA	localization of GFP/YFP fusion protein	NONE	Publication:501779759|PMID:29749929  	TAIR	2018-11-06
AT4G19640	locus:2123010	AT4G19640	enables	GTP binding	GO:0005525	1035	F	nucleotide binding	IEA	none	InterPro:IPR001806|InterPro:IPR005225	AnalysisReference:501756966		2023-06-11
AT4G19640	locus:2123010	AT4G19640	is active in	endocytic vesicle	GO:0030139	7876	C	other intracellular components	IBA	none	PANTHER:PTN000634566|UniProtKB:Q9H0T7|WB:WBGene00004268|dictyBase:DDB_G0271984|MGI:MGI:105938	Communication:501741973		2023-06-07
AT4G19640	locus:2123010	AT4G19640	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:F4I312	Publication:501778849|PMID:29463724  		2021-04-01
AT4G19640	locus:2123010	AT4G19640	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	yeast two-hybrid assay	AGI_LocusCode:AT1G24560	Publication:501779759|PMID:29749929  	TAIR	2018-05-25
AT4G19640	locus:2123010	AT4G19640	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:Q9LT31	Publication:501739925|PMID:20833725  		2021-04-01
AT4G19640	locus:2123010	AT4G19640	is active in	endosome	GO:0005768	272	C	endosome	IBA	none	PANTHER:PTN000634566|UniProtKB:O96193|CGD:CAL0000181874|TAIR:locus:2123010|SGD:S000005615|dictyBase:DDB_G0271984|SGD:S000005037|MGI:MGI:104640|WB:WBGene00004268|UniProtKB:Q5B3G5|TAIR:locus:2082672|FB:FBgn0014010|UniProtKB:P61020	Communication:501741973		2023-06-07
AT4G19640	locus:2123010	AT4G19640	involved in	vacuole organization	GO:0007033	7551	P	cellular component organization	IMP	none		Publication:501760216|PMID:24824487  		2021-03-31
AT4G19640	locus:2123010	AT4G19640	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:Q9LSB9	Publication:501770516|PMID:27288222  		2021-04-01
AT4G19640	locus:2123010	AT4G19640	enables	GTP binding	GO:0005525	1035	F	nucleotide binding	IDA	none		Publication:501739925|PMID:20833725  		2021-04-01
AT4G19640	locus:2123010	AT4G19640	enables	GTP binding	GO:0005525	1035	F	other binding	IDA	none		Publication:501739925|PMID:20833725  		2021-04-01
AT4G19640	locus:2123010	AT4G19640	is active in	endocytic vesicle	GO:0030139	7876	C	cytoplasm	IBA	none	PANTHER:PTN000634566|UniProtKB:Q9H0T7|WB:WBGene00004268|dictyBase:DDB_G0271984|MGI:MGI:105938	Communication:501741973		2023-06-07
AT4G19640	locus:2123010	AT4G19640	located in	endosome	GO:0005768	272	C	endosome	IDA	localization of GFP/YFP fusion protein		Publication:501736075|PMID:20181757  	mpiz_zhang	2010-04-13
AT4G19640	locus:2123010	AT4G19640	enables	GTPase activity	GO:0003924	1041	F	hydrolase activity	IEA	none	InterPro:IPR001806	AnalysisReference:501756966		2023-06-11
AT4G19640	locus:2123010	AT4G19640	involved in	intracellular protein transport	GO:0006886	6091	P	transport	IBA	none	PANTHER:PTN001292747|SGD:S000005037|SGD:S000004252|UniProtKB:Q5B6I8|FB:FBgn0015797|SGD:S000001856|SGD:S000005615|SGD:S000001722|CGD:CAL0000181874|TAIR:locus:2123010|CGD:CAL0000201225|UniProtKB:Q0ILQ6|CGD:CAL0000194609	Communication:501741973		2023-06-07
AT4G19640	locus:2123010	AT4G19640	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:P31265	Publication:501739721|PMID:20736351  		2021-04-01
AT4G19640	locus:2123010	AT4G19640	involved in	intracellular protein transport	GO:0006886	6091	P	other cellular processes	IBA	none	PANTHER:PTN001292747|SGD:S000005037|SGD:S000004252|UniProtKB:Q5B6I8|FB:FBgn0015797|SGD:S000001856|SGD:S000005615|SGD:S000001722|CGD:CAL0000181874|TAIR:locus:2123010|CGD:CAL0000201225|UniProtKB:Q0ILQ6|CGD:CAL0000194609	Communication:501741973		2023-06-07
AT4G19640	gene:2123009	AT4G19640.1	located in	plasma membrane	GO:0005886	570	C	plasma membrane	HDA	none	NONE	Publication:501722708|PMID:17644812  	kvanwijk	2021-04-07
AT4G19640	locus:2123010	AT4G19640	involved in	late endosome to vacuole transport	GO:0045324	11761	P	other cellular processes	IMP	none		Publication:501760216|PMID:24824487  		2021-03-31
AT4G19640	locus:2123010	AT4G19640	located in	cytoplasm	GO:0005737	231	C	cytoplasm	IDA	none		Publication:501778849|PMID:29463724  		2018-04-25
AT4G19640	locus:2123010	AT4G19640	involved in	intracellular protein transport	GO:0006886	6091	P	transport	IMP	none	UniProtKB:Q7X659	Publication:501760216|PMID:24824487  		2021-03-31
AT4G19645	locus:504955499	AT4G19645	involved in	lipid homeostasis	GO:0055088	28517	P	other biological processes	IBA	none	PANTHER:PTN001412312|WB:WBGene00013407|UniProtKB:Q96CP7|UniProtKB:A6NGC4|ZFIN:ZDB-GENE-030131-2046	Communication:501741973		2023-06-07
AT4G19645	locus:504955499	AT4G19645	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	none	NONE	Communication:1345790	TIGR	2003-10-04
AT4G19645	locus:504955499	AT4G19645	is active in	endoplasmic reticulum	GO:0005783	268	C	endoplasmic reticulum	IBA	none	PANTHER:PTN000335242|UniProtKB:Q71RH2|MGI:MGI:1349447|UniProtKB:Q9UBY8	Communication:501741973		2023-06-07
AT4G19650	locus:2133920	AT4G19650	involved in	regulation of DNA-templated transcription	GO:0006355	7461	P	other metabolic processes	IEA	none	InterPro:IPR003690	AnalysisReference:501756966		2023-05-23
AT4G19650	locus:2133920	AT4G19650	involved in	regulation of DNA-templated transcription	GO:0006355	7461	P	nucleobase-containing compound metabolic process	IEA	none	InterPro:IPR003690	AnalysisReference:501756966		2023-05-23
AT4G19650	locus:2133920	AT4G19650	involved in	regulation of DNA-templated transcription	GO:0006355	7461	P	biosynthetic process	IEA	none	InterPro:IPR003690	AnalysisReference:501756966		2023-05-23
AT4G19650	locus:2133920	AT4G19650	acts upstream of or within	embryo development ending in seed dormancy	GO:0009793	11994	P	anatomical structure development	IEA	traceable computational prediction	AGI_LocusCode:AT1G21970	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19650	gene:2133919	AT4G19650.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19650	locus:2133920	AT4G19650	acts upstream of or within	regulation of nitrogen compound metabolic process	GO:0051171	19379	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G21970|AGI_LocusCode:AT4G27410	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19650	locus:2133920	AT4G19650	acts upstream of or within	embryo development ending in seed dormancy	GO:0009793	11994	P	multicellular organism development	IEA	traceable computational prediction	AGI_LocusCode:AT1G21970	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19650	gene:6532550031	AT4G19650.2	located in	mitochondrion	GO:0005739	486	C	mitochondrion	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19650	locus:2133920	AT4G19650	acts upstream of or within	regulation of gene expression	GO:0010468	28541	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G21970|AGI_LocusCode:AT4G27410	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19650	locus:2133920	AT4G19650	acts upstream of or within	embryo development ending in seed dormancy	GO:0009793	11994	P	post-embryonic development	IEA	traceable computational prediction	AGI_LocusCode:AT1G21970	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19650	locus:2133920	AT4G19650	acts upstream of or within	RNA metabolic process	GO:0016070	4899	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G21970|AGI_LocusCode:AT4G27410	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19650	locus:2133920	AT4G19650	involved in	regulation of DNA-templated transcription	GO:0006355	7461	P	other cellular processes	IEA	none	InterPro:IPR003690	AnalysisReference:501756966		2023-05-23
AT4G19650	locus:2133920	AT4G19650	acts upstream of or within	embryo development ending in seed dormancy	GO:0009793	11994	P	reproduction	IEA	traceable computational prediction	AGI_LocusCode:AT1G21970	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19650	locus:2133920	AT4G19650	acts upstream of or within	embryo development ending in seed dormancy	GO:0009793	11994	P	embryo development	IEA	traceable computational prediction	AGI_LocusCode:AT1G21970	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19650	locus:2133920	AT4G19650	acts upstream of or within	RNA metabolic process	GO:0016070	4899	P	nucleobase-containing compound metabolic process	IEA	traceable computational prediction	AGI_LocusCode:AT1G21970|AGI_LocusCode:AT4G27410	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19650	locus:2133920	AT4G19650	enables	double-stranded DNA binding	GO:0003690	2198	F	DNA binding	IEA	none	InterPro:IPR003690	AnalysisReference:501756966		2023-05-23
AT4G19660	locus:2133925	AT4G19660	acts upstream of or within	response to bacterium	GO:0009617	7120	P	response to biotic stimulus	IEP	Transcript levels (e.g. Northerns)	NONE	Publication:501714391|PMID:15634206  	TAIR	2006-09-20
AT4G19660	locus:2133925	AT4G19660	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:Q9SCK1	Publication:501790246|PMID:32612234  		2023-05-23
AT4G19660	locus:2133925	AT4G19660	acts upstream of or within	response to bacterium	GO:0009617	7120	P	response to external stimulus	IMP	analysis of physiological response	TAIR:TGA2	Publication:501714391|PMID:15634206  	TAIR	2006-09-20
AT4G19660	locus:2133925	AT4G19660	involved in	regulation of jasmonic acid mediated signaling pathway	GO:2000022	35570	P	response to endogenous stimulus	IEA	none	InterPro:IPR044292	AnalysisReference:501756966		2023-06-11
AT4G19660	locus:2133925	AT4G19660	involved in	defense response to fungus	GO:0050832	18013	P	response to stress	IBA	none	PANTHER:PTN002925302|UniProtKB:Q9FDY4|TAIR:locus:2014200|TAIR:locus:2133925|TAIR:locus:2153192	Communication:501741973		2023-06-07
AT4G19660	locus:2133925	AT4G19660	involved in	regulation of jasmonic acid mediated signaling pathway	GO:2000022	35570	P	other cellular processes	IEA	none	InterPro:IPR044292	AnalysisReference:501756966		2023-06-11
AT4G19660	locus:2133925	AT4G19660	involved in	regulation of jasmonic acid mediated signaling pathway	GO:2000022	35570	P	signal transduction	IMP	none		Publication:501714391|PMID:15634206  		2021-03-31
AT4G19660	locus:2133925	AT4G19660	involved in	regulation of jasmonic acid mediated signaling pathway	GO:2000022	35570	P	cell communication	IMP	none		Publication:501714391|PMID:15634206  		2021-03-31
AT4G19660	locus:2133925	AT4G19660	involved in	systemic acquired resistance, salicylic acid mediated signaling pathway	GO:0009862	8839	P	signal transduction	IEA	none	InterPro:IPR044292	AnalysisReference:501756966		2023-06-11
AT4G19660	locus:2133925	AT4G19660	involved in	systemic acquired resistance, salicylic acid mediated signaling pathway	GO:0009862	8839	P	response to external stimulus	IEA	none	InterPro:IPR044292	AnalysisReference:501756966		2023-06-11
AT4G19660	locus:2133925	AT4G19660	involved in	regulation of jasmonic acid mediated signaling pathway	GO:2000022	35570	P	other cellular processes	IBA	none	PANTHER:PTN002925302|TAIR:locus:2014200|TAIR:locus:2133925	Communication:501741973		2023-04-13
AT4G19660	locus:2133925	AT4G19660	acts upstream of or within	response to bacterium	GO:0009617	7120	P	response to biotic stimulus	IMP	analysis of physiological response	TAIR:TGA2	Publication:501714391|PMID:15634206  	TAIR	2006-09-20
AT4G19660	locus:2133925	AT4G19660	involved in	defense response to bacterium	GO:0042742	14972	P	response to external stimulus	IMP	none		Publication:501714391|PMID:15634206  		2021-03-31
AT4G19660	locus:2133925	AT4G19660	enables	identical protein binding	GO:0042802	15210	F	protein binding	IPI	none	UniProtKB:Q5ICL9	Publication:501750116|PMID:22699612  		2023-05-23
AT4G19660	locus:2133925	AT4G19660	involved in	regulation of jasmonic acid mediated signaling pathway	GO:2000022	35570	P	response to chemical	IMP	none		Publication:501714391|PMID:15634206  		2021-03-31
AT4G19660	locus:2133925	AT4G19660	acts upstream of or within	systemic acquired resistance	GO:0009627	7392	P	response to stress	IGI	double mutant analysis	AGI_LocusCode:AT5G45110	Publication:501750116|PMID:22699612  	TAIR	2012-09-17
AT4G19660	locus:2133925	AT4G19660	acts upstream of or within	systemic acquired resistance	GO:0009627	7392	P	response to external stimulus	IGI	double mutant analysis	AGI_LocusCode:AT5G45110	Publication:501750116|PMID:22699612  	TAIR	2012-09-17
AT4G19660	gene:6532560772	AT4G19660.2	located in	chloroplast	GO:0009507	175	C	chloroplast	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19660	locus:2133925	AT4G19660	is active in	nucleus	GO:0005634	537	C	nucleus	IBA	none	PANTHER:PTN002925302|TAIR:locus:2133925|UniProtKB:Q9FDY4|TAIR:locus:2014200	Communication:501741973		2023-06-07
AT4G19660	locus:2133925	AT4G19660	acts upstream of or within	response to fungus	GO:0009620	7124	P	response to external stimulus	IMP	analysis of physiological response	NONE	Publication:501714391|PMID:15634206  	TAIR	2006-09-20
AT4G19660	locus:2133925	AT4G19660	involved in	defense response to bacterium	GO:0042742	14972	P	response to stress	IBA	none	PANTHER:PTN002925302|UniProtKB:Q9FDY4|TAIR:locus:2014200|UniProtKB:Q75HA6|TAIR:locus:2153192|TAIR:locus:2133925	Communication:501741973		2023-06-07
AT4G19660	locus:2133925	AT4G19660	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:Q9FMS4	Publication:501790246|PMID:32612234  		2023-05-23
AT4G19660	locus:2133925	AT4G19660	involved in	regulation of jasmonic acid mediated signaling pathway	GO:2000022	35570	P	other cellular processes	IMP	none		Publication:501714391|PMID:15634206  		2021-03-31
AT4G19660	locus:2133925	AT4G19660	involved in	defense response to fungus	GO:0050832	18013	P	response to biotic stimulus	IMP	none		Publication:501714391|PMID:15634206  		2021-03-31
AT4G19660	locus:2133925	AT4G19660	involved in	regulation of salicylic acid mediated signaling pathway	GO:2000031	35579	P	response to chemical	IEA	none	InterPro:IPR044292	AnalysisReference:501756966		2023-06-11
AT4G19660	locus:2133925	AT4G19660	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:Q8L746	Publication:501750116|PMID:22699612  		2023-05-23
AT4G19660	locus:2133925	AT4G19660	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:Q39234	Publication:501714391|PMID:15634206  		2021-04-01
AT4G19660	locus:2133925	AT4G19660	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:Q9FNH6	Publication:501790246|PMID:32612234  		2023-05-23
AT4G19660	locus:2133925	AT4G19660	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:Q39163	Publication:501714391|PMID:15634206  		2021-04-01
AT4G19660	locus:2133925	AT4G19660	involved in	defense response to fungus	GO:0050832	18013	P	response to external stimulus	IBA	none	PANTHER:PTN002925302|UniProtKB:Q9FDY4|TAIR:locus:2014200|TAIR:locus:2133925|TAIR:locus:2153192	Communication:501741973		2023-06-07
AT4G19660	locus:2133925	AT4G19660	involved in	defense response to fungus	GO:0050832	18013	P	response to biotic stimulus	IBA	none	PANTHER:PTN002925302|UniProtKB:Q9FDY4|TAIR:locus:2014200|TAIR:locus:2133925|TAIR:locus:2153192	Communication:501741973		2023-06-07
AT4G19660	locus:2133925	AT4G19660	involved in	regulation of jasmonic acid mediated signaling pathway	GO:2000022	35570	P	signal transduction	IEA	none	InterPro:IPR044292	AnalysisReference:501756966		2023-06-11
AT4G19660	locus:2133925	AT4G19660	involved in	regulation of jasmonic acid mediated signaling pathway	GO:2000022	35570	P	response to endogenous stimulus	IMP	none		Publication:501714391|PMID:15634206  		2021-03-31
AT4G19660	locus:2133925	AT4G19660	involved in	regulation of jasmonic acid mediated signaling pathway	GO:2000022	35570	P	cell communication	IBA	none	PANTHER:PTN002925302|TAIR:locus:2014200|TAIR:locus:2133925	Communication:501741973		2023-04-13
AT4G19660	locus:2133925	AT4G19660	acts upstream of or within	response to bacterium	GO:0009617	7120	P	response to external stimulus	IEP	Transcript levels (e.g. Northerns)	NONE	Publication:501714391|PMID:15634206  	TAIR	2006-09-20
AT4G19660	locus:2133925	AT4G19660	involved in	regulation of jasmonic acid mediated signaling pathway	GO:2000022	35570	P	response to chemical	IEA	none	InterPro:IPR044292	AnalysisReference:501756966		2023-06-11
AT4G19660	locus:2133925	AT4G19660	acts upstream of or within	defense response to bacterium	GO:0042742	14972	P	response to external stimulus	IMP	analysis of physiological response	NONE	Publication:501720485|PMID:17076807  	TAIR	2020-12-18
AT4G19660	locus:2133925	AT4G19660	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:Q0WLB5	Publication:501790246|PMID:32612234  		2023-05-23
AT4G19660	locus:2133925	AT4G19660	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:Q8L746	Publication:501790246|PMID:32612234  		2023-05-23
AT4G19660	locus:2133925	AT4G19660	acts upstream of or within	defense response to bacterium	GO:0042742	14972	P	response to stress	IMP	analysis of physiological response	NONE	Publication:501720485|PMID:17076807  	TAIR	2020-12-18
AT4G19660	locus:2133925	AT4G19660	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:O64471	Publication:501790246|PMID:32612234  		2023-05-23
AT4G19660	locus:2133925	AT4G19660	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:Q39163	Publication:501720485|PMID:17076807  		2021-04-01
AT4G19660	locus:2133925	AT4G19660	involved in	systemic acquired resistance, salicylic acid mediated signaling pathway	GO:0009862	8839	P	response to stress	IEA	none	InterPro:IPR044292	AnalysisReference:501756966		2023-06-11
AT4G19660	locus:2133925	AT4G19660	involved in	systemic acquired resistance, salicylic acid mediated signaling pathway	GO:0009862	8839	P	other cellular processes	IEA	none	InterPro:IPR044292	AnalysisReference:501756966		2023-06-11
AT4G19660	locus:2133925	AT4G19660	involved in	systemic acquired resistance, salicylic acid mediated signaling pathway	GO:0009862	8839	P	response to chemical	IEA	none	InterPro:IPR044292	AnalysisReference:501756966		2023-06-11
AT4G19660	locus:2133925	AT4G19660	involved in	defense response to bacterium	GO:0042742	14972	P	response to stress	IMP	none		Publication:501714391|PMID:15634206  		2021-03-31
AT4G19660	locus:2133925	AT4G19660	located in	nucleus	GO:0005634	537	C	nucleus	IDA	localization of GFP/YFP fusion protein		Publication:501714391|PMID:15634206  	TAIR	2005-03-21
AT4G19660	locus:2133925	AT4G19660	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:P93002	Publication:501750116|PMID:22699612  		2023-05-23
AT4G19660	locus:2133925	AT4G19660	involved in	defense response to bacterium	GO:0042742	14972	P	response to external stimulus	IBA	none	PANTHER:PTN002925302|UniProtKB:Q9FDY4|TAIR:locus:2014200|UniProtKB:Q75HA6|TAIR:locus:2153192|TAIR:locus:2133925	Communication:501741973		2023-06-07
AT4G19660	locus:2133925	AT4G19660	involved in	regulation of jasmonic acid mediated signaling pathway	GO:2000022	35570	P	cell communication	IEA	none	InterPro:IPR044292	AnalysisReference:501756966		2023-06-11
AT4G19660	locus:2133925	AT4G19660	involved in	defense response to bacterium	GO:0042742	14972	P	response to biotic stimulus	IMP	none		Publication:501714391|PMID:15634206  		2021-03-31
AT4G19660	locus:2133925	AT4G19660	involved in	regulation of salicylic acid mediated signaling pathway	GO:2000031	35579	P	other cellular processes	IMP	none		Publication:501714391|PMID:15634206  		2021-03-31
AT4G19660	locus:2133925	AT4G19660	involved in	regulation of jasmonic acid mediated signaling pathway	GO:2000022	35570	P	response to endogenous stimulus	IBA	none	PANTHER:PTN002925302|TAIR:locus:2014200|TAIR:locus:2133925	Communication:501741973		2023-04-13
AT4G19660	locus:2133925	AT4G19660	involved in	regulation of salicylic acid mediated signaling pathway	GO:2000031	35579	P	signal transduction	IMP	none		Publication:501714391|PMID:15634206  		2021-03-31
AT4G19660	locus:2133925	AT4G19660	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:Q39140	Publication:501720485|PMID:17076807  		2021-04-01
AT4G19660	locus:2133925	AT4G19660	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:F4IDX2	Publication:501790246|PMID:32612234  		2023-05-23
AT4G19660	locus:2133925	AT4G19660	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:P43273	Publication:501714391|PMID:15634206  		2021-04-01
AT4G19660	locus:2133925	AT4G19660	enables	salicylic acid binding	GO:1901149	43231	F	other binding	IDA	in vitro binding assay	NONE	Publication:501750116|PMID:22699612  	TAIR	2012-09-17
AT4G19660	locus:2133925	AT4G19660	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:P43273	Publication:501720485|PMID:17076807  		2021-04-01
AT4G19660	locus:2133925	AT4G19660	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:Q93ZE2	Publication:501714391|PMID:15634206  		2021-04-01
AT4G19660	locus:2133925	AT4G19660	acts upstream of or within	defense response to bacterium	GO:0042742	14972	P	response to biotic stimulus	IMP	analysis of physiological response	NONE	Publication:501720485|PMID:17076807  	TAIR	2020-12-18
AT4G19660	gene:2133924	AT4G19660.1	located in	chloroplast	GO:0009507	175	C	chloroplast	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19660	locus:2133925	AT4G19660	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:F4JRR1	Publication:501790246|PMID:32612234  		2023-05-23
AT4G19660	locus:2133925	AT4G19660	involved in	regulation of salicylic acid mediated signaling pathway	GO:2000031	35579	P	cell communication	IEA	none	InterPro:IPR044292	AnalysisReference:501756966		2023-06-11
AT4G19660	locus:2133925	AT4G19660	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:P25269	Publication:501790246|PMID:32612234  		2023-05-23
AT4G19660	locus:2133925	AT4G19660	involved in	systemic acquired resistance, salicylic acid mediated signaling pathway	GO:0009862	8839	P	response to biotic stimulus	IEA	none	InterPro:IPR044292	AnalysisReference:501756966		2023-06-11
AT4G19660	locus:2133925	AT4G19660	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:Q9FIR9	Publication:501790246|PMID:32612234  		2023-05-23
AT4G19660	locus:2133925	AT4G19660	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:Q39140	Publication:501714391|PMID:15634206  		2021-04-01
AT4G19660	locus:2133925	AT4G19660	involved in	regulation of jasmonic acid mediated signaling pathway	GO:2000022	35570	P	response to chemical	IBA	none	PANTHER:PTN002925302|TAIR:locus:2014200|TAIR:locus:2133925	Communication:501741973		2023-04-13
AT4G19660	locus:2133925	AT4G19660	involved in	defense response to fungus	GO:0050832	18013	P	response to stress	IMP	none		Publication:501714391|PMID:15634206  		2021-03-31
AT4G19660	locus:2133925	AT4G19660	involved in	regulation of salicylic acid mediated signaling pathway	GO:2000031	35579	P	other cellular processes	IEA	none	InterPro:IPR044292	AnalysisReference:501756966		2023-06-11
AT4G19660	locus:2133925	AT4G19660	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:Q8VZE5	Publication:501790246|PMID:32612234  		2023-05-23
AT4G19660	locus:2133925	AT4G19660	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:Q39234	Publication:501720485|PMID:17076807  		2021-04-01
AT4G19660	locus:2133925	AT4G19660	involved in	defense response to bacterium	GO:0042742	14972	P	response to biotic stimulus	IBA	none	PANTHER:PTN002925302|UniProtKB:Q9FDY4|TAIR:locus:2014200|UniProtKB:Q75HA6|TAIR:locus:2153192|TAIR:locus:2133925	Communication:501741973		2023-06-07
AT4G19660	locus:2133925	AT4G19660	acts upstream of or within	systemic acquired resistance	GO:0009627	7392	P	response to biotic stimulus	IGI	double mutant analysis	AGI_LocusCode:AT5G45110	Publication:501750116|PMID:22699612  	TAIR	2012-09-17
AT4G19660	locus:2133925	AT4G19660	involved in	regulation of salicylic acid mediated signaling pathway	GO:2000031	35579	P	signal transduction	IEA	none	InterPro:IPR044292	AnalysisReference:501756966		2023-06-11
AT4G19660	locus:2133925	AT4G19660	involved in	regulation of jasmonic acid mediated signaling pathway	GO:2000022	35570	P	signal transduction	IBA	none	PANTHER:PTN002925302|TAIR:locus:2014200|TAIR:locus:2133925	Communication:501741973		2023-04-13
AT4G19660	locus:2133925	AT4G19660	acts upstream of or within	response to fungus	GO:0009620	7124	P	response to biotic stimulus	IMP	analysis of physiological response	NONE	Publication:501714391|PMID:15634206  	TAIR	2006-09-20
AT4G19660	locus:2133925	AT4G19660	involved in	regulation of salicylic acid mediated signaling pathway	GO:2000031	35579	P	cell communication	IMP	none		Publication:501714391|PMID:15634206  		2021-03-31
AT4G19660	locus:2133925	AT4G19660	involved in	regulation of salicylic acid mediated signaling pathway	GO:2000031	35579	P	response to chemical	IMP	none		Publication:501714391|PMID:15634206  		2021-03-31
AT4G19660	locus:2133925	AT4G19660	involved in	defense response to fungus	GO:0050832	18013	P	response to external stimulus	IMP	none		Publication:501714391|PMID:15634206  		2021-03-31
AT4G19670	gene:6532551011	AT4G19670.4	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19670	locus:2133935	AT4G19670	enables	RNA-DNA hybrid ribonuclease activity	GO:0004523	4049	F	nuclease activity	IEA	none	InterPro:IPR002156	AnalysisReference:501756966		2023-06-11
AT4G19670	gene:4010712910	AT4G19670.2	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19670	gene:6532551040	AT4G19670.3	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19670	locus:2133935	AT4G19670	enables	zinc ion binding	GO:0008270	4661	F	other binding	IEA	none	InterPro:IPR002867	AnalysisReference:501756966		2023-06-11
AT4G19670	gene:6532552491	AT4G19670.5	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19670	gene:2133934	AT4G19670.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19670	locus:2133935	AT4G19670	involved in	ubiquitin-dependent protein catabolic process	GO:0006511	7522	P	other metabolic processes	IBA	none	PANTHER:PTN000188395|FB:FBgn0017418|WB:WBGene00003967|FB:FBgn0041100|UniProtKB:Q9UBS8|UniProtKB:O95376|UniProtKB:O60260|MGI:MGI:1344361|UniProtKB:Q7Z419|TAIR:locus:2057401|MGI:MGI:1344363	Communication:501741973		2023-06-07
AT4G19670	locus:2133935	AT4G19670	involved in	ubiquitin-dependent protein catabolic process	GO:0006511	7522	P	catabolic process	IBA	none	PANTHER:PTN000188395|FB:FBgn0017418|WB:WBGene00003967|FB:FBgn0041100|UniProtKB:Q9UBS8|UniProtKB:O95376|UniProtKB:O60260|MGI:MGI:1344361|UniProtKB:Q7Z419|TAIR:locus:2057401|MGI:MGI:1344363	Communication:501741973		2023-06-07
AT4G19670	locus:2133935	AT4G19670	enables	ubiquitin protein ligase activity	GO:0061630	45660	F	transferase activity	IBA	none	PANTHER:PTN000188395|UniProtKB:Q9UBS8|FB:FBgn0041100|MGI:MGI:1355296|UniProtKB:P50876|FB:FBgn0017418|TAIR:locus:2057401|TAIR:locus:2101074|SGD:S000001725|TAIR:locus:2050100|UniProtKB:O60260|UniProtKB:Q6ZMZ0	Communication:501741973		2023-06-07
AT4G19670	locus:2133935	AT4G19670	enables	nucleic acid binding	GO:0003676	3376	F	nucleic acid binding	IEA	none	InterPro:IPR002156|InterPro:IPR036397	AnalysisReference:501756966		2023-06-11
AT4G19670	locus:2133935	AT4G19670	involved in	positive regulation of proteasomal ubiquitin-dependent protein catabolic process	GO:0032436	25430	P	protein metabolic process	IBA	none	PANTHER:PTN000188395|MGI:MGI:1355296	Communication:501741973		2021-03-31
AT4G19670	gene:6532552492	AT4G19670.6	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19670	locus:2133935	AT4G19670	involved in	positive regulation of proteasomal ubiquitin-dependent protein catabolic process	GO:0032436	25430	P	other cellular processes	IBA	none	PANTHER:PTN000188395|MGI:MGI:1355296	Communication:501741973		2021-03-31
AT4G19670	locus:2133935	AT4G19670	involved in	ubiquitin-dependent protein catabolic process	GO:0006511	7522	P	protein metabolic process	IBA	none	PANTHER:PTN000188395|FB:FBgn0017418|WB:WBGene00003967|FB:FBgn0041100|UniProtKB:Q9UBS8|UniProtKB:O95376|UniProtKB:O60260|MGI:MGI:1344361|UniProtKB:Q7Z419|TAIR:locus:2057401|MGI:MGI:1344363	Communication:501741973		2023-06-07
AT4G19670	locus:2133935	AT4G19670	enables	ubiquitin protein ligase activity	GO:0061630	45660	F	catalytic activity	IBA	none	PANTHER:PTN000188395|UniProtKB:Q9UBS8|FB:FBgn0041100|MGI:MGI:1355296|UniProtKB:P50876|FB:FBgn0017418|TAIR:locus:2057401|TAIR:locus:2101074|SGD:S000001725|TAIR:locus:2050100|UniProtKB:O60260|UniProtKB:Q6ZMZ0	Communication:501741973		2023-06-07
AT4G19670	locus:2133935	AT4G19670	enables	RNA-DNA hybrid ribonuclease activity	GO:0004523	4049	F	catalytic activity	IEA	none	InterPro:IPR002156	AnalysisReference:501756966		2023-06-11
AT4G19670	locus:2133935	AT4G19670	part of	ubiquitin ligase complex	GO:0000151	725	C	other cellular components	IBA	none	PANTHER:PTN000188395|UniProtKB:O60260|WB:WBGene00003967	Communication:501741973		2023-04-13
AT4G19670	locus:2133935	AT4G19670	involved in	positive regulation of proteasomal ubiquitin-dependent protein catabolic process	GO:0032436	25430	P	other metabolic processes	IBA	none	PANTHER:PTN000188395|MGI:MGI:1355296	Communication:501741973		2021-03-31
AT4G19670	locus:2133935	AT4G19670	involved in	positive regulation of proteasomal ubiquitin-dependent protein catabolic process	GO:0032436	25430	P	catabolic process	IBA	none	PANTHER:PTN000188395|MGI:MGI:1355296	Communication:501741973		2021-03-31
AT4G19670	locus:2133935	AT4G19670	is active in	cytoplasm	GO:0005737	231	C	cytoplasm	IBA	none	PANTHER:PTN000188395|ZFIN:ZDB-GENE-050417-109|UniProtKB:Q9Y4X5|UniProtKB:O60260|TAIR:locus:2101074|TAIR:locus:2057401|WB:WBGene00003967|FB:FBgn0017418|WB:WBGene00016158|UniProtKB:O95376|RGD:61797|UniProtKB:Q6ZMZ0|UniProtKB:Q7Z419|UniProtKB:P50876|MGI:MGI:1922484|UniProtKB:Q9UBS8|MGI:MGI:1355296|FB:FBgn0041100	Communication:501741973		2023-06-07
AT4G19670	locus:2133935	AT4G19670	enables	ubiquitin conjugating enzyme binding	GO:0031624	21915	F	protein binding	IBA	none	PANTHER:PTN000188395|UniProtKB:O60260|UniProtKB:Q9Y4X5|FB:FBgn0041100|UniProtKB:O95376|WB:WBGene00003967|FB:FBgn0017418	Communication:501741973		2023-06-07
AT4G19670	locus:2133935	AT4G19670	involved in	ubiquitin-dependent protein catabolic process	GO:0006511	7522	P	other cellular processes	IBA	none	PANTHER:PTN000188395|FB:FBgn0017418|WB:WBGene00003967|FB:FBgn0041100|UniProtKB:Q9UBS8|UniProtKB:O95376|UniProtKB:O60260|MGI:MGI:1344361|UniProtKB:Q7Z419|TAIR:locus:2057401|MGI:MGI:1344363	Communication:501741973		2023-06-07
AT4G19670	gene:6532556650	AT4G19670.7	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19680	locus:2133950	AT4G19680	enables	zinc ion transmembrane transporter activity	GO:0005385	4663	F	transporter activity	IBA	none	PANTHER:PTN000115815|UniProtKB:Q9NY26|PomBase:SPBC16D10.06|MGI:MGI:2147269|TAIR:locus:2087705|TAIR:locus:2206011|MGI:MGI:2684326|SGD:S000001658|UniProtKB:Q4WHX5|SGD:S000004120|UniProtKB:O04089|UniProtKB:P0A8H3|FB:FBgn0033097|FB:FBgn0033096|MGI:MGI:1917056|SGD:S000003224|TAIR:locus:2062576|UniProtKB:Q9NP94|MGI:MGI:1353474	Communication:501741973		2023-06-07
AT4G19680	locus:2133950	AT4G19680	is active in	plasma membrane	GO:0005886	570	C	plasma membrane	IBA	none	PANTHER:PTN000867664|TAIR:locus:2206011|UniProtKB:A3BI11|UniProtKB:Q6L8G0|TAIR:locus:2133965|UniProtKB:Q6ZJ91|UniProtKB:Q75HB1|SGD:S000003224|PomBase:SPBC16D10.06|SGD:S000004120|UniProtKB:Q6L8G1	Communication:501741973		2023-06-07
AT4G19680	locus:2133950	AT4G19680	acts upstream of or within	iron ion transport	GO:0006826	6103	P	transport	TAS	none	NONE	Publication:501680620|PMID:11500563  	TIGR	2003-04-17
AT4G19680	locus:2133950	AT4G19680	enables	iron ion transmembrane transporter activity	GO:0005381	2923	F	transporter activity	IGI	Functional complementation in heterologous system		Publication:501680206|PMID:11389759  	TAIR	2003-10-11
AT4G19680	locus:2133950	AT4G19680	involved in	zinc ion transmembrane transport	GO:0071577	34306	P	transport	IBA	none	PANTHER:PTN000115815|UniProtKB:P0A8H3|FB:FBgn0033097|SGD:S000004120|TAIR:locus:2123787|UniProtKB:Q4WKR5|UniProtKB:Q4WHX5|TAIR:locus:2062576|SGD:S000003224|UniProtKB:Q9NP94|CGD:CAL0000174214|FB:FBgn0033096|TAIR:locus:2087705|PomBase:SPBC16D10.06|MGI:MGI:2147269|UniProtKB:Q9NY26|TAIR:locus:2174098|MGI:MGI:2684326	Communication:501741973		2023-06-07
AT4G19680	locus:2133950	AT4G19680	involved in	zinc ion transmembrane transport	GO:0071577	34306	P	other cellular processes	IBA	none	PANTHER:PTN000115815|UniProtKB:P0A8H3|FB:FBgn0033097|SGD:S000004120|TAIR:locus:2123787|UniProtKB:Q4WKR5|UniProtKB:Q4WHX5|TAIR:locus:2062576|SGD:S000003224|UniProtKB:Q9NP94|CGD:CAL0000174214|FB:FBgn0033096|TAIR:locus:2087705|PomBase:SPBC16D10.06|MGI:MGI:2147269|UniProtKB:Q9NY26|TAIR:locus:2174098|MGI:MGI:2684326	Communication:501741973		2023-06-07
AT4G19680	locus:2133950	AT4G19680	acts upstream of or within	zinc ion transport	GO:0006829	7618	P	transport	TAS	none	NONE	Publication:501680620|PMID:11500563  	TIGR	2003-04-17
AT4G19680	locus:2133950	AT4G19680	enables	iron ion transmembrane transporter activity	GO:0005381	2923	F	transporter activity	TAS	none		Publication:501680620|PMID:11500563  	TIGR	2003-04-17
AT4G19680	locus:2133950	AT4G19680	enables	zinc ion transmembrane transporter activity	GO:0005385	4663	F	transporter activity	TAS	none		Publication:501680620|PMID:11500563  	TIGR	2003-04-17
AT4G19690	locus:2133965	AT4G19690	enables	zinc ion transmembrane transporter activity	GO:0005385	4663	F	transporter activity	IBA	none	PANTHER:PTN000115815|UniProtKB:Q9NY26|PomBase:SPBC16D10.06|MGI:MGI:2147269|TAIR:locus:2087705|TAIR:locus:2206011|MGI:MGI:2684326|SGD:S000001658|UniProtKB:Q4WHX5|SGD:S000004120|UniProtKB:O04089|UniProtKB:P0A8H3|FB:FBgn0033097|FB:FBgn0033096|MGI:MGI:1917056|SGD:S000003224|TAIR:locus:2062576|UniProtKB:Q9NP94|MGI:MGI:1353474	Communication:501741973		2023-06-07
AT4G19690	locus:2133965	AT4G19690	enables	iron ion transmembrane transporter activity	GO:0005381	2923	F	transporter activity	IGI	Functional complementation in heterologous system	SGD:S000004662|SGD:S000004938	Publication:501721148|PMID:17337631  	TAIR	2018-10-31
AT4G19690	locus:2133965	AT4G19690	enables	iron ion transmembrane transporter activity	GO:0005381	2923	F	transporter activity	TAS	none		Publication:501680620|PMID:11500563  	TIGR	2003-04-17
AT4G19690	locus:2133965	AT4G19690	acts upstream of or within	response to bacterium	GO:0009617	7120	P	response to external stimulus	IEP	Protein levels (e.g. Western blots)	NONE	Publication:501729634|PMID:19121106  	TAIR	2009-02-04
AT4G19690	locus:2133965	AT4G19690	enables	manganese ion transmembrane transporter activity	GO:0005384	3130	F	transporter activity	TAS	none		Publication:501680620|PMID:11500563  	TIGR	2003-04-17
AT4G19690	locus:2133965	AT4G19690	acts upstream of or within	zinc ion transport	GO:0006829	7618	P	transport	TAS	none	NONE	Publication:501680620|PMID:11500563  	TIGR	2003-04-17
AT4G19690	locus:2133965	AT4G19690	involved in	zinc ion transmembrane transport	GO:0071577	34306	P	other cellular processes	IBA	none	PANTHER:PTN000115815|UniProtKB:P0A8H3|FB:FBgn0033097|SGD:S000004120|TAIR:locus:2123787|UniProtKB:Q4WKR5|UniProtKB:Q4WHX5|TAIR:locus:2062576|SGD:S000003224|UniProtKB:Q9NP94|CGD:CAL0000174214|FB:FBgn0033096|TAIR:locus:2087705|PomBase:SPBC16D10.06|MGI:MGI:2147269|UniProtKB:Q9NY26|TAIR:locus:2174098|MGI:MGI:2684326	Communication:501741973		2023-06-07
AT4G19690	locus:2133965	AT4G19690	enables	cadmium ion transmembrane transporter activity	GO:0015086	1749	F	transporter activity	TAS	none		Publication:501680620|PMID:11500563  	TIGR	2003-04-17
AT4G19690	gene:1005027765	AT4G19690.2	located in	chloroplast	GO:0009507	175	C	chloroplast	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19690	locus:2133965	AT4G19690	involved in	zinc ion transmembrane transport	GO:0071577	34306	P	transport	IBA	none	PANTHER:PTN000115815|UniProtKB:P0A8H3|FB:FBgn0033097|SGD:S000004120|TAIR:locus:2123787|UniProtKB:Q4WKR5|UniProtKB:Q4WHX5|TAIR:locus:2062576|SGD:S000003224|UniProtKB:Q9NP94|CGD:CAL0000174214|FB:FBgn0033096|TAIR:locus:2087705|PomBase:SPBC16D10.06|MGI:MGI:2147269|UniProtKB:Q9NY26|TAIR:locus:2174098|MGI:MGI:2684326	Communication:501741973		2023-06-07
AT4G19690	locus:2133965	AT4G19690	acts upstream of or within	response to bacterium	GO:0009617	7120	P	response to biotic stimulus	IEP	Protein levels (e.g. Western blots)	NONE	Publication:501729634|PMID:19121106  	TAIR	2009-02-04
AT4G19690	locus:2133965	AT4G19690	located in	trans-Golgi network	GO:0005802	57	C	other intracellular components	IDA	immunolocalization		Publication:501743278|PMID:21628566  	mnhindt	2011-08-11
AT4G19690	locus:2133965	AT4G19690	located in	trans-Golgi network	GO:0005802	57	C	Golgi apparatus	IDA	immunolocalization		Publication:501743278|PMID:21628566  	mnhindt	2011-08-11
AT4G19690	locus:2133965	AT4G19690	enables	zinc ion transmembrane transporter activity	GO:0005385	4663	F	transporter activity	TAS	none		Publication:501680620|PMID:11500563  	TIGR	2003-04-17
AT4G19690	locus:2133965	AT4G19690	enables	iron ion transmembrane transporter activity	GO:0005381	2923	F	transporter activity	TAS	none		Publication:1376|PMID:10394943  	TIGR	2003-04-17
AT4G19690	locus:2133965	AT4G19690	located in	endosome	GO:0005768	272	C	endosome	IDA	immunolocalization		Publication:501760930|PMID:24843126  	TAIR	2015-08-24
AT4G19690	locus:2133965	AT4G19690	located in	plasma membrane	GO:0005886	570	C	plasma membrane	IDA	immunolocalization		Publication:501743278|PMID:21628566  	mnhindt	2011-08-11
AT4G19690	locus:2133965	AT4G19690	acts upstream of or within	manganese ion transport	GO:0006828	6274	P	transport	TAS	none	NONE	Publication:501680620|PMID:11500563  	TIGR	2003-04-17
AT4G19690	locus:2133965	AT4G19690	acts upstream of or within	nickel cation transport	GO:0015675	6484	P	transport	IMP	biochemical/chemical analysis	NONE	Publication:501743529|PMID:21742768  	TAIR	2011-09-26
AT4G19690	locus:2133965	AT4G19690	located in	trans-Golgi network	GO:0005802	57	C	other intracellular components	IDA	immunolocalization		Publication:501760930|PMID:24843126  	TAIR	2015-08-24
AT4G19690	locus:2133965	AT4G19690	located in	trans-Golgi network	GO:0005802	57	C	Golgi apparatus	IDA	immunolocalization		Publication:501760930|PMID:24843126  	TAIR	2015-08-24
AT4G19690	locus:2133965	AT4G19690	enables	copper ion transmembrane transporter activity	GO:0005375	1993	F	transporter activity	IGI	Functional complementation in heterologous system	SGD:S000006328	Publication:501721148|PMID:17337631  	TAIR	2020-12-21
AT4G19690	locus:2133965	AT4G19690	acts upstream of or within	cadmium ion transport	GO:0015691	5276	P	transport	TAS	none	NONE	Publication:501680620|PMID:11500563  	TIGR	2003-04-17
AT4G19690	locus:2133965	AT4G19690	acts upstream of or within	metal ion transport	GO:0030001	6337	P	transport	IMP	Analysis of overexpression/ectopic expression phenotype	NONE	Publication:501743278|PMID:21628566  	mnhindt	2011-08-11
AT4G19690	locus:2133965	AT4G19690	acts upstream of or within	iron ion transport	GO:0006826	6103	P	transport	TAS	none	NONE	Publication:501680620|PMID:11500563  	TIGR	2003-04-17
AT4G19690	locus:2133965	AT4G19690	enables	iron ion transmembrane transporter activity	GO:0005381	2923	F	transporter activity	TAS	original experiments are traceable through an article		Publication:1376|PMID:10394943  	TAIR	2003-03-29
AT4G19690	gene:2133964	AT4G19690.1	located in	chloroplast	GO:0009507	175	C	chloroplast	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19690	locus:2133965	AT4G19690	is active in	plasma membrane	GO:0005886	570	C	plasma membrane	IBA	none	PANTHER:PTN000867664|TAIR:locus:2206011|UniProtKB:A3BI11|UniProtKB:Q6L8G0|TAIR:locus:2133965|UniProtKB:Q6ZJ91|UniProtKB:Q75HB1|SGD:S000003224|PomBase:SPBC16D10.06|SGD:S000004120|UniProtKB:Q6L8G1	Communication:501741973		2023-06-07
AT4G19690	locus:2133965	AT4G19690	acts upstream of or within	intracellular iron ion homeostasis	GO:0006879	6100	P	cellular homeostasis	IMP	none	NONE	Publication:501682382|PMID:12207649  	TIGR	2003-04-17
AT4G19700	locus:2133990	AT4G19700	acts upstream of or within	response to gibberellin	GO:0009739	11405	P	response to endogenous stimulus	IEP	Transcript levels (e.g.microarray data)	NONE	Publication:501723369|PMID:17933900  	TAIR	2009-02-18
AT4G19700	locus:2133990	AT4G19700	enables	cysteine-type endopeptidase inhibitor activity	GO:0004869	2037	F	enzyme regulator activity	IDA	Enzyme assays		Publication:501745152|PMID:21926169  	TAIR	2023-02-24
AT4G19700	locus:2133990	AT4G19700	acts upstream of or within	defense response to fungus	GO:0050832	18013	P	response to stress	IMP	RNAi experiments	NONE	Publication:501740087|PMID:20921156  	TAIR	2011-03-22
AT4G19700	locus:2133990	AT4G19700	involved in	proteasome-mediated ubiquitin-dependent protein catabolic process	GO:0043161	18924	P	other cellular processes	IEA	none	UniPathway:UPA00144	AnalysisReference:501757242		2023-06-11
AT4G19700	locus:2133990	AT4G19700	located in	nucleus	GO:0005634	537	C	nucleus	IDA	localization of GFP/YFP fusion protein		Publication:501740087|PMID:20921156  	TAIR	2011-03-22
AT4G19700	locus:2133990	AT4G19700	acts upstream of or within	defense response to bacterium	GO:0042742	14972	P	response to external stimulus	IMP	RNAi experiments	NONE	Publication:501740087|PMID:20921156  	TAIR	2011-03-22
AT4G19700	locus:2133990	AT4G19700	enables	ubiquitin-protein transferase activity	GO:0004842	4573	F	catalytic activity	IDA	Enzyme assays		Publication:501740087|PMID:20921156  	TAIR	2011-03-22
AT4G19700	locus:2133990	AT4G19700	acts upstream of or within	negative regulation of programmed cell death	GO:0043069	17979	P	other cellular processes	IMP	Analysis of overexpression/ectopic expression phenotype	NONE	Publication:501745152|PMID:21926169  	TAIR	2012-08-20
AT4G19700	locus:2133990	AT4G19700	acts upstream of or within	response to gibberellin	GO:0009739	11405	P	response to endogenous stimulus	IEP	Transcript levels (e.g. RT-PCR)	NONE	Publication:501723369|PMID:17933900  	TAIR	2009-02-18
AT4G19700	locus:2133990	AT4G19700	acts upstream of or within	defense response to fungus	GO:0050832	18013	P	response to external stimulus	IMP	RNAi experiments	NONE	Publication:501740087|PMID:20921156  	TAIR	2011-03-22
AT4G19700	gene:2133989	AT4G19700.1	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19700	locus:2133990	AT4G19700	involved in	regulation of programmed cell death	GO:0043067	17978	P	other cellular processes	IBA	none	PANTHER:PTN002447940|TAIR:locus:2153227|TAIR:locus:2133990|TAIR:locus:2207385|TAIR:locus:2089225	Communication:501741973		2023-06-07
AT4G19700	locus:2133990	AT4G19700	acts upstream of or within	response to gibberellin	GO:0009739	11405	P	response to chemical	IEP	Transcript levels (e.g.microarray data)	NONE	Publication:501723369|PMID:17933900  	TAIR	2009-02-18
AT4G19700	locus:2133990	AT4G19700	involved in	proteasome-mediated ubiquitin-dependent protein catabolic process	GO:0043161	18924	P	other metabolic processes	IEA	none	UniPathway:UPA00144	AnalysisReference:501757242		2023-06-11
AT4G19700	locus:2133990	AT4G19700	acts upstream of or within	defense response to bacterium	GO:0042742	14972	P	response to stress	IMP	RNAi experiments	NONE	Publication:501740087|PMID:20921156  	TAIR	2011-03-22
AT4G19700	locus:2133990	AT4G19700	acts upstream of or within	defense response to fungus	GO:0050832	18013	P	response to biotic stimulus	IMP	RNAi experiments	NONE	Publication:501740087|PMID:20921156  	TAIR	2011-03-22
AT4G19700	locus:2133990	AT4G19700	enables	ubiquitin-protein transferase activity	GO:0004842	4573	F	catalytic activity	IBA	none	PANTHER:PTN001802598|TAIR:locus:2153227|TAIR:locus:2133990	Communication:501741973		2022-04-20
AT4G19700	locus:2133990	AT4G19700	involved in	proteasome-mediated ubiquitin-dependent protein catabolic process	GO:0043161	18924	P	protein metabolic process	IEA	none	UniPathway:UPA00144	AnalysisReference:501757242		2023-06-11
AT4G19700	locus:2133990	AT4G19700	enables	ubiquitin-protein transferase activity	GO:0004842	4573	F	transferase activity	IBA	none	PANTHER:PTN001802598|TAIR:locus:2153227|TAIR:locus:2133990	Communication:501741973		2022-04-20
AT4G19700	locus:2133990	AT4G19700	located in	nucleus	GO:0005634	537	C	nucleus	IDA	localization of GFP/YFP fusion protein		Publication:501745152|PMID:21926169  	TAIR	2012-08-20
AT4G19700	locus:2133990	AT4G19700	involved in	proteasome-mediated ubiquitin-dependent protein catabolic process	GO:0043161	18924	P	catabolic process	IEA	none	UniPathway:UPA00144	AnalysisReference:501757242		2023-06-11
AT4G19700	locus:2133990	AT4G19700	acts upstream of or within	defense response to bacterium	GO:0042742	14972	P	response to biotic stimulus	IMP	RNAi experiments	NONE	Publication:501740087|PMID:20921156  	TAIR	2011-03-22
AT4G19700	locus:2133990	AT4G19700	enables	ubiquitin-protein transferase activity	GO:0004842	4573	F	transferase activity	IDA	Enzyme assays		Publication:501740087|PMID:20921156  	TAIR	2011-03-22
AT4G19700	locus:2133990	AT4G19700	acts upstream of or within	response to gibberellin	GO:0009739	11405	P	response to chemical	IEP	Transcript levels (e.g. RT-PCR)	NONE	Publication:501723369|PMID:17933900  	TAIR	2009-02-18
AT4G19700	locus:2133990	AT4G19700	involved in	regulation of programmed cell death	GO:0043067	17978	P	cell death	IBA	none	PANTHER:PTN002447940|TAIR:locus:2153227|TAIR:locus:2133990|TAIR:locus:2207385|TAIR:locus:2089225	Communication:501741973		2023-06-07
AT4G19700	locus:2133990	AT4G19700	acts upstream of or within	negative regulation of programmed cell death	GO:0043069	17979	P	cell death	IMP	Analysis of overexpression/ectopic expression phenotype	NONE	Publication:501745152|PMID:21926169  	TAIR	2012-08-20
AT4G19710	locus:2133995	AT4G19710	involved in	lysine biosynthetic process via diaminopimelate	GO:0009089	6213	P	biosynthetic process	IEA	none	UniPathway:UPA00034	AnalysisReference:501757242		2023-06-11
AT4G19710	locus:2133995	AT4G19710	acts upstream of or within	aspartate family amino acid biosynthetic process	GO:0009067	5179	P	other metabolic processes	TAS	none	NONE	Publication:4919|PMID:8204822   	TIGR	2003-04-17
AT4G19710	locus:2133995	AT4G19710	involved in	homoserine biosynthetic process	GO:0009090	6019	P	biosynthetic process	IBA	none	PANTHER:PTN000487079|CGD:CAL0000188452|SGD:S000003900	Communication:501741973		2022-11-23
AT4G19710	locus:2133995	AT4G19710	enables	homoserine dehydrogenase activity	GO:0004412	2735	F	catalytic activity	IBA	none	PANTHER:PTN000487079|TAIR:locus:2029564|TAIR:locus:2133995|SGD:S000003900	Communication:501741973		2023-06-07
AT4G19710	locus:2133995	AT4G19710	involved in	homoserine biosynthetic process	GO:0009090	6019	P	other metabolic processes	IBA	none	PANTHER:PTN000487079|CGD:CAL0000188452|SGD:S000003900	Communication:501741973		2022-11-23
AT4G19710	locus:2133995	AT4G19710	involved in	threonine biosynthetic process	GO:0009088	7433	P	biosynthetic process	IEA	none	UniPathway:UPA00050	AnalysisReference:501757242		2023-06-11
AT4G19710	locus:2133995	AT4G19710	involved in	threonine biosynthetic process	GO:0009088	7433	P	other metabolic processes	IEA	none	UniPathway:UPA00050	AnalysisReference:501757242		2023-06-11
AT4G19710	locus:2133995	AT4G19710	enables	aspartate kinase activity	GO:0004072	1615	F	kinase activity	IDA	Enzyme assays		Publication:501717923|PMID:16216875  	TAIR	2005-11-21
AT4G19710	locus:2133995	AT4G19710	involved in	aspartate family amino acid biosynthetic process	GO:0009067	5179	P	other cellular processes	IBA	none	PANTHER:PTN000487079|SGD:S000003900|CGD:CAL0000188452	Communication:501741973		2023-06-07
AT4G19710	locus:2133995	AT4G19710	involved in	lysine biosynthetic process via diaminopimelate	GO:0009089	6213	P	other metabolic processes	IEA	none	UniPathway:UPA00034	AnalysisReference:501757242		2023-06-11
AT4G19710	locus:2133995	AT4G19710	involved in	threonine biosynthetic process	GO:0009088	7433	P	other cellular processes	IEA	none	UniPathway:UPA00050	AnalysisReference:501757242		2023-06-11
AT4G19710	locus:2133995	AT4G19710	involved in	lysine biosynthetic process via diaminopimelate	GO:0009089	6213	P	other cellular processes	IEA	none	UniPathway:UPA00034	AnalysisReference:501757242		2023-06-11
AT4G19710	locus:2133995	AT4G19710	acts upstream of or within	aspartate family amino acid biosynthetic process	GO:0009067	5179	P	other cellular processes	TAS	none	NONE	Publication:4919|PMID:8204822   	TIGR	2003-04-17
AT4G19710	locus:2133995	AT4G19710	involved in	aspartate family amino acid biosynthetic process	GO:0009067	5179	P	other metabolic processes	IBA	none	PANTHER:PTN000487079|SGD:S000003900|CGD:CAL0000188452	Communication:501741973		2023-06-07
AT4G19710	locus:2133995	AT4G19710	acts upstream of or within	aspartate family amino acid biosynthetic process	GO:0009067	5179	P	biosynthetic process	TAS	none	NONE	Publication:4919|PMID:8204822   	TIGR	2003-04-17
AT4G19710	locus:2133995	AT4G19710	involved in	aspartate family amino acid biosynthetic process	GO:0009067	5179	P	biosynthetic process	IBA	none	PANTHER:PTN000487079|SGD:S000003900|CGD:CAL0000188452	Communication:501741973		2023-06-07
AT4G19710	gene:2133994	AT4G19710.1	located in	chloroplast	GO:0009507	175	C	chloroplast	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19710	gene:1006228954	AT4G19710.2	located in	chloroplast stroma	GO:0009570	178	C	chloroplast	HDA	none	NONE	Publication:501735990|PMID:20061580  	nrolland	2021-04-07
AT4G19710	locus:2133995	AT4G19710	enables	aspartate kinase activity	GO:0004072	1615	F	transferase activity	IDA	Enzyme assays		Publication:501717923|PMID:16216875  	TAIR	2005-11-21
AT4G19710	gene:1006228954	AT4G19710.2	located in	chloroplast stroma	GO:0009570	178	C	plastid	HDA	none	NONE	Publication:501735990|PMID:20061580  	nrolland	2021-04-07
AT4G19710	locus:2133995	AT4G19710	enables	NADP binding	GO:0050661	17565	F	nucleotide binding	IEA	none	InterPro:IPR005106	AnalysisReference:501756966		2023-06-11
AT4G19710	gene:1006228954	AT4G19710.2	located in	chloroplast	GO:0009507	175	C	chloroplast	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19710	locus:2133995	AT4G19710	enables	homoserine dehydrogenase activity	GO:0004412	2735	F	catalytic activity	IDA	Enzyme assays		Publication:501717923|PMID:16216875  	TAIR	2005-11-21
AT4G19710	gene:2133994	AT4G19710.1	located in	chloroplast	GO:0009507	175	C	chloroplast	HDA	none	NONE	Publication:501724486|PMID:18431481  	kvanwijk	2021-04-07
AT4G19710	locus:2133995	AT4G19710	involved in	homoserine biosynthetic process	GO:0009090	6019	P	other cellular processes	IBA	none	PANTHER:PTN000487079|CGD:CAL0000188452|SGD:S000003900	Communication:501741973		2022-11-23
AT4G19710	gene:1006228954	AT4G19710.2	located in	chloroplast	GO:0009507	175	C	chloroplast	HDA	none	NONE	Publication:501724486|PMID:18431481  	kvanwijk	2021-04-07
AT4G19720	locus:2134010	AT4G19720	involved in	chitin catabolic process	GO:0006032	5376	P	catabolic process	IBA	none	PANTHER:PTN000126545|FB:FBgn0022700|TAIR:locus:2134025|UniProtKB:Q8I4R4|UniProtKB:Q9BZP6|RGD:1303058|WB:WBGene00000503|MGI:MGI:1932052|FB:FBgn0034582|FB:FBgn0038180|UniProtKB:A0A072VEP0|UniProtKB:E9QRF2|UniProtKB:A0A072UR65|UniProtKB:Q13231	Communication:501741973		2023-06-07
AT4G19720	locus:2134010	AT4G19720	is active in	extracellular region	GO:0005576	294	C	extracellular region	IBA	none	PANTHER:PTN000126545|FB:FBgn0038180|FB:FBgn0034582|UniProtKB:Q13231|UniProtKB:C8VAA5|FB:FBgn0022702|MGI:MGI:106661|RGD:620874|FB:FBgn0013763|FB:FBgn0022700|RGD:1303058|UniProtKB:Q9BZP6|UniProtKB:P36222|FB:FBgn0020415	Communication:501741973		2023-06-07
AT4G19720	locus:2134010	AT4G19720	enables	chitin binding	GO:0008061	1899	F	other binding	IBA	none	PANTHER:PTN000126545|UniProtKB:P36222|FB:FBgn0013763|MGI:MGI:1932052|FB:FBgn0022700|UniProtKB:Q15782|FB:FBgn0038180|UniProtKB:Q8I4R4	Communication:501741973		2023-06-07
AT4G19720	gene:2134009	AT4G19720.1	located in	extracellular region	GO:0005576	294	C	extracellular region	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19720	locus:2134010	AT4G19720	involved in	chitin catabolic process	GO:0006032	5376	P	other metabolic processes	IBA	none	PANTHER:PTN000126545|FB:FBgn0022700|TAIR:locus:2134025|UniProtKB:Q8I4R4|UniProtKB:Q9BZP6|RGD:1303058|WB:WBGene00000503|MGI:MGI:1932052|FB:FBgn0034582|FB:FBgn0038180|UniProtKB:A0A072VEP0|UniProtKB:E9QRF2|UniProtKB:A0A072UR65|UniProtKB:Q13231	Communication:501741973		2023-06-07
AT4G19720	locus:2134010	AT4G19720	involved in	carbohydrate metabolic process	GO:0005975	5291	P	carbohydrate metabolic process	IEA	none	InterPro:IPR001223	AnalysisReference:501756966		2023-06-11
AT4G19720	locus:2134010	AT4G19720	enables	chitinase activity	GO:0004568	1902	F	hydrolase activity	IBA	none	PANTHER:PTN000126545|UniProtKB:Q8I4R4|TAIR:locus:2134025|FB:FBgn0022700|RGD:1303058|UniProtKB:Q9BZP6|UniProtKB:A0A072VEP0|FB:FBgn0038180|FB:FBgn0034582|MGI:MGI:1932052|WB:WBGene00000503|UniProtKB:Q13231|UniProtKB:A0A072UR65|UniProtKB:E9QRF2	Communication:501741973		2023-06-07
AT4G19730	locus:2134020	AT4G19730	involved in	chitin catabolic process	GO:0006032	5376	P	catabolic process	IBA	none	PANTHER:PTN000126545|FB:FBgn0022700|TAIR:locus:2134025|UniProtKB:Q8I4R4|UniProtKB:Q9BZP6|RGD:1303058|WB:WBGene00000503|MGI:MGI:1932052|FB:FBgn0034582|FB:FBgn0038180|UniProtKB:A0A072VEP0|UniProtKB:E9QRF2|UniProtKB:A0A072UR65|UniProtKB:Q13231	Communication:501741973		2023-06-07
AT4G19730	locus:2134020	AT4G19730	enables	chitinase activity	GO:0004568	1902	F	hydrolase activity	IBA	none	PANTHER:PTN000126545|UniProtKB:Q8I4R4|TAIR:locus:2134025|FB:FBgn0022700|RGD:1303058|UniProtKB:Q9BZP6|UniProtKB:A0A072VEP0|FB:FBgn0038180|FB:FBgn0034582|MGI:MGI:1932052|WB:WBGene00000503|UniProtKB:Q13231|UniProtKB:A0A072UR65|UniProtKB:E9QRF2	Communication:501741973		2023-06-07
AT4G19730	locus:2134020	AT4G19730	involved in	chitin catabolic process	GO:0006032	5376	P	other metabolic processes	IBA	none	PANTHER:PTN000126545|FB:FBgn0022700|TAIR:locus:2134025|UniProtKB:Q8I4R4|UniProtKB:Q9BZP6|RGD:1303058|WB:WBGene00000503|MGI:MGI:1932052|FB:FBgn0034582|FB:FBgn0038180|UniProtKB:A0A072VEP0|UniProtKB:E9QRF2|UniProtKB:A0A072UR65|UniProtKB:Q13231	Communication:501741973		2023-06-07
AT4G19730	locus:2134020	AT4G19730	involved in	carbohydrate metabolic process	GO:0005975	5291	P	carbohydrate metabolic process	IEA	none	InterPro:IPR001223	AnalysisReference:501756966		2023-06-11
AT4G19730	locus:2134020	AT4G19730	enables	chitin binding	GO:0008061	1899	F	other binding	IBA	none	PANTHER:PTN000126545|UniProtKB:P36222|FB:FBgn0013763|MGI:MGI:1932052|FB:FBgn0022700|UniProtKB:Q15782|FB:FBgn0038180|UniProtKB:Q8I4R4	Communication:501741973		2023-06-07
AT4G19730	gene:2134019	AT4G19730.1	located in	extracellular region	GO:0005576	294	C	extracellular region	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19730	locus:2134020	AT4G19730	is active in	extracellular region	GO:0005576	294	C	extracellular region	IBA	none	PANTHER:PTN000126545|FB:FBgn0038180|FB:FBgn0034582|UniProtKB:Q13231|UniProtKB:C8VAA5|FB:FBgn0022702|MGI:MGI:106661|RGD:620874|FB:FBgn0013763|FB:FBgn0022700|RGD:1303058|UniProtKB:Q9BZP6|UniProtKB:P36222|FB:FBgn0020415	Communication:501741973		2023-06-07
AT4G19740	locus:2133930	AT4G19740	enables	chitin binding	GO:0008061	1899	F	other binding	IBA	none	PANTHER:PTN000126545|UniProtKB:P36222|FB:FBgn0013763|MGI:MGI:1932052|FB:FBgn0022700|UniProtKB:Q15782|FB:FBgn0038180|UniProtKB:Q8I4R4	Communication:501741973		2023-06-07
AT4G19740	locus:2133930	AT4G19740	involved in	chitin catabolic process	GO:0006032	5376	P	catabolic process	IBA	none	PANTHER:PTN000126545|FB:FBgn0022700|TAIR:locus:2134025|UniProtKB:Q8I4R4|UniProtKB:Q9BZP6|RGD:1303058|WB:WBGene00000503|MGI:MGI:1932052|FB:FBgn0034582|FB:FBgn0038180|UniProtKB:A0A072VEP0|UniProtKB:E9QRF2|UniProtKB:A0A072UR65|UniProtKB:Q13231	Communication:501741973		2023-06-07
AT4G19740	locus:2133930	AT4G19740	involved in	chitin catabolic process	GO:0006032	5376	P	other metabolic processes	IBA	none	PANTHER:PTN000126545|FB:FBgn0022700|TAIR:locus:2134025|UniProtKB:Q8I4R4|UniProtKB:Q9BZP6|RGD:1303058|WB:WBGene00000503|MGI:MGI:1932052|FB:FBgn0034582|FB:FBgn0038180|UniProtKB:A0A072VEP0|UniProtKB:E9QRF2|UniProtKB:A0A072UR65|UniProtKB:Q13231	Communication:501741973		2023-06-07
AT4G19740	locus:2133930	AT4G19740	enables	chitinase activity	GO:0004568	1902	F	hydrolase activity	IBA	none	PANTHER:PTN000126545|UniProtKB:Q8I4R4|TAIR:locus:2134025|FB:FBgn0022700|RGD:1303058|UniProtKB:Q9BZP6|UniProtKB:A0A072VEP0|FB:FBgn0038180|FB:FBgn0034582|MGI:MGI:1932052|WB:WBGene00000503|UniProtKB:Q13231|UniProtKB:A0A072UR65|UniProtKB:E9QRF2	Communication:501741973		2023-06-07
AT4G19740	gene:2133929	AT4G19740.1	located in	extracellular region	GO:0005576	294	C	extracellular region	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19740	locus:2133930	AT4G19740	is active in	extracellular region	GO:0005576	294	C	extracellular region	IBA	none	PANTHER:PTN000126545|FB:FBgn0038180|FB:FBgn0034582|UniProtKB:Q13231|UniProtKB:C8VAA5|FB:FBgn0022702|MGI:MGI:106661|RGD:620874|FB:FBgn0013763|FB:FBgn0022700|RGD:1303058|UniProtKB:Q9BZP6|UniProtKB:P36222|FB:FBgn0020415	Communication:501741973		2023-06-07
AT4G19740	locus:2133930	AT4G19740	involved in	carbohydrate metabolic process	GO:0005975	5291	P	carbohydrate metabolic process	IEA	none	InterPro:IPR001223	AnalysisReference:501756966		2023-06-11
AT4G19750	locus:2133940	AT4G19750	enables	chitin binding	GO:0008061	1899	F	other binding	IBA	none	PANTHER:PTN000126545|UniProtKB:P36222|FB:FBgn0013763|MGI:MGI:1932052|FB:FBgn0022700|UniProtKB:Q15782|FB:FBgn0038180|UniProtKB:Q8I4R4	Communication:501741973		2023-06-07
AT4G19750	locus:2133940	AT4G19750	involved in	chitin catabolic process	GO:0006032	5376	P	other metabolic processes	IBA	none	PANTHER:PTN000126545|FB:FBgn0022700|TAIR:locus:2134025|UniProtKB:Q8I4R4|UniProtKB:Q9BZP6|RGD:1303058|WB:WBGene00000503|MGI:MGI:1932052|FB:FBgn0034582|FB:FBgn0038180|UniProtKB:A0A072VEP0|UniProtKB:E9QRF2|UniProtKB:A0A072UR65|UniProtKB:Q13231	Communication:501741973		2023-06-07
AT4G19750	gene:2133939	AT4G19750.1	located in	extracellular region	GO:0005576	294	C	extracellular region	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19750	locus:2133940	AT4G19750	involved in	carbohydrate metabolic process	GO:0005975	5291	P	carbohydrate metabolic process	IEA	none	InterPro:IPR001223	AnalysisReference:501756966		2023-06-11
AT4G19750	locus:2133940	AT4G19750	is active in	extracellular region	GO:0005576	294	C	extracellular region	IBA	none	PANTHER:PTN000126545|FB:FBgn0038180|FB:FBgn0034582|UniProtKB:Q13231|UniProtKB:C8VAA5|FB:FBgn0022702|MGI:MGI:106661|RGD:620874|FB:FBgn0013763|FB:FBgn0022700|RGD:1303058|UniProtKB:Q9BZP6|UniProtKB:P36222|FB:FBgn0020415	Communication:501741973		2023-06-07
AT4G19750	locus:2133940	AT4G19750	involved in	chitin catabolic process	GO:0006032	5376	P	catabolic process	IBA	none	PANTHER:PTN000126545|FB:FBgn0022700|TAIR:locus:2134025|UniProtKB:Q8I4R4|UniProtKB:Q9BZP6|RGD:1303058|WB:WBGene00000503|MGI:MGI:1932052|FB:FBgn0034582|FB:FBgn0038180|UniProtKB:A0A072VEP0|UniProtKB:E9QRF2|UniProtKB:A0A072UR65|UniProtKB:Q13231	Communication:501741973		2023-06-07
AT4G19750	locus:2133940	AT4G19750	enables	chitinase activity	GO:0004568	1902	F	hydrolase activity	IBA	none	PANTHER:PTN000126545|UniProtKB:Q8I4R4|TAIR:locus:2134025|FB:FBgn0022700|RGD:1303058|UniProtKB:Q9BZP6|UniProtKB:A0A072VEP0|FB:FBgn0038180|FB:FBgn0034582|MGI:MGI:1932052|WB:WBGene00000503|UniProtKB:Q13231|UniProtKB:A0A072UR65|UniProtKB:E9QRF2	Communication:501741973		2023-06-07
AT4G19760	locus:2133955	AT4G19760	involved in	chitin catabolic process	GO:0006032	5376	P	other metabolic processes	IBA	none	PANTHER:PTN000126545|FB:FBgn0022700|TAIR:locus:2134025|UniProtKB:Q8I4R4|UniProtKB:Q9BZP6|RGD:1303058|WB:WBGene00000503|MGI:MGI:1932052|FB:FBgn0034582|FB:FBgn0038180|UniProtKB:A0A072VEP0|UniProtKB:E9QRF2|UniProtKB:A0A072UR65|UniProtKB:Q13231	Communication:501741973		2023-06-07
AT4G19760	locus:2133955	AT4G19760	involved in	chitin catabolic process	GO:0006032	5376	P	catabolic process	IBA	none	PANTHER:PTN000126545|FB:FBgn0022700|TAIR:locus:2134025|UniProtKB:Q8I4R4|UniProtKB:Q9BZP6|RGD:1303058|WB:WBGene00000503|MGI:MGI:1932052|FB:FBgn0034582|FB:FBgn0038180|UniProtKB:A0A072VEP0|UniProtKB:E9QRF2|UniProtKB:A0A072UR65|UniProtKB:Q13231	Communication:501741973		2023-06-07
AT4G19760	locus:2133955	AT4G19760	involved in	carbohydrate metabolic process	GO:0005975	5291	P	carbohydrate metabolic process	IEA	none	InterPro:IPR001223	AnalysisReference:501756966		2023-06-11
AT4G19760	locus:2133955	AT4G19760	enables	chitinase activity	GO:0004568	1902	F	hydrolase activity	IBA	none	PANTHER:PTN000126545|UniProtKB:Q8I4R4|TAIR:locus:2134025|FB:FBgn0022700|RGD:1303058|UniProtKB:Q9BZP6|UniProtKB:A0A072VEP0|FB:FBgn0038180|FB:FBgn0034582|MGI:MGI:1932052|WB:WBGene00000503|UniProtKB:Q13231|UniProtKB:A0A072UR65|UniProtKB:E9QRF2	Communication:501741973		2023-06-07
AT4G19760	locus:2133955	AT4G19760	is active in	extracellular region	GO:0005576	294	C	extracellular region	IBA	none	PANTHER:PTN000126545|FB:FBgn0038180|FB:FBgn0034582|UniProtKB:Q13231|UniProtKB:C8VAA5|FB:FBgn0022702|MGI:MGI:106661|RGD:620874|FB:FBgn0013763|FB:FBgn0022700|RGD:1303058|UniProtKB:Q9BZP6|UniProtKB:P36222|FB:FBgn0020415	Communication:501741973		2023-06-07
AT4G19760	locus:2133955	AT4G19760	enables	chitin binding	GO:0008061	1899	F	other binding	IBA	none	PANTHER:PTN000126545|UniProtKB:P36222|FB:FBgn0013763|MGI:MGI:1932052|FB:FBgn0022700|UniProtKB:Q15782|FB:FBgn0038180|UniProtKB:Q8I4R4	Communication:501741973		2023-06-07
AT4G19760	gene:2133954	AT4G19760.1	located in	extracellular region	GO:0005576	294	C	extracellular region	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19770	locus:2133970	AT4G19770	is active in	extracellular region	GO:0005576	294	C	extracellular region	IBA	none	PANTHER:PTN000126545|FB:FBgn0038180|FB:FBgn0034582|UniProtKB:Q13231|UniProtKB:C8VAA5|FB:FBgn0022702|MGI:MGI:106661|RGD:620874|FB:FBgn0013763|FB:FBgn0022700|RGD:1303058|UniProtKB:Q9BZP6|UniProtKB:P36222|FB:FBgn0020415	Communication:501741973		2023-06-07
AT4G19770	locus:2133970	AT4G19770	involved in	chitin catabolic process	GO:0006032	5376	P	catabolic process	IBA	none	PANTHER:PTN000126545|FB:FBgn0022700|TAIR:locus:2134025|UniProtKB:Q8I4R4|UniProtKB:Q9BZP6|RGD:1303058|WB:WBGene00000503|MGI:MGI:1932052|FB:FBgn0034582|FB:FBgn0038180|UniProtKB:A0A072VEP0|UniProtKB:E9QRF2|UniProtKB:A0A072UR65|UniProtKB:Q13231	Communication:501741973		2023-06-07
AT4G19770	locus:2133970	AT4G19770	involved in	chitin catabolic process	GO:0006032	5376	P	other metabolic processes	IBA	none	PANTHER:PTN000126545|FB:FBgn0022700|TAIR:locus:2134025|UniProtKB:Q8I4R4|UniProtKB:Q9BZP6|RGD:1303058|WB:WBGene00000503|MGI:MGI:1932052|FB:FBgn0034582|FB:FBgn0038180|UniProtKB:A0A072VEP0|UniProtKB:E9QRF2|UniProtKB:A0A072UR65|UniProtKB:Q13231	Communication:501741973		2023-06-07
AT4G19770	gene:2133969	AT4G19770.1	located in	extracellular region	GO:0005576	294	C	extracellular region	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19770	locus:2133970	AT4G19770	involved in	carbohydrate metabolic process	GO:0005975	5291	P	carbohydrate metabolic process	IEA	none	InterPro:IPR001223|InterPro:IPR001579	AnalysisReference:501756966		2023-06-11
AT4G19770	locus:2133970	AT4G19770	enables	chitin binding	GO:0008061	1899	F	other binding	IBA	none	PANTHER:PTN000126545|UniProtKB:P36222|FB:FBgn0013763|MGI:MGI:1932052|FB:FBgn0022700|UniProtKB:Q15782|FB:FBgn0038180|UniProtKB:Q8I4R4	Communication:501741973		2023-06-07
AT4G19770	locus:2133970	AT4G19770	enables	chitinase activity	GO:0004568	1902	F	hydrolase activity	IBA	none	PANTHER:PTN000126545|UniProtKB:Q8I4R4|TAIR:locus:2134025|FB:FBgn0022700|RGD:1303058|UniProtKB:Q9BZP6|UniProtKB:A0A072VEP0|FB:FBgn0038180|FB:FBgn0034582|MGI:MGI:1932052|WB:WBGene00000503|UniProtKB:Q13231|UniProtKB:A0A072UR65|UniProtKB:E9QRF2	Communication:501741973		2023-06-07
AT4G19800	locus:2134015	AT4G19800	involved in	chitin catabolic process	GO:0006032	5376	P	catabolic process	IBA	none	PANTHER:PTN000126545|FB:FBgn0022700|TAIR:locus:2134025|UniProtKB:Q8I4R4|UniProtKB:Q9BZP6|RGD:1303058|WB:WBGene00000503|MGI:MGI:1932052|FB:FBgn0034582|FB:FBgn0038180|UniProtKB:A0A072VEP0|UniProtKB:E9QRF2|UniProtKB:A0A072UR65|UniProtKB:Q13231	Communication:501741973		2023-06-07
AT4G19800	locus:2134015	AT4G19800	involved in	carbohydrate metabolic process	GO:0005975	5291	P	carbohydrate metabolic process	IEA	none	InterPro:IPR001223	AnalysisReference:501756966		2023-06-11
AT4G19800	locus:2134015	AT4G19800	is active in	extracellular region	GO:0005576	294	C	extracellular region	IBA	none	PANTHER:PTN000126545|FB:FBgn0038180|FB:FBgn0034582|UniProtKB:Q13231|UniProtKB:C8VAA5|FB:FBgn0022702|MGI:MGI:106661|RGD:620874|FB:FBgn0013763|FB:FBgn0022700|RGD:1303058|UniProtKB:Q9BZP6|UniProtKB:P36222|FB:FBgn0020415	Communication:501741973		2023-06-07
AT4G19800	gene:2134014	AT4G19800.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19800	locus:2134015	AT4G19800	involved in	chitin catabolic process	GO:0006032	5376	P	other metabolic processes	IBA	none	PANTHER:PTN000126545|FB:FBgn0022700|TAIR:locus:2134025|UniProtKB:Q8I4R4|UniProtKB:Q9BZP6|RGD:1303058|WB:WBGene00000503|MGI:MGI:1932052|FB:FBgn0034582|FB:FBgn0038180|UniProtKB:A0A072VEP0|UniProtKB:E9QRF2|UniProtKB:A0A072UR65|UniProtKB:Q13231	Communication:501741973		2023-06-07
AT4G19800	locus:2134015	AT4G19800	enables	chitinase activity	GO:0004568	1902	F	hydrolase activity	IBA	none	PANTHER:PTN000126545|UniProtKB:Q8I4R4|TAIR:locus:2134025|FB:FBgn0022700|RGD:1303058|UniProtKB:Q9BZP6|UniProtKB:A0A072VEP0|FB:FBgn0038180|FB:FBgn0034582|MGI:MGI:1932052|WB:WBGene00000503|UniProtKB:Q13231|UniProtKB:A0A072UR65|UniProtKB:E9QRF2	Communication:501741973		2023-06-07
AT4G19800	locus:2134015	AT4G19800	enables	chitin binding	GO:0008061	1899	F	other binding	IBA	none	PANTHER:PTN000126545|UniProtKB:P36222|FB:FBgn0013763|MGI:MGI:1932052|FB:FBgn0022700|UniProtKB:Q15782|FB:FBgn0038180|UniProtKB:Q8I4R4	Communication:501741973		2023-06-07
AT4G19810	locus:2134025	AT4G19810	acts upstream of or within	response to jasmonic acid	GO:0009753	11419	P	response to chemical	IEP	Transcript levels (e.g. RT-PCR)	NONE	Publication:501742192|PMID:21390509  	TAIR	2011-04-08
AT4G19810	locus:2134025	AT4G19810	enables	endochitinase activity	GO:0008843	2232	F	hydrolase activity	IDA	in vitro assay		Publication:501742192|PMID:21390509  	TAIR	2011-04-08
AT4G19810	locus:2134025	AT4G19810	acts upstream of or within	chitin catabolic process	GO:0006032	5376	P	other metabolic processes	IDA	in vitro assay	NONE	Publication:501742192|PMID:21390509  	TAIR	2011-04-08
AT4G19810	locus:2134025	AT4G19810	enables	chitinase activity	GO:0004568	1902	F	hydrolase activity	IDA	none		Publication:501751277|PMID:22936594  		2021-04-01
AT4G19810	locus:2134025	AT4G19810	involved in	chitin catabolic process	GO:0006032	5376	P	catabolic process	IBA	none	PANTHER:PTN000126545|FB:FBgn0022700|TAIR:locus:2134025|UniProtKB:Q8I4R4|UniProtKB:Q9BZP6|RGD:1303058|WB:WBGene00000503|MGI:MGI:1932052|FB:FBgn0034582|FB:FBgn0038180|UniProtKB:A0A072VEP0|UniProtKB:E9QRF2|UniProtKB:A0A072UR65|UniProtKB:Q13231	Communication:501741973		2023-06-07
AT4G19810	locus:2134025	AT4G19810	acts upstream of or within	response to abscisic acid	GO:0009737	11395	P	response to endogenous stimulus	IEP	Transcript levels (e.g. RT-PCR)	NONE	Publication:501742192|PMID:21390509  	TAIR	2011-04-08
AT4G19810	locus:2134025	AT4G19810	enables	chitinase activity	GO:0004568	1902	F	hydrolase activity	IBA	none	PANTHER:PTN000126545|UniProtKB:Q8I4R4|TAIR:locus:2134025|FB:FBgn0022700|RGD:1303058|UniProtKB:Q9BZP6|UniProtKB:A0A072VEP0|FB:FBgn0038180|FB:FBgn0034582|MGI:MGI:1932052|WB:WBGene00000503|UniProtKB:Q13231|UniProtKB:A0A072UR65|UniProtKB:E9QRF2	Communication:501741973		2023-06-07
AT4G19810	locus:2134025	AT4G19810	acts upstream of or within	chitin catabolic process	GO:0006032	5376	P	catabolic process	IDA	in vitro assay	NONE	Publication:501742192|PMID:21390509  	TAIR	2011-04-08
AT4G19810	locus:2134025	AT4G19810	enables	chitin binding	GO:0008061	1899	F	other binding	IBA	none	PANTHER:PTN000126545|UniProtKB:P36222|FB:FBgn0013763|MGI:MGI:1932052|FB:FBgn0022700|UniProtKB:Q15782|FB:FBgn0038180|UniProtKB:Q8I4R4	Communication:501741973		2023-06-07
AT4G19810	locus:2134025	AT4G19810	acts upstream of or within	response to jasmonic acid	GO:0009753	11419	P	response to endogenous stimulus	IEP	Transcript levels (e.g. RT-PCR)	NONE	Publication:501742192|PMID:21390509  	TAIR	2011-04-08
AT4G19810	locus:2134025	AT4G19810	located in	secretory vesicle	GO:0099503	51212	C	cytoplasm	HDA	none		Publication:501761766|PMID:25293756  	dszymanski	2019-04-04
AT4G19810	locus:2134025	AT4G19810	acts upstream of or within	response to salt stress	GO:0009651	7182	P	response to abiotic stimulus	IEP	Transcript levels (e.g. RT-PCR)	NONE	Publication:501742192|PMID:21390509  	TAIR	2011-04-08
AT4G19810	gene:2134024	AT4G19810.1	located in	extracellular region	GO:0005576	294	C	extracellular region	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19810	locus:2134025	AT4G19810	acts upstream of or within	response to salt stress	GO:0009651	7182	P	response to stress	IEP	Transcript levels (e.g. RT-PCR)	NONE	Publication:501742192|PMID:21390509  	TAIR	2011-04-08
AT4G19810	locus:2134025	AT4G19810	involved in	chitin catabolic process	GO:0006032	5376	P	other metabolic processes	IBA	none	PANTHER:PTN000126545|FB:FBgn0022700|TAIR:locus:2134025|UniProtKB:Q8I4R4|UniProtKB:Q9BZP6|RGD:1303058|WB:WBGene00000503|MGI:MGI:1932052|FB:FBgn0034582|FB:FBgn0038180|UniProtKB:A0A072VEP0|UniProtKB:E9QRF2|UniProtKB:A0A072UR65|UniProtKB:Q13231	Communication:501741973		2023-06-07
AT4G19810	gene:2134024	AT4G19810.1	located in	plant-type cell wall	GO:0009505	156	C	cell wall	HDA	none	NONE	Publication:501714811|PMID:15593128  	kvanwijk	2022-02-08
AT4G19810	locus:2134025	AT4G19810	involved in	chitin catabolic process	GO:0006032	5376	P	other metabolic processes	IEA	none	UniPathway:UPA00349	AnalysisReference:501757242		2023-06-11
AT4G19810	locus:2134025	AT4G19810	acts upstream of or within	response to abscisic acid	GO:0009737	11395	P	response to chemical	IEP	Transcript levels (e.g. RT-PCR)	NONE	Publication:501742192|PMID:21390509  	TAIR	2011-04-08
AT4G19810	locus:2134025	AT4G19810	is active in	extracellular region	GO:0005576	294	C	extracellular region	IBA	none	PANTHER:PTN000126545|FB:FBgn0038180|FB:FBgn0034582|UniProtKB:Q13231|UniProtKB:C8VAA5|FB:FBgn0022702|MGI:MGI:106661|RGD:620874|FB:FBgn0013763|FB:FBgn0022700|RGD:1303058|UniProtKB:Q9BZP6|UniProtKB:P36222|FB:FBgn0020415	Communication:501741973		2023-06-07
AT4G19810	locus:2134025	AT4G19810	involved in	chitin catabolic process	GO:0006032	5376	P	catabolic process	IEA	none	UniPathway:UPA00349	AnalysisReference:501757242		2023-06-11
AT4G19810	locus:2134025	AT4G19810	located in	secretory vesicle	GO:0099503	51212	C	other intracellular components	HDA	none		Publication:501761766|PMID:25293756  	dszymanski	2019-04-04
AT4G19810	locus:2134025	AT4G19810	enables	exochitinase activity	GO:0035885	38082	F	hydrolase activity	IDA	in vitro assay		Publication:501742192|PMID:21390509  	TAIR	2012-06-13
AT4G19810	gene:6532545683	AT4G19810.2	located in	extracellular region	GO:0005576	294	C	extracellular region	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19820	gene:2134029	AT4G19820.1	located in	extracellular region	GO:0005576	294	C	extracellular region	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19820	locus:2134030	AT4G19820	involved in	chitin catabolic process	GO:0006032	5376	P	other metabolic processes	IBA	none	PANTHER:PTN000126545|FB:FBgn0022700|TAIR:locus:2134025|UniProtKB:Q8I4R4|UniProtKB:Q9BZP6|RGD:1303058|WB:WBGene00000503|MGI:MGI:1932052|FB:FBgn0034582|FB:FBgn0038180|UniProtKB:A0A072VEP0|UniProtKB:E9QRF2|UniProtKB:A0A072UR65|UniProtKB:Q13231	Communication:501741973		2023-06-07
AT4G19820	locus:2134030	AT4G19820	enables	chitinase activity	GO:0004568	1902	F	hydrolase activity	IBA	none	PANTHER:PTN000126545|UniProtKB:Q8I4R4|TAIR:locus:2134025|FB:FBgn0022700|RGD:1303058|UniProtKB:Q9BZP6|UniProtKB:A0A072VEP0|FB:FBgn0038180|FB:FBgn0034582|MGI:MGI:1932052|WB:WBGene00000503|UniProtKB:Q13231|UniProtKB:A0A072UR65|UniProtKB:E9QRF2	Communication:501741973		2023-06-07
AT4G19820	locus:2134030	AT4G19820	involved in	carbohydrate metabolic process	GO:0005975	5291	P	carbohydrate metabolic process	IEA	none	InterPro:IPR001223	AnalysisReference:501756966		2023-06-11
AT4G19820	locus:2134030	AT4G19820	is active in	extracellular region	GO:0005576	294	C	extracellular region	IBA	none	PANTHER:PTN000126545|FB:FBgn0038180|FB:FBgn0034582|UniProtKB:Q13231|UniProtKB:C8VAA5|FB:FBgn0022702|MGI:MGI:106661|RGD:620874|FB:FBgn0013763|FB:FBgn0022700|RGD:1303058|UniProtKB:Q9BZP6|UniProtKB:P36222|FB:FBgn0020415	Communication:501741973		2023-06-07
AT4G19820	locus:2134030	AT4G19820	involved in	chitin catabolic process	GO:0006032	5376	P	catabolic process	IBA	none	PANTHER:PTN000126545|FB:FBgn0022700|TAIR:locus:2134025|UniProtKB:Q8I4R4|UniProtKB:Q9BZP6|RGD:1303058|WB:WBGene00000503|MGI:MGI:1932052|FB:FBgn0034582|FB:FBgn0038180|UniProtKB:A0A072VEP0|UniProtKB:E9QRF2|UniProtKB:A0A072UR65|UniProtKB:Q13231	Communication:501741973		2023-06-07
AT4G19820	locus:2134030	AT4G19820	enables	chitin binding	GO:0008061	1899	F	other binding	IBA	none	PANTHER:PTN000126545|UniProtKB:P36222|FB:FBgn0013763|MGI:MGI:1932052|FB:FBgn0022700|UniProtKB:Q15782|FB:FBgn0038180|UniProtKB:Q8I4R4	Communication:501741973		2023-06-07
AT4G19829	locus:4515103425	AT4G19829	is active in	cellular_component	GO:0005575	163	C	unknown cellular components	ND	Unknown cellular component	NONE	Communication:1345790	TAIR	2022-02-01
AT4G19829	locus:4515103425	AT4G19829	involved in	biological_process	GO:0008150	5239	P	unknown biological processes	ND	Unknown biological process	NONE	Communication:1345790	TAIR	2022-02-01
AT4G19829	locus:4515103425	AT4G19829	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	Unknown molecular function		Communication:1345790	TAIR	2011-10-26
AT4G19830	locus:2134035	AT4G19830	located in	chloroplast thylakoid lumen	GO:0009543	705	C	plastid	IDA	protein separation and fragment identification	NONE	Publication:501795501|PMID:34360890  	TAIR	2021-10-19
AT4G19830	locus:2134035	AT4G19830	acts upstream of or within	response to light stimulus	GO:0009416	6176	P	response to light stimulus	IEA	traceable computational prediction	AGI_LocusCode:AT5G47640	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19830	locus:2134035	AT4G19830	located in	chloroplast thylakoid lumen	GO:0009543	705	C	other intracellular components	IDA	protein separation and fragment identification	NONE	Publication:501795501|PMID:34360890  	TAIR	2021-10-19
AT4G19830	locus:2134035	AT4G19830	located in	chloroplast thylakoid lumen	GO:0009543	705	C	thylakoid	IDA	protein separation and fragment identification	NONE	Publication:501795501|PMID:34360890  	TAIR	2021-10-19
AT4G19830	locus:2134035	AT4G19830	enables	peptidyl-prolyl cis-trans isomerase activity	GO:0003755	3623	F	catalytic activity	IEA	none	EC:5.2.1.8	AnalysisReference:501756967		2023-05-23
AT4G19830	locus:2134035	AT4G19830	located in	chloroplast thylakoid lumen	GO:0009543	705	C	chloroplast	IDA	protein separation and fragment identification	NONE	Publication:501795501|PMID:34360890  	TAIR	2021-10-19
AT4G19840	locus:2133945	AT4G19840	acts upstream of or within	response to insect	GO:0009625	7128	P	response to biotic stimulus	IDA	bioassay	NONE	Publication:501738008|PMID:20442276  	sdinant	2010-06-28
AT4G19840	locus:2133945	AT4G19840	enables	proteoglycan binding	GO:0043394	21082	F	protein binding	IDA	in vitro binding assay	NONE	Publication:501738008|PMID:20442276  	sdinant	2010-06-28
AT4G19840	locus:2133945	AT4G19840	acts upstream of or within	response to insect	GO:0009625	7128	P	response to external stimulus	IDA	bioassay	NONE	Publication:501738008|PMID:20442276  	sdinant	2010-06-28
AT4G19840	locus:2133945	AT4G19840	enables	carbohydrate binding	GO:0030246	8584	F	carbohydrate binding	ISS	Recognized domains	PP2 domain	Publication:501682998|PMID:12529520  	TAIR	2004-04-20
AT4G19840	locus:2133945	AT4G19840	enables	proteoglycan binding	GO:0043394	21082	F	other binding	IDA	in vitro binding assay	NONE	Publication:501738008|PMID:20442276  	sdinant	2010-06-28
AT4G19840	locus:2133945	AT4G19840	enables	carbohydrate binding	GO:0030246	8584	F	carbohydrate binding	ISS	Sequence similarity (homologue of/most closely related to)	GB:L31550	Publication:501682998|PMID:12529520  	TAIR	2008-03-13
AT4G19850	locus:2133960	AT4G19850	enables	carbohydrate binding	GO:0030246	8584	F	carbohydrate binding	ISS	Sequence similarity (homologue of/most closely related to)	GB:L31550	Publication:501682998|PMID:12529520  	TAIR	2008-03-13
AT4G19850	gene:5019474410	AT4G19850.2	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19850	gene:6532556759	AT4G19850.3	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19850	locus:2133960	AT4G19850	involved in	biological_process	GO:0008150	5239	P	unknown biological processes	ND	none	NONE	Communication:1345790	TIGR	2022-02-01
AT4G19860	locus:2133975	AT4G19860	acts upstream of or within	phospholipid catabolic process	GO:0009395	6740	P	catabolic process	IDA	Enzyme assays	NONE	Publication:501747132|PMID:22245677  	TAIR	2012-02-15
AT4G19860	locus:2133975	AT4G19860	located in	cytosol	GO:0005829	241	C	cytosol	IDA	Cell fractionation (for cellular component)		Publication:501747132|PMID:22245677  	TAIR	2012-02-15
AT4G19860	locus:2133975	AT4G19860	involved in	lipid metabolic process	GO:0006629	6186	P	lipid metabolic process	IBA	none	PANTHER:PTN000154155|UniProtKB:P04180|PomBase:SPBC776.14|TAIR:locus:2173244|MGI:MGI:2178076|SGD:S000002465|RGD:1302982|MGI:MGI:96755|SGD:S000005291|TAIR:locus:2024117|RGD:2993|UniProtKB:Q8NCC3|TAIR:locus:2133975	Communication:501741973		2023-06-07
AT4G19860	locus:2133975	AT4G19860	enables	phospholipase activity	GO:0004620	3702	F	hydrolase activity	IBA	none	PANTHER:PTN000154141|TAIR:locus:2133975	Communication:501741973		2021-06-03
AT4G19860	locus:2133975	AT4G19860	acts upstream of or within	phospholipid catabolic process	GO:0009395	6740	P	other cellular processes	IDA	Enzyme assays	NONE	Publication:501747132|PMID:22245677  	TAIR	2012-02-15
AT4G19860	locus:2133975	AT4G19860	acts upstream of or within	phospholipid catabolic process	GO:0009395	6740	P	lipid metabolic process	IDA	Enzyme assays	NONE	Publication:501747132|PMID:22245677  	TAIR	2012-02-15
AT4G19860	locus:2133975	AT4G19860	enables	phospholipase activity	GO:0004620	3702	F	hydrolase activity	IDA	Enzyme assays		Publication:501747132|PMID:22245677  	TAIR	2012-02-15
AT4G19860	locus:2133975	AT4G19860	enables	O-acyltransferase activity	GO:0008374	1172	F	transferase activity	IEA	none	InterPro:IPR003386	AnalysisReference:501756966		2023-05-23
AT4G19860	locus:2133975	AT4G19860	acts upstream of or within	phospholipid catabolic process	GO:0009395	6740	P	other metabolic processes	IDA	Enzyme assays	NONE	Publication:501747132|PMID:22245677  	TAIR	2012-02-15
AT4G19865	locus:1005716274	AT4G19865	involved in	biological_process	GO:0008150	5239	P	unknown biological processes	ND	none	NONE	Communication:1345790	TIGR	2022-02-01
AT4G19865	locus:1005716274	AT4G19865	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	Unknown molecular function		Communication:1345790	TAIR	2009-05-08
AT4G19865	gene:1005713975	AT4G19865.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19870	locus:2133985	AT4G19870	acts upstream of or within	regulation of gene expression	GO:0010468	28541	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT3G11100	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19870	locus:2133985	AT4G19870	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	none	NONE	Communication:1345790	TIGR	2003-04-17
AT4G19870	gene:2133984	AT4G19870.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19870	gene:1005713976	AT4G19870.2	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19880	locus:2134005	AT4G19880	enables	glutathione transferase activity	GO:0004364	2545	F	transferase activity	IBA	none	PANTHER:PTN000792402|SGD:S000004863|SGD:S000003386|CGD:CAL0000196854|SGD:S000001784	Communication:501741973		2023-06-07
AT4G19880	gene:1009022141	AT4G19880.3	located in	chloroplast	GO:0009507	175	C	chloroplast	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19880	locus:2134005	AT4G19880	acts upstream of or within	cellular response to hypoxia	GO:0071456	33992	P	response to abiotic stimulus	HEP	none	NONE	Publication:501786480|PMID:31519798  	baileyserr	2019-09-19
AT4G19880	locus:2134005	AT4G19880	located in	cytosol	GO:0005829	241	C	cytosol	HDA	none		Publication:501776792|PMID:28887381  	dszymanski	2019-04-04
AT4G19880	locus:2134005	AT4G19880	acts upstream of or within	cellular response to hypoxia	GO:0071456	33992	P	response to stress	HEP	none	NONE	Publication:501786480|PMID:31519798  	baileyserr	2019-09-19
AT4G19880	locus:2134005	AT4G19880	acts upstream of or within	cellular response to hypoxia	GO:0071456	33992	P	response to chemical	HEP	none	NONE	Publication:501786480|PMID:31519798  	baileyserr	2019-09-19
AT4G19880	locus:2134005	AT4G19880	acts upstream of or within	response to inorganic substance	GO:0010035	14767	P	response to chemical	IEA	traceable computational prediction	AGI_LocusCode:AT3G16857|AGI_LocusCode:AT4G31920	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19880	locus:2134005	AT4G19880	acts upstream of or within	cellular response to hypoxia	GO:0071456	33992	P	other cellular processes	HEP	none	NONE	Publication:501786480|PMID:31519798  	baileyserr	2019-09-19
AT4G19880	locus:2134005	AT4G19880	located in	chloroplast	GO:0009507	175	C	chloroplast	HDA	none		Publication:501761766|PMID:25293756  	dszymanski	2019-04-04
AT4G19880	locus:2134005	AT4G19880	is active in	cytoplasm	GO:0005737	231	C	cytoplasm	IBA	none	PANTHER:PTN000792402|SGD:S000004863|SGD:S000003386|CGD:CAL0000196854|SGD:S000001784	Communication:501741973		2023-06-07
AT4G19890	locus:6530298213	AT4G19890	enables	mRNA binding	GO:0003729	3091	F	RNA binding	IBA	none	PANTHER:PTN008722623|TAIR:locus:2151236|UniProtKB:O82178|TAIR:locus:2206420|TAIR:locus:2197424|TAIR:locus:2074924|UniProtKB:A0A1D6IEG9|TAIR:locus:2181286|TAIR:locus:2157607	Communication:501741973		2023-06-07
AT4G19890	locus:6530298213	AT4G19890	is active in	cytoplasm	GO:0005737	231	C	cytoplasm	IBA	none	PANTHER:PTN008722623|TAIR:locus:2197424|UniProtKB:A0A1D6IEG9|TAIR:locus:2074924|TAIR:locus:2157607|TAIR:locus:2125152|TAIR:locus:2181286|TAIR:locus:2157732|TAIR:locus:2139732|TAIR:locus:2086032	Communication:501741973		2023-06-07
AT4G19890	locus:6530298213	AT4G19890	involved in	mRNA processing	GO:0006397	6230	P	nucleobase-containing compound metabolic process	IBA	none	PANTHER:PTN008722623|TAIR:locus:2132937|TAIR:locus:2181286|TAIR:locus:2157607|TAIR:locus:2074924|UniProtKB:O82178	Communication:501741973		2023-06-07
AT4G19890	gene:6530297455	AT4G19890.1	located in	chloroplast	GO:0009507	175	C	chloroplast	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19890	locus:6530298213	AT4G19890	involved in	mRNA processing	GO:0006397	6230	P	other metabolic processes	IBA	none	PANTHER:PTN008722623|TAIR:locus:2132937|TAIR:locus:2181286|TAIR:locus:2157607|TAIR:locus:2074924|UniProtKB:O82178	Communication:501741973		2023-06-07
AT4G19900	locus:2119752	AT4G19900	enables	glycosyltransferase activity	GO:0016757	14916	F	transferase activity	ISS	Recognized domains		Communication:1674999	TAIR	2004-04-21
AT4G19900	gene:6532563067	AT4G19900.2	located in	mitochondrion	GO:0005739	486	C	mitochondrion	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19900	gene:2119751	AT4G19900.1	located in	Golgi apparatus	GO:0005794	48	C	Golgi apparatus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19905	locus:1009023388	AT4G19905	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	Unknown molecular function		Communication:1345790	TAIR	2021-12-15
AT4G19905	locus:1009023388	AT4G19905	involved in	biological_process	GO:0008150	5239	P	unknown biological processes	ND	Unknown biological process	NONE	Communication:1345790	TAIR	2022-02-01
AT4G19905	gene:1009022284	AT4G19905.1	located in	extracellular region	GO:0005576	294	C	extracellular region	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19910	locus:1006230307	AT4G19910	involved in	signal transduction	GO:0007165	7243	P	signal transduction	IEA	none	InterPro:IPR000157	AnalysisReference:501756966		2023-05-23
AT4G19910	gene:1006227849	AT4G19910.1	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19920	locus:2119762	AT4G19920	involved in	signal transduction	GO:0007165	7243	P	signal transduction	IEA	none	InterPro:IPR000157	AnalysisReference:501756966		2023-05-23
AT4G19920	locus:2119762	AT4G19920	is active in	cellular_component	GO:0005575	163	C	unknown cellular components	ND	Unknown cellular component	NONE	Communication:1345790	TAIR	2022-02-01
AT4G19925	gene:1006227850	AT4G19925.1	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19925	locus:1006230308	AT4G19925	involved in	signal transduction	GO:0007165	7243	P	signal transduction	IEA	none	InterPro:IPR000157	AnalysisReference:501756966		2023-05-23
AT4G19930	gene:2119771	AT4G19930.1	located in	mitochondrion	GO:0005739	486	C	mitochondrion	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19930	locus:2119772	AT4G19930	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	none	NONE	Communication:1345790	TIGR	2003-09-13
AT4G19930	locus:2119772	AT4G19930	involved in	biological_process	GO:0008150	5239	P	unknown biological processes	ND	Unknown biological process	NONE	Communication:1345790	TAIR	2022-02-01
AT4G19940	locus:2119787	AT4G19940	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	none	NONE	Communication:1345790	TIGR	2003-09-13
AT4G19940	gene:2119786	AT4G19940.1	located in	mitochondrion	GO:0005739	486	C	mitochondrion	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19940	locus:2119787	AT4G19940	involved in	biological_process	GO:0008150	5239	P	unknown biological processes	ND	Unknown biological process	NONE	Communication:1345790	TAIR	2022-02-01
AT4G19950	locus:2119807	AT4G19950	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	none	NONE	Communication:1345790	TIGR	2003-10-04
AT4G19950	locus:2119807	AT4G19950	acts upstream of or within	cellular developmental process	GO:0048869	25767	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT5G03150|AGI_LocusCode:AT5G16560	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19950	locus:2119807	AT4G19950	acts upstream of or within	growth	GO:0040007	10293	P	growth	IEA	traceable computational prediction	AGI_LocusCode:AT5G03150	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19950	gene:2119806	AT4G19950.1	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19960	locus:2119812	AT4G19960	acts upstream of or within	cotyledon development	GO:0048825	23442	P	embryo development	IMP	analysis of visible trait	NONE	Publication:501780715|PMID:30037141  	TAIR	2020-05-27
AT4G19960	locus:2119812	AT4G19960	acts upstream of or within	potassium ion transmembrane transport	GO:0071805	36074	P	other cellular processes	IMP	none		Publication:501736003|PMID:20057125  		2021-04-01
AT4G19960	locus:2119812	AT4G19960	involved in	potassium ion transmembrane transport	GO:0071805	36074	P	transport	IEA	none	InterPro:IPR003855	AnalysisReference:501756966		2023-06-11
AT4G19960	locus:2119812	AT4G19960	is active in	membrane	GO:0016020	453	C	other membranes	IBA	none	PANTHER:PTN000776785|TAIR:locus:2184722	Communication:501741973		2021-09-10
AT4G19960	locus:2119812	AT4G19960	enables	potassium ion transmembrane transporter activity	GO:0015079	3807	F	transporter activity	IBA	none	PANTHER:PTN000776785|UniProtKB:O49423|CGD:CAL0000188660|TAIR:locus:2128399|TAIR:locus:2142110|TAIR:locus:2184722|TAIR:locus:2045639	Communication:501741973		2023-06-07
AT4G19960	locus:2119812	AT4G19960	enables	potassium ion transmembrane transporter activity	GO:0015079	3807	F	transporter activity	IMP	Functional complementation		Publication:501736003|PMID:20057125  	TAIR	2010-02-04
AT4G19960	locus:2119812	AT4G19960	involved in	potassium ion transport	GO:0006813	6833	P	transport	IBA	none	PANTHER:PTN000776785|UniProtKB:O49423|TAIR:locus:2184722|CGD:CAL0000188660|TAIR:locus:2045639|TAIR:locus:2142110|UniProtKB:Q6VVA6	Communication:501741973		2023-06-07
AT4G19960	locus:2119812	AT4G19960	acts upstream of or within	cotyledon development	GO:0048825	23442	P	multicellular organism development	IMP	analysis of visible trait	NONE	Publication:501780715|PMID:30037141  	TAIR	2020-05-27
AT4G19960	locus:2119812	AT4G19960	involved in	potassium ion transmembrane transport	GO:0071805	36074	P	other cellular processes	IEA	none	InterPro:IPR003855	AnalysisReference:501756966		2023-06-11
AT4G19960	locus:2119812	AT4G19960	acts upstream of or within	potassium ion transmembrane transport	GO:0071805	36074	P	transport	IMP	none		Publication:501736003|PMID:20057125  		2021-04-01
AT4G19960	locus:2119812	AT4G19960	enables	potassium ion transmembrane transporter activity	GO:0015079	3807	F	transporter activity	ISS	Sequence similarity (homologue of/most closely related to)		Communication:1674994	TAIR	2003-03-29
AT4G19960	locus:2119812	AT4G19960	acts upstream of or within	cotyledon development	GO:0048825	23442	P	anatomical structure development	IMP	analysis of visible trait	NONE	Publication:501780715|PMID:30037141  	TAIR	2020-05-27
AT4G19960	locus:2119812	AT4G19960	acts upstream of or within	cotyledon development	GO:0048825	23442	P	reproduction	IMP	analysis of visible trait	NONE	Publication:501780715|PMID:30037141  	TAIR	2020-05-27
AT4G19960	locus:2119812	AT4G19960	acts upstream of or within	cotyledon development	GO:0048825	23442	P	post-embryonic development	IMP	analysis of visible trait	NONE	Publication:501780715|PMID:30037141  	TAIR	2020-05-27
AT4G19960	gene:6532546594	AT4G19960.4	located in	extracellular region	GO:0005576	294	C	extracellular region	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19970	locus:2119827	AT4G19970	acts upstream of or within	defense response to other organism	GO:0098542	46569	P	response to stress	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19970	locus:2119827	AT4G19970	acts upstream of or within	defense response to other organism	GO:0098542	46569	P	response to external stimulus	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19970	locus:2119827	AT4G19970	acts upstream of or within	response to oxidative stress	GO:0006979	6625	P	response to stress	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19970	locus:2119827	AT4G19970	acts upstream of or within	tissue development	GO:0009888	11998	P	anatomical structure development	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19970	locus:2119827	AT4G19970	acts upstream of or within	defense response to other organism	GO:0098542	46569	P	response to biotic stimulus	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19970	gene:2119826	AT4G19970.1	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19970	locus:2119827	AT4G19970	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	Unknown molecular function		Communication:1345790	TAIR	2021-10-28
AT4G19975	gene:6532562767	AT4G19975.1	located in	chloroplast	GO:0009507	175	C	chloroplast	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19975	locus:6532565056	AT4G19975	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	Unknown molecular function		Communication:1345790	TAIR	2021-10-28
AT4G19975	locus:6532565056	AT4G19975	acts upstream of or within	response to hypoxia	GO:0001666	10824	P	response to abiotic stimulus	IEA	traceable computational prediction	AGI_LocusCode:AT5G63790	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19975	locus:6532565056	AT4G19975	acts upstream of or within	response to hypoxia	GO:0001666	10824	P	response to stress	IEA	traceable computational prediction	AGI_LocusCode:AT5G63790	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19980	locus:2119727	AT4G19980	acts upstream of or within	plant epidermis development	GO:0090558	47687	P	anatomical structure development	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19980	gene:2119726	AT4G19980.1	located in	mitochondrion	GO:0005739	486	C	mitochondrion	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19980	locus:2119727	AT4G19980	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	none	NONE	Communication:1345790	TIGR	2003-10-04
AT4G19985	locus:505006501	AT4G19985	is active in	endoplasmic reticulum	GO:0005783	268	C	endoplasmic reticulum	IBA	none	PANTHER:PTN000331677|TAIR:locus:2143226	Communication:501741973		2021-04-02
AT4G19985	locus:505006501	AT4G19985	enables	glucosamine 6-phosphate N-acetyltransferase activity	GO:0004343	2478	F	transferase activity	IEA	none	EC:2.3.1.4	AnalysisReference:501756967		2023-06-11
AT4G19985	locus:505006501	AT4G19985	is active in	endoplasmic reticulum-Golgi intermediate compartment	GO:0005793	45	C	other intracellular components	IBA	none	PANTHER:PTN000331677|MGI:MGI:1858963	Communication:501741973		2021-04-02
AT4G19985	locus:505006501	AT4G19985	involved in	UDP-N-acetylglucosamine biosynthetic process	GO:0006048	4946	P	other cellular processes	IBA	none	PANTHER:PTN000331677|UniProtKB:Q5AHF9|SGD:S000001877	Communication:501741973		2022-09-29
AT4G19985	locus:505006501	AT4G19985	involved in	UDP-N-acetylglucosamine biosynthetic process	GO:0006048	4946	P	nucleobase-containing compound metabolic process	IEA	none	UniPathway:UPA00113	AnalysisReference:501757242		2023-05-23
AT4G19985	locus:505006501	AT4G19985	involved in	UDP-N-acetylglucosamine biosynthetic process	GO:0006048	4946	P	biosynthetic process	IEA	none	UniPathway:UPA00113	AnalysisReference:501757242		2023-05-23
AT4G19985	locus:505006501	AT4G19985	involved in	N-terminal protein amino acid acetylation	GO:0006474	4852	P	other metabolic processes	IDA	none		Publication:501792535|PMID:32633465  		2023-02-07
AT4G19985	gene:3703851	AT4G19985.1	located in	mitochondrion	GO:0005739	486	C	mitochondrion	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19985	locus:505006501	AT4G19985	involved in	UDP-N-acetylglucosamine biosynthetic process	GO:0006048	4946	P	other metabolic processes	IBA	none	PANTHER:PTN000331677|UniProtKB:Q5AHF9|SGD:S000001877	Communication:501741973		2022-09-29
AT4G19985	locus:505006501	AT4G19985	involved in	UDP-N-acetylglucosamine biosynthetic process	GO:0006048	4946	P	biosynthetic process	IBA	none	PANTHER:PTN000331677|UniProtKB:Q5AHF9|SGD:S000001877	Communication:501741973		2022-09-29
AT4G19985	locus:505006501	AT4G19985	involved in	UDP-N-acetylglucosamine biosynthetic process	GO:0006048	4946	P	biosynthetic process	IEA	none	InterPro:IPR039143	AnalysisReference:501756966		2023-05-23
AT4G19985	gene:6532552937	AT4G19985.2	located in	mitochondrion	GO:0005739	486	C	mitochondrion	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19985	locus:505006501	AT4G19985	enables	monosaccharide binding	GO:0048029	14714	F	carbohydrate binding	IBA	none	PANTHER:PTN000331677|RGD:1563144	Communication:501741973		2021-04-02
AT4G19985	gene:6532552920	AT4G19985.6	located in	mitochondrion	GO:0005739	486	C	mitochondrion	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19985	gene:6532559040	AT4G19985.7	located in	mitochondrion	GO:0005739	486	C	mitochondrion	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19985	gene:6532559042	AT4G19985.5	located in	mitochondrion	GO:0005739	486	C	mitochondrion	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19985	locus:505006501	AT4G19985	involved in	UDP-N-acetylglucosamine biosynthetic process	GO:0006048	4946	P	other metabolic processes	IEA	none	InterPro:IPR039143	AnalysisReference:501756966		2023-05-23
AT4G19985	locus:505006501	AT4G19985	enables	N-acetyltransferase activity	GO:0008080	1124	F	transferase activity	IDA	none		Publication:501792535|PMID:32633465  		2023-02-07
AT4G19985	locus:505006501	AT4G19985	enables	glucosamine 6-phosphate N-acetyltransferase activity	GO:0004343	2478	F	transferase activity	IBA	none	PANTHER:PTN000331677|TAIR:locus:2143226|UniProtKB:Q5AHF9|MGI:MGI:1858963|UniProtKB:Q4WCU5|UniProtKB:Q5U9F2|SGD:S000001877|RGD:1563144	Communication:501741973		2023-06-07
AT4G19985	locus:505006501	AT4G19985	located in	chloroplast	GO:0009507	175	C	chloroplast	IDA	localization of GFP/YFP fusion protein		Publication:501792535|PMID:32633465  	ifinkemeier	2021-01-06
AT4G19985	locus:505006501	AT4G19985	enables	N-acetyltransferase activity	GO:0008080	1124	F	transferase activity	IDA	protein separation and fragment identification		Publication:501792535|PMID:32633465  	ifinkemeier	2021-01-06
AT4G19985	locus:505006501	AT4G19985	involved in	UDP-N-acetylglucosamine biosynthetic process	GO:0006048	4946	P	nucleobase-containing compound metabolic process	IEA	none	InterPro:IPR039143	AnalysisReference:501756966		2023-05-23
AT4G19985	locus:505006501	AT4G19985	involved in	N-terminal protein amino acid acetylation	GO:0006474	4852	P	protein metabolic process	IDA	none		Publication:501792535|PMID:32633465  		2023-02-07
AT4G19985	locus:505006501	AT4G19985	is active in	endoplasmic reticulum-Golgi intermediate compartment	GO:0005793	45	C	cytoplasm	IBA	none	PANTHER:PTN000331677|MGI:MGI:1858963	Communication:501741973		2021-04-02
AT4G19985	locus:505006501	AT4G19985	involved in	UDP-N-acetylglucosamine biosynthetic process	GO:0006048	4946	P	other cellular processes	IEA	none	InterPro:IPR039143	AnalysisReference:501756966		2023-05-23
AT4G19985	locus:505006501	AT4G19985	is active in	Golgi apparatus	GO:0005794	48	C	Golgi apparatus	IBA	none	PANTHER:PTN000331677|MGI:MGI:1858963	Communication:501741973		2021-04-02
AT4G19985	locus:505006501	AT4G19985	involved in	UDP-N-acetylglucosamine biosynthetic process	GO:0006048	4946	P	nucleobase-containing compound metabolic process	IBA	none	PANTHER:PTN000331677|UniProtKB:Q5AHF9|SGD:S000001877	Communication:501741973		2022-09-29
AT4G19985	gene:6532559043	AT4G19985.4	located in	chloroplast	GO:0009507	175	C	chloroplast	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19985	locus:505006501	AT4G19985	enables	monosaccharide binding	GO:0048029	14714	F	other binding	IBA	none	PANTHER:PTN000331677|RGD:1563144	Communication:501741973		2021-04-02
AT4G19985	locus:505006501	AT4G19985	involved in	UDP-N-acetylglucosamine biosynthetic process	GO:0006048	4946	P	other metabolic processes	IEA	none	UniPathway:UPA00113	AnalysisReference:501757242		2023-05-23
AT4G19985	gene:6532559044	AT4G19985.3	located in	mitochondrion	GO:0005739	486	C	mitochondrion	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19985	locus:505006501	AT4G19985	involved in	UDP-N-acetylglucosamine biosynthetic process	GO:0006048	4946	P	other cellular processes	IEA	none	UniPathway:UPA00113	AnalysisReference:501757242		2023-05-23
AT4G19990	locus:2119737	AT4G19990	acts upstream of or within	regulation of defense response	GO:0031347	20899	P	response to stress	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT1G80840|AGI_LocusCode:AT4G31800|AGI_LocusCode:AT5G67300	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19990	gene:6532546888	AT4G19990.3	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19990	locus:2119737	AT4G19990	acts upstream of or within	RNA metabolic process	GO:0016070	4899	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT1G80840|AGI_LocusCode:AT2G22430|AGI_LocusCode:AT4G27410|AGI_LocusCode:AT5G65310|AGI_LocusCode:AT5G67300|AGI_LocusCode:AT4G15090	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19990	locus:2119737	AT4G19990	acts upstream of or within	defense response to fungus	GO:0050832	18013	P	response to stress	IEA	traceable computational prediction	AGI_LocusCode:AT1G80840|AGI_LocusCode:AT2G38470|AGI_LocusCode:AT4G31800|AGI_LocusCode:AT5G67300	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19990	locus:2119737	AT4G19990	acts upstream of or within	defense response to fungus	GO:0050832	18013	P	response to biotic stimulus	IEA	traceable computational prediction	AGI_LocusCode:AT1G80840|AGI_LocusCode:AT2G38470|AGI_LocusCode:AT4G31800|AGI_LocusCode:AT5G67300	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19990	gene:2119736	AT4G19990.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19990	locus:2119737	AT4G19990	acts upstream of or within	defense response to bacterium	GO:0042742	14972	P	response to stress	IEA	traceable computational prediction	AGI_LocusCode:AT1G80840|AGI_LocusCode:AT2G38470|AGI_LocusCode:AT4G31800|AGI_LocusCode:AT5G67300	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19990	locus:2119737	AT4G19990	acts upstream of or within	RNA metabolic process	GO:0016070	4899	P	nucleobase-containing compound metabolic process	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT1G80840|AGI_LocusCode:AT2G22430|AGI_LocusCode:AT4G27410|AGI_LocusCode:AT5G65310|AGI_LocusCode:AT5G67300|AGI_LocusCode:AT4G15090	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19990	locus:2119737	AT4G19990	acts upstream of or within	defense response to fungus	GO:0050832	18013	P	response to external stimulus	IEA	traceable computational prediction	AGI_LocusCode:AT1G80840|AGI_LocusCode:AT2G38470|AGI_LocusCode:AT4G31800|AGI_LocusCode:AT5G67300	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19990	gene:6530297457	AT4G19990.2	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19990	gene:6532546896	AT4G19990.5	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19990	locus:2119737	AT4G19990	acts upstream of or within	defense response to bacterium	GO:0042742	14972	P	response to external stimulus	IEA	traceable computational prediction	AGI_LocusCode:AT1G80840|AGI_LocusCode:AT2G38470|AGI_LocusCode:AT4G31800|AGI_LocusCode:AT5G67300	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19990	locus:2119737	AT4G19990	acts upstream of or within	defense response to bacterium	GO:0042742	14972	P	response to biotic stimulus	IEA	traceable computational prediction	AGI_LocusCode:AT1G80840|AGI_LocusCode:AT2G38470|AGI_LocusCode:AT4G31800|AGI_LocusCode:AT5G67300	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G19990	gene:6532557562	AT4G19990.7	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19990	gene:6532559539	AT4G19990.4	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19990	gene:6532545526	AT4G19990.6	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G19990	locus:2119737	AT4G19990	acts upstream of or within	response to organic cyclic compound	GO:0014070	26665	P	response to chemical	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT1G80840|AGI_LocusCode:AT4G31800|AGI_LocusCode:AT5G67300	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G20000	gene:2119756	AT4G20000.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G20000	locus:2119757	AT4G20000	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	none	NONE	Communication:1345790	TIGR	2003-09-27
AT4G20000	locus:2119757	AT4G20000	involved in	response to fungus	GO:0009620	7124	P	response to biotic stimulus	IDA	localization of GFP/YFP fusion protein	NONE	Publication:501798511|PMID:35406838  	TAIR	2022-04-24
AT4G20000	locus:2119757	AT4G20000	involved in	response to fungus	GO:0009620	7124	P	response to external stimulus	IEP	Transcript levels (e.g. RT-PCR)	NONE	Publication:501798511|PMID:35406838  	TAIR	2022-04-24
AT4G20000	locus:2119757	AT4G20000	involved in	response to fungus	GO:0009620	7124	P	response to biotic stimulus	IEP	Transcript levels (e.g. RT-PCR)	NONE	Publication:501798511|PMID:35406838  	TAIR	2022-04-24
AT4G20000	locus:2119757	AT4G20000	involved in	response to fungus	GO:0009620	7124	P	response to external stimulus	IDA	localization of GFP/YFP fusion protein	NONE	Publication:501798511|PMID:35406838  	TAIR	2022-04-24
AT4G20010	locus:2119767	AT4G20010	acts upstream of or within	seed development	GO:0048316	18540	P	reproduction	IEA	traceable computational prediction	AGI_LocusCode:AT2G25180|AGI_LocusCode:AT3G16857|AGI_LocusCode:AT4G31920	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G20010	locus:2119767	AT4G20010	acts upstream of or within	seed development	GO:0048316	18540	P	multicellular organism development	IEA	traceable computational prediction	AGI_LocusCode:AT2G25180|AGI_LocusCode:AT3G16857|AGI_LocusCode:AT4G31920	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G20010	locus:2119767	AT4G20010	acts upstream of or within	macromolecule modification	GO:0043412	21458	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT3G11100	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G20010	locus:2119767	AT4G20010	involved in	positive regulation of helicase activity	GO:0051096	19066	P	regulation of molecular function	IBA	none	PANTHER:PTN000033807|UniProtKB:Q04837	Communication:501741973		2021-03-31
AT4G20010	locus:2119767	AT4G20010	enables	single-stranded DNA binding	GO:0003697	4154	F	DNA binding	IBA	none	PANTHER:PTN000033807|PomBase:SPAC2F3.04c|UniProtKB:P9WGD5|UniProtKB:Q8GWJ4|TAIR:locus:2086889|UniProtKB:Q8I415|TAIR:locus:2015353|UniProtKB:Q04837|FB:FBgn0010438|UniProtKB:P0AGE0|TAIR:locus:2119767|SGD:S000007222|TAIR:locus:2136083	Communication:501741973		2023-06-07
AT4G20010	gene:1009022144	AT4G20010.2	located in	chloroplast	GO:0009507	175	C	chloroplast	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G20010	gene:1009022144	AT4G20010.2	located in	chloroplast nucleoid	GO:0042644	14200	C	other intracellular components	HDA	none	NONE	Publication:501718303|PMID:16326926  	kvanwijk	2021-04-15
AT4G20010	locus:2119767	AT4G20010	involved in	mitochondrial DNA replication	GO:0006264	14013	P	other cellular processes	IBA	none	PANTHER:PTN000033809|FB:FBgn0010438	Communication:501741973		2021-03-31
AT4G20010	locus:2119767	AT4G20010	acts upstream of or within	defense response to bacterium	GO:0042742	14972	P	response to external stimulus	IEA	traceable computational prediction	AGI_LocusCode:AT1G80840|AGI_LocusCode:AT2G38470|AGI_LocusCode:AT4G31800|AGI_LocusCode:AT5G67300	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G20010	gene:2119766	AT4G20010.1	located in	chloroplast nucleoid	GO:0042644	14200	C	plastid	HDA	none	NONE	Publication:501718303|PMID:16326926  	kvanwijk	2021-04-15
AT4G20010	gene:2119766	AT4G20010.1	located in	chloroplast nucleoid	GO:0042644	14200	C	chloroplast	HDA	none	NONE	Publication:501718303|PMID:16326926  	kvanwijk	2021-04-15
AT4G20010	locus:2119767	AT4G20010	acts upstream of or within	cellular nitrogen compound biosynthetic process	GO:0044271	19681	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT1G80840|AGI_LocusCode:AT2G22430|AGI_LocusCode:AT2G25180|AGI_LocusCode:AT2G38470|AGI_LocusCode:AT3G16857|AGI_LocusCode:AT4G27410|AGI_LocusCode:AT4G31920|AGI_LocusCode:AT4G34000|AGI_LocusCode:AT5G46760|AGI_LocusCode:AT5G65310|AGI_LocusCode:AT5G67300	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G20010	gene:2119766	AT4G20010.1	located in	chloroplast nucleoid	GO:0042644	14200	C	other intracellular components	HDA	none	NONE	Publication:501718303|PMID:16326926  	kvanwijk	2021-04-15
AT4G20010	locus:2119767	AT4G20010	acts upstream of or within	defense response to fungus	GO:0050832	18013	P	response to stress	IEA	traceable computational prediction	AGI_LocusCode:AT1G80840|AGI_LocusCode:AT2G38470|AGI_LocusCode:AT4G31800|AGI_LocusCode:AT5G67300	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G20010	locus:2119767	AT4G20010	acts upstream of or within	RNA metabolic process	GO:0016070	4899	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT1G80840|AGI_LocusCode:AT2G22430|AGI_LocusCode:AT4G27410|AGI_LocusCode:AT4G34000|AGI_LocusCode:AT5G46760|AGI_LocusCode:AT5G65310|AGI_LocusCode:AT5G67300	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G20010	locus:2119767	AT4G20010	is active in	mitochondrial nucleoid	GO:0042645	14199	C	other intracellular components	IBA	none	PANTHER:PTN000033809|FB:FBgn0010438|SGD:S000007222|UniProtKB:Q04837	Communication:501741973		2023-06-07
AT4G20010	locus:2119767	AT4G20010	acts upstream of or within	cellular nitrogen compound biosynthetic process	GO:0044271	19681	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT1G80840|AGI_LocusCode:AT2G22430|AGI_LocusCode:AT2G25180|AGI_LocusCode:AT2G38470|AGI_LocusCode:AT3G16857|AGI_LocusCode:AT4G27410|AGI_LocusCode:AT4G31920|AGI_LocusCode:AT4G34000|AGI_LocusCode:AT5G46760|AGI_LocusCode:AT5G65310|AGI_LocusCode:AT5G67300	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G20010	locus:2119767	AT4G20010	acts upstream of or within	response to inorganic substance	GO:0010035	14767	P	response to chemical	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT2G22430|AGI_LocusCode:AT2G25180|AGI_LocusCode:AT2G38470|AGI_LocusCode:AT3G16857|AGI_LocusCode:AT4G27410|AGI_LocusCode:AT4G31920|AGI_LocusCode:AT4G34000|AGI_LocusCode:AT5G47640|AGI_LocusCode:AT5G67300	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G20010	locus:2119767	AT4G20010	involved in	mitochondrial DNA replication	GO:0006264	14013	P	other metabolic processes	IBA	none	PANTHER:PTN000033809|FB:FBgn0010438	Communication:501741973		2021-03-31
AT4G20010	locus:2119767	AT4G20010	involved in	mitochondrial DNA replication	GO:0006264	14013	P	biosynthetic process	IBA	none	PANTHER:PTN000033809|FB:FBgn0010438	Communication:501741973		2021-03-31
AT4G20010	locus:2119767	AT4G20010	acts upstream of or within	seed development	GO:0048316	18540	P	post-embryonic development	IEA	traceable computational prediction	AGI_LocusCode:AT2G25180|AGI_LocusCode:AT3G16857|AGI_LocusCode:AT4G31920	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G20010	locus:2119767	AT4G20010	enables	single-stranded DNA binding	GO:0003697	4154	F	DNA binding	IDA	in vitro binding assay	NONE	Publication:501783790|PMID:30698803  	TAIR	2019-03-13
AT4G20010	gene:1009022144	AT4G20010.2	located in	chloroplast nucleoid	GO:0042644	14200	C	chloroplast	HDA	none	NONE	Publication:501718303|PMID:16326926  	kvanwijk	2021-04-15
AT4G20010	locus:2119767	AT4G20010	acts upstream of or within	defense response to bacterium	GO:0042742	14972	P	response to biotic stimulus	IEA	traceable computational prediction	AGI_LocusCode:AT1G80840|AGI_LocusCode:AT2G38470|AGI_LocusCode:AT4G31800|AGI_LocusCode:AT5G67300	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G20010	locus:2119767	AT4G20010	acts upstream of or within	cellular nitrogen compound biosynthetic process	GO:0044271	19681	P	biosynthetic process	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT1G80840|AGI_LocusCode:AT2G22430|AGI_LocusCode:AT2G25180|AGI_LocusCode:AT2G38470|AGI_LocusCode:AT3G16857|AGI_LocusCode:AT4G27410|AGI_LocusCode:AT4G31920|AGI_LocusCode:AT4G34000|AGI_LocusCode:AT5G46760|AGI_LocusCode:AT5G65310|AGI_LocusCode:AT5G67300	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G20010	gene:2119766	AT4G20010.1	located in	chloroplast	GO:0009507	175	C	chloroplast	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G20010	locus:2119767	AT4G20010	acts upstream of or within	regulation of defense response	GO:0031347	20899	P	response to stress	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT1G80840|AGI_LocusCode:AT4G31800|AGI_LocusCode:AT5G67300	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G20010	locus:2119767	AT4G20010	is active in	mitochondrial nucleoid	GO:0042645	14199	C	mitochondrion	IBA	none	PANTHER:PTN000033809|FB:FBgn0010438|SGD:S000007222|UniProtKB:Q04837	Communication:501741973		2023-06-07
AT4G20010	locus:2119767	AT4G20010	acts upstream of or within	defense response to fungus	GO:0050832	18013	P	response to biotic stimulus	IEA	traceable computational prediction	AGI_LocusCode:AT1G80840|AGI_LocusCode:AT2G38470|AGI_LocusCode:AT4G31800|AGI_LocusCode:AT5G67300	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G20010	locus:2119767	AT4G20010	acts upstream of or within	RNA metabolic process	GO:0016070	4899	P	nucleobase-containing compound metabolic process	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT1G80840|AGI_LocusCode:AT2G22430|AGI_LocusCode:AT4G27410|AGI_LocusCode:AT4G34000|AGI_LocusCode:AT5G46760|AGI_LocusCode:AT5G65310|AGI_LocusCode:AT5G67300	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G20010	locus:2119767	AT4G20010	acts upstream of or within	seed development	GO:0048316	18540	P	anatomical structure development	IEA	traceable computational prediction	AGI_LocusCode:AT2G25180|AGI_LocusCode:AT3G16857|AGI_LocusCode:AT4G31920	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G20010	locus:2119767	AT4G20010	acts upstream of or within	defense response to bacterium	GO:0042742	14972	P	response to stress	IEA	traceable computational prediction	AGI_LocusCode:AT1G80840|AGI_LocusCode:AT2G38470|AGI_LocusCode:AT4G31800|AGI_LocusCode:AT5G67300	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G20010	locus:2119767	AT4G20010	acts upstream of or within	defense response to fungus	GO:0050832	18013	P	response to external stimulus	IEA	traceable computational prediction	AGI_LocusCode:AT1G80840|AGI_LocusCode:AT2G38470|AGI_LocusCode:AT4G31800|AGI_LocusCode:AT5G67300	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G20010	gene:1009022144	AT4G20010.2	located in	chloroplast nucleoid	GO:0042644	14200	C	plastid	HDA	none	NONE	Publication:501718303|PMID:16326926  	kvanwijk	2021-04-15
AT4G20010	locus:2119767	AT4G20010	acts upstream of or within	cellular component biogenesis	GO:0044085	31671	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G20020	locus:2119782	AT4G20020	acts upstream of or within	chloroplast RNA modification	GO:1900865	41806	P	other metabolic processes	IMP	biochemical/chemical analysis	NONE	Publication:501748768|PMID:22411807  	TAIR	2012-06-13
AT4G20020	locus:2119782	AT4G20020	involved in	mitochondrial mRNA modification	GO:0080156	35965	P	other metabolic processes	IBA	none	PANTHER:PTN000772312|TAIR:locus:2086310|UniProtKB:Q9CAH0|TAIR:locus:2206639|TAIR:locus:2083348|TAIR:locus:2156344|TAIR:locus:2063389	Communication:501741973		2023-06-07
AT4G20020	gene:1006228810	AT4G20020.2	located in	chloroplast	GO:0009507	175	C	chloroplast	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G20020	locus:2119782	AT4G20020	located in	mitochondrion	GO:0005739	486	C	mitochondrion	HDA	none	NONE	Publication:501750648|PMID:22923678  	dupree	2021-04-07
AT4G20020	locus:2119782	AT4G20020	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:O49429	Publication:501748768|PMID:22411807  		2021-04-01
AT4G20020	locus:2119782	AT4G20020	enables	protein dimerization activity	GO:0046983	15253	F	protein binding	IPI	Bimolecular fluorescence complementation (BiFC)	AGI_LocusCode:AT3G06790	Publication:501762829|PMID:25583991  	TAIR	2018-06-08
AT4G20020	gene:2119781	AT4G20020.1	located in	chloroplast	GO:0009507	175	C	chloroplast	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G20020	locus:2119782	AT4G20020	enables	protein dimerization activity	GO:0046983	15253	F	protein binding	IPI	yeast two-hybrid assay	AGI_LocusCode:AT1G32580	Publication:501762829|PMID:25583991  	TAIR	2018-06-08
AT4G20020	locus:2119782	AT4G20020	enables	protein dimerization activity	GO:0046983	15253	F	protein binding	IPI	Bimolecular fluorescence complementation (BiFC)	AGI_LocusCode:AT3G15000	Publication:501762829|PMID:25583991  	TAIR	2018-06-08
AT4G20020	locus:2119782	AT4G20020	acts upstream of or within	mitochondrial RNA modification	GO:1900864	41805	P	nucleobase-containing compound metabolic process	IMP	biochemical/chemical analysis	NONE	Publication:501748768|PMID:22411807  	TAIR	2012-06-13
AT4G20020	locus:2119782	AT4G20020	acts upstream of or within	chloroplast RNA modification	GO:1900865	41806	P	nucleobase-containing compound metabolic process	IMP	biochemical/chemical analysis	NONE	Publication:501748768|PMID:22411807  	TAIR	2012-06-13
AT4G20020	locus:2119782	AT4G20020	involved in	cytidine to uridine editing	GO:0016554	5487	P	nucleobase-containing compound metabolic process	IEA	none	InterPro:IPR039206	AnalysisReference:501756966		2023-06-11
AT4G20020	locus:2119782	AT4G20020	enables	protein dimerization activity	GO:0046983	15253	F	protein binding	IPI	Bimolecular fluorescence complementation (BiFC)	AGI_LocusCode:AT2G35240	Publication:501762829|PMID:25583991  	TAIR	2018-06-08
AT4G20020	locus:2119782	AT4G20020	enables	protein dimerization activity	GO:0046983	15253	F	protein binding	IPI	Bimolecular fluorescence complementation (BiFC)	AGI_LocusCode:AT5G44780	Publication:501762829|PMID:25583991  	TAIR	2018-06-08
AT4G20020	locus:2119782	AT4G20020	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	yeast two-hybrid assay	AGI_LocusCode:AT1G13900	Publication:501775428|PMID:28478183  	TAIR	2018-05-25
AT4G20020	locus:2119782	AT4G20020	colocalizes with	mitochondrion	GO:0005739	486	C	mitochondrion	IPI	Bimolecular fluorescence complementation (BiFC)	AGI_LocusCode:AT1G13900	Publication:501775428|PMID:28478183  	TAIR	2018-05-25
AT4G20020	locus:2119782	AT4G20020	enables	protein dimerization activity	GO:0046983	15253	F	protein binding	IPI	yeast two-hybrid assay	AGI_LocusCode:AT1G72530	Publication:501762829|PMID:25583991  	TAIR	2018-06-08
AT4G20020	locus:2119782	AT4G20020	acts upstream of or within	mitochondrial RNA modification	GO:1900864	41805	P	other metabolic processes	IMP	biochemical/chemical analysis	NONE	Publication:501748768|PMID:22411807  	TAIR	2012-06-13
AT4G20020	locus:2119782	AT4G20020	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:Q9FWA6	Publication:501764625|PMID:26048647  		2021-04-01
AT4G20020	locus:2119782	AT4G20020	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:O23266	Publication:501766753|PMID:26470017  		2021-04-01
AT4G20020	locus:2119782	AT4G20020	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:Q84JZ6	Publication:501748768|PMID:22411807  		2021-04-01
AT4G20020	locus:2119782	AT4G20020	involved in	mitochondrial mRNA modification	GO:0080156	35965	P	nucleobase-containing compound metabolic process	IBA	none	PANTHER:PTN000772312|TAIR:locus:2086310|UniProtKB:Q9CAH0|TAIR:locus:2206639|TAIR:locus:2083348|TAIR:locus:2156344|TAIR:locus:2063389	Communication:501741973		2023-06-07
AT4G20020	locus:2119782	AT4G20020	involved in	cytidine to uridine editing	GO:0016554	5487	P	other metabolic processes	IEA	none	InterPro:IPR039206	AnalysisReference:501756966		2023-06-11
AT4G20030	gene:2119796	AT4G20030.1	located in	chloroplast	GO:0009507	175	C	chloroplast	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G20030	locus:2119797	AT4G20030	is active in	nuclear speck	GO:0016607	524	C	nucleoplasm	IBA	none	PANTHER:PTN000567445|UniProtKB:Q96PK6|MGI:MGI:98284|UniProtKB:Q07955|UniProtKB:Q08170|TAIR:locus:2144945|MGI:MGI:98287|TAIR:locus:2049766|TAIR:locus:2024770|TAIR:locus:2086804|RGD:1359422|UniProtKB:Q13247|TAIR:locus:2199837|UniProtKB:P84103|TAIR:locus:2035651|UniProtKB:Q01130|TAIR:locus:2099931|TAIR:locus:2079502|TAIR:locus:2131864|UniProtKB:Q9BRL6|TAIR:locus:2124799|MGI:MGI:98283|TAIR:locus:2173338|FB:FBgn0265298|UniProtKB:O75494|UniProtKB:Q9BWF3|MGI:MGI:98285|UniProtKB:Q586R7	Communication:501741973		2023-06-07
AT4G20030	locus:2119797	AT4G20030	involved in	mRNA splicing, via spliceosome	GO:0000398	17088	P	other metabolic processes	IBA	none	PANTHER:PTN000567445|UniProtKB:Q8WXF0|TAIR:locus:2035651|UniProtKB:O75494|TAIR:locus:2079502|UniProtKB:Q6K9C3|FB:FBgn0260944|UniProtKB:Q07955|UniProtKB:Q13247|TAIR:locus:2086804|RGD:1587490	Communication:501741973		2023-06-07
AT4G20030	locus:2119797	AT4G20030	involved in	mRNA splicing, via spliceosome	GO:0000398	17088	P	nucleobase-containing compound metabolic process	IBA	none	PANTHER:PTN000567445|UniProtKB:Q8WXF0|TAIR:locus:2035651|UniProtKB:O75494|TAIR:locus:2079502|UniProtKB:Q6K9C3|FB:FBgn0260944|UniProtKB:Q07955|UniProtKB:Q13247|TAIR:locus:2086804|RGD:1587490	Communication:501741973		2023-06-07
AT4G20030	locus:2119797	AT4G20030	is active in	nucleolus	GO:0005730	527	C	nucleolus	IBA	none	PANTHER:PTN001924395|MGI:MGI:1921361|UniProtKB:Q9Y4C8	Communication:501741973		2023-06-07
AT4G20030	gene:2119796	AT4G20030.1	located in	nucleus	GO:0005634	537	C	nucleus	HDA	none	NONE	Publication:501711432|PMID:14617066  	kvanwijk	2021-04-22
AT4G20030	locus:2119797	AT4G20030	enables	RNA binding	GO:0003723	1217	F	RNA binding	IBA	none	PANTHER:PTN000567445|TAIR:locus:2049766|FB:FBgn0011640|FB:FBgn0028554|TAIR:locus:2114643|UniProtKB:Q07955|FB:FBgn0260944|TAIR:locus:2083936|MGI:MGI:98284|FB:FBgn0011305|ZFIN:ZDB-GENE-030131-3019|TAIR:locus:2173088|TAIR:locus:2046638|MGI:MGI:1100865|UniProtKB:Q57VT4|RGD:1309233|UniProtKB:P84103|TAIR:locus:2199837|TAIR:locus:2086804|TAIR:locus:2024770|UniProtKB:Q13247|RGD:3664|TAIR:locus:2124799|FB:FBgn0030479|TAIR:locus:2079502|FB:FBgn0283477|UniProtKB:Q9BWF3|UniProtKB:Q586R7|FB:FBgn0265298|UniProtKB:O75494|TAIR:locus:2173338|MGI:MGI:98283	Communication:501741973		2023-06-07
AT4G20040	locus:2119817	AT4G20040	enables	polygalacturonase activity	GO:0004650	3789	F	hydrolase activity	IBA	none	PANTHER:PTN002142759|TAIR:locus:2119832	Communication:501741973		2021-04-02
AT4G20040	locus:2119817	AT4G20040	acts upstream of or within	small molecule metabolic process	GO:0044281	34292	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G21970|AGI_LocusCode:AT3G19290|AGI_LocusCode:AT4G01120	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT4G20040	gene:2119816	AT4G20040.1	located in	extracellular region	GO:0005576	294	C	extracellular region	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G20050	locus:2119832	AT4G20050	acts upstream of or within	pollen exine formation	GO:0010584	29496	P	other cellular processes	IMP	analysis of visible trait	NONE	Publication:501744802|PMID:21849515  	TAIR	2012-01-12
AT4G20050	locus:2119832	AT4G20050	acts upstream of or within	pollen exine formation	GO:0010584	29496	P	anatomical structure development	IMP	analysis of visible trait	NONE	Communication:501724251	TAIR	2008-08-22
AT4G20050	locus:2119832	AT4G20050	enables	polygalacturonase activity	GO:0004650	3789	F	hydrolase activity	IDA	Enzyme assays		Publication:501711301|PMID:14551328  	TAIR	2004-07-27
AT4G20050	locus:2119832	AT4G20050	acts upstream of or within	microsporogenesis	GO:0009556	11331	P	multicellular organism development	IMP	analysis of visible trait	NONE	Publication:501711301|PMID:14551328  	TAIR	2004-07-27
AT4G20050	locus:2119832	AT4G20050	enables	polygalacturonase activity	GO:0004650	3789	F	hydrolase activity	IBA	none	PANTHER:PTN002142759|TAIR:locus:2119832	Communication:501741973		2021-04-01
AT4G20050	locus:2119832	AT4G20050	acts upstream of or within	pollen exine formation	GO:0010584	29496	P	cellular component organization	IMP	analysis of visible trait	NONE	Communication:501724251	TAIR	2008-08-22
AT4G20050	locus:2119832	AT4G20050	acts upstream of or within	pollen exine formation	GO:0010584	29496	P	multicellular organism development	IMP	analysis of visible trait	NONE	Communication:501724251	TAIR	2008-08-22
AT4G20050	gene:2119831	AT4G20050.1	located in	extracellular region	GO:0005576	294	C	extracellular region	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G20050	gene:4010712911	AT4G20050.2	located in	extracellular region	GO:0005576	294	C	extracellular region	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G20050	locus:2119832	AT4G20050	acts upstream of or within	pollen exine formation	GO:0010584	29496	P	multicellular organism development	IMP	analysis of visible trait	NONE	Publication:501744802|PMID:21849515  	TAIR	2012-01-12
AT4G20050	locus:2119832	AT4G20050	acts upstream of or within	microsporogenesis	GO:0009556	11331	P	other cellular processes	IMP	analysis of visible trait	NONE	Publication:501711301|PMID:14551328  	TAIR	2004-07-27
AT4G20050	gene:6532557506	AT4G20050.6	located in	extracellular region	GO:0005576	294	C	extracellular region	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G20050	locus:2119832	AT4G20050	acts upstream of or within	microsporogenesis	GO:0009556	11331	P	reproduction	IMP	analysis of visible trait	NONE	Publication:501711301|PMID:14551328  	TAIR	2004-07-27
AT4G20050	locus:2119832	AT4G20050	acts upstream of or within	microsporogenesis	GO:0009556	11331	P	cell cycle	IMP	analysis of visible trait	NONE	Publication:501711301|PMID:14551328  	TAIR	2004-07-27
AT4G20050	gene:6532557507	AT4G20050.7	located in	extracellular region	GO:0005576	294	C	extracellular region	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT4G20050	locus:2119832	AT4G20050	enables	polygalacturonase activity	GO:0004650	3789	F	hydrolase activity	IEA	none	InterPro:IPR039279	AnalysisReference:501756966		2023-05-23
AT4G20050	locus:2119832	AT4G20050	acts upstream of or within	pollen exine formation	GO:0010584	29496	P	anatomical structure development	IMP	analysis of visible trait	NONE	Publication:501744802|PMID:21849515  	TAIR	2012-01-12
AT4G20050	locus:2119832	AT4G20050	acts upstream of or within	pollen exine formation	GO:0010584	29496	P	cellular component organization	IMP	analysis of visible trait	NONE	Publication:501744802|PMID:21849515  	TAIR	2012-01-12
AT4G20050	locus:2119832	AT4G20050	acts upstream of or within	pollen exine formation	GO:0010584	29496	P	other cellular processes	IMP	analysis of visible trait	NONE	Communication:501724251	TAIR	2008-08-22
AT4G20060	locus:2119722	AT4G20060	acts upstream of or within	embryo development ending in seed dormancy	GO:0009793	11994	P	anatomical structure development	NAS	Database entries that dont cite a paper (e.g. SwissProt records)</li>
</ul>
<h2 id="function-related-keywords">Function-related keywords:</h2>
<ul>
<li><a href="/tags/root/"



 


>root</a>
</li>
</ul>
<h2 id="literature">Literature:</h2>
<ul>
<li><a href="https://www.doi.org/10.1104/pp.103.020925"




 target="_blank"
 


>Comparative analysis of the Arabidopsis pollen transcriptome.</a>
  DOI:  <a href="https://www.doi.org/10.1104/pp.103.020925"




 target="_blank"
 


>10.1104/pp.103.020925</a>
;  PMID:  <a href="https://pubmed.ncbi.nlm.nih.gov/12805594/"




 target="_blank"
 


>12805594</a>
</li>
<li><a href="https://www.doi.org/10.1105/tpc.104.022830"




 target="_blank"
 


>Novel and stress-regulated microRNAs and other small RNAs from Arabidopsis.</a>
  DOI:  <a href="https://www.doi.org/10.1105/tpc.104.022830"




 target="_blank"
 


>10.1105/tpc.104.022830</a>
;  PMID:  <a href="https://pubmed.ncbi.nlm.nih.gov/15258262/"




 target="_blank"
 


>15258262</a>
</li>
<li><a href="https://www.doi.org/10.1093/nar/gkad493"




 target="_blank"
 


>Interplay of endonucleolytic and exonucleolytic processing in the 3'-end  formation of a mitochondrial nad2 RNA precursor in Arabidopsis.</a>
  DOI:  <a href="https://www.doi.org/10.1093/nar/gkad493"




 target="_blank"
 


>10.1093/nar/gkad493</a>
;  PMID:  <a href="https://pubmed.ncbi.nlm.nih.gov/37293952/"




 target="_blank"
 


>37293952</a>
</li>
</ul>
<h2 id="sequence">Sequence:</h2>
<p>





  



<div class="accordion ">
  <div class="accordion-header " data-accordion>
    cDNA Sequence
    <svg
      class="accordion-icon"
      x="0px"
      y="0px"
      viewBox="0 0 512 512"
      xmlspace="preserve">
      <path
        fill="currentColor"
        d="M505.755,123.592c-8.341-8.341-21.824-8.341-30.165,0L256.005,343.176L36.421,123.592c-8.341-8.341-21.824-8.341-30.165,0 s-8.341,21.824,0,30.165l234.667,234.667c4.16,4.16,9.621,6.251,15.083,6.251c5.462,0,10.923-2.091,15.083-6.251l234.667-234.667 C514.096,145.416,514.096,131.933,505.755,123.592z"></path>
    </svg>
</div>


  <div class="accordion-content ">
    <div class="content-wrapper">
    <button class="copy-btn" aria-label="Copy Content">Copy</button>
    </div>
  
      <div class="text-to-copy">
    <ul>
<li>&gt;AT4G19440.1
TTTGCCCAAATCGCCTCCTCTCGAGAATTTCTCCGTCTCTCTCTCTCTCTCCAATGGGGTGGCTCAACACCATCTTCAATTCTTCTTCTTCGACATTGATCAATGGCGGCTCTTCTCTATTTCCCTAAAATCTCTTCGCAAATGACTTCCTCACATTTCATCTCTTTCTCACCAATGGATTTGAGGAGACTTTCTCGCGCAAGCTACTTATTCACTCGCCGCTTGAAATTCATAGCGAAATCTCGAAAATGCTTTCATACAAGCCGATACCTGCAACAGTGTGTTCATCGGCCGGATAAATCGGAAGAGACTTCCTCTGATCGGCATTTACACGAACGATTATCTTCAGTTTTGTCGAAGAGATCTCTAGATTACGAGCAGTGTAAACAACTGATAACTGTTCTATCGCCTCTCGAATTCGATAGGTTGTTTCCTGAGTTTCGATCCAAAGTTAATCCCAAGACGGCTTTGGATTTCTTCCGATTAGCTTCTGATTCGTTTAGTTTCTCCTTTAGTCTCCGTTCTTATTGTTTATTGATAGGTTTATTGCTTGATGCGAATCTGTTATCGGCTGCTAGGGTAGTTTTGATTCGTTTGATCAATGGGAATGTACCGGTTTTGCCTTGTGGTTTGAGAGATAGTCGAGTTGCGATTGCGGATGCAATGGCGAGTTTGAGTTTGTGTTTTGATGAAGAAATCAGAAGGAAAATGTCGGATTTGTTGATTGAAGTTTATTGCACACAGTTCAAGCGTGATGGTTGTTATTTAGCTCTTGATGTATTTCCTGTTCTTGCAAACAAGGGTATGTTTCCTTCTAAGACGACTTGCAACATTTTGTTGACTTCTCTAGTGAGAGCTAATGAGTTTCAGAAGTGTTGTGAAGCTTTTGATGTTGTTTGCAAAGGGGTTTCTCCGGATGTTTACTTGTTCACTACAGCGATTAATGCGTTTTGCAAGGGAGGTAAAGTTGAGGAGGCCGTTAAGTTGTTCTCAAAAATGGAAGAAGCGGGTGTTGCACCAAACGTTGTCACTTTTAATACTGTTATTGATGGACTTGGGATGTGTGGAAGGTATGATGAGGCGTTTATGTTTAAAGAAAAGATGGTAGAAAGAGGCATGGAACCAACTCTTATCACTTATAGTATTCTTGTCAAGGGTTTGACCAGGGCTAAGAGGATTGGTGATGCTTATTTTGTCTTGAAGGAAATGACAAAGAAAGGGTTTCCTCCTAATGTTATTGTATATAATAATTTGATTGATAGTTTCATCGAGGCTGGAAGTTTGAACAAAGCTATTGAGATAAAGGATCTCATGGTTTCGAAAGGGTTGTCTCTCACATCGTCGACGTATAATACTCTCATAAAGGGATACTGCAAGAACGGTCAAGCAGATAATGCGGAGCGTCTCCTGAAGGAAATGCTGTCGATTGGTTTTAATGTAAATCAGGGCTCCTTTACATCTGTGATTTGCTTGTTGTGCAGTCATCTTATGTTTGACTCAGCACTTCGTTTTGTTGGAGAGATGCTATTACGCAACATGAGTCCTGGTGGTGGATTACTTACTACATTGATTTCTGGTCTCTGTAAGCATGGGAAACACTCAAAGGCACTAGAACTTTGGTTTCAATTTCTAAATAAAGGTTTTGTAGTGGACACGAGGACTTCAAATGCTTTACTTCATGGACTCTGTGAAGCAGGGAAGCTGGATGAAGCTTTTAGGATTCAAAAGGAGATATTAGGGCGTGGCTGTGTTATGGATAGAGTTTCCTATAACACGTTGATTTCTGGATGTTGCGGAAAGAAGAAACTAGATGAAGCTTTTATGTTCTTGGATGAAATGGTTAAGAGAGGACTTAAGCCTGACAATTATACTTATAGCATATTGATTTGCGGGCTGTTTAATATGAATAAAGTTGAGGAGGCCATACAGTTTTGGGATGATTGTAAACGGAATGGCATGCTTCCAGATGTTTATACATATTCGGTGATGATAGATGGATGTTGTAAAGCTGAAAGAACAGAAGAGGGTCAGGAATTTTTTGATGAGATGATGAGCAAAAACGTGCAGCCAAATACTGTTGTTTACAATCATCTTATCAGAGCATACTGTAGAAGCGGAAGGCTATCGATGGCCTTGGAACTCCGTGAAGATATGAAACACAAAGGAATTTCACCTAATTCTGCTACATACACTTCACTAATAAAGGGGATGTCAATCATTAGCCGTGTTGAGGAGGCAAAACTCCTCTTCGAGGAGATGAGGATGGAGGGTTTGGAACCGAATGTCTTTCATTATACAGCACTTATTGATGGATATGGTAAATTGGGTCAGATGGTCAAAGTTGAATGTCTTCTGCGTGAGATGCATTCAAAGAACGTACACCCAAATAAGATTACCTACACTGTGATGATTGGTGGGTATGCTAGAGATGGGAATGTAACGGAAGCTTCTAGGCTTCTAAATGAGATGAGGGAAAAAGGAATTGTTCCGGATAGTATCACATACAAAGAATTTATATACGGATATCTTAAGCAGGGAGGTGTGTTGGAAGCTTTTAAAGGTTCTGATGAAGAAAATTATGCAGCAATCATTGAAGGATGGAACAAACTCATTCAGTAACTGATGAATAAGAAGAAGAACAGTTGGAAATTGAAATCTAGTAGCCATCTCAGTTTTGATTCACGCCGGGGATCTACGTCTTCTTCTTAGAGGAGCACGTTAGGATGGAGTTACTTGGTAAATGATGCATATGGTTCATTGCATCTTACAAGTAGGGTAGTGTGTTCTACTTGATTCTCCAGCCCATTCAGTTAACAGCTTTGGCCTGCTTCAATCTTAATGAGAAAAGTTGTTTCTTTTAGCATCAAAGGCTATATGGGTTTATGATTTACAAATCCACATTACTGGTACTCGTCAGGTTTAGGATTGAGTGCCACAGAATCAAAGGCATATGATAGATTCTGGATCAGAGCAGGTACTTCAGACAGCAGTCCTGAAAAGATTATACAGATTTAGACACACATCATGGATAGTCAATTAAAAGTTTACACATGGATATTCATGGGAGCTATGAGAATGGATCAATAGGAGGCTCCTCGGTTTACAGGTCACATAAATAGAGCTTAAGCGAATTCTCATCGGAGTAGTTTAGAACTAATTCATGCCTGACACATCGCTGGGGAGAATTTCCTCAGCCGTGCAGAATTTTTTGCCCAAGGAGATTAATGGACTGATTTGAAAGGTAACAGTTTACAACGGTATCAACTAATTCGACAACATTCACCTTAGAGTATAGACTTTCAGTGGGGTGAATACTGTTTATTCATCTTGTTTACGATTCTTTAAATTTGTTAATTCTCTTAAAGAGGTGGAAAAAGTGTTTTAGATGTACGTAGAAAGAAAGAAGGAAAACAAGAATAAAATTTGAAATACTCTTCATGATTTTTGGTTTTATTTTTTAACTTATACTACTAGTCAAGATACCCTAAACAGAGATTGTTATGAGAAGATTGTAAAAAAGCCAATATGAAA</li>
<li>&gt;AT4G19440.3
TTTGCCCAAATCGCCTCCTCTCGAGAATTTCTCCGTCTCTCTCTCTCTCTCCAATGGGGTGGCTCAACACCATCTTCAATTCTTCTTCTTCGACATTGATCAATGGCGGCTCTTCTCTATTTCCCTAAAATCTCTTCGCAAATGACTTCCTCACATTTCATCTCTTTCTCACCAATGGATTTGAGGAGACTTTCTCGCGCAAGCTACTTATTCACTCGCCGCTTGAAATTCATAGCGAAATCTCGAAAATGCTTTCATACAAGCCGATACCTGCAACAGTGTGTTCATCGGCCGGATAAATCGGAAGAGACTTCCTCTGATCGGCATTTACACGAACGATTATCTTCAGTTTTGTCGAAGAGATCTCTAGATTACGAGCAGTGTAAACAACTGATAACTGTTCTATCGCCTCTCGAATTCGATAGGTTGTTTCCTGAGTTTCGATCCAAAGTTAATCCCAAGACGGCTTTGGATTTCTTCCGATTAGCTTCTGATTCGTTTAGTTTCTCCTTTAGTCTCCGTTCTTATTGTTTATTGATAGGTTTATTGCTTGATGCGAATCTGTTATCGGCTGCTAGGGTAGTTTTGATTCGTTTGATCAATGGGAATGTACCGGTTTTGCCTTGTGGTTTGAGAGATAGTCGAGTTGCGATTGCGGATGCAATGGCGAGTTTGAGTTTGTGTTTTGATGAAGAAATCAGAAGGAAAATGTCGGATTTGTTGATTGAAGTTTATTGCACACAGTTCAAGCGTGATGGTTGTTATTTAGCTCTTGATGTATTTCCTGTTCTTGCAAACAAGGGTATGTTTCCTTCTAAGACGACTTGCAACATTTTGTTGACTTCTCTAGTGAGAGCTAATGAGTTTCAGAAGTGTTGTGAAGCTTTTGATGTTGTTTGCAAAGGGGTTTCTCCGGATGTTTACTTGTTCACTACAGCGATTAATGCGTTTTGCAAGGGAGGTAAAGTTGAGGAGGCCGTTAAGTTGTTCTCAAAAATGGAAGAAGCGGGTGTTGCACCAAACGTTGTCACTTTTAATACTGTTATTGATGGACTTGGGATGTGTGGAAGGTATGATGAGGCGTTTATGTTTAAAGAAAAGATGGTAGAAAGAGGCATGGAACCAACTCTTATCACTTATAGTATTCTTGTCAAGGGTTTGACCAGGGCTAAGAGGATTGGTGATGCTTATTTTGTCTTGAAGGAAATGACAAAGAAAGGGTTTCCTCCTAATGTTATTGTATATAATAATTTGATTGATAGTTTCATCGAGGCTGGAAGTTTGAACAAAGCTATTGAGATAAAGGATCTCATGGTTTCGAAAGGGTTGTCTCTCACATCGTCGACGTATAATACTCTCATAAAGGGATACTGCAAGAACGGTCAAGCAGATAATGCGGAGCGTCTCCTGAAGGAAATGCTGTCGATTGGTTTTAATGTAAATCAGGGCTCCTTTACATCTGTGATTTGCTTGTTGTGCAGTCATCTTATGTTTGACTCAGCACTTCGTTTTGTTGGAGAGATGCTATTACGCAACATGAGTCCTGGTGGTGGATTACTTACTACATTGATTTCTGGTCTCTGTAAGCATGGGAAACACTCAAAGGCACTAGAACTTTGGTTTCAATTTCTAAATAAAGGTTTTGTAGTGGACACGAGGACTTCAAATGCTTTACTTCATGGACTCTGTGAAGCAGGGAAGCTGGATGAAGCTTTTAGGATTCAAAAGGAGATATTAGGGCGTGGCTGTGTTATGGATAGAGTTTCCTATAACACGTTGATTTCTGGATGTTGCGGAAAGAAGAAACTAGATGAAGCTTTTATGTTCTTGGATGAAATGGTTAAGAGAGGACTTAAGCCTGACAATTATACTTATAGCATATTGATTTGCGGGCTGTTTAATATGAATAAAGTTGAGGAGGCCATACAGTTTTGGGATGATTGTAAACGGAATGGCATGCTTCCAGATGTTTATACATATTCGGTGATGATAGATGGATGTTGTAAAGCTGAAAGAACAGAAGAGGGTCAGGAATTTTTTGATGAGATGATGAGCAAAAACGTGCAGCCAAATACTGTTGTTTACAATCATCTTATCAGAGCATACTGTAGAAGCGGAAGGCTATCGATGGCCTTGGAACTCCGTGAAGATATGAAACACAAAGGAATTTCACCTAATTCTGCTACATACACTTCACTAATAAAGGGGATGTCAATCATTAGCCGTGTTGAGGAGGCAAAACTCCTCTTCGAGGAGATGAGGATGGAGGGTTTGGAACCGAATGTCTTTCATTATACAGCACTTATTGATGGATATGGTAAATTGGGTCAGATGGTCAAAGTTGAATGTCTTCTGCGTGAGATGCATTCAAAGAACGTACACCCAAATAAGATTACCTACACTGTGATGATTGGTGGGTATGCTAGAGATGGGAATGTAACGGAAGCTTCTAGGCTTCTAAATGAGATGAGGGAAAAAGGAATTGTTCCGGATAGTATCACATACAAAGAATTTATATACGGATATCTTAAGCAGGGAGGTGTGTTGGAAGCTTTTAAAGGTTCTGATGAAGAAAATTATGCAGCAATCATTGAAGGATGGAACAAACTCATTCAGTAACTGATGAATAAGAAGAAGAACAGTTGGAAATTGAAATCTAGTAGCCATCTCAGTTTTGATTCACGCCGGGGATCTACGTCTTCTTCTTAGAGGAGCACGTTAGGATGGAGTTACTTGGTAAATGATGCATATGGTTCATTGCATCTTACAAGTAGGGTAGTGTGTTCTACTTGATTCTCCAGCCCATTCAGTTAACAGCTTTGGCCTGCTTCAATCTTAATGAGAAAAGTTGTTTCTTTTAGCATCAAAGGCTATATGGGTTTATGATTTACAAATCCACATTACTGGTACTCGTCAGGTTTAGGATTGAGTGCCACAGAATCAAAGGCATATGATAGATTCTGGATCAGAGCAGGTACTTCAGACAGCAGTCCTGAAAAGATTATACAGATTTAGACACACATCATGGATAGTCAATTAAAAGTTTACACATGGATATTCATGGGAGCTATGAGAATGGATCAATAGGAGGCTCCTCGGTTTACAGGTCACATAAATAGAGCTTAAGCGAATTCTCATCGGAGTAGTTTAGAACTAATTCATGCCTGACACATCGCTGGGGAGAATTTCCTCAGCCGTGCAGAATTTTTTGCCCAAGGAGATTAATGGACTGATTTGAAAGGTAACAGTTTACAACGGTATCAACTAATTCGACAACATTCACCTTAGAGTATAGACTTTCAGTGGGGTGAATACTGTTTATTCATCTTGTTTACGATTCTTTAAATTTGTTAATTCTCTTAAAGAGGTGGAAAAAGTGTTTTAGATGTACGTAGAAAGAAAGAAGGAAAACAAGAATAAAATTTGAAATACTCTTCATGATTTTTGGTTTTATTTTTTAACTTATACTACTAGTCAAGATACCCTAAACAGAGATTGTTATGAGAAGATTGTAAAAAAGCCAATATGAAA</li>
<li>&gt;AT4G19440.2
TTTGCCCAAATCGCCTCCTCTCGAGAATTTCTCCGTCTCTCTCTCTCTCTCCAATGGGGTGGCTCAACACCATCTTCAATTCTTCTTCTTCGACATTGATCAATGGCGGCTCTTCTCTATTTCCCTAAAATCTCTTCGCAAATGACTTCCTCACATTTCATCTCTTTCTCACCAATGGATTTGAGGAGACTTTCTCGCGCAAGCTACTTATTCACTCGCCGCTTGAAATTCATAGCGAAATCTCGAAAATGCTTTCATACAAGCCGATACCTGCAACAGTGTGTTCATCGGCCGGATAAATCGGAAGAGACTTCCTCTGATCGGCATTTACACGAACGATTATCTTCAGTTTTGTCGAAGAGATCTCTAGATTACGAGCAGTGTAAACAACTGATAACTGTTCTATCGCCTCTCGAATTCGATAGGTTGTTTCCTGAGTTTCGATCCAAAGTTAATCCCAAGACGGCTTTGGATTTCTTCCGATTAGCTTCTGATTCGTTTAGTTTCTCCTTTAGTCTCCGTTCTTATTGTTTATTGATAGGTTTATTGCTTGATGCGAATCTGTTATCGGCTGCTAGGGTAGTTTTGATTCGTTTGATCAATGGGAATGTACCGGTTTTGCCTTGTGGTTTGAGAGATAGTCGAGTTGCGATTGCGGATGCAATGGCGAGTTTGAGTTTGTGTTTTGATGAAGAAATCAGAAGGAAAATGTCGGATTTGTTGATTGAAGTTTATTGCACACAGTTCAAGCGTGATGGTTGTTATTTAGCTCTTGATGTATTTCCTGTTCTTGCAAACAAGGGTATGTTTCCTTCTAAGACGACTTGCAACATTTTGTTGACTTCTCTAGTGAGAGCTAATGAGTTTCAGAAGTGTTGTGAAGCTTTTGATGTTGTTTGCAAAGGGGTTTCTCCGGATGTTTACTTGTTCACTACAGCGATTAATGCGTTTTGCAAGGGAGGTAAAGTTGAGGAGGCCGTTAAGTTGTTCTCAAAAATGGAAGAAGCGGGTGTTGCACCAAACGTTGTCACTTTTAATACTGTTATTGATGGACTTGGGATGTGTGGAAGGTATGATGAGGCGTTTATGTTTAAAGAAAAGATGGTAGAAAGAGGCATGGAACCAACTCTTATCACTTATAGTATTCTTGTCAAGGGTTTGACCAGGGCTAAGAGGATTGGTGATGCTTATTTTGTCTTGAAGGAAATGACAAAGAAAGGGTTTCCTCCTAATGTTATTGTATATAATAATTTGATTGATAGTTTCATCGAGGCTGGAAGTTTGAACAAAGCTATTGAGATAAAGGATCTCATGGTTTCGAAAGGGTTGTCTCTCACATCGTCGACGTATAATACTCTCATAAAGGGATACTGCAAGAACGGTCAAGCAGATAATGCGGAGCGTCTCCTGAAGGAAATGCTGTCGATTGGTTTTAATGTAAATCAGGGCTCCTTTACATCTGTGATTTGCTTGTTGTGCAGTCATCTTATGTTTGACTCAGCACTTCGTTTTGTTGGAGAGATGCTATTACGCAACATGAGTCCTGGTGGTGGATTACTTACTACATTGATTTCTGGTCTCTGTAAGCATGGGAAACACTCAAAGGCACTAGAACTTTGGTTTCAATTTCTAAATAAAGGTTTTGTAGTGGACACGAGGACTTCAAATGCTTTACTTCATGGACTCTGTGAAGCAGGGAAGCTGGATGAAGCTTTTAGGATTCAAAAGGAGATATTAGGGCGTGGCTGTGTTATGGATAGAGTTTCCTATAACACGTTGATTTCTGGATGTTGCGGAAAGAAGAAACTAGATGAAGCTTTTATGTTCTTGGATGAAATGGTTAAGAGAGGACTTAAGCCTGACAATTATACTTATAGCATATTGATTTGCGGGCTGTTTAATATGAATAAAGTTGAGGAGGCCATACAGTTTTGGGATGATTGTAAACGGAATGGCATGCTTCCAGATGTTTATACATATTCGGTGATGATAGATGGATGTTGTAAAGCTGAAAGAACAGAAGAGGGTCAGGAATTTTTTGATGAGATGATGAGCAAAAACGTGCAGCCAAATACTGTTGTTTACAATCATCTTATCAGAGCATACTGTAGAAGCGGAAGGCTATCGATGGCCTTGGAACTCCGTGAAGATATGAAACACAAAGGAATTTCACCTAATTCTGCTACATACACTTCACTAATAAAGGGGATGTCAATCATTAGCCGTGTTGAGGAGGCAAAACTCCTCTTCGAGGAGATGAGGATGGAGGGTTTGGAACCGAATGTCTTTCATTATACAGCACTTATTGATGGATATGGTAAATTGGGTCAGATGGTCAAAGTTGAATGTCTTCTGCGTGAGATGCATTCAAAGAACGTACACCCAAATAAGATTACCTACACTGTGATGATTGGTGGGTATGCTAGAGATGGGAATGTAACGGAAGCTTCTAGGCTTCTAAATGAGATGAGGGAAAAAGGAATTGTTCCGGATAGTATCACATACAAAGAATTTATATACGGATATCTTAAGCAGGGAGGTGTGTTGGAAGCTTTTAAAGGTTCTGATGAAGAAAATTATGCAGCAATCATTGAAGGATGGAACAAACTCATTCAGTAACTGATGAATAAGAAGAAGAACAGTTGGAAATTGAAATCTAGTAGCCATCTCAGTTTTGATTCACGCCGGGGATCTACGTCTTCTTCTTAGAGGAGCACGTTAGGATGGAGTTACTTGGTAAATGATGCATATGGTTCATTGCATCTTACAAGTAGGGTAGTGTGTTCTACTTGATTCTCCAGCCCATTCAGTTAACAGCTTTGGCCTGCTTCAATCTTAATGAGAAAAGTTGTTTCTTTTAGCATCAAAGGCTATATGGGTTTATGATTTACAAATCCACATTACTGGTACTCGTCAGGTTTAGGATTGAGTGCCACAGAATCAAAGGCATATGATAGATTCTGGATCAGAGCAGGTACTTCAGACAGCAGTCCTGAAAAGATTATACAGATTTAGACACACATCATGGATAGTCAATTAAAAGTTTACACATGGATATTCATGGGAGCTATGAGAATGGATCAATAGGAGGCTCCTCGGTTTACAGGTCACATAAATAGAGCTTAAGCGAATTCTCATCGGAGTAGTTTAGAACTAATTCATGCCTGACACATCGCTGGGGAGAATTTCCTCAGCCGTGCAGAATTTTTTGCCCAAGGAGATTAATGGACTGATTTGAAAGGTAACAGTTTACAACGGTATCAACTAATTCGACAACATTCACCTTAGAGTATAGACTTTCAGTGGGGTGAATACTGTTTATTCATCTTGTTTACGATTCTTTAAATTTGTTAATTCTCTTAAAGAGGTGGAAAAAGTGTTTTAGATGTACGTAGAAAGAAAGAAGGAAAACAAGAATAAAATTTGAAATACTCTTCATGATTTTTGGTTTTATTTTTTAACTTATACTACTAGTCAAGATACCCTAAACAGAGATTGTTATGAGAAGATTGTAAAAAAGCCAATATGAAA</li>
<li>&gt;AT4G19440.4
TTTGCCCAAATCGCCTCCTCTCGAGAATTTCTCCGTCTCTCTCTCTCTCTCCAATGGGGTGGCTCAACACCATCTTCAATTCTTCTTCTTCGACATTGATCAATGGCGGCTCTTCTCTATTTCCCTAAAATCTCTTCGCAAATGACTTCCTCACATTTCATCTCTTTCTCACCAATGGATTTGAGGAGACTTTCTCGCGCAAGCTACTTATTCACTCGCCGCTTGAAATTCATAGCGAAATCTCGAAAATGCTTTCATACAAGCCGATACCTGCAACAGTGTGTTCATCGGCCGGATAAATCGGAAGAGACTTCCTCTGATCGGCATTTACACGAACGATTATCTTCAGTTTTGTCGAAGAGATCTCTAGATTACGAGCAGTGTAAACAACTGATAACTGTTCTATCGCCTCTCGAATTCGATAGGTTGTTTCCTGAGTTTCGATCCAAAGTTAATCCCAAGACGGCTTTGGATTTCTTCCGATTAGCTTCTGATTCGTTTAGTTTCTCCTTTAGTCTCCGTTCTTATTGTTTATTGATAGGTTTATTGCTTGATGCGAATCTGTTATCGGCTGCTAGGGTAGTTTTGATTCGTTTGATCAATGGGAATGTACCGGTTTTGCCTTGTGGTTTGAGAGATAGTCGAGTTGCGATTGCGGATGCAATGGCGAGTTTGAGTTTGTGTTTTGATGAAGAAATCAGAAGGAAAATGTCGGATTTGTTGATTGAAGTTTATTGCACACAGTTCAAGCGTGATGGTTGTTATTTAGCTCTTGATGTATTTCCTGTTCTTGCAAACAAGGGTATGTTTCCTTCTAAGACGACTTGCAACATTTTGTTGACTTCTCTAGTGAGAGCTAATGAGTTTCAGAAGTGTTGTGAAGCTTTTGATGTTGTTTGCAAAGGGGTTTCTCCGGATGTTTACTTGTTCACTACAGCGATTAATGCGTTTTGCAAGGGAGGTAAAGTTGAGGAGGCCGTTAAGTTGTTCTCAAAAATGGAAGAAGCGGGTGTTGCACCAAACGTTGTCACTTTTAATACTGTTATTGATGGACTTGGGATGTGTGGAAGGTATGATGAGGCGTTTATGTTTAAAGAAAAGATGGTAGAAAGAGGCATGGAACCAACTCTTATCACTTATAGTATTCTTGTCAAGGGTTTGACCAGGGCTAAGAGGATTGGTGATGCTTATTTTGTCTTGAAGGAAATGACAAAGAAAGGGTTTCCTCCTAATGTTATTGTATATAATAATTTGATTGATAGTTTCATCGAGGCTGGAAGTTTGAACAAAGCTATTGAGATAAAGGATCTCATGGTTTCGAAAGGGTTGTCTCTCACATCGTCGACGTATAATACTCTCATAAAGGGATACTGCAAGAACGGTCAAGCAGATAATGCGGAGCGTCTCCTGAAGGAAATGCTGTCGATTGGTTTTAATGTAAATCAGGGCTCCTTTACATCTGTGATTTGCTTGTTGTGCAGTCATCTTATGTTTGACTCAGCACTTCGTTTTGTTGGAGAGATGCTATTACGCAACATGAGTCCTGGTGGTGGATTACTTACTACATTGATTTCTGGTCTCTGTAAGCATGGGAAACACTCAAAGGCACTAGAACTTTGGTTTCAATTTCTAAATAAAGGTTTTGTAGTGGACACGAGGACTTCAAATGCTTTACTTCATGGACTCTGTGAAGCAGGGAAGCTGGATGAAGCTTTTAGGATTCAAAAGGAGATATTAGGGCGTGGCTGTGTTATGGATAGAGTTTCCTATAACACGTTGATTTCTGGATGTTGCGGAAAGAAGAAACTAGATGAAGCTTTTATGTTCTTGGATGAAATGGTTAAGAGAGGACTTAAGCCTGACAATTATACTTATAGCATATTGATTTGCGGGCTGTTTAATATGAATAAAGTTGAGGAGGCCATACAGTTTTGGGATGATTGTAAACGGAATGGCATGCTTCCAGATGTTTATACATATTCGGTGATGATAGATGGATGTTGTAAAGCTGAAAGAACAGAAGAGGGTCAGGAATTTTTTGATGAGATGATGAGCAAAAACGTGCAGCCAAATACTGTTGTTTACAATCATCTTATCAGAGCATACTGTAGAAGCGGAAGGCTATCGATGGCCTTGGAACTCCGTGAAGATATGAAACACAAAGGAATTTCACCTAATTCTGCTACATACACTTCACTAATAAAGGGGATGTCAATCATTAGCCGTGTTGAGGAGGCAAAACTCCTCTTCGAGGAGATGAGGATGGAGGGTTTGGAACCGAATGTCTTTCATTATACAGCACTTATTGATGGATATGGTAAATTGGGTCAGATGGTCAAAGTTGAATGTCTTCTGCGTGAGATGCATTCAAAGAACGTACACCCAAATAAGATTACCTACACTGTGATGATTGGTGGGTATGCTAGAGATGGGAATGTAACGGAAGCTTCTAGGCTTCTAAATGAGATGAGGGAAAAAGGAATTGTTCCGGATAGTATCACATACAAAGAATTTATATACGGATATCTTAAGCAGGGAGGTGTGTTGGAAGCTTTTAAAGGTTCTGATGAAGAAAATTATGCAGCAATCATTGAAGGATGGAACAAACTCATTCAGTAACTGATGAATAAGAAGAAGAACAGTTGGAAATTGAAATCTAGTAGCCATCTCAGTTTTGATTCACGCCGGGGATCTACGTCTTCTTCTTAGAGGAGCACGTTAGGATGGAGTTACTTGGTAAATGATGCATATGGTTCATTGCATCTTACAAGTAGGGTAGTGTGTTCTACTTGATTCTCCAGCCCATTCAGTTAACAGCTTTGGCCTGCTTCAATCTTAATGAGAAAAGTTGTTTCTTTTAGCATCAAAGGCTATATGGGTTTATGATTTACAAATCCACATTACTGGTACTCGTCAGGTTTAGGATTGAGTGCCACAGAATCAAAGGCATATGATAGATTCTGGATCAGAGCAGGTACTTCAGACAGCAGTCCTGAAAAGATTATACAGATTTAGACACACATCATGGATAGTCAATTAAAAGTTTACACATGGATATTCATGGGAGCTATGAGAATGGATCAATAGGAGGCTCCTCGGTTTACAGGTCACATAAATAGAGCTTAAGCGAATTCTCATCGGAGTAGTTTAGAACTAATTCATGCCTGACACATCGCTGGGGAGAATTTCCTCAGCCGTGCAGAATTTTTTGCCCAAGGAGATTAATGGACTGATTTGAAAGGTAACAGTTTACAACGGTATCAACTAATTCGACAACATTCACCTTAGAGTATAGACTTTCAGTGGGGTGAATACTGTTTATTCATCTTGTTTACGATTCTTTAAATTTGTTAATTCTCTTAAAGAGGTGGAAAAAGTGTTTTAGATGTACGTAGAAAGAAAGAAGGAAAACAAGAATAAAATTTGAAATACTCTTCATGATTTTTGGTTTTATTTTTTAACTTATACTACTAGTCAAGATACCCTAAACAGAGATTGTTATGAGAAGATTGTAAAAAAGCCAATATGAAA</li>
</ul>

      </div>
  </div>
</div>








  



<div class="accordion ">
  <div class="accordion-header " data-accordion>
    CDS Sequence
    <svg
      class="accordion-icon"
      x="0px"
      y="0px"
      viewBox="0 0 512 512"
      xmlspace="preserve">
      <path
        fill="currentColor"
        d="M505.755,123.592c-8.341-8.341-21.824-8.341-30.165,0L256.005,343.176L36.421,123.592c-8.341-8.341-21.824-8.341-30.165,0 s-8.341,21.824,0,30.165l234.667,234.667c4.16,4.16,9.621,6.251,15.083,6.251c5.462,0,10.923-2.091,15.083-6.251l234.667-234.667 C514.096,145.416,514.096,131.933,505.755,123.592z"></path>
    </svg>
</div>


  <div class="accordion-content ">
    <div class="content-wrapper">
    <button class="copy-btn" aria-label="Copy Content">Copy</button>
    </div>
  
      <div class="text-to-copy">
    
      </div>
  </div>
</div>








  



<div class="accordion ">
  <div class="accordion-header " data-accordion>
    Protein Sequence
    <svg
      class="accordion-icon"
      x="0px"
      y="0px"
      viewBox="0 0 512 512"
      xmlspace="preserve">
      <path
        fill="currentColor"
        d="M505.755,123.592c-8.341-8.341-21.824-8.341-30.165,0L256.005,343.176L36.421,123.592c-8.341-8.341-21.824-8.341-30.165,0 s-8.341,21.824,0,30.165l234.667,234.667c4.16,4.16,9.621,6.251,15.083,6.251c5.462,0,10.923-2.091,15.083-6.251l234.667-234.667 C514.096,145.416,514.096,131.933,505.755,123.592z"></path>
    </svg>
</div>


  <div class="accordion-content ">
    <div class="content-wrapper">
    <button class="copy-btn" aria-label="Copy Content">Copy</button>
    </div>
  
      <div class="text-to-copy">
    
      </div>
  </div>
</div>

</p>

          </div>
        </div>
      </div>
    </div>
  </section>

    </main>
  
    
    
      <footer class="bg-theme-light dark:bg-darkmode-theme-light">
  <div class="container">
    <div class="row items-center py-0">
      <div class="lg:col-6 mb-8 text-center lg:mb-0">
        <ul>
          
        </ul>
      </div>
      <div class="lg:col-3 mb-8 text-center lg:mb-0 lg:mt-0 lg:text-right">
        <ul class="social-icons">
          
        </ul>
      </div>
    </div>
  </div>
  <div class="border-border dark:border-darkmode-border border-t py-7">
    <div class="text-light dark:text-darkmode-light container text-center">
      <p>
        Designed &amp; Developed by <a href="https://gohugo.io/"




 target="_blank"
 


>Hugo</a>
 and <a href="https://www.netlify.com/"




 target="_blank"
 


>netlify</a>
 | 2024 Jinbiao Cao (曹金彪)
      </p>
    </div>
  </div>
</footer>
      



  
    
      
    
  

  
    
      
    
  

  
    
      
    
  

  
    
      
    
  

  
    
      
    
  

  
    
      
    
  

  
    
      
    
  

  
    
      
    
  

  
    
      
    
  










  
  



<script
  crossorigin="anonymous"
  integrity="sha256-qB/nHiPX28YVBLSaTV&#43;c5NTxlhkrSM0wXi7NrqdMnrw="
  src="/js/script.min.a81fe71e23d7dbc61504b49a4d5f9ce4d4f196192b48cd305e2ecdaea74c9ebc.js"></script>


<script
  defer
  async
  crossorigin="anonymous"
  integrity="sha256-9e9iyznGCSfFZB4ElW&#43;WFonrJpHJr5pYxsaD7VVh4wc="
  src="/js/script-lazy.min.f5ef62cb39c60927c5641e04956f961689eb2691c9af9a58c6c683ed5561e307.js"></script>



<script>
  if ('serviceWorker' in navigator){navigator.serviceWorker.register("/service-worker.js");}
</script>






  








    
  </body>
  
</html>
