<!doctype html>
<html
  itemscope
  class=""
  lang="en-us"
  itemtype="http://schema.org/WebPage">
  <head>
    
    <meta charset="utf-8" />


<meta
  name="viewport"
  content="width=device-width, initial-scale=1, maximum-scale=5" />


<meta name="theme-name" content="hugoplate" />















<link rel="shortcut icon" href="/images/fpg_logo_hu8359c659081f4f0de026e43505d3fe2b_71406_96x0_resize_lanczos_3.png" type="image/x-icon">
<link rel="icon" href="/images/fpg_logo_hu8359c659081f4f0de026e43505d3fe2b_71406_96x0_resize_lanczos_3.png" type="image/x-icon">
<link rel="icon" type="image/png" sizes="48x48" href="/images/fpg_logo_hu8359c659081f4f0de026e43505d3fe2b_71406_48x0_resize_lanczos_3.png">
<link rel="icon" type="image/png" sizes="96x96" href="/images/fpg_logo_hu8359c659081f4f0de026e43505d3fe2b_71406_96x0_resize_lanczos_3.png">
<link rel="apple-touch-icon" sizes="144x144" href="/images/fpg_logo_hu8359c659081f4f0de026e43505d3fe2b_71406_144x0_resize_lanczos_3.png">








<link rel="manifest" href="/manifest.webmanifest" />
<meta
  name="msapplication-TileColor"
  content="#ddd" />
<meta
  name="theme-color"
  content="#ffffff" />





























  


<title>AT5G16750</title>















  <meta
    name="keywords"
    content="Boilerplate, Hugo" />




<meta
  name="description"
  content="AT5G16750 ; TOZ ; Arabidopsis thaliana" />



  <meta name="author" content="zeon.studio" />






  
  












  

  
  
  


  
  


  





<meta property="og:title" content="AT5G16750" />
<meta property="og:description" content="AT5G16750 ; TOZ ; Arabidopsis thaliana" />
<meta property="og:type" content="website" />
<meta property="og:url" content="/genes/arabidopsis_thaliana/at5g16750/" />


<meta name="twitter:title" content="AT5G16750" />
<meta name="twitter:description" content="AT5G16750 ; TOZ ; Arabidopsis thaliana" />





















<script>
  let indexURL = "/searchindex.json";
  let includeSectionsInSearch = ["genes","pages","sections","submit"];
  let no_results_for = "No results for";
  let empty_search_results_placeholder = "Type something to search..";
</script>




















    
    
<meta http-equiv="x-dns-prefetch-control" content="on" />
<link rel="preconnect" href="https://use.fontawesome.com" crossorigin />
<link rel="preconnect" href="//cdnjs.cloudflare.com" />
<link rel="preconnect" href="//www.googletagmanager.com" />
<link rel="preconnect" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="https://use.fontawesome.com" />
<link rel="dns-prefetch" href="//ajax.googleapis.com" />
<link rel="dns-prefetch" href="//cdnjs.cloudflare.com" />
<link rel="dns-prefetch" href="//www.googletagmanager.com" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//connect.facebook.net" />
<link rel="dns-prefetch" href="//platform.linkedin.com" />
<link rel="dns-prefetch" href="//platform.twitter.com" />




<link rel="preconnect" href="https://fonts.googleapis.com" />
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
<script>
  (function () {
    const googleFont = document.createElement("link");
    googleFont.href = "https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Montserrat:wght@500;700&display=swap";
    googleFont.type = "text/css";
    googleFont.rel = "stylesheet";
    document.head.appendChild(googleFont);
  })();
</script>





  
    
      
    
  

  
    
      
    
  

  
    
      
    
  

  
    
      
    
  

  
    
      
    
  











  
  



<link
  href="/css/style.min.ad82ade437df81f5c5a48af63c5d6c5da442a19f83f854c3b4168220f83e4f5f.css"
  integrity="sha256-rYKt5DffgfXFpIr2PF1sXaRCoZ+D+FTDtBaCIPg+T18="
  rel="stylesheet" />


<link
  defer
  async
  rel="stylesheet"
  href="/css/style-lazy.min.901e5cbd6a6883c52a645fd5c91bb3fec340173bc306529fe6cb027c818eb6bf.css"
  integrity="sha256-kB5cvWpog8UqZF/VyRuz/sNAFzvDBlKf5ssCfIGOtr8="
  media="print"
  onload="this.media='all'; this.onload=null;" />

  </head>

  <body style="padding-top: 2px; padding-bottom: 2px;">
    
    
      



      

    
  
    
    <header
  class="header sticky top-0 z-30">
  <nav class="navbar container">
    
    <div class="order-0">
      
      <a class="navbar-brand block" href="/">
        






















  
  funPlantGenes


      </a>
    </div>
    
    <input id="nav-toggle" type="checkbox" class="hidden" />
    <label
      for="nav-toggle"
      class="order-3 cursor-pointer flex items-center lg:hidden text-dark dark:text-white lg:order-1">
      <svg id="show-button" class="h-6 fill-current block" viewBox="0 0 20 20">
        <title>Menu Open</title>
        <path d="M0 3h20v2H0V3z m0 6h20v2H0V9z m0 6h20v2H0V0z"></path>
      </svg>
      <svg id="hide-button" class="h-6 fill-current hidden" viewBox="0 0 20 20">
        <title>Menu Close</title>
        <polygon
          points="11 9 22 9 22 11 11 11 11 22 9 22 9 11 -2 11 -2 9 9 9 9 -2 11 -2"
          transform="rotate(45 10 10)"></polygon>
      </svg>
    </label>
    

    
    <ul
      id="nav-menu"
      class="navbar-nav order-3 hidden lg:flex w-full pb-6 lg:order-1 lg:w-auto lg:space-x-2 lg:pb-0 xl:space-x-8">
      
      
        
        
        
        
          <li class="nav-item">
            <a
              class="nav-link "
              
              href="/species/"
              >SPECIES</a
            >
          </li>
        
      
        
        
        
        
          <li class="nav-item">
            <a
              class="nav-link "
              
              href="/genes/"
              >GENES</a
            >
          </li>
        
      
        
        
        
        
          <li class="nav-item">
            <a
              class="nav-link "
              
              href="/tags/"
              >FUNCTION</a
            >
          </li>
        
      
        
        
        
        
          <li class="nav-item nav-dropdown group relative">
            <span
              class="nav-link 
                
                
                
              
                
                
                
               inline-flex items-center">
              HELP
              <svg class="h-4 w-4 fill-current" viewBox="0 0 20 20">
                <path
                  d="M9.293 12.95l.707.707L15.657 8l-1.414-1.414L10 10.828 5.757 6.586 4.343 8z" />
              </svg>
            </span>
            <ul
              class="nav-dropdown-list lg:group-hover:visible lg:group-hover:opacity-100">
              
                
                
                <li class="nav-dropdown-item">
                  <a
                    class="nav-dropdown-link "
                    
                    href="/link/">
                    LINKS
                  </a>
                </li>
              
                
                
                <li class="nav-dropdown-item">
                  <a
                    class="nav-dropdown-link "
                    
                    href="/submit/">
                    SUBMIT
                  </a>
                </li>
              
            </ul>
          </li>
        
      
      
        <li class="mt-4 inline-block lg:hidden">
          <a
            class="btn btn-outline-primary btn-sm"
            href="/submit">
            Submit
          </a>
        </li>
      
    </ul>

    
     

  <div class="order-1 ml-auto flex items-center md:order-2 lg:ml-0" style="margin-right: 20px;">

    <input
  id="search-modal-input"
  type="text"
  placeholder="Search"
  autocomplete="on"
  aria-label="Search"
  aria-describedby="Search"
  placeholder="Search Post ..."
  style="width: 120px; /* 设置宽度为800像素 */
         padding: 5px;
         border: 2px solid #ccc;
         border-radius: 20px;
         box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
         outline: none;
         transition: border-color 0.3s ease-in-out;
         margin-right: 20px; /* 添加右边距 */"/>
    <button
        id="search-button"
        aria-label="search"
        class="border-border text-dark hover:text-primary dark:border-darkmode-border mr-5 inline-block border-r pr-5 text-xl dark:text-white dark:hover:text-darkmode-primary"
        data-target="search-modal"
        onclick="search()">
        <i class="fa-solid fa-search"></i>
    </button>

   
   </div>

    <div class="order-1 ml-auto flex items-center md:order-2 lg:ml-0" style="margin-right: 20px;"></div>
          









          


  <div class="theme-switcher mr-5">
    <input id="theme-switcher" data-theme-switcher type="checkbox" />
    <label for="theme-switcher">
      <span class="sr-only">theme switcher</span>
      <span>
        
        <svg
          class="absolute left-[4px] top-[4px] z-10 opacity-100 dark:opacity-0"
          viewBox="0 0 56 56"
          fill="#fff"
          height="16"
          width="16">
          <path
            d="M30 4.6c0-1-.9-2-2-2a2 2 0 0 0-2 2v5c0 1 .9 2 2 2s2-1 2-2Zm9.6 9a2 2 0 0 0 0 2.8c.8.8 2 .8 2.9 0L46 13a2 2 0 0 0 0-2.9 2 2 0 0 0-3 0Zm-26 2.8c.7.8 2 .8 2.8 0 .8-.7.8-2 0-2.9L13 10c-.7-.7-2-.8-2.9 0-.7.8-.7 2.1 0 3ZM28 16a12 12 0 0 0-12 12 12 12 0 0 0 12 12 12 12 0 0 0 12-12 12 12 0 0 0-12-12Zm23.3 14c1.1 0 2-.9 2-2s-.9-2-2-2h-4.9a2 2 0 0 0-2 2c0 1.1 1 2 2 2ZM4.7 26a2 2 0 0 0-2 2c0 1.1.9 2 2 2h4.9c1 0 2-.9 2-2s-1-2-2-2Zm37.8 13.6a2 2 0 0 0-3 0 2 2 0 0 0 0 2.9l3.6 3.5a2 2 0 0 0 2.9 0c.8-.8.8-2.1 0-3ZM10 43.1a2 2 0 0 0 0 2.9c.8.7 2.1.8 3 0l3.4-3.5c.8-.8.8-2.1 0-2.9-.8-.8-2-.8-2.9 0Zm20 3.4c0-1.1-.9-2-2-2a2 2 0 0 0-2 2v4.9c0 1 .9 2 2 2s2-1 2-2Z" />
        </svg>
        
        <svg
          class="absolute left-[4px] top-[4px] z-10 opacity-0 dark:opacity-100"
          viewBox="0 0 24 24"
          fill="none"
          height="16"
          width="16">
          <path
            fill="#000"
            fill-rule="evenodd"
            clip-rule="evenodd"
            d="M8.2 2.2c1-.4 2 .6 1.6 1.5-1 3-.4 6.4 1.8 8.7a8.4 8.4 0 0 0 8.7 1.8c1-.3 2 .5 1.5 1.5v.1a10.3 10.3 0 0 1-9.4 6.2A10.3 10.3 0 0 1 3.2 6.7c1-2 2.9-3.5 4.9-4.4Z" />
        </svg>
      </span>
    </label>
  </div>

  
  <script>
    var darkMode = false;

    
    if (window.matchMedia("(prefers-color-scheme: dark)").matches){darkMode = true}
    

    if (localStorage.getItem("theme") === "dark"){darkMode = true}
    else if (localStorage.getItem("theme") === "light"){darkMode = false}
    if (darkMode){document.documentElement.classList.toggle("dark")}
    var themeSwitch = document.querySelectorAll("[data-theme-switcher]");

    document.addEventListener("DOMContentLoaded", () => {
      [].forEach.call(themeSwitch, function (ts) {
        ts.checked = darkMode ? true : false;
        ts.addEventListener("click", () => {
          document.documentElement.classList.toggle("dark");
          localStorage.setItem(
            "theme",
            document.documentElement.classList.contains("dark") ? "dark" : "light"
          );
        });
      });
    });
  </script>


    </div>

  </nav>
       
</header>

    




  
    <div
      class="search-modal "
      aria-hidden="true"
      style="--color-primary: #121212">
      <div data-target="close-search-modal" class="search-modal-overlay"></div>
      <div
        class="search-wrapper"
        #data-image="true"
        data-description="true"
        data-tags="true"
        data-categories="true">
        <div class="search-wrapper-header">
          <label for="search-modal-input" style="margin-top:-1px">
            <span class="sr-only">search icon</span>
          </label>
        </div>   
        <div class="search-wrapper-body">
          <div class="search-result" data-search-result></div>
          <span class="search-result-empty">
            Type something to search..
          </span>
        </div>
        <div class="search-wrapper-footer">
          <span>
            <kbd>
              <svg
                xmlns="http://www.w3.org/2000/svg"
                width="14"
                height="14"
                fill="currentColor"
                viewBox="0 0 16 16">
                <path
                  d="M3.204 11h9.592L8 5.519 3.204 11zm-.753-.659 4.796-5.48a1 1 0 0 1 1.506 0l4.796 5.48c.566.647.106 1.659-.753 1.659H3.204a1 1 0 0 1-.753-1.659z" />
              </svg>
            </kbd>
            <kbd>
              <svg
                xmlns="http://www.w3.org/2000/svg"
                width="14"
                height="14"
                fill="currentColor"
                style="margin-top:1px;"
                viewBox="0 0 16 16">
                <path
                  d="M3.204 5h9.592L8 10.481 3.204 5zm-.753.659 4.796 5.48a1 1 0 0 0 1.506 0l4.796-5.48c.566-.647.106-1.659-.753-1.659H3.204a1 1 0 0 0-.753 1.659z" />
              </svg>
            </kbd>
            to navigate
          </span>
          <span>
            <kbd>
              <svg
                xmlns="http://www.w3.org/2000/svg"
                width="12"
                height="12"
                fill="currentColor"
                style="display:inline-block;"
                viewBox="0 0 16 16">
                <path
                  fill-rule="evenodd"
                  d="M14.5 1.5a.5.5 0 0 1 .5.5v4.8a2.5 2.5 0 0 1-2.5 2.5H2.707l3.347 3.346a.5.5 0 0 1-.708.708l-4.2-4.2a.5.5 0 0 1 0-.708l4-4a.5.5 0 1 1 .708.708L2.707 8.3H12.5A1.5 1.5 0 0 0 14 6.8V2a.5.5 0 0 1 .5-.5z" />
              </svg>
            </kbd>
            to select
          </span>
          <span class="search-result-info"></span>
          <span data-target="close-search-modal">
            <kbd>ESC</kbd> to close
          </span>
        </div>
      </div>
    </div>
  


  
    <main>
      
  <section class="section-sm" style="padding-top: 15px; padding-bottom: 15px;">
    <div class="container">
      <div class="row justify-center">
        <div class="lg:col-11">
          <div class="content">
            <h2 id="gene-details">Gene Details:</h2>
<ul>
<li><strong>Gene ID:</strong> <a href="https://www.arabidopsis.org/locus?name=AT5G16750"




 target="_blank"
 


>AT5G16750</a>
</li>
<li><strong>Gene Symbol:</strong> <u>TOZ</u></li>
<li><strong>Gene Name:</strong> TORMOZEMBRYO DEFECTIVE</li>
<li><strong>Description:</strong>   Transducin family protein / WD-40 repeat family protein;(source:Araport11)</li>
<li><strong>TAIR Accession:</strong> locus:2148965</li>
<li><strong>Genome:</strong> <a href="https://www.arabidopsis.org/download/list?dir=Genes%2FAraport11_genome_release"




 target="_blank"
 


>Araport11_genome_release</a>
</li>
<li><strong>Species:</strong> <em>Arabidopsis thaliana</em></li>
</ul>
<h2 id="transcripts">Transcripts:</h2>
<ul>
<li><a href="https://www.arabidopsis.org/gene?name=AT5G16750.1"




 target="_blank"
 


>AT5G16750.1</a>
</li>
</ul>
<h2 id="plant-ontology-annotations">Plant Ontology Annotations:</h2>
<ul>
<li><a href="https://browser.planteome.org/amigo/term/PO:0009005"




 target="_blank"
 


>PO:0009005</a>
 &mdash; root &mdash; raíz (Spanish, exact), radices (exact, plural), radix (exact), 根 (Japanese, exact), aerial root (narrow), climbing root (narrow)</li>
</ul>
<h2 id="gene-ontology">Gene Ontology:</h2>
<ul>
<li><a href="https://amigo.geneontology.org/amigo/term/GO:0009880"




 target="_blank"
 


>GO:0009880</a>
 &mdash; acts upstream of or within &mdash; embryonic pattern specification</li>
<li><a href="https://amigo.geneontology.org/amigo/term/GO:0009793"




 target="_blank"
 


>GO:0009793</a>
 &mdash; acts upstream of or within &mdash; embryo development ending in seed dormancy</li>
<li><a href="https://amigo.geneontology.org/amigo/term/GO:0005634"




 target="_blank"
 


>GO:0005634</a>
 &mdash; located in &mdash; nucleus</li>
<li><a href="https://amigo.geneontology.org/amigo/term/GO:0051301"




 target="_blank"
 


>GO:0051301</a>
 &mdash; acts upstream of or within &mdash; cell division</li>
<li><a href="https://amigo.geneontology.org/amigo/term/GO:0000472"




 target="_blank"
 


>GO:0000472</a>
 &mdash; involved in &mdash; endonucleolytic cleavage in 5-ETS of tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)	GO:0000480	27824	P	other metabolic processes	IBA	none	PANTHER:PTN000457905|SGD:S000004212	Communication:501741973		2021-03-31
AT5G16750	locus:2148965	AT5G16750	part of	small-subunit processome	GO:0032040	23326	C	other cellular components	IEA	none	InterPro:IPR013934	AnalysisReference:501756966		2023-06-11
AT5G16750	locus:2148965	AT5G16750	involved in	endonucleolytic cleavage to generate mature 5-end of SSU-rRNA from (SSU-rRNA, 5.8S rRNA, LSU-rRNA)</li>
<li><a href="https://amigo.geneontology.org/amigo/term/GO:0005730"




 target="_blank"
 


>GO:0005730</a>
 &mdash; is active in &mdash; nucleolus</li>
<li><a href="https://amigo.geneontology.org/amigo/term/GO:0000472"




 target="_blank"
 


>GO:0000472</a>
 &mdash; involved in &mdash; endonucleolytic cleavage to generate mature 5-end of SSU-rRNA from (SSU-rRNA, 5.8S rRNA, LSU-rRNA)	GO:0000472	27817	P	other metabolic processes	IBA	none	PANTHER:PTN000457905|SGD:S000004212	Communication:501741973		2021-03-31</li>
<li><a href="https://amigo.geneontology.org/amigo/term/GO:0005730"




 target="_blank"
 


>GO:0005730</a>
 &mdash; located in &mdash; nucleolus</li>
<li><a href="https://amigo.geneontology.org/amigo/term/"




 target="_blank"
 


></a>
 &mdash; involved in &mdash; endonucleolytic cleavage in 5-ETS of tricistronic rRNA transcript (SSU-rRNA, 5.8S rRNA, LSU-rRNA)	GO:0000480	27824	P	nucleobase-containing compound metabolic process	IBA	none	PANTHER:PTN000457905|SGD:S000004212	Communication:501741973		2021-03-31
AT5G16750	locus:2148965	AT5G16750	acts upstream of or within	embryonic pattern specification	GO:0009880	11997	P	embryo development	IMP	analysis of visible trait	NONE	Publication:501722778|PMID:17616738  	TAIR	2007-09-11
AT5G16750	locus:2148965	AT5G16750	enables	U3 snoRNA binding	GO:0034511	29638	F	RNA binding	IBA	none	PANTHER:PTN000457905|SGD:S000004212	Communication:501741973		2021-04-02
AT5G16750	locus:2148965	AT5G16750	acts upstream of or within	embryo development ending in seed dormancy	GO:0009793	11994	P	embryo development	IMP	analysis of visible trait	NONE	Publication:501722778|PMID:17616738  	TAIR	2007-09-11
AT5G16760	locus:2148990	AT5G16760	enables	inositol tetrakisphosphate 1-kinase activity	GO:0047325	15923	F	transferase activity	IBA	none	PANTHER:PTN001022327|UniProtKB:A7X672|UniProtKB:Q84Y01|TAIR:locus:2051744|UniProtKB:Q13572|TAIR:locus:2134253|UniProtKB:A7X657|UniProtKB:A7X680|UniProtKB:A7X665|TAIR:locus:2132487	Communication:501741973		2023-06-07
AT5G16760	locus:2148990	AT5G16760	involved in	inositol trisphosphate metabolic process	GO:0032957	26623	P	other cellular processes	IDA	Enzyme assays		Publication:3347|PMID:9126335   	TAIR	2023-01-10
AT5G16760	locus:2148990	AT5G16760	enables	inositol-1,3,4-trisphosphate 6-kinase activity	GO:0052725	38930	F	kinase activity	IBA	none	PANTHER:PTN001022327|UniProtKB:A7X665|UniProtKB:A7X680|TAIR:locus:2132487|UniProtKB:A7X657|TAIR:locus:2134253|TAIR:locus:2051744|UniProtKB:Q13572|UniProtKB:A7X672|UniProtKB:Q84Y01|TAIR:locus:2148990	Communication:501741973		2023-06-07
AT5G16760	locus:2148990	AT5G16760	involved in	inositol trisphosphate metabolic process	GO:0032957	26623	P	other metabolic processes	IEA	none	InterPro:IPR008656	AnalysisReference:501756966		2023-05-23
AT5G16760	locus:2148990	AT5G16760	involved in	auxin-activated signaling pathway	GO:0009734	11415	P	signal transduction	IMP	biochemical/chemical analysis		Publication:501800588|PMID:36124979  	TAIR	2022-10-20
AT5G16760	locus:2148990	AT5G16760	enables	inositol-1,3,4-trisphosphate 6-kinase activity	GO:0052725	38930	F	transferase activity	IDA	Enzyme assays		Publication:3347|PMID:9126335   	TAIR	2011-09-15
AT5G16760	locus:2148990	AT5G16760	enables	inositol-1,3,4-trisphosphate 6-kinase activity	GO:0052725	38930	F	kinase activity	IDA	Enzyme assays		Publication:3347|PMID:9126335   	TAIR	2011-09-15
AT5G16760	locus:2148990	AT5G16760	acts upstream of or within	myo-inositol hexakisphosphate biosynthetic process	GO:0010264	23388	P	other metabolic processes	IMP	biochemical/chemical analysis	NONE	Publication:501743051|PMID:21698461  	TAIR	2012-06-21
AT5G16760	locus:2148990	AT5G16760	involved in	auxin-activated signaling pathway	GO:0009734	11415	P	response to endogenous stimulus	IMP	biochemical/chemical analysis		Publication:501800588|PMID:36124979  	TAIR	2022-10-20
AT5G16760	locus:2148990	AT5G16760	acts upstream of or within	myo-inositol hexakisphosphate biosynthetic process	GO:0010264	23388	P	other cellular processes	IMP	biochemical/chemical analysis	NONE	Publication:501743051|PMID:21698461  	TAIR	2012-06-21
AT5G16760	locus:2148990	AT5G16760	involved in	auxin-activated signaling pathway	GO:0009734	11415	P	other cellular processes	IMP	biochemical/chemical analysis		Publication:501800588|PMID:36124979  	TAIR	2022-10-20
AT5G16760	locus:2148990	AT5G16760	enables	inositol-1,3,4-trisphosphate 5-kinase activity	GO:0052726	38931	F	kinase activity	IBA	none	PANTHER:PTN001022327|TAIR:locus:2051744|UniProtKB:Q13572|TAIR:locus:2134253|UniProtKB:A7X657|UniProtKB:A7X672|TAIR:locus:2148990|UniProtKB:Q84Y01|UniProtKB:A7X680|UniProtKB:A7X665|TAIR:locus:2132487	Communication:501741973		2023-06-07
AT5G16760	locus:2148990	AT5G16760	enables	inositol-1,3,4-trisphosphate 5-kinase activity	GO:0052726	38931	F	kinase activity	IDA	Enzyme assays		Publication:3347|PMID:9126335   	TAIR	2011-09-15
AT5G16760	locus:2148990	AT5G16760	involved in	auxin-activated signaling pathway	GO:0009734	11415	P	response to chemical	IMP	biochemical/chemical analysis		Publication:501800588|PMID:36124979  	TAIR	2022-10-20
AT5G16760	locus:2148990	AT5G16760	involved in	inositol trisphosphate metabolic process	GO:0032957	26623	P	other metabolic processes	IDA	Enzyme assays		Publication:3347|PMID:9126335   	TAIR	2023-01-10
AT5G16760	locus:2148990	AT5G16760	enables	inositol-1,3,4-trisphosphate 6-kinase activity	GO:0052725	38930	F	transferase activity	IBA	none	PANTHER:PTN001022327|UniProtKB:A7X665|UniProtKB:A7X680|TAIR:locus:2132487|UniProtKB:A7X657|TAIR:locus:2134253|TAIR:locus:2051744|UniProtKB:Q13572|UniProtKB:A7X672|UniProtKB:Q84Y01|TAIR:locus:2148990	Communication:501741973		2023-06-07
AT5G16760	locus:2148990	AT5G16760	enables	signaling receptor binding	GO:0005102	3002	F	signaling receptor binding	IPI	Bimolecular fluorescence complementation (BiFC)	AGI_LocusCode:AT3G62980	Publication:501800588|PMID:36124979  	TAIR	2022-10-20
AT5G16760	locus:2148990	AT5G16760	acts upstream of or within	myo-inositol hexakisphosphate biosynthetic process	GO:0010264	23388	P	biosynthetic process	IMP	biochemical/chemical analysis	NONE	Publication:501743051|PMID:21698461  	TAIR	2012-06-21
AT5G16760	locus:2148990	AT5G16760	is active in	nucleus	GO:0005634	537	C	nucleus	IPI	Bimolecular fluorescence complementation (BiFC)	AGI_LocusCode:AT3G62980	Publication:501800588|PMID:36124979  	TAIR	2022-10-20
AT5G16760	locus:2148990	AT5G16760	enables	inositol-1,3,4-trisphosphate 5-kinase activity	GO:0052726	38931	F	transferase activity	IDA	Enzyme assays		Publication:3347|PMID:9126335   	TAIR	2011-09-15
AT5G16760	locus:2148990	AT5G16760	involved in	inositol trisphosphate metabolic process	GO:0032957	26623	P	other cellular processes	IEA	none	InterPro:IPR008656	AnalysisReference:501756966		2023-05-23
AT5G16760	locus:2148990	AT5G16760	enables	magnesium ion binding	GO:0000287	3101	F	other binding	IEA	none	InterPro:IPR008656	AnalysisReference:501756966		2023-06-11
AT5G16760	locus:2148990	AT5G16760	enables	inositol-1,3,4-trisphosphate 5-kinase activity	GO:0052726	38931	F	transferase activity	IBA	none	PANTHER:PTN001022327|TAIR:locus:2051744|UniProtKB:Q13572|TAIR:locus:2134253|UniProtKB:A7X657|UniProtKB:A7X672|TAIR:locus:2148990|UniProtKB:Q84Y01|UniProtKB:A7X680|UniProtKB:A7X665|TAIR:locus:2132487	Communication:501741973		2023-06-07
AT5G16760	locus:2148990	AT5G16760	enables	inositol tetrakisphosphate 1-kinase activity	GO:0047325	15923	F	kinase activity	IBA	none	PANTHER:PTN001022327|UniProtKB:A7X672|UniProtKB:Q84Y01|TAIR:locus:2051744|UniProtKB:Q13572|TAIR:locus:2134253|UniProtKB:A7X657|UniProtKB:A7X680|UniProtKB:A7X665|TAIR:locus:2132487	Communication:501741973		2023-06-07
AT5G16770	gene:6532549738	AT5G16770.4	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G16770	locus:2149000	AT5G16770	enables	transcription cis-regulatory region binding	GO:0000976	35767	F	nucleic acid binding	IPI	yeast one-hybrid assay	AGI_LocusCode:AT5G53460|AGI_LocusCode:AT1G22710	Publication:501786139|PMID:30356219  	sibrady	2020-12-18
AT5G16770	locus:2149000	AT5G16770	enables	DNA-binding transcription factor activity	GO:0003700	4449	F	DNA-binding transcription factor activity	ISS	Recognized domains		Publication:1345963|PMID:11118137  	TAIR	2003-06-06
AT5G16770	locus:2149000	AT5G16770	acts upstream of or within	response to water deprivation	GO:0009414	5647	P	response to stress	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT3G16857	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16770	locus:2149000	AT5G16770	acts upstream of or within	organic cyclic compound biosynthetic process	GO:1901362	43684	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT3G16857	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16770	gene:1006229165	AT5G16770.2	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G16770	locus:2149000	AT5G16770	acts upstream of or within	response to abscisic acid	GO:0009737	11395	P	response to chemical	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT2G46270	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16770	locus:2149000	AT5G16770	acts upstream of or within	aromatic compound biosynthetic process	GO:0019438	10128	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT3G16857	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16770	gene:6532557503	AT5G16770.3	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G16770	locus:2149000	AT5G16770	enables	transcription cis-regulatory region binding	GO:0000976	35767	F	DNA binding	IPI	yeast one-hybrid assay	AGI_LocusCode:AT1G68810	Publication:501787543|PMID:31806676  	petchells	2020-04-17
AT5G16770	locus:2149000	AT5G16770	acts upstream of or within	organic cyclic compound biosynthetic process	GO:1901362	43684	P	biosynthetic process	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT3G16857	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16770	locus:2149000	AT5G16770	acts upstream of or within	secondary metabolite biosynthetic process	GO:0044550	40579	P	secondary metabolic process	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16770	locus:2149000	AT5G16770	acts upstream of or within	response to water deprivation	GO:0009414	5647	P	response to chemical	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT3G16857	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16770	locus:2149000	AT5G16770	enables	DNA-binding transcription factor activity	GO:0003700	4449	F	DNA-binding transcription factor activity	IDA	transcriptional activation assay		Publication:501789048|PMID:32246506  	sbaud	2020-04-14
AT5G16770	locus:2149000	AT5G16770	acts upstream of or within	response to abscisic acid	GO:0009737	11395	P	response to endogenous stimulus	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT2G46270	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16770	locus:2149000	AT5G16770	acts upstream of or within	aromatic compound biosynthetic process	GO:0019438	10128	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT3G16857	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16770	locus:2149000	AT5G16770	enables	transcription cis-regulatory region binding	GO:0000976	35767	F	nucleic acid binding	IPI	yeast one-hybrid assay	AGI_LocusCode:AT1G68810	Publication:501787543|PMID:31806676  	petchells	2020-04-17
AT5G16770	locus:2149000	AT5G16770	enables	transcription cis-regulatory region binding	GO:0000976	35767	F	DNA binding	IPI	yeast one-hybrid assay	AGI_LocusCode:AT5G53460|AGI_LocusCode:AT1G22710	Publication:501786139|PMID:30356219  	sibrady	2020-12-18
AT5G16770	gene:2148999	AT5G16770.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G16770	locus:2149000	AT5G16770	acts upstream of or within	aromatic compound biosynthetic process	GO:0019438	10128	P	biosynthetic process	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT3G16857	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16770	locus:2149000	AT5G16770	acts upstream of or within	response to water deprivation	GO:0009414	5647	P	response to abiotic stimulus	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT3G16857	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16770	locus:2149000	AT5G16770	acts upstream of or within	secondary metabolite biosynthetic process	GO:0044550	40579	P	biosynthetic process	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16780	locus:2149010	AT5G16780	acts upstream of or within	cotyledon vascular tissue pattern formation	GO:0010588	29504	P	anatomical structure development	IMP	analysis of visible trait	NONE	Publication:501727292|PMID:18643975  	TAIR	2008-09-23
AT5G16780	locus:2149010	AT5G16780	acts upstream of or within	root development	GO:0048364	18902	P	anatomical structure development	IMP	analysis of visible trait	NONE	Publication:501727292|PMID:18643975  	TAIR	2008-09-23
AT5G16780	gene:2149009	AT5G16780.1	acts upstream of or within	meristem structural organization	GO:0009933	13132	P	anatomical structure development	IMP	analysis of visible trait	NONE	Publication:501729155|PMID:19000164  	TAIR	2008-12-16
AT5G16780	locus:2149010	AT5G16780	acts upstream of or within	flower development	GO:0009908	11347	P	flower development	IMP	analysis of visible trait	NONE	Publication:501727292|PMID:18643975  	TAIR	2008-09-23
AT5G16780	locus:2149010	AT5G16780	acts upstream of or within	cotyledon vascular tissue pattern formation	GO:0010588	29504	P	multicellular organism development	IMP	analysis of visible trait	NONE	Publication:501727292|PMID:18643975  	TAIR	2008-09-23
AT5G16780	locus:2149010	AT5G16780	acts upstream of or within	cotyledon vascular tissue pattern formation	GO:0010588	29504	P	post-embryonic development	IMP	analysis of visible trait	NONE	Publication:501727292|PMID:18643975  	TAIR	2008-09-23
AT5G16780	locus:2149010	AT5G16780	part of	U4/U6 x U5 tri-snRNP complex	GO:0046540	13420	C	nucleus	IBA	none	PANTHER:PTN000365724|PomBase:SPAC167.03c|UniProtKB:O43290|SGD:S000005835	Communication:501741973		2023-06-07
AT5G16780	locus:2149010	AT5G16780	acts upstream of or within	cotyledon vascular tissue pattern formation	GO:0010588	29504	P	reproduction	IMP	analysis of visible trait	NONE	Publication:501727292|PMID:18643975  	TAIR	2008-09-23
AT5G16780	locus:2149010	AT5G16780	acts upstream of or within	leaf vascular tissue pattern formation	GO:0010305	25145	P	multicellular organism development	IMP	analysis of visible trait	NONE	Publication:501727292|PMID:18643975  	TAIR	2008-09-23
AT5G16780	locus:2149010	AT5G16780	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	Unknown molecular function		Communication:1345790	TAIR	2006-10-20
AT5G16780	gene:2149009	AT5G16780.1	acts upstream of or within	post-embryonic root development	GO:0048528	20803	P	post-embryonic development	IMP	analysis of visible trait	NONE	Publication:501729155|PMID:19000164  	TAIR	2008-12-16
AT5G16780	gene:6532557068	AT5G16780.3	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G16780	gene:2149009	AT5G16780.1	acts upstream of or within	post-embryonic root development	GO:0048528	20803	P	anatomical structure development	IMP	analysis of visible trait	NONE	Publication:501729155|PMID:19000164  	TAIR	2008-12-16
AT5G16780	locus:2149010	AT5G16780	acts upstream of or within	shoot system development	GO:0048367	18906	P	multicellular organism development	IMP	analysis of visible trait	NONE	Publication:501727292|PMID:18643975  	TAIR	2008-09-23
AT5G16780	locus:2149010	AT5G16780	acts upstream of or within	root development	GO:0048364	18902	P	multicellular organism development	IMP	analysis of visible trait	NONE	Publication:501727292|PMID:18643975  	TAIR	2008-09-23
AT5G16780	gene:2149009	AT5G16780.1	acts upstream of or within	post-embryonic root development	GO:0048528	20803	P	multicellular organism development	IMP	analysis of visible trait	NONE	Publication:501729155|PMID:19000164  	TAIR	2008-12-16
AT5G16780	gene:2149009	AT5G16780.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G16780	locus:2149010	AT5G16780	acts upstream of or within	shoot system development	GO:0048367	18906	P	anatomical structure development	IMP	analysis of visible trait	NONE	Publication:501727292|PMID:18643975  	TAIR	2008-09-23
AT5G16780	gene:2149009	AT5G16780.1	located in	nucleus	GO:0005634	537	C	nucleus	IDA	localization of GFP/YFP fusion protein		Publication:501729155|PMID:19000164  	TAIR	2008-12-16
AT5G16780	locus:2149010	AT5G16780	acts upstream of or within	phloem or xylem histogenesis	GO:0010087	14841	P	anatomical structure development	IMP	analysis of visible trait	NONE	Publication:501727292|PMID:18643975  	TAIR	2008-09-23
AT5G16780	locus:2149010	AT5G16780	involved in	maturation of 5S rRNA	GO:0000481	27825	P	other metabolic processes	IBA	none	PANTHER:PTN000365724|SGD:S000005835	Communication:501741973		2021-03-31
AT5G16780	gene:2149009	AT5G16780.1	located in	nucleolus	GO:0005730	527	C	nucleolus	HDA	none	NONE	Publication:501714224|PMID:15496452  	kvanwijk	2021-05-10
AT5G16780	locus:2149010	AT5G16780	acts upstream of or within	leaf development	GO:0048366	18907	P	multicellular organism development	IMP	analysis of visible trait	NONE	Publication:501727292|PMID:18643975  	TAIR	2008-09-23
AT5G16780	locus:2149010	AT5G16780	involved in	mRNA cis splicing, via spliceosome	GO:0045292	11531	P	nucleobase-containing compound metabolic process	IBA	none	PANTHER:PTN000365724|PomBase:SPAC167.03c	Communication:501741973		2021-03-31
AT5G16780	locus:2149010	AT5G16780	involved in	mRNA cis splicing, via spliceosome	GO:0045292	11531	P	other metabolic processes	IBA	none	PANTHER:PTN000365724|PomBase:SPAC167.03c	Communication:501741973		2021-03-31
AT5G16780	gene:6532553520	AT5G16780.2	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G16780	locus:2149010	AT5G16780	involved in	maturation of 5S rRNA	GO:0000481	27825	P	other cellular processes	IBA	none	PANTHER:PTN000365724|SGD:S000005835	Communication:501741973		2021-03-31
AT5G16780	locus:2149010	AT5G16780	acts upstream of or within	cotyledon vascular tissue pattern formation	GO:0010588	29504	P	embryo development	IMP	analysis of visible trait	NONE	Publication:501727292|PMID:18643975  	TAIR	2008-09-23
AT5G16780	locus:2149010	AT5G16780	involved in	maturation of 5S rRNA	GO:0000481	27825	P	nucleobase-containing compound metabolic process	IBA	none	PANTHER:PTN000365724|SGD:S000005835	Communication:501741973		2021-03-31
AT5G16780	locus:2149010	AT5G16780	acts upstream of or within	leaf development	GO:0048366	18907	P	anatomical structure development	IMP	analysis of visible trait	NONE	Publication:501727292|PMID:18643975  	TAIR	2008-09-23
AT5G16790	locus:2149020	AT5G16790	part of	THO complex	GO:0000347	13955	C	nucleus	IDA	none		Publication:501738451|PMID:20634427  		2021-04-01
AT5G16790	locus:2149020	AT5G16790	involved in	mRNA export from nucleus	GO:0006406	6235	P	other metabolic processes	IBA	none	PANTHER:PTN000381981|UniProtKB:Q6I9Y2	Communication:501741973		2021-03-31
AT5G16790	locus:2149020	AT5G16790	involved in	mRNA export from nucleus	GO:0006406	6235	P	transport	IBA	none	PANTHER:PTN000381981|UniProtKB:Q6I9Y2	Communication:501741973		2021-03-31
AT5G16790	locus:2149020	AT5G16790	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	Unknown molecular function		Communication:1345790	TAIR	2021-10-28
AT5G16790	locus:2149020	AT5G16790	involved in	mRNA export from nucleus	GO:0006406	6235	P	other cellular processes	IBA	none	PANTHER:PTN000381981|UniProtKB:Q6I9Y2	Communication:501741973		2021-03-31
AT5G16790	locus:2149020	AT5G16790	part of	THO complex part of transcription export complex	GO:0000445	27145	C	nucleus	IBA	none	PANTHER:PTN000381981|UniProtKB:Q6I9Y2	Communication:501741973		2021-04-01
AT5G16790	gene:2149019	AT5G16790.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G16790	locus:2149020	AT5G16790	acts upstream of or within	regulation of gene expression	GO:0010468	28541	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT3G11100	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16790	gene:6532559102	AT5G16790.2	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G16800	locus:2149030	AT5G16800	enables	histone acetyltransferase activity	GO:0004402	2723	F	catalytic activity	IBA	none	PANTHER:PTN000380943|UniProtKB:Q9H7X0|FB:FBgn0036039	Communication:501741973		2023-04-13
AT5G16800	locus:2149030	AT5G16800	acts upstream of or within	N-terminal protein amino acid acetylation	GO:0006474	4852	P	other metabolic processes	IDA	in vitro assay	NONE	Publication:501790064|PMID:32548857  	TAIR	2020-06-28
AT5G16800	gene:2149029	AT5G16800.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G16800	locus:2149030	AT5G16800	enables	histone acetyltransferase activity	GO:0004402	2723	F	transferase activity	IBA	none	PANTHER:PTN000380943|UniProtKB:Q9H7X0|FB:FBgn0036039	Communication:501741973		2023-04-13
AT5G16800	locus:2149030	AT5G16800	is active in	Golgi membrane	GO:0000139	54	C	Golgi apparatus	IBA	none	PANTHER:PTN000380943|UniProtKB:Q9H7X0	Communication:501741973		2021-04-02
AT5G16800	locus:2149030	AT5G16800	located in	membrane	GO:0016020	453	C	other membranes	IDA	localization of GFP/YFP fusion protein	NONE	Publication:501790064|PMID:32548857  	TAIR	2020-06-28
AT5G16800	locus:2149030	AT5G16800	involved in	N-terminal peptidyl-methionine acetylation	GO:0017196	9135	P	other metabolic processes	IBA	none	PANTHER:PTN000380943|FB:FBgn0036039|UniProtKB:Q9H7X0	Communication:501741973		2023-04-13
AT5G16800	locus:2149030	AT5G16800	acts upstream of or within	N-terminal protein amino acid acetylation	GO:0006474	4852	P	protein metabolic process	IDA	in vitro assay	NONE	Publication:501790064|PMID:32548857  	TAIR	2020-06-28
AT5G16800	gene:1006229166	AT5G16800.2	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G16800	gene:6530297798	AT5G16800.3	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G16800	locus:2149030	AT5G16800	involved in	N-terminal peptidyl-methionine acetylation	GO:0017196	9135	P	protein metabolic process	IBA	none	PANTHER:PTN000380943|FB:FBgn0036039|UniProtKB:Q9H7X0	Communication:501741973		2023-04-13
AT5G16800	locus:2149030	AT5G16800	enables	peptide alpha-N-acetyltransferase activity	GO:0004596	3604	F	transferase activity	IBA	none	PANTHER:PTN000380943|UniProtKB:Q9H7X0|FB:FBgn0036039	Communication:501741973		2022-11-23
AT5G16800	locus:2149030	AT5G16800	is active in	Golgi membrane	GO:0000139	54	C	other intracellular components	IBA	none	PANTHER:PTN000380943|UniProtKB:Q9H7X0	Communication:501741973		2021-04-02
AT5G16800	locus:2149030	AT5G16800	is active in	Golgi membrane	GO:0000139	54	C	other membranes	IBA	none	PANTHER:PTN000380943|UniProtKB:Q9H7X0	Communication:501741973		2021-04-02
AT5G16810	locus:2149040	AT5G16810	acts upstream of or within	response to wounding	GO:0009611	7144	P	response to stress	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT3G16857	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16810	locus:2149040	AT5G16810	enables	protein kinase activity	GO:0004672	3889	F	transferase activity	IEA	none	InterPro:IPR000719	AnalysisReference:501756966		2023-05-23
AT5G16810	locus:2149040	AT5G16810	acts upstream of or within	flavonoid metabolic process	GO:0009812	5786	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT3G16857	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16810	locus:2149040	AT5G16810	enables	protein kinase activity	GO:0004672	3889	F	catalytic activity	IEA	none	InterPro:IPR000719	AnalysisReference:501756966		2023-05-23
AT5G16810	locus:2149040	AT5G16810	acts upstream of or within	cellular nitrogen compound biosynthetic process	GO:0044271	19681	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT1G49720|AGI_LocusCode:AT2G22430|AGI_LocusCode:AT3G16857|AGI_LocusCode:AT3G19290|AGI_LocusCode:AT4G27410|AGI_LocusCode:AT4G34000|AGI_LocusCode:AT5G65310|AGI_LocusCode:AT5G67300	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16810	locus:2149040	AT5G16810	acts upstream of or within	RNA metabolic process	GO:0016070	4899	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT1G49720|AGI_LocusCode:AT2G22430|AGI_LocusCode:AT3G19290|AGI_LocusCode:AT4G27410|AGI_LocusCode:AT4G34000|AGI_LocusCode:AT5G65310|AGI_LocusCode:AT5G67300	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16810	locus:2149040	AT5G16810	acts upstream of or within	protein-containing complex assembly	GO:0065003	25779	P	cellular component organization	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16810	locus:2149040	AT5G16810	acts upstream of or within	chlorophyll metabolic process	GO:0015994	5383	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT3G16857|AGI_LocusCode:AT3G19290	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16810	locus:2149040	AT5G16810	acts upstream of or within	organonitrogen compound biosynthetic process	GO:1901566	44205	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT3G16857	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16810	locus:2149040	AT5G16810	acts upstream of or within	response to jasmonic acid	GO:0009753	11419	P	response to endogenous stimulus	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT5G67300	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16810	locus:2149040	AT5G16810	enables	ATP binding	GO:0005524	894	F	nucleotide binding	IEA	none	InterPro:IPR000719	AnalysisReference:501756966		2023-05-23
AT5G16810	locus:2149040	AT5G16810	acts upstream of or within	glucosinolate metabolic process	GO:0019760	10522	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT3G19290|AGI_LocusCode:AT4G01120	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16810	locus:2149040	AT5G16810	acts upstream of or within	macromolecule biosynthetic process	GO:0009059	6237	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT1G49720|AGI_LocusCode:AT2G22430|AGI_LocusCode:AT3G19290|AGI_LocusCode:AT4G27410|AGI_LocusCode:AT4G34000|AGI_LocusCode:AT5G65310|AGI_LocusCode:AT5G67300	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16810	locus:2149040	AT5G16810	involved in	protein phosphorylation	GO:0006468	6897	P	other cellular processes	IEA	none	InterPro:IPR000719	AnalysisReference:501756966		2023-05-23
AT5G16810	locus:2149040	AT5G16810	acts upstream of or within	cellular nitrogen compound biosynthetic process	GO:0044271	19681	P	biosynthetic process	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT1G49720|AGI_LocusCode:AT2G22430|AGI_LocusCode:AT3G16857|AGI_LocusCode:AT3G19290|AGI_LocusCode:AT4G27410|AGI_LocusCode:AT4G34000|AGI_LocusCode:AT5G65310|AGI_LocusCode:AT5G67300	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16810	locus:2149040	AT5G16810	acts upstream of or within	cellular nitrogen compound biosynthetic process	GO:0044271	19681	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT1G49720|AGI_LocusCode:AT2G22430|AGI_LocusCode:AT3G16857|AGI_LocusCode:AT3G19290|AGI_LocusCode:AT4G27410|AGI_LocusCode:AT4G34000|AGI_LocusCode:AT5G65310|AGI_LocusCode:AT5G67300	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16810	locus:2149040	AT5G16810	acts upstream of or within	glucosinolate metabolic process	GO:0019760	10522	P	secondary metabolic process	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT3G19290|AGI_LocusCode:AT4G01120	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16810	locus:2149040	AT5G16810	located in	chloroplast	GO:0009507	175	C	chloroplast	IDA	localization of GFP/YFP fusion protein		Publication:501744974|PMID:21365755  	TAIR	2012-07-19
AT5G16810	locus:2149040	AT5G16810	acts upstream of or within	response to low light intensity stimulus	GO:0009645	7131	P	response to light stimulus	IEA	traceable computational prediction	AGI_LocusCode:AT2G43010	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16810	locus:2149040	AT5G16810	acts upstream of or within	RNA metabolic process	GO:0016070	4899	P	nucleobase-containing compound metabolic process	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT1G49720|AGI_LocusCode:AT2G22430|AGI_LocusCode:AT3G19290|AGI_LocusCode:AT4G27410|AGI_LocusCode:AT4G34000|AGI_LocusCode:AT5G65310|AGI_LocusCode:AT5G67300	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16810	locus:2149040	AT5G16810	acts upstream of or within	protein-containing complex assembly	GO:0065003	25779	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16810	locus:2149040	AT5G16810	enables	protein kinase activity	GO:0004672	3889	F	kinase activity	IEA	none	InterPro:IPR000719	AnalysisReference:501756966		2023-05-23
AT5G16810	locus:2149040	AT5G16810	acts upstream of or within	response to temperature stimulus	GO:0009266	7411	P	response to abiotic stimulus	IEA	traceable computational prediction	AGI_LocusCode:AT5G67300	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16810	locus:2149040	AT5G16810	acts upstream of or within	macromolecule biosynthetic process	GO:0009059	6237	P	biosynthetic process	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT1G49720|AGI_LocusCode:AT2G22430|AGI_LocusCode:AT3G19290|AGI_LocusCode:AT4G27410|AGI_LocusCode:AT4G34000|AGI_LocusCode:AT5G65310|AGI_LocusCode:AT5G67300	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16810	locus:2149040	AT5G16810	acts upstream of or within	chlorophyll metabolic process	GO:0015994	5383	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT3G16857|AGI_LocusCode:AT3G19290	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16810	locus:2149040	AT5G16810	enables	ATP binding	GO:0005524	894	F	other binding	IEA	none	InterPro:IPR000719	AnalysisReference:501756966		2023-05-23
AT5G16810	locus:2149040	AT5G16810	is active in	chloroplast	GO:0009507	175	C	chloroplast	IBA	none	PANTHER:PTN002197770|TAIR:locus:2149040	Communication:501741973		2021-04-02
AT5G16810	locus:2149040	AT5G16810	acts upstream of or within	response to jasmonic acid	GO:0009753	11419	P	response to chemical	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT5G67300	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16810	locus:2149040	AT5G16810	involved in	protein phosphorylation	GO:0006468	6897	P	other metabolic processes	IEA	none	InterPro:IPR000719	AnalysisReference:501756966		2023-05-23
AT5G16810	locus:2149040	AT5G16810	acts upstream of or within	organonitrogen compound biosynthetic process	GO:1901566	44205	P	biosynthetic process	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT3G16857	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16810	locus:2149040	AT5G16810	acts upstream of or within	glucosinolate metabolic process	GO:0019760	10522	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT3G19290|AGI_LocusCode:AT4G01120	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16820	locus:2149050	AT5G16820	enables	RNA polymerase II cis-regulatory region sequence-specific DNA binding	GO:0000978	35769	F	DNA binding	IBA	none	PANTHER:PTN008492349|MGI:MGI:96239|MGI:MGI:96238|UniProtKB:Q9ULV5|UniProtKB:Q00613|PomBase:SPAC8C9.14|UniProtKB:Q03933|PomBase:SPAC2E12.02|UniProtKB:P38531|SGD:S000005666|UniProtKB:P38529	Communication:501741973		2023-06-07
AT5G16820	gene:2149049	AT5G16820.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G16820	gene:1005713689	AT5G16820.2	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G16820	locus:2149050	AT5G16820	enables	DNA-binding transcription factor activity	GO:0003700	4449	F	DNA-binding transcription factor activity	IBA	none	PANTHER:PTN008492349|SGD:S000001249|SGD:S000003041|MGI:MGI:1347058|UniProtKB:Q9ULV5|UniProtKB:Q03933|TAIR:locus:2057371|SGD:S000005666|WB:WBGene00002004|UniProtKB:P38529|CGD:CAL0000195882|PomBase:SPAC8C9.14|UniProtKB:Q00613|UniProtKB:Q5KMX8|TAIR:locus:2149050|PomBase:SPAC2E12.02|FB:FBgn0001222|UniProtKB:P38531	Communication:501741973		2023-06-07
AT5G16820	locus:2149050	AT5G16820	involved in	regulation of transcription by RNA polymerase II	GO:0006357	7463	P	other cellular processes	IBA	none	PANTHER:PTN008492349|SGD:S000001249|SGD:S000003041|MGI:MGI:3045337|UniProtKB:Q9ULV5|UniProtKB:Q03933|CGD:CAL0000178027|SGD:S000005666|UniProtKB:A0A024DAP6|WB:WBGene00002004|UniProtKB:P38529|CGD:CAL0000175059|UniProtKB:Q00613|PomBase:SPAC8C9.14|CGD:CAL0000195882|FB:FBgn0001222|UniProtKB:A0A024DB95|PomBase:SPAC2E12.02|UniProtKB:P38531	Communication:501741973		2023-06-07
AT5G16820	locus:2149050	AT5G16820	enables	RNA polymerase II cis-regulatory region sequence-specific DNA binding	GO:0000978	35769	F	nucleic acid binding	IBA	none	PANTHER:PTN008492349|MGI:MGI:96239|MGI:MGI:96238|UniProtKB:Q9ULV5|UniProtKB:Q00613|PomBase:SPAC8C9.14|UniProtKB:Q03933|PomBase:SPAC2E12.02|UniProtKB:P38531|SGD:S000005666|UniProtKB:P38529	Communication:501741973		2023-06-07
AT5G16820	locus:2149050	AT5G16820	located in	nucleus	GO:0005634	537	C	nucleus	IDA	localization of GFP/YFP fusion protein		Publication:501735835|PMID:19945192  	TAIR	2010-02-24
AT5G16820	locus:2149050	AT5G16820	is active in	nucleus	GO:0005634	537	C	nucleus	IBA	none	PANTHER:PTN008492349|PomBase:SPAC8C9.14|TAIR:locus:2144603|UniProtKB:Q9UBD0|UniProtKB:K7LH36|PomBase:SPAC2E12.02|UniProtKB:Q43457|SGD:S000005666|TAIR:locus:2057371|WB:WBGene00002004|TAIR:locus:2117139|UniProtKB:I1LGH5|UniProtKB:Q03933|UniProtKB:P38530|SGD:S000003041|UniProtKB:I1MPZ9|UniProtKB:Q00613|CGD:CAL0000195882|UniProtKB:P38531|TAIR:locus:2075447|UniProtKB:Q942D6|FB:FBgn0001222|TAIR:locus:2149050|CGD:CAL0000178027|TAIR:locus:2204750|UniProtKB:P38529|UniProtKB:Q9ULV5|MGI:MGI:96239|MGI:MGI:96238|UniProtKB:Q96LI6|SGD:S000001249|MGI:MGI:3045337|TAIR:locus:2005495	Communication:501741973		2023-06-07
AT5G16820	locus:2149050	AT5G16820	enables	DNA-binding transcription factor activity	GO:0003700	4449	F	DNA-binding transcription factor activity	IMP	Analysis of overexpression/ectopic expression phenotype		Publication:2459|PMID:9645433   	TAIR	2003-04-24
AT5G16820	locus:2149050	AT5G16820	involved in	cellular response to heat	GO:0034605	29752	P	response to stress	IBA	none	PANTHER:PTN000797409|TAIR:locus:2057371	Communication:501741973		2022-11-23
AT5G16820	locus:2149050	AT5G16820	acts upstream of or within	response to heat	GO:0009408	5962	P	response to stress	IEP	Transcript levels (e.g. RT-PCR)	NONE	Publication:501736437|PMID:20229063  	TAIR	2010-04-15
AT5G16820	locus:2149050	AT5G16820	involved in	cellular response to heat	GO:0034605	29752	P	response to abiotic stimulus	IBA	none	PANTHER:PTN000797409|TAIR:locus:2057371	Communication:501741973		2022-11-23
AT5G16820	locus:2149050	AT5G16820	involved in	regulation of transcription by RNA polymerase II	GO:0006357	7463	P	other metabolic processes	IBA	none	PANTHER:PTN008492349|SGD:S000001249|SGD:S000003041|MGI:MGI:3045337|UniProtKB:Q9ULV5|UniProtKB:Q03933|CGD:CAL0000178027|SGD:S000005666|UniProtKB:A0A024DAP6|WB:WBGene00002004|UniProtKB:P38529|CGD:CAL0000175059|UniProtKB:Q00613|PomBase:SPAC8C9.14|CGD:CAL0000195882|FB:FBgn0001222|UniProtKB:A0A024DB95|PomBase:SPAC2E12.02|UniProtKB:P38531	Communication:501741973		2023-06-07
AT5G16820	locus:2149050	AT5G16820	acts upstream of or within	response to heat	GO:0009408	5962	P	response to abiotic stimulus	IEP	Transcript levels (e.g. RT-PCR)	NONE	Publication:501736437|PMID:20229063  	TAIR	2010-04-15
AT5G16820	locus:2149050	AT5G16820	located in	nucleus	GO:0005634	537	C	nucleus	IDA	localization of GFP/YFP fusion protein		Publication:501745135|PMID:21931939  	TAIR	2013-03-22
AT5G16820	locus:2149050	AT5G16820	involved in	regulation of transcription by RNA polymerase II	GO:0006357	7463	P	biosynthetic process	IBA	none	PANTHER:PTN008492349|SGD:S000001249|SGD:S000003041|MGI:MGI:3045337|UniProtKB:Q9ULV5|UniProtKB:Q03933|CGD:CAL0000178027|SGD:S000005666|UniProtKB:A0A024DAP6|WB:WBGene00002004|UniProtKB:P38529|CGD:CAL0000175059|UniProtKB:Q00613|PomBase:SPAC8C9.14|CGD:CAL0000195882|FB:FBgn0001222|UniProtKB:A0A024DB95|PomBase:SPAC2E12.02|UniProtKB:P38531	Communication:501741973		2023-06-07
AT5G16820	locus:2149050	AT5G16820	acts upstream of or within	response to heat	GO:0009408	5962	P	response to abiotic stimulus	IMP	Analysis of overexpression/ectopic expression phenotype	NONE	Publication:2459|PMID:9645433   	TAIR	2003-04-24
AT5G16820	locus:2149050	AT5G16820	acts upstream of or within	response to heat	GO:0009408	5962	P	response to stress	IMP	Analysis of overexpression/ectopic expression phenotype	NONE	Publication:2459|PMID:9645433   	TAIR	2003-04-24
AT5G16820	locus:2149050	AT5G16820	involved in	cellular response to heat	GO:0034605	29752	P	other cellular processes	IBA	none	PANTHER:PTN000797409|TAIR:locus:2057371	Communication:501741973		2022-11-23
AT5G16820	locus:2149050	AT5G16820	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:Q8GW48	Publication:501736740|PMID:20388662  		2021-04-01
AT5G16820	locus:2149050	AT5G16820	enables	DNA binding	GO:0003677	961	F	DNA binding	IDA	electrophoretic mobility shift (EMSA)	NONE	Publication:2459|PMID:9645433   	TAIR	2006-10-04
AT5G16820	locus:2149050	AT5G16820	enables	DNA-binding transcription factor activity	GO:0003700	4449	F	DNA-binding transcription factor activity	ISS	Recognized domains		Publication:1345963|PMID:11118137  	TAIR	2003-06-06
AT5G16820	locus:2149050	AT5G16820	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:Q8GW48	Publication:501738390|PMID:20657173  		2021-04-01
AT5G16820	locus:2149050	AT5G16820	involved in	regulation of transcription by RNA polymerase II	GO:0006357	7463	P	nucleobase-containing compound metabolic process	IBA	none	PANTHER:PTN008492349|SGD:S000001249|SGD:S000003041|MGI:MGI:3045337|UniProtKB:Q9ULV5|UniProtKB:Q03933|CGD:CAL0000178027|SGD:S000005666|UniProtKB:A0A024DAP6|WB:WBGene00002004|UniProtKB:P38529|CGD:CAL0000175059|UniProtKB:Q00613|PomBase:SPAC8C9.14|CGD:CAL0000195882|FB:FBgn0001222|UniProtKB:A0A024DB95|PomBase:SPAC2E12.02|UniProtKB:P38531	Communication:501741973		2023-06-07
AT5G16830	locus:2148960	AT5G16830	involved in	intracellular protein transport	GO:0006886	6091	P	transport	IBA	none	PANTHER:PTN000461349|SGD:S000005562|TAIR:locus:2075054|SGD:S000003180|SGD:S000005378|TAIR:locus:2079894|TAIR:locus:2081476|TAIR:locus:2148960|TAIR:locus:2148513|TAIR:locus:2032569|TAIR:locus:2078042|TAIR:locus:505006427	Communication:501741973		2023-06-07
AT5G16830	locus:2148960	AT5G16830	part of	SNARE complex	GO:0031201	20634	C	cytoplasm	IBA	none	PANTHER:PTN000461349|RGD:621005|MGI:MGI:103077|MGI:MGI:109355|UniProtKB:Q12846|FB:FBgn0035540|RGD:619747|SGD:S000005378|MGI:MGI:893577|SGD:S000006153|SGD:S000004016|TAIR:locus:2032569|UniProtKB:Q16623|SGD:S000005632|UniProtKB:P56962|RGD:69430|TAIR:locus:2148513|UniProtKB:Q86Y82|TAIR:locus:2081476|SGD:S000002876|UniProtKB:O14662|FB:FBgn0013343|RGD:621019|RGD:61917|UniProtKB:O43752|RGD:68426|SGD:S000003180	Communication:501741973		2023-06-07
AT5G16830	locus:2148960	AT5G16830	involved in	vesicle docking	GO:0048278	18030	P	transport	IBA	none	PANTHER:PTN000461349|SGD:S000005632|RGD:621019|RGD:69430|MGI:MGI:109355|MGI:MGI:1930705	Communication:501741973		2023-06-07
AT5G16830	locus:2148960	AT5G16830	enables	SNARE binding	GO:0000149	1257	F	protein binding	IBA	none	PANTHER:PTN000461349|UniProtKB:O43752|ZFIN:ZDB-GENE-040426-1360|RGD:621019|UniProtKB:O15400|UniProtKB:O14662|RGD:2558|FB:FBgn0013343|MGI:MGI:893577|RGD:708499|RGD:619747|FB:FBgn0035540|UniProtKB:Q9UNK0|MGI:MGI:109355|UniProtKB:O60499|UniProtKB:P56962|RGD:69430	Communication:501741973		2023-06-07
AT5G16830	locus:2148960	AT5G16830	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:Q9SEL6	Publication:501758894|PMID:24556609  		2023-05-23
AT5G16830	locus:2148960	AT5G16830	enables	SNAP receptor activity	GO:0005484	1256	F	other molecular functions	IDA	transport assay		Publication:1147|PMID:10504581  	TAIR	2007-07-12
AT5G16830	locus:2148960	AT5G16830	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:Q9SEL6	Publication:1369|PMID:10397763  		2023-05-23
AT5G16830	locus:2148960	AT5G16830	enables	SNAP receptor activity	GO:0005484	1256	F	other molecular functions	IMP	Functional complementation		Publication:4330|PMID:7638178   	TAIR	2007-07-12
AT5G16830	locus:2148960	AT5G16830	located in	trans-Golgi network transport vesicle	GO:0030140	7878	C	other intracellular components	IDA	immunogold labeling		Publication:3321|PMID:9144962   	TAIR	2005-04-25
AT5G16830	locus:2148960	AT5G16830	involved in	intracellular protein transport	GO:0006886	6091	P	other cellular processes	IBA	none	PANTHER:PTN000461349|SGD:S000005562|TAIR:locus:2075054|SGD:S000003180|SGD:S000005378|TAIR:locus:2079894|TAIR:locus:2081476|TAIR:locus:2148960|TAIR:locus:2148513|TAIR:locus:2032569|TAIR:locus:2078042|TAIR:locus:505006427	Communication:501741973		2023-06-07
AT5G16830	locus:2148960	AT5G16830	enables	SNAP receptor activity	GO:0005484	1256	F	other molecular functions	TAS	none		Publication:1345989|PMID:11115874  	TIGR	2007-07-12
AT5G16830	locus:2148960	AT5G16830	acts upstream of or within	intracellular protein transport	GO:0006886	6091	P	transport	TAS	none	NONE	Publication:1345989|PMID:11115874  	TIGR	2003-04-17
AT5G16830	locus:2148960	AT5G16830	is active in	endomembrane system	GO:0012505	9932	C	other cellular components	IBA	none	PANTHER:PTN000461349|RGD:708499|UniProtKB:C8VLM4|MGI:MGI:893577|SGD:S000004016|PomBase:SPAC6F12.03c|TAIR:locus:2148960|MGI:MGI:109355|MGI:MGI:1858210|UniProtKB:Q9UNK0|UniProtKB:Q12846|RGD:619747|SGD:S000005378|TAIR:locus:2025620|SGD:S000000012|FB:FBgn0013343|TAIR:locus:2078282|RGD:620977|SGD:S000005562|PomBase:SPBC31E1.04|RGD:61917|FB:FBgn0011708|UniProtKB:O43752|UniProtKB:Q8VZU2|UniProtKB:Q94KK5|UniProtKB:Q86Y82|PomBase:SPCC825.03c|UniProtKB:Q13277|FB:FBgn0267849|UniProtKB:O15400|TAIR:locus:2032569|MGI:MGI:1926235|UniProtKB:Q13190|MGI:MGI:103077|dictyBase:DDB_G0287733|TAIR:locus:2166419|MGI:MGI:1923396|TAIR:locus:505006427|FB:FBgn0031106|TAIR:locus:2075054|PomBase:SPCC594.06c|SGD:S000002876|TAIR:locus:2081476|UniProtKB:O14662|UniProtKB:Q9LK09|TAIR:locus:2078042|TAIR:locus:2197749|RGD:61915|RGD:68426|UniProtKB:P56962|UniProtKB:O60499|RGD:69430|TAIR:locus:2148513|WB:WBGene00006374|UniProtKB:Q8IE08|RGD:1309423	Communication:501741973		2023-06-07
AT5G16830	locus:2148960	AT5G16830	located in	trans-Golgi network transport vesicle	GO:0030140	7878	C	cytoplasm	TAS	none		Publication:1345989|PMID:11115874  	TIGR	2003-04-17
AT5G16830	locus:2148960	AT5G16830	located in	trans-Golgi network transport vesicle	GO:0030140	7878	C	cytoplasm	IDA	immunogold labeling		Publication:3321|PMID:9144962   	TAIR	2005-04-25
AT5G16830	locus:2148960	AT5G16830	located in	late endosome	GO:0005770	428	C	endosome	IDA	localization of GFP/YFP fusion protein		Publication:501719655|PMID:16935987  	TAIR	2006-10-06
AT5G16830	locus:2148960	AT5G16830	enables	SNAP receptor activity	GO:0005484	1256	F	other molecular functions	IBA	none	PANTHER:PTN000461349|TAIR:locus:2148960|UniProtKB:P56962|RGD:621005|SGD:S000005632|FB:FBgn0031106|RGD:619747|SGD:S000004795|SGD:S000005378|FB:FBgn0035540|SGD:S000006153|SGD:S000000012|SGD:S000002876|SGD:S000004016|FB:FBgn0013343|UniProtKB:O14662|UniProtKB:Q13190|SGD:S000005562|RGD:621019|SGD:S000003180|FB:FBgn0011708	Communication:501741973		2023-06-07
AT5G16830	locus:2148960	AT5G16830	located in	trans-Golgi network transport vesicle	GO:0030140	7878	C	other intracellular components	TAS	none		Publication:1345989|PMID:11115874  	TIGR	2003-04-17
AT5G16830	locus:2148960	AT5G16830	located in	late endosome	GO:0005770	428	C	endosome	TAS	original experiments are traceable through an article		Publication:1345989|PMID:11115874  	TAIR	2003-02-26
AT5G16830	locus:2148960	AT5G16830	involved in	vesicle fusion	GO:0006906	6505	P	transport	IBA	none	PANTHER:PTN000461349|RGD:68426|SGD:S000003180|PomBase:SPAC6F12.03c|UniProtKB:O43752|RGD:61917|SGD:S000004016|TAIR:locus:2078042|TAIR:locus:2032569|FB:FBgn0013343|MGI:MGI:893577|SGD:S000006153|SGD:S000002876|FB:FBgn0267849|RGD:619747|MGI:MGI:1930705|SGD:S000005378|TAIR:locus:505006427|TAIR:locus:2148513|dictyBase:DDB_G0287733|RGD:69430|SGD:S000005632	Communication:501741973		2023-06-07
AT5G16830	locus:2148960	AT5G16830	acts upstream of or within	late endosome to vacuole transport	GO:0045324	11761	P	transport	IMP	Analysis of overexpression/ectopic expression phenotype	NONE	Publication:501719655|PMID:16935987  	TAIR	2006-10-06
AT5G16830	locus:2148960	AT5G16830	part of	SNARE complex	GO:0031201	20634	C	other membranes	IBA	none	PANTHER:PTN000461349|RGD:621005|MGI:MGI:103077|MGI:MGI:109355|UniProtKB:Q12846|FB:FBgn0035540|RGD:619747|SGD:S000005378|MGI:MGI:893577|SGD:S000006153|SGD:S000004016|TAIR:locus:2032569|UniProtKB:Q16623|SGD:S000005632|UniProtKB:P56962|RGD:69430|TAIR:locus:2148513|UniProtKB:Q86Y82|TAIR:locus:2081476|SGD:S000002876|UniProtKB:O14662|FB:FBgn0013343|RGD:621019|RGD:61917|UniProtKB:O43752|RGD:68426|SGD:S000003180	Communication:501741973		2023-06-07
AT5G16830	locus:2148960	AT5G16830	acts upstream of or within	Golgi to vacuole transport	GO:0006896	4808	P	transport	TAS	inferred by the author, from expression pattern	NONE	Publication:1147|PMID:10504581  	TAIR	2005-04-25
AT5G16830	locus:2148960	AT5G16830	located in	late endosome	GO:0005770	428	C	endosome	TAS	none		Publication:1345989|PMID:11115874  	TIGR	2003-04-17
AT5G16830	locus:2148960	AT5G16830	acts upstream of or within	protein targeting to vacuole	GO:0006623	6931	P	transport	IDA	localization of GFP/YFP fusion protein	NONE	Publication:501722658|PMID:17662029  	TAIR	2007-12-12
AT5G16830	locus:2148960	AT5G16830	acts upstream of or within	Golgi to vacuole transport	GO:0006896	4808	P	other cellular processes	TAS	inferred by the author, from expression pattern	NONE	Publication:1147|PMID:10504581  	TAIR	2005-04-25
AT5G16830	locus:2148960	AT5G16830	located in	membrane	GO:0016020	453	C	other membranes	IDA	protein expression in heterologous system		Publication:501719655|PMID:16935987  	TAIR	2006-10-06
AT5G16830	locus:2148960	AT5G16830	involved in	vesicle fusion	GO:0006906	6505	P	other cellular processes	IBA	none	PANTHER:PTN000461349|RGD:68426|SGD:S000003180|PomBase:SPAC6F12.03c|UniProtKB:O43752|RGD:61917|SGD:S000004016|TAIR:locus:2078042|TAIR:locus:2032569|FB:FBgn0013343|MGI:MGI:893577|SGD:S000006153|SGD:S000002876|FB:FBgn0267849|RGD:619747|MGI:MGI:1930705|SGD:S000005378|TAIR:locus:505006427|TAIR:locus:2148513|dictyBase:DDB_G0287733|RGD:69430|SGD:S000005632	Communication:501741973		2023-06-07
AT5G16830	locus:2148960	AT5G16830	acts upstream of or within	intracellular protein transport	GO:0006886	6091	P	other cellular processes	TAS	none	NONE	Publication:1345989|PMID:11115874  	TIGR	2003-04-17
AT5G16830	gene:2148959	AT5G16830.1	located in	Golgi apparatus	GO:0005794	48	C	Golgi apparatus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G16830	locus:2148960	AT5G16830	involved in	vesicle docking	GO:0048278	18030	P	other cellular processes	IBA	none	PANTHER:PTN000461349|SGD:S000005632|RGD:621019|RGD:69430|MGI:MGI:109355|MGI:MGI:1930705	Communication:501741973		2023-06-07
AT5G16830	locus:2148960	AT5G16830	acts upstream of or within	protein targeting to vacuole	GO:0006623	6931	P	other cellular processes	IDA	localization of GFP/YFP fusion protein	NONE	Publication:501722658|PMID:17662029  	TAIR	2007-12-12
AT5G16830	locus:2148960	AT5G16830	acts upstream of or within	late endosome to vacuole transport	GO:0045324	11761	P	other cellular processes	IMP	Analysis of overexpression/ectopic expression phenotype	NONE	Publication:501719655|PMID:16935987  	TAIR	2006-10-06
AT5G16830	locus:2148960	AT5G16830	involved in	vesicle fusion	GO:0006906	6505	P	cellular component organization	IBA	none	PANTHER:PTN000461349|RGD:68426|SGD:S000003180|PomBase:SPAC6F12.03c|UniProtKB:O43752|RGD:61917|SGD:S000004016|TAIR:locus:2078042|TAIR:locus:2032569|FB:FBgn0013343|MGI:MGI:893577|SGD:S000006153|SGD:S000002876|FB:FBgn0267849|RGD:619747|MGI:MGI:1930705|SGD:S000005378|TAIR:locus:505006427|TAIR:locus:2148513|dictyBase:DDB_G0287733|RGD:69430|SGD:S000005632	Communication:501741973		2023-06-07
AT5G16830	locus:2148960	AT5G16830	acts upstream of or within	membrane fusion	GO:0061025	34381	P	cellular component organization	TAS	none	NONE	Publication:1345989|PMID:11115874  	TIGR	2014-07-18
AT5G16840	locus:2148970	AT5G16840	located in	plastid	GO:0009536	576	C	plastid	HDA	none		Publication:501776792|PMID:28887381  	dszymanski	2019-04-04
AT5G16840	locus:2148970	AT5G16840	acts upstream of or within	negative regulation of plant-type hypersensitive response	GO:0034051	28884	P	cell death	IMP	analysis of physiological response	NONE	Publication:501783771|PMID:30703564  	TAIR	2019-08-20
AT5G16840	locus:2148970	AT5G16840	acts upstream of or within	negative regulation of plant-type hypersensitive response	GO:0034051	28884	P	response to external stimulus	IMP	analysis of physiological response	NONE	Publication:501783771|PMID:30703564  	TAIR	2019-08-20
AT5G16840	locus:2148970	AT5G16840	acts upstream of or within	negative regulation of plant-type hypersensitive response	GO:0034051	28884	P	other cellular processes	IMP	analysis of physiological response	NONE	Publication:501783771|PMID:30703564  	TAIR	2019-08-20
AT5G16840	locus:2148970	AT5G16840	located in	plasma membrane	GO:0005886	570	C	plasma membrane	IDA	localization of GFP/YFP fusion protein		Publication:501783771|PMID:30703564  	TAIR	2019-08-14
AT5G16840	locus:2148970	AT5G16840	located in	cytoplasmic stress granule	GO:0010494	28661	C	cytoplasm	IDA	affinity capture		Publication:501783732|PMID:30664249  	aimer.gutierrez.diaz	2022-10-28
AT5G16840	gene:4010713135	AT5G16840.2	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G16840	gene:4010713136	AT5G16840.3	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G16840	locus:2148970	AT5G16840	acts upstream of or within	negative regulation of plant-type hypersensitive response	GO:0034051	28884	P	response to stress	IMP	analysis of physiological response	NONE	Publication:501783771|PMID:30703564  	TAIR	2019-08-20
AT5G16840	locus:2148970	AT5G16840	acts upstream of or within	defense response to fungus	GO:0050832	18013	P	response to stress	IMP	analysis of physiological response	NONE	Publication:501783771|PMID:30703564  	TAIR	2019-08-14
AT5G16840	locus:2148970	AT5G16840	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	Co-immunoprecipitation	AGI_LocusCode:AT2G34690	Publication:501783771|PMID:30703564  	TAIR	2021-09-09
AT5G16840	locus:2148970	AT5G16840	acts upstream of or within	negative regulation of plant-type hypersensitive response	GO:0034051	28884	P	response to biotic stimulus	IMP	analysis of physiological response	NONE	Publication:501783771|PMID:30703564  	TAIR	2019-08-20
AT5G16840	locus:2148970	AT5G16840	enables	nucleic acid binding	GO:0003676	3376	F	nucleic acid binding	ISS	manually reviewed TIGR computational analysis	INTERPRO:IPR000504	Communication:501714663	TIGR	2019-08-14
AT5G16840	locus:2148970	AT5G16840	located in	cytoplasmic stress granule	GO:0010494	28661	C	other intracellular components	IDA	affinity capture		Publication:501783732|PMID:30664249  	aimer.gutierrez.diaz	2022-10-28
AT5G16840	locus:2148970	AT5G16840	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:O64587	Publication:501728985|PMID:18845362  		2023-05-23
AT5G16840	locus:2148970	AT5G16840	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	yeast two-hybrid assay	AGI_LocusCode:AT2G34690	Publication:501783771|PMID:30703564  	TAIR	2021-09-09
AT5G16840	locus:2148970	AT5G16840	acts upstream of or within	defense response to fungus	GO:0050832	18013	P	response to external stimulus	IMP	analysis of physiological response	NONE	Publication:501783771|PMID:30703564  	TAIR	2019-08-14
AT5G16840	gene:2148969	AT5G16840.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G16840	locus:2148970	AT5G16840	acts upstream of or within	defense response to fungus	GO:0050832	18013	P	response to biotic stimulus	IMP	analysis of physiological response	NONE	Publication:501783771|PMID:30703564  	TAIR	2019-08-14
AT5G16850	gene:2148974	AT5G16850.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G16850	locus:2148975	AT5G16850	involved in	telomere maintenance via telomerase	GO:0007004	7410	P	biosynthetic process	IBA	none	PANTHER:PTN000249074|UniProtKB:O14746|PomBase:SPBC29A3.14c|TAIR:locus:2148975|CGD:CAL0000184329|UniProtKB:Q8IEH2|SGD:S000004310|ZFIN:ZDB-GENE-080405-1	Communication:501741973		2023-06-07
AT5G16850	locus:2148975	AT5G16850	involved in	telomere maintenance via telomerase	GO:0007004	7410	P	nucleobase-containing compound metabolic process	IBA	none	PANTHER:PTN000249074|UniProtKB:O14746|PomBase:SPBC29A3.14c|TAIR:locus:2148975|CGD:CAL0000184329|UniProtKB:Q8IEH2|SGD:S000004310|ZFIN:ZDB-GENE-080405-1	Communication:501741973		2023-06-07
AT5G16850	locus:2148975	AT5G16850	acts upstream of or within	root meristem growth	GO:0010449	28043	P	anatomical structure development	IMP	analysis of visible trait		Publication:501764048|PMID:25937286  	MaryPaz	2022-04-15
AT5G16850	locus:2148975	AT5G16850	involved in	telomere maintenance via telomerase	GO:0007004	7410	P	DNA metabolic process	IGI	double mutant analysis	Tair:gene:2164480	Publication:501717740|PMID:16166376  	TAIR	2021-04-13
AT5G16850	locus:2148975	AT5G16850	involved in	telomere maintenance via telomerase	GO:0007004	7410	P	DNA metabolic process	IMP	analysis of visible trait	AGI_LocusCode:AT5G16850	Publication:883|PMID:10611295  	TAIR	2021-04-13
AT5G16850	locus:2148975	AT5G16850	involved in	telomere maintenance via telomerase	GO:0007004	7410	P	nucleobase-containing compound metabolic process	IGI	double mutant analysis	Tair:gene:2164480	Publication:501717740|PMID:16166376  	TAIR	2021-04-13
AT5G16850	locus:2148975	AT5G16850	involved in	telomere maintenance	GO:0000723	13303	P	DNA metabolic process	IDA	in vitro assay		Publication:501744724|PMID:21865176  	sykorova	2021-04-13
AT5G16850	locus:2148975	AT5G16850	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	yeast two-hybrid assay	AGI_LocusCode:AT3G49850|AGI_LocusCode:AT1G49950|AGI_LocusCode:AT5G67580	Publication:501757658|PMID:24397874  	SchPetra	2022-08-18
AT5G16850	locus:2148975	AT5G16850	enables	telomerase activity	GO:0003720	4379	F	catalytic activity	IDA	in vitro assay		Publication:501744724|PMID:21865176  	sykorova	2012-01-19
AT5G16850	locus:2148975	AT5G16850	involved in	telomere maintenance via telomerase	GO:0007004	7410	P	other cellular processes	IGI	double mutant analysis	Tair:gene:2164480	Publication:501717740|PMID:16166376  	TAIR	2021-04-13
AT5G16850	locus:2148975	AT5G16850	involved in	telomere maintenance via telomerase	GO:0007004	7410	P	cellular component organization	IBA	none	PANTHER:PTN000249074|UniProtKB:O14746|PomBase:SPBC29A3.14c|TAIR:locus:2148975|CGD:CAL0000184329|UniProtKB:Q8IEH2|SGD:S000004310|ZFIN:ZDB-GENE-080405-1	Communication:501741973		2023-06-07
AT5G16850	locus:2148975	AT5G16850	involved in	telomere maintenance via telomerase	GO:0007004	7410	P	other metabolic processes	IGI	double mutant analysis	Tair:gene:2164480	Publication:501717740|PMID:16166376  	TAIR	2021-04-13
AT5G16850	locus:2148975	AT5G16850	located in	nucleus	GO:0005634	537	C	nucleus	IDA	Immunofluorescence(for Cellular Component)		Publication:501764048|PMID:25937286  	MaryPaz	2015-05-29
AT5G16850	locus:2148975	AT5G16850	acts upstream of or within	root meristem growth	GO:0010449	28043	P	multicellular organism development	IMP	analysis of visible trait		Publication:501764048|PMID:25937286  	MaryPaz	2022-04-15
AT5G16850	locus:2148975	AT5G16850	acts upstream of or within	meristem maintenance	GO:0010073	14818	P	anatomical structure development	IMP	analysis of visible trait		Publication:501764048|PMID:25937286  	MaryPaz	2022-04-15
AT5G16850	gene:6532550199	AT5G16850.2	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G16850	locus:2148975	AT5G16850	enables	telomerase RNA reverse transcriptase activity	GO:0003721	4380	F	transferase activity	IBA	none	PANTHER:PTN000249074|CGD:CAL0000184329|PomBase:SPBC29A3.14c|UniProtKB:O14746|UniProtKB:Q1PS67|SGD:S000004310	Communication:501741973		2023-06-07
AT5G16850	locus:2148975	AT5G16850	acts upstream of or within	root meristem growth	GO:0010449	28043	P	growth	IMP	analysis of visible trait		Publication:501764048|PMID:25937286  	MaryPaz	2022-04-15
AT5G16850	locus:2148975	AT5G16850	acts upstream of or within	stem cell population maintenance	GO:0019827	10911	P	other biological processes	IMP	analysis of visible trait		Publication:501764048|PMID:25937286  	MaryPaz	2022-04-15
AT5G16850	locus:2148975	AT5G16850	enables	telomerase RNA reverse transcriptase activity	GO:0003721	4380	F	catalytic activity	IEA	none	InterPro:IPR003545	AnalysisReference:501756966		2023-06-11
AT5G16850	locus:2148975	AT5G16850	enables	telomerase activity	GO:0003720	4379	F	transferase activity	IDA	in vitro assay		Publication:501744724|PMID:21865176  	sykorova	2012-01-19
AT5G16850	locus:2148975	AT5G16850	involved in	telomere maintenance via telomerase	GO:0007004	7410	P	nucleobase-containing compound metabolic process	IMP	analysis of visible trait	AGI_LocusCode:AT5G16850	Publication:883|PMID:10611295  	TAIR	2021-04-13
AT5G16850	locus:2148975	AT5G16850	involved in	telomere maintenance via telomerase	GO:0007004	7410	P	other cellular processes	IMP	analysis of visible trait	AGI_LocusCode:AT5G16850	Publication:883|PMID:10611295  	TAIR	2021-04-13
AT5G16850	locus:2148975	AT5G16850	involved in	telomere maintenance via telomerase	GO:0007004	7410	P	other cellular processes	IBA	none	PANTHER:PTN000249074|UniProtKB:O14746|PomBase:SPBC29A3.14c|TAIR:locus:2148975|CGD:CAL0000184329|UniProtKB:Q8IEH2|SGD:S000004310|ZFIN:ZDB-GENE-080405-1	Communication:501741973		2023-06-07
AT5G16850	locus:2148975	AT5G16850	enables	telomerase RNA binding	GO:0070034	30370	F	RNA binding	IBA	none	PANTHER:PTN000249074|UniProtKB:Q1PS67|MGI:MGI:1202709|PomBase:SPBC29A3.14c|UniProtKB:O14746	Communication:501741973		2023-06-07
AT5G16850	locus:2148975	AT5G16850	involved in	telomere maintenance via telomerase	GO:0007004	7410	P	other metabolic processes	IBA	none	PANTHER:PTN000249074|UniProtKB:O14746|PomBase:SPBC29A3.14c|TAIR:locus:2148975|CGD:CAL0000184329|UniProtKB:Q8IEH2|SGD:S000004310|ZFIN:ZDB-GENE-080405-1	Communication:501741973		2023-06-07
AT5G16850	locus:2148975	AT5G16850	enables	telomerase RNA reverse transcriptase activity	GO:0003721	4380	F	catalytic activity	IBA	none	PANTHER:PTN000249074|CGD:CAL0000184329|PomBase:SPBC29A3.14c|UniProtKB:O14746|UniProtKB:Q1PS67|SGD:S000004310	Communication:501741973		2023-06-07
AT5G16850	locus:2148975	AT5G16850	involved in	telomere maintenance via telomerase	GO:0007004	7410	P	cellular component organization	IGI	double mutant analysis	Tair:gene:2164480	Publication:501717740|PMID:16166376  	TAIR	2021-04-13
AT5G16850	locus:2148975	AT5G16850	part of	telomerase catalytic core complex	GO:0000333	11718	C	nucleus	IBA	none	PANTHER:PTN000249074|MGI:MGI:1202709|UniProtKB:O14746|PomBase:SPBC29A3.14c|UniProtKB:Q1PS67|CGD:CAL0000184329|SGD:S000004310	Communication:501741973		2023-06-07
AT5G16850	locus:2148975	AT5G16850	enables	telomerase activity	GO:0003720	4379	F	catalytic activity	IDA	Enzyme assays		Publication:883|PMID:10611295  	TAIR	2005-10-27
AT5G16850	locus:2148975	AT5G16850	involved in	telomere maintenance via telomerase	GO:0007004	7410	P	cellular component organization	IMP	analysis of visible trait	AGI_LocusCode:AT5G16850	Publication:883|PMID:10611295  	TAIR	2021-04-13
AT5G16850	locus:2148975	AT5G16850	involved in	telomere maintenance via telomerase	GO:0007004	7410	P	other metabolic processes	IMP	analysis of visible trait	AGI_LocusCode:AT5G16850	Publication:883|PMID:10611295  	TAIR	2021-04-13
AT5G16850	locus:2148975	AT5G16850	enables	telomerase RNA reverse transcriptase activity	GO:0003721	4380	F	transferase activity	IEA	none	InterPro:IPR003545	AnalysisReference:501756966		2023-06-11
AT5G16850	locus:2148975	AT5G16850	involved in	telomere maintenance via telomerase	GO:0007004	7410	P	biosynthetic process	IMP	analysis of visible trait	AGI_LocusCode:AT5G16850	Publication:883|PMID:10611295  	TAIR	2021-04-13
AT5G16850	locus:2148975	AT5G16850	involved in	telomere maintenance via telomerase	GO:0007004	7410	P	biosynthetic process	IGI	double mutant analysis	Tair:gene:2164480	Publication:501717740|PMID:16166376  	TAIR	2021-04-13
AT5G16850	locus:2148975	AT5G16850	enables	telomeric DNA binding	GO:0042162	10982	F	DNA binding	IBA	none	PANTHER:PTN000249074|SGD:S000004310	Communication:501741973		2021-04-02
AT5G16850	locus:2148975	AT5G16850	involved in	telomere maintenance via telomerase	GO:0007004	7410	P	DNA metabolic process	IBA	none	PANTHER:PTN000249074|UniProtKB:O14746|PomBase:SPBC29A3.14c|TAIR:locus:2148975|CGD:CAL0000184329|UniProtKB:Q8IEH2|SGD:S000004310|ZFIN:ZDB-GENE-080405-1	Communication:501741973		2023-06-07
AT5G16850	locus:2148975	AT5G16850	enables	DNA binding	GO:0003677	961	F	DNA binding	IEA	none	InterPro:IPR003545	AnalysisReference:501756966		2023-06-11
AT5G16850	locus:2148975	AT5G16850	acts upstream of or within	chromosome localization	GO:0050000	20694	P	other cellular processes	IMP	analysis of visible trait		Publication:501710504|PMID:14579127  	TAIR	2022-04-15
AT5G16850	locus:2148975	AT5G16850	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:Q56Y52	Publication:501723443|PMID:17911168  		2023-05-23
AT5G16850	locus:2148975	AT5G16850	enables	telomerase activity	GO:0003720	4379	F	transferase activity	IDA	Enzyme assays		Publication:883|PMID:10611295  	TAIR	2005-10-27
AT5G16850	locus:2148975	AT5G16850	involved in	telomere maintenance	GO:0000723	13303	P	cellular component organization	IDA	in vitro assay		Publication:501744724|PMID:21865176  	sykorova	2021-04-13
AT5G16850	locus:2148975	AT5G16850	acts upstream of or within	chromosome organization	GO:0051276	19651	P	cellular component organization	IGI	double mutant analysis	Tair:gene:2164480	Publication:501717740|PMID:16166376  	TAIR	2022-04-15
AT5G16860	locus:2148101	AT5G16860	enables	RNA binding	GO:0003723	1217	F	RNA binding	IBA	none	PANTHER:PTN000755749|TAIR:locus:2077279|TAIR:locus:2124603|TAIR:locus:2130389|TAIR:locus:2053659|TAIR:locus:2090444|TAIR:locus:2013079|TAIR:locus:2162207|TAIR:locus:2825319|TAIR:locus:2103483|TAIR:locus:2090857|TAIR:locus:2122634	Communication:501741973		2023-06-07
AT5G16860	locus:2148101	AT5G16860	is active in	intracellular membrane-bounded organelle	GO:0043231	19388	C	other intracellular components	IBA	none	PANTHER:PTN000755749|TAIR:locus:2130389|TAIR:locus:2196583|TAIR:locus:2053659|TAIR:locus:2090444|TAIR:locus:2077279|TAIR:locus:2062939|TAIR:locus:2131939|TAIR:locus:2063771|TAIR:locus:2054131|TAIR:locus:2198678|TAIR:locus:2124603|TAIR:locus:2080727|TAIR:locus:2027589|TAIR:locus:2127801|TAIR:locus:2159582|TAIR:locus:2103483|TAIR:locus:2019130	Communication:501741973		2023-06-07
AT5G16860	gene:2148100	AT5G16860.1	located in	mitochondrion	GO:0005739	486	C	mitochondrion	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G16860	locus:2148101	AT5G16860	involved in	RNA modification	GO:0009451	4901	P	other metabolic processes	IBA	none	PANTHER:PTN000755749|TAIR:locus:2054131|TAIR:locus:2162207|TAIR:locus:2131939|TAIR:locus:2062939|TAIR:locus:2018092|TAIR:locus:2090444|TAIR:locus:2196583|TAIR:locus:2178188|TAIR:locus:2019130|TAIR:locus:2159582|TAIR:locus:2080727|TAIR:locus:2124603|TAIR:locus:2087969|TAIR:locus:2198678	Communication:501741973		2023-06-07
AT5G16860	locus:2148101	AT5G16860	involved in	RNA modification	GO:0009451	4901	P	nucleobase-containing compound metabolic process	IBA	none	PANTHER:PTN000755749|TAIR:locus:2054131|TAIR:locus:2162207|TAIR:locus:2131939|TAIR:locus:2062939|TAIR:locus:2018092|TAIR:locus:2090444|TAIR:locus:2196583|TAIR:locus:2178188|TAIR:locus:2019130|TAIR:locus:2159582|TAIR:locus:2080727|TAIR:locus:2124603|TAIR:locus:2087969|TAIR:locus:2198678	Communication:501741973		2023-06-07
AT5G16860	locus:2148101	AT5G16860	enables	zinc ion binding	GO:0008270	4661	F	other binding	IEA	none	InterPro:IPR032867	AnalysisReference:501756966		2023-06-11
AT5G16870	locus:2148111	AT5G16870	is active in	cytosol	GO:0005829	241	C	cytosol	IBA	none	PANTHER:PTN000291358|UniProtKB:Q9Y3E5	Communication:501741973		2021-04-02
AT5G16870	locus:2148111	AT5G16870	enables	aminoacyl-tRNA hydrolase activity	GO:0004045	1515	F	catalytic activity	IBA	none	PANTHER:PTN000291358|SGD:S000000153|UniProtKB:Q9Y3E5	Communication:501741973		2023-06-07
AT5G16870	locus:2148111	AT5G16870	enables	aminoacyl-tRNA hydrolase activity	GO:0004045	1515	F	hydrolase activity	IBA	none	PANTHER:PTN000291358|SGD:S000000153|UniProtKB:Q9Y3E5	Communication:501741973		2023-06-07
AT5G16870	locus:2148111	AT5G16870	acts upstream of or within	carboxylic acid biosynthetic process	GO:0046394	13251	P	biosynthetic process	IEA	traceable computational prediction	AGI_LocusCode:AT5G04760	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16870	gene:2148110	AT5G16870.1	located in	mitochondrion	GO:0005739	486	C	mitochondrion	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G16870	locus:2148111	AT5G16870	acts upstream of or within	carboxylic acid biosynthetic process	GO:0046394	13251	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT5G04760	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16870	locus:2148111	AT5G16870	acts upstream of or within	carboxylic acid biosynthetic process	GO:0046394	13251	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT5G04760	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16870	locus:2148111	AT5G16870	is active in	mitochondrion	GO:0005739	486	C	mitochondrion	IBA	none	PANTHER:PTN000291358|UniProtKB:Q9Y3E5	Communication:501741973		2021-04-02
AT5G16880	gene:6532556458	AT5G16880.4	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G16880	locus:2148121	AT5G16880	acts upstream of or within	vesicle-mediated transport	GO:0016192	7567	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT3G13720	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16880	locus:2148121	AT5G16880	acts upstream of or within	cellular response to oxygen-containing compound	GO:1901701	44644	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT2G01570	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16880	locus:2148121	AT5G16880	enables	ubiquitin binding	GO:0043130	18481	F	protein binding	IEA	none	InterPro:IPR002014|InterPro:IPR004152|InterPro:IPR044836	AnalysisReference:501756966		2023-05-23
AT5G16880	locus:2148121	AT5G16880	acts upstream of or within	vesicle-mediated transport	GO:0016192	7567	P	transport	IEA	traceable computational prediction	AGI_LocusCode:AT3G13720	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16880	locus:2148121	AT5G16880	involved in	protein transport to vacuole involved in ubiquitin-dependent protein catabolic process via the multivesicular body sorting pathway	GO:0043328	19963	P	catabolic process	IEA	none	InterPro:IPR044836	AnalysisReference:501756966		2023-05-23
AT5G16880	locus:2148121	AT5G16880	acts upstream of or within	signal transduction	GO:0007165	7243	P	signal transduction	IEA	traceable computational prediction	AGI_LocusCode:AT2G01570	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16880	locus:2148121	AT5G16880	involved in	protein transport to vacuole involved in ubiquitin-dependent protein catabolic process via the multivesicular body sorting pathway	GO:0043328	19963	P	protein metabolic process	IEA	none	InterPro:IPR044836	AnalysisReference:501756966		2023-05-23
AT5G16880	locus:2148121	AT5G16880	acts upstream of or within	protein catabolic process	GO:0030163	9035	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT2G01570	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16880	gene:1005713711	AT5G16880.3	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G16880	locus:2148121	AT5G16880	acts upstream of or within	protein metabolic process	GO:0019538	10586	P	protein metabolic process	IEA	traceable computational prediction	AGI_LocusCode:AT2G01570|AGI_LocusCode:AT5G37890	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16880	locus:2148121	AT5G16880	involved in	protein transport to vacuole involved in ubiquitin-dependent protein catabolic process via the multivesicular body sorting pathway	GO:0043328	19963	P	other cellular processes	IEA	none	InterPro:IPR044836	AnalysisReference:501756966		2023-05-23
AT5G16880	locus:2148121	AT5G16880	involved in	protein transport to vacuole involved in ubiquitin-dependent protein catabolic process via the multivesicular body sorting pathway	GO:0043328	19963	P	other metabolic processes	IEA	none	InterPro:IPR044836	AnalysisReference:501756966		2023-05-23
AT5G16880	locus:2148121	AT5G16880	located in	plastid	GO:0009536	576	C	plastid	HDA	none		Publication:501776792|PMID:28887381  	dszymanski	2019-04-04
AT5G16880	locus:2148121	AT5G16880	acts upstream of or within	protein modification process	GO:0036211	41017	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT5G37890	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16880	gene:1005713712	AT5G16880.2	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G16880	locus:2148121	AT5G16880	acts upstream of or within	protein modification process	GO:0036211	41017	P	protein metabolic process	IEA	traceable computational prediction	AGI_LocusCode:AT5G37890	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16880	locus:2148121	AT5G16880	acts upstream of or within	cellular response to organic substance	GO:0071310	33835	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT2G01570	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16880	locus:2148121	AT5G16880	acts upstream of or within	protein catabolic process	GO:0030163	9035	P	protein metabolic process	IEA	traceable computational prediction	AGI_LocusCode:AT2G01570	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16880	locus:2148121	AT5G16880	acts upstream of or within	cellular response to organic substance	GO:0071310	33835	P	response to chemical	IEA	traceable computational prediction	AGI_LocusCode:AT2G01570	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16880	locus:2148121	AT5G16880	involved in	protein transport to vacuole involved in ubiquitin-dependent protein catabolic process via the multivesicular body sorting pathway	GO:0043328	19963	P	transport	IEA	none	InterPro:IPR044836	AnalysisReference:501756966		2023-05-23
AT5G16880	locus:2148121	AT5G16880	acts upstream of or within	cellular response to oxygen-containing compound	GO:1901701	44644	P	response to chemical	IEA	traceable computational prediction	AGI_LocusCode:AT2G01570	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16880	locus:2148121	AT5G16880	acts upstream of or within	protein catabolic process	GO:0030163	9035	P	catabolic process	IEA	traceable computational prediction	AGI_LocusCode:AT2G01570	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16880	locus:2148121	AT5G16880	enables	phosphatidylinositol binding	GO:0035091	17931	F	lipid binding	IEA	none	InterPro:IPR002014|InterPro:IPR004152|InterPro:IPR044836	AnalysisReference:501756966		2023-05-23
AT5G16880	gene:2148120	AT5G16880.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G16890	locus:2148136	AT5G16890	involved in	protein glycosylation	GO:0006486	6886	P	biosynthetic process	IEA	none	InterPro:IPR004263	AnalysisReference:501756966		2023-05-23
AT5G16890	gene:2148135	AT5G16890.1	located in	Golgi trans cisterna	GO:0000138	56	C	Golgi apparatus	HDA	none		Publication:501785785|PMID:31266899  	tempeparsons	2019-08-16
AT5G16890	locus:2148136	AT5G16890	involved in	protein glycosylation	GO:0006486	6886	P	other metabolic processes	IEA	none	InterPro:IPR004263	AnalysisReference:501756966		2023-05-23
AT5G16890	locus:2148136	AT5G16890	involved in	protein glycosylation	GO:0006486	6886	P	other cellular processes	IEA	none	InterPro:IPR004263	AnalysisReference:501756966		2023-05-23
AT5G16890	gene:2148135	AT5G16890.1	located in	Golgi apparatus	GO:0005794	48	C	Golgi apparatus	HDA	none	NONE	Publication:501747810|PMID:22430844  	jheazlewoo	2021-05-12
AT5G16890	gene:2148135	AT5G16890.1	located in	Golgi apparatus	GO:0005794	48	C	Golgi apparatus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G16890	locus:2148136	AT5G16890	involved in	protein glycosylation	GO:0006486	6886	P	protein metabolic process	IEA	none	InterPro:IPR004263	AnalysisReference:501756966		2023-05-23
AT5G16890	gene:2148135	AT5G16890.1	located in	Golgi trans cisterna	GO:0000138	56	C	other intracellular components	HDA	none		Publication:501785785|PMID:31266899  	tempeparsons	2019-08-16
AT5G16900	locus:2148151	AT5G16900	acts upstream of or within	developmental cell growth	GO:0048588	21322	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G50610|AGI_LocusCode:AT4G14455	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16900	locus:2148151	AT5G16900	acts upstream of or within	plant epidermis development	GO:0090558	47687	P	anatomical structure development	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT5G46760	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16900	locus:2148151	AT5G16900	enables	protein serine/threonine kinase activity	GO:0004674	3907	F	transferase activity	IEA	none	EC:2.7.11.1	AnalysisReference:501756967		2023-05-23
AT5G16900	locus:2148151	AT5G16900	enables	protein kinase activity	GO:0004672	3889	F	catalytic activity	IEA	none	InterPro:IPR000719|InterPro:IPR008271	AnalysisReference:501756966		2023-05-23
AT5G16900	locus:2148151	AT5G16900	acts upstream of or within	developmental growth involved in morphogenesis	GO:0060560	32023	P	growth	IEA	traceable computational prediction	AGI_LocusCode:AT1G50610|AGI_LocusCode:AT4G14455	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16900	locus:2148151	AT5G16900	acts upstream of or within	developmental cell growth	GO:0048588	21322	P	anatomical structure development	IEA	traceable computational prediction	AGI_LocusCode:AT1G50610|AGI_LocusCode:AT4G14455	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16900	locus:2148151	AT5G16900	acts upstream of or within	phosphorylation	GO:0016310	6748	P	other metabolic processes	ISS	none		Communication:1674994		2021-04-01
AT5G16900	locus:2148151	AT5G16900	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:Q8W4S5	Publication:501778368|PMID:29320478  		2023-05-23
AT5G16900	locus:2148151	AT5G16900	enables	protein serine/threonine kinase activity	GO:0004674	3907	F	catalytic activity	IEA	none	EC:2.7.11.1	AnalysisReference:501756967		2023-05-23
AT5G16900	locus:2148151	AT5G16900	involved in	protein phosphorylation	GO:0006468	6897	P	other cellular processes	IEA	none	InterPro:IPR000719|InterPro:IPR008271	AnalysisReference:501756966		2023-05-23
AT5G16900	locus:2148151	AT5G16900	enables	protein kinase activity	GO:0004672	3889	F	kinase activity	IEA	none	InterPro:IPR000719|InterPro:IPR001245	AnalysisReference:501756966		2023-05-23
AT5G16900	locus:2148151	AT5G16900	enables	kinase activity	GO:0016301	2953	F	kinase activity	ISS	Sequence similarity (homologue of/most closely related to)		Communication:1674994	TAIR	2003-03-29
AT5G16900	locus:2148151	AT5G16900	acts upstream of or within	macromolecule biosynthetic process	GO:0009059	6237	P	biosynthetic process	IEA	traceable computational prediction	AGI_LocusCode:AT1G21970|AGI_LocusCode:AT1G32640|AGI_LocusCode:AT4G34000|AGI_LocusCode:AT5G46760|AGI_LocusCode:AT3G03210	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16900	locus:2148151	AT5G16900	acts upstream of or within	polysaccharide biosynthetic process	GO:0000271	6809	P	carbohydrate metabolic process	IEA	traceable computational prediction	AGI_LocusCode:AT3G03210	Publication:501796011|PMID:34562334  	klaasvdp	2023-01-08
AT5G16900	locus:2148151	AT5G16900	acts upstream of or within	root development	GO:0048364	18902	P	multicellular organism development	IEA	traceable computational prediction	AGI_LocusCode:AT2G25180|AGI_LocusCode:AT3G16857|AGI_LocusCode:AT1G72180	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16900	locus:2148151	AT5G16900	acts upstream of or within	polysaccharide biosynthetic process	GO:0000271	6809	P	biosynthetic process	IEA	traceable computational prediction	AGI_LocusCode:AT3G03210	Publication:501796011|PMID:34562334  	klaasvdp	2023-01-08
AT5G16900	locus:2148151	AT5G16900	enables	protein serine/threonine kinase activity	GO:0004674	3907	F	kinase activity	IEA	none	EC:2.7.11.1	AnalysisReference:501756967		2023-05-23
AT5G16900	locus:2148151	AT5G16900	acts upstream of or within	root morphogenesis	GO:0010015	13857	P	anatomical structure development	IEA	traceable computational prediction	AGI_LocusCode:AT2G25180|AGI_LocusCode:AT3G16857	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16900	locus:2148151	AT5G16900	enables	protein kinase activity	GO:0004672	3889	F	kinase activity	IEA	none	InterPro:IPR000719|InterPro:IPR008271	AnalysisReference:501756966		2023-05-23
AT5G16900	locus:2148151	AT5G16900	enables	protein kinase activity	GO:0004672	3889	F	transferase activity	IEA	none	InterPro:IPR000719|InterPro:IPR008271	AnalysisReference:501756966		2023-05-23
AT5G16900	locus:2148151	AT5G16900	involved in	protein phosphorylation	GO:0006468	6897	P	other metabolic processes	IEA	none	InterPro:IPR000719|InterPro:IPR008271	AnalysisReference:501756966		2023-05-23
AT5G16900	locus:2148151	AT5G16900	acts upstream of or within	developmental growth	GO:0048589	21420	P	growth	IEA	traceable computational prediction	AGI_LocusCode:AT2G25180|AGI_LocusCode:AT3G16857|AGI_LocusCode:AT1G50610|AGI_LocusCode:AT1G60800|AGI_LocusCode:AT2G23950|AGI_LocusCode:AT4G14455|AGI_LocusCode:AT5G45780|AGI_LocusCode:AT5G65700	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16900	locus:2148151	AT5G16900	acts upstream of or within	root morphogenesis	GO:0010015	13857	P	multicellular organism development	IEA	traceable computational prediction	AGI_LocusCode:AT2G25180|AGI_LocusCode:AT3G16857	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16900	locus:2148151	AT5G16900	acts upstream of or within	phosphorylation	GO:0016310	6748	P	other cellular processes	ISS	none		Communication:1674994		2021-04-01
AT5G16900	locus:2148151	AT5G16900	acts upstream of or within	cell wall macromolecule metabolic process	GO:0044036	31378	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT3G03210	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16900	locus:2148151	AT5G16900	acts upstream of or within	developmental maturation	GO:0021700	23895	P	other biological processes	IEA	traceable computational prediction	AGI_LocusCode:AT3G02130	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16900	locus:2148151	AT5G16900	acts upstream of or within	response to hypoxia	GO:0001666	10824	P	response to abiotic stimulus	IEA	traceable computational prediction	AGI_LocusCode:AT5G25110	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16900	locus:2148151	AT5G16900	involved in	protein phosphorylation	GO:0006468	6897	P	other metabolic processes	IEA	none	InterPro:IPR000719|InterPro:IPR001245	AnalysisReference:501756966		2023-05-23
AT5G16900	locus:2148151	AT5G16900	acts upstream of or within	response to hypoxia	GO:0001666	10824	P	response to stress	IEA	traceable computational prediction	AGI_LocusCode:AT5G25110	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16900	locus:2148151	AT5G16900	acts upstream of or within	cell wall organization or biogenesis	GO:0071554	34183	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT3G03210	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16900	locus:2148151	AT5G16900	acts upstream of or within	macromolecule biosynthetic process	GO:0009059	6237	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G21970|AGI_LocusCode:AT1G32640|AGI_LocusCode:AT4G34000|AGI_LocusCode:AT5G46760|AGI_LocusCode:AT3G03210	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16900	locus:2148151	AT5G16900	acts upstream of or within	root development	GO:0048364	18902	P	anatomical structure development	IEA	traceable computational prediction	AGI_LocusCode:AT2G25180|AGI_LocusCode:AT3G16857|AGI_LocusCode:AT1G72180	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16900	locus:2148151	AT5G16900	acts upstream of or within	cell differentiation	GO:0030154	9398	P	cell differentiation	IEA	traceable computational prediction	AGI_LocusCode:AT5G16560|AGI_LocusCode:AT1G50610|AGI_LocusCode:AT4G14455|AGI_LocusCode:AT5G65700	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16900	locus:2148151	AT5G16900	acts upstream of or within	cell wall macromolecule metabolic process	GO:0044036	31378	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT3G03210	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16900	locus:2148151	AT5G16900	acts upstream of or within	developmental cell growth	GO:0048588	21322	P	growth	IEA	traceable computational prediction	AGI_LocusCode:AT1G50610|AGI_LocusCode:AT4G14455	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16900	locus:2148151	AT5G16900	enables	protein kinase activity	GO:0004672	3889	F	catalytic activity	IEA	none	InterPro:IPR000719|InterPro:IPR001245	AnalysisReference:501756966		2023-05-23
AT5G16900	locus:2148151	AT5G16900	involved in	protein phosphorylation	GO:0006468	6897	P	other cellular processes	IEA	none	InterPro:IPR000719|InterPro:IPR001245	AnalysisReference:501756966		2023-05-23
AT5G16900	gene:6532561545	AT5G16900.4	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G16900	locus:2148151	AT5G16900	acts upstream of or within	developmental cell growth	GO:0048588	21322	P	cell differentiation	IEA	traceable computational prediction	AGI_LocusCode:AT1G50610|AGI_LocusCode:AT4G14455	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16900	locus:2148151	AT5G16900	acts upstream of or within	plant-type cell wall organization or biogenesis	GO:0071669	34585	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16900	locus:2148151	AT5G16900	acts upstream of or within	polysaccharide biosynthetic process	GO:0000271	6809	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT3G03210	Publication:501796011|PMID:34562334  	klaasvdp	2023-01-08
AT5G16900	locus:2148151	AT5G16900	acts upstream of or within	tissue development	GO:0009888	11998	P	anatomical structure development	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT2G25180|AGI_LocusCode:AT3G16857|AGI_LocusCode:AT5G46760|AGI_LocusCode:AT1G60800|AGI_LocusCode:AT2G23950|AGI_LocusCode:AT3G02130|AGI_LocusCode:AT5G45780|AGI_LocusCode:AT5G65700	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16900	locus:2148151	AT5G16900	enables	protein kinase activity	GO:0004672	3889	F	transferase activity	IEA	none	InterPro:IPR000719|InterPro:IPR001245	AnalysisReference:501756966		2023-05-23
AT5G16900	locus:2148151	AT5G16900	acts upstream of or within	developmental growth involved in morphogenesis	GO:0060560	32023	P	anatomical structure development	IEA	traceable computational prediction	AGI_LocusCode:AT1G50610|AGI_LocusCode:AT4G14455	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16900	locus:2148151	AT5G16900	acts upstream of or within	developmental cell growth	GO:0048588	21322	P	cell growth	IEA	traceable computational prediction	AGI_LocusCode:AT1G50610|AGI_LocusCode:AT4G14455	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16910	locus:2148171	AT5G16910	is active in	plasma membrane	GO:0005886	570	C	plasma membrane	IBA	none	PANTHER:PTN001410221|TAIR:locus:2176090|TAIR:locus:2178935	Communication:501741973		2023-04-13
AT5G16910	locus:2148171	AT5G16910	involved in	cellulose biosynthetic process	GO:0030244	8923	P	carbohydrate metabolic process	IEA	none	InterPro:IPR005150	AnalysisReference:501756966		2023-05-23
AT5G16910	locus:2148171	AT5G16910	located in	Golgi apparatus	GO:0005794	48	C	Golgi apparatus	IDA	localization of GFP/YFP fusion protein		Publication:501728637|PMID:18768911  	pectin	2008-09-22
AT5G16910	locus:2148171	AT5G16910	involved in	plant-type primary cell wall biogenesis	GO:0009833	10266	P	cell cycle	IBA	none	PANTHER:PTN001410221|TAIR:locus:2156789|TAIR:locus:2176090|TAIR:locus:2127776	Communication:501741973		2023-04-13
AT5G16910	locus:2148171	AT5G16910	acts upstream of or within	response to cold	GO:0009409	5433	P	response to abiotic stimulus	IEP	Transcript levels (e.g. RT-PCR)	NONE	Publication:501728637|PMID:18768911  	pectin	2008-09-22
AT5G16910	locus:2148171	AT5G16910	enables	cellulose synthase activity	GO:0016759	1882	F	transferase activity	ISS	Sequence similarity (homologue of/most closely related to)		Communication:1674989	TAIR	2003-03-29
AT5G16910	locus:2148171	AT5G16910	acts upstream of or within	root hair elongation	GO:0048767	22772	P	cell differentiation	IMP	analysis of visible trait	NONE	Publication:501728637|PMID:18768911  	pectin	2012-10-26
AT5G16910	locus:2148171	AT5G16910	acts upstream of or within	root hair elongation	GO:0048767	22772	P	cell growth	IMP	analysis of visible trait	NONE	Publication:501728637|PMID:18768911  	pectin	2012-10-26
AT5G16910	locus:2148171	AT5G16910	located in	trans-Golgi network	GO:0005802	57	C	other intracellular components	HDA	none	NONE	Publication:501750648|PMID:22923678  	dupree	2021-04-07
AT5G16910	locus:2148171	AT5G16910	located in	Golgi membrane	GO:0000139	54	C	other intracellular components	IDA	Cell fractionation (for cellular component)	NONE	Publication:501727305|PMID:18642024  	TAIR	2022-08-25
AT5G16910	gene:2148170	AT5G16910.1	located in	Golgi apparatus	GO:0005794	48	C	Golgi apparatus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G16910	locus:2148171	AT5G16910	enables	mannan synthase activity	GO:0051753	22358	F	transferase activity	IBA	none	PANTHER:PTN002665465|TAIR:locus:2097700|TAIR:locus:2024745|TAIR:locus:2148171	Communication:501741973		2023-06-07
AT5G16910	locus:2148171	AT5G16910	enables	cellulose synthase (UDP-forming) activity	GO:0016760	1884	F	transferase activity	IEA	none	InterPro:IPR005150	AnalysisReference:501756966		2023-06-11
AT5G16910	locus:2148171	AT5G16910	located in	endosome	GO:0005768	272	C	endosome	HDA	none	NONE	Publication:501750648|PMID:22923678  	dupree	2021-04-07
AT5G16910	locus:2148171	AT5G16910	located in	trans-Golgi network	GO:0005802	57	C	Golgi apparatus	HDA	none	NONE	Publication:501750648|PMID:22923678  	dupree	2021-04-07
AT5G16910	locus:2148171	AT5G16910	located in	Golgi membrane	GO:0000139	54	C	Golgi apparatus	IDA	Cell fractionation (for cellular component)	NONE	Publication:501727305|PMID:18642024  	TAIR	2022-08-25
AT5G16910	locus:2148171	AT5G16910	acts upstream of or within	root hair elongation	GO:0048767	22772	P	growth	IMP	analysis of visible trait	NONE	Publication:501728637|PMID:18768911  	pectin	2012-10-26
AT5G16910	locus:2148171	AT5G16910	involved in	cellulose biosynthetic process	GO:0030244	8923	P	biosynthetic process	IEA	none	InterPro:IPR005150	AnalysisReference:501756966		2023-05-23
AT5G16910	locus:2148171	AT5G16910	involved in	cellulose biosynthetic process	GO:0030244	8923	P	other cellular processes	IEA	none	InterPro:IPR005150	AnalysisReference:501756966		2023-05-23
AT5G16910	locus:2148171	AT5G16910	located in	Golgi membrane	GO:0000139	54	C	other membranes	IDA	Cell fractionation (for cellular component)	NONE	Publication:501727305|PMID:18642024  	TAIR	2022-08-25
AT5G16910	locus:2148171	AT5G16910	acts upstream of or within	response to cold	GO:0009409	5433	P	response to stress	IEP	Transcript levels (e.g. RT-PCR)	NONE	Publication:501728637|PMID:18768911  	pectin	2008-09-22
AT5G16910	locus:2148171	AT5G16910	involved in	cellulose biosynthetic process	GO:0030244	8923	P	other metabolic processes	IEA	none	InterPro:IPR005150	AnalysisReference:501756966		2023-05-23
AT5G16910	locus:2148171	AT5G16910	acts upstream of or within	root hair elongation	GO:0048767	22772	P	anatomical structure development	IMP	analysis of visible trait	NONE	Publication:501728637|PMID:18768911  	pectin	2012-10-26
AT5G16910	locus:2148171	AT5G16910	acts upstream of or within	root hair elongation	GO:0048767	22772	P	multicellular organism development	IMP	analysis of visible trait	NONE	Publication:501728637|PMID:18768911  	pectin	2012-10-26
AT5G16910	locus:2148171	AT5G16910	acts upstream of or within	root hair elongation	GO:0048767	22772	P	other cellular processes	IMP	analysis of visible trait	NONE	Publication:501728637|PMID:18768911  	pectin	2012-10-26
AT5G16910	locus:2148171	AT5G16910	located in	Golgi apparatus	GO:0005794	48	C	Golgi apparatus	HDA	none	NONE	Publication:501718909|PMID:16618929  	TAIR	2021-05-12
AT5G16910	locus:2148171	AT5G16910	involved in	plant-type primary cell wall biogenesis	GO:0009833	10266	P	other cellular processes	IBA	none	PANTHER:PTN001410221|TAIR:locus:2156789|TAIR:locus:2176090|TAIR:locus:2127776	Communication:501741973		2023-04-13
AT5G16910	locus:2148171	AT5G16910	enables	mannan synthase activity	GO:0051753	22358	F	transferase activity	IDA	Enzyme assays		Publication:501745229|PMID:21904114  	TAIR	2012-10-26
AT5G16910	locus:2148171	AT5G16910	located in	Golgi apparatus	GO:0005794	48	C	Golgi apparatus	HDA	none	NONE	Publication:501750648|PMID:22923678  	dupree	2021-04-07
AT5G16910	locus:2148171	AT5G16910	enables	glycosyltransferase activity	GO:0016757	14916	F	transferase activity	ISS	Recognized domains		Communication:1674999	TAIR	2004-04-21
AT5G16920	locus:2148191	AT5G16920	acts upstream of or within	oxoacid metabolic process	GO:0043436	21524	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT4G01120	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16920	locus:2148191	AT5G16920	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	none	NONE	Communication:1345790	TIGR	2003-10-04
AT5G16920	locus:2148191	AT5G16920	acts upstream of or within	oxoacid metabolic process	GO:0043436	21524	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT4G01120	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16920	locus:2148191	AT5G16920	is active in	cellular_component	GO:0005575	163	C	unknown cellular components	ND	Unknown cellular component	NONE	Communication:1345790	TAIR	2022-02-01
AT5G16930	locus:2148206	AT5G16930	is active in	mitochondrion	GO:0005739	486	C	mitochondrion	IBA	none	PANTHER:PTN000554019|UniProtKB:Q9NVI7|WB:WBGene00010015|MGI:MGI:1919214|UniProtKB:Q5T9A4|UniProtKB:C0H4M1|UniProtKB:Q5T2N8	Communication:501741973		2023-06-07
AT5G16930	locus:2148206	AT5G16930	located in	mitochondrion	GO:0005739	486	C	mitochondrion	HDA	none	NONE	Publication:501750648|PMID:22923678  	dupree	2021-04-07
AT5G16930	locus:2148206	AT5G16930	involved in	mitochondrion organization	GO:0007005	6383	P	cellular component organization	IBA	none	PANTHER:PTN000554019|WB:WBGene00010015|FB:FBgn0287225	Communication:501741973		2023-06-07
AT5G16930	locus:2148206	AT5G16930	enables	ATP hydrolysis activity	GO:0016887	907	F	hydrolase activity	IEA	none	InterPro:IPR003959	AnalysisReference:501756966		2023-05-23
AT5G16930	gene:2148205	AT5G16930.1	located in	chloroplast	GO:0009507	175	C	chloroplast	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G16930	locus:2148206	AT5G16930	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	Co-immunoprecipitation	AGI_LocusCode:AT3G60400	Publication:501795864|PMID:34482561  	TAIR	2021-09-26
AT5G16940	locus:2148221	AT5G16940	involved in	biological_process	GO:0008150	5239	P	unknown biological processes	ND	Unknown biological process	NONE	Communication:1345790	TAIR	2022-02-01
AT5G16940	gene:5019474489	AT5G16940.2	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G16940	gene:2148220	AT5G16940.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G16940	locus:2148221	AT5G16940	enables	carbon-sulfur lyase activity	GO:0016846	1808	F	catalytic activity	IEA	none	InterPro:IPR006913	AnalysisReference:501756966		2023-05-23
AT5G16950	locus:2148106	AT5G16950	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	none	NONE	Communication:1345790	TIGR	2003-10-04
AT5G16950	locus:2148106	AT5G16950	acts upstream of or within	RNA metabolic process	GO:0016070	4899	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G19350|AGI_LocusCode:AT5G22290	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16950	locus:2148106	AT5G16950	is active in	cellular_component	GO:0005575	163	C	unknown cellular components	ND	Unknown cellular component	NONE	Communication:1345790	TAIR	2022-02-01
AT5G16950	locus:2148106	AT5G16950	acts upstream of or within	RNA metabolic process	GO:0016070	4899	P	nucleobase-containing compound metabolic process	IEA	traceable computational prediction	AGI_LocusCode:AT1G19350|AGI_LocusCode:AT5G22290	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G16960	locus:2148116	AT5G16960	involved in	response to oxidative stress	GO:0006979	6625	P	response to stress	IBA	none	PANTHER:PTN005151586|TAIR:locus:2028736|TAIR:locus:2148131|TAIR:locus:2148146|TAIR:locus:2153759|TAIR:locus:2148116|TAIR:locus:2148166|TAIR:locus:2153799|TAIR:locus:505006405|TAIR:locus:2148186|TAIR:locus:2154041	Communication:501741973		2023-06-07
AT5G16960	locus:2148116	AT5G16960	located in	plastid	GO:0009536	576	C	plastid	HDA	none		Publication:501776792|PMID:28887381  	dszymanski	2019-04-04
AT5G16960	gene:2148115	AT5G16960.1	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G16960	locus:2148116	AT5G16960	enables	2-alkenal reductase [NAD(P)+] activity	GO:0032440	25434	F	catalytic activity	IBA	none	PANTHER:PTN000190595|TAIR:locus:2148131	Communication:501741973		2021-04-02
AT5G16960	gene:6532553429	AT5G16960.2	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G16960	locus:2148116	AT5G16960	acts upstream of or within	response to oxidative stress	GO:0006979	6625	P	response to stress	IEP	none	NONE	Publication:4173|PMID:7592828   	TIGR	2003-04-17
AT5G16970	locus:2148131	AT5G16970	enables	2-alkenal reductase [NAD(P)+] activity	GO:0032440	25434	F	catalytic activity	IDA	Enzyme assays		Publication:501718040|PMID:16299173  	TAIR	2006-12-21
AT5G16970	locus:2148131	AT5G16970	located in	cytosol	GO:0005829	241	C	cytosol	HDA	none		Publication:501761766|PMID:25293756  	dszymanski	2019-04-04
AT5G16970	locus:2148131	AT5G16970	enables	2-alkenal reductase [NAD(P)+] activity	GO:0032440	25434	F	catalytic activity	IBA	none	PANTHER:PTN000190595|TAIR:locus:2148131	Communication:501741973		2021-04-02
AT5G16970	locus:2148131	AT5G16970	acts upstream of or within	response to oxidative stress	GO:0006979	6625	P	response to stress	IGI	Functional complementation in heterologous system	NONE	Publication:4173|PMID:7592828   	TAIR	2009-01-23
AT5G16970	locus:2148131	AT5G16970	acts upstream of or within	response to oxidative stress	GO:0006979	6625	P	response to stress	IMP	Analysis of overexpression/ectopic expression phenotype	NONE	Publication:501718040|PMID:16299173  	TAIR	2005-12-23
AT5G16970	gene:2148130	AT5G16970.1	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G16970	locus:2148131	AT5G16970	located in	cytosol	GO:0005829	241	C	cytosol	IDA	localization of GFP/YFP fusion protein		Publication:501718040|PMID:16299173  	TAIR	2005-12-23
AT5G16970	locus:2148131	AT5G16970	located in	plastid	GO:0009536	576	C	plastid	HDA	none		Publication:501776792|PMID:28887381  	dszymanski	2019-04-04
AT5G16970	locus:2148131	AT5G16970	acts upstream of or within	response to oxidative stress	GO:0006979	6625	P	response to stress	IEP	none	NONE	Publication:4173|PMID:7592828   	TIGR	2003-04-17
AT5G16970	locus:2148131	AT5G16970	located in	nucleus	GO:0005634	537	C	nucleus	IDA	localization of GFP/YFP fusion protein		Publication:501718040|PMID:16299173  	TAIR	2005-12-23
AT5G16970	locus:2148131	AT5G16970	involved in	response to oxidative stress	GO:0006979	6625	P	response to stress	IBA	none	PANTHER:PTN005151586|TAIR:locus:2028736|TAIR:locus:2148131|TAIR:locus:2148146|TAIR:locus:2153759|TAIR:locus:2148116|TAIR:locus:2148166|TAIR:locus:2153799|TAIR:locus:505006405|TAIR:locus:2148186|TAIR:locus:2154041	Communication:501741973		2023-06-07
AT5G16980	locus:2148146	AT5G16980	enables	2-alkenal reductase [NAD(P)+] activity	GO:0032440	25434	F	catalytic activity	IBA	none	PANTHER:PTN000190595|TAIR:locus:2148131	Communication:501741973		2021-04-02
AT5G16980	locus:2148146	AT5G16980	located in	plastid	GO:0009536	576	C	plastid	HDA	none		Publication:501776792|PMID:28887381  	dszymanski	2019-04-04
AT5G16980	locus:2148146	AT5G16980	acts upstream of or within	response to oxidative stress	GO:0006979	6625	P	response to stress	IEP	none	NONE	Publication:4173|PMID:7592828   	TIGR	2003-04-17
AT5G16980	gene:6532556387	AT5G16980.3	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G16980	gene:2148145	AT5G16980.1	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G16980	gene:6530297799	AT5G16980.2	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G16980	locus:2148146	AT5G16980	involved in	response to oxidative stress	GO:0006979	6625	P	response to stress	IBA	none	PANTHER:PTN005151586|TAIR:locus:2028736|TAIR:locus:2148131|TAIR:locus:2148146|TAIR:locus:2153759|TAIR:locus:2148116|TAIR:locus:2148166|TAIR:locus:2153799|TAIR:locus:505006405|TAIR:locus:2148186|TAIR:locus:2154041	Communication:501741973		2023-06-07
AT5G16990	locus:2148166	AT5G16990	involved in	response to oxidative stress	GO:0006979	6625	P	response to stress	IBA	none	PANTHER:PTN005151586|TAIR:locus:2028736|TAIR:locus:2148131|TAIR:locus:2148146|TAIR:locus:2153759|TAIR:locus:2148116|TAIR:locus:2148166|TAIR:locus:2153799|TAIR:locus:505006405|TAIR:locus:2148186|TAIR:locus:2154041	Communication:501741973		2023-06-07
AT5G16990	gene:2148165	AT5G16990.1	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G16990	locus:2148166	AT5G16990	acts upstream of or within	response to oxidative stress	GO:0006979	6625	P	response to stress	IEP	none	NONE	Publication:4173|PMID:7592828   	TAIR	2009-01-23
AT5G16990	locus:2148166	AT5G16990	located in	plastid	GO:0009536	576	C	plastid	HDA	none		Publication:501776792|PMID:28887381  	dszymanski	2019-04-04
AT5G16990	gene:2148165	AT5G16990.1	located in	plasma membrane	GO:0005886	570	C	plasma membrane	HDA	none	NONE	Publication:501722708|PMID:17644812  	kvanwijk	2021-04-07
AT5G16990	locus:2148166	AT5G16990	enables	2-alkenal reductase [NAD(P)+] activity	GO:0032440	25434	F	catalytic activity	IBA	none	PANTHER:PTN000190595|TAIR:locus:2148131	Communication:501741973		2021-04-02
AT5G16990	locus:2148166	AT5G16990	acts upstream of or within	response to oxidative stress	GO:0006979	6625	P	response to stress	IGI	Functional complementation in heterologous system	NONE	Publication:4173|PMID:7592828   	TAIR	2009-01-23
AT5G17000	gene:6532550485	AT5G17000.2	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G17000	gene:2148185	AT5G17000.1	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G17000	locus:2148186	AT5G17000	located in	plastid	GO:0009536	576	C	plastid	HDA	none		Publication:501776792|PMID:28887381  	dszymanski	2019-04-04
AT5G17000	locus:2148186	AT5G17000	acts upstream of or within	response to oxidative stress	GO:0006979	6625	P	response to stress	IEP	none	NONE	Publication:4173|PMID:7592828   	TIGR	2003-04-17
AT5G17000	locus:2148186	AT5G17000	enables	2-alkenal reductase [NAD(P)+] activity	GO:0032440	25434	F	catalytic activity	IBA	none	PANTHER:PTN000190595|TAIR:locus:2148131	Communication:501741973		2021-04-02
AT5G17000	locus:2148186	AT5G17000	involved in	response to oxidative stress	GO:0006979	6625	P	response to stress	IBA	none	PANTHER:PTN005151586|TAIR:locus:2028736|TAIR:locus:2148131|TAIR:locus:2148146|TAIR:locus:2153759|TAIR:locus:2148116|TAIR:locus:2148166|TAIR:locus:2153799|TAIR:locus:505006405|TAIR:locus:2148186|TAIR:locus:2154041	Communication:501741973		2023-06-07
AT5G17010	locus:2148201	AT5G17010	enables	transmembrane transporter activity	GO:0022857	27431	F	transporter activity	IEA	none	InterPro:IPR003663|InterPro:IPR005828|InterPro:IPR005829|InterPro:IPR020846	AnalysisReference:501756966		2023-05-23
AT5G17010	locus:2148201	AT5G17010	enables	symporter activity	GO:0015293	4325	F	transporter activity	IBA	none	PANTHER:PTN008729220|UniProtKB:P0AEP1|UniProtKB:P0AE24	Communication:501741973		2022-11-23
AT5G17010	gene:6530297800	AT5G17010.4	located in	plasma membrane	GO:0005886	570	C	plasma membrane	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G17010	locus:2148201	AT5G17010	is active in	membrane	GO:0016020	453	C	other membranes	IBA	none	PANTHER:PTN008729220|MGI:MGI:3052471|TAIR:locus:2097780|UniProtKB:P0AE24|UniProtKB:P0AEP1|UniProtKB:O95528	Communication:501741973		2023-06-07
AT5G17010	gene:2148200	AT5G17010.1	located in	plant-type vacuole	GO:0000325	11545	C	vacuole	HDA	none	NONE	Publication:501720626|PMID:17151019  	kvanwijk	2021-05-12
AT5G17010	gene:1005713713	AT5G17010.3	located in	plasma membrane	GO:0005886	570	C	plasma membrane	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G17010	locus:2148201	AT5G17010	involved in	transmembrane transport	GO:0055085	28452	P	other cellular processes	IEA	none	InterPro:IPR003663|InterPro:IPR005828|InterPro:IPR005829	AnalysisReference:501756966		2023-06-11
AT5G17010	gene:1005713714	AT5G17010.2	located in	chloroplast	GO:0009507	175	C	chloroplast	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G17010	locus:2148201	AT5G17010	involved in	glucose transmembrane transport	GO:1904659	50497	P	other cellular processes	IBA	none	PANTHER:PTN002512237|TAIR:locus:2168494|UniProtKB:O95528|ZFIN:ZDB-GENE-040426-1513|MGI:MGI:3052471|TAIR:locus:2097780	Communication:501741973		2023-06-07
AT5G17010	locus:2148201	AT5G17010	involved in	glucose transmembrane transport	GO:1904659	50497	P	transport	IBA	none	PANTHER:PTN002512237|TAIR:locus:2168494|UniProtKB:O95528|ZFIN:ZDB-GENE-040426-1513|MGI:MGI:3052471|TAIR:locus:2097780	Communication:501741973		2023-06-07
AT5G17010	gene:1005713713	AT5G17010.3	located in	plant-type vacuole	GO:0000325	11545	C	vacuole	HDA	none	NONE	Publication:501720626|PMID:17151019  	kvanwijk	2021-05-12
AT5G17010	locus:2148201	AT5G17010	enables	glucose transmembrane transporter activity	GO:0005355	2485	F	transporter activity	IBA	none	PANTHER:PTN002512237|TAIR:locus:2097780|TAIR:locus:2168494|UniProtKB:O95528	Communication:501741973		2023-04-13
AT5G17010	locus:2148201	AT5G17010	located in	plastid	GO:0009536	576	C	plastid	HDA	none		Publication:501776792|PMID:28887381  	dszymanski	2019-04-04
AT5G17010	gene:1005713714	AT5G17010.2	located in	plant-type vacuole	GO:0000325	11545	C	vacuole	HDA	none	NONE	Publication:501720626|PMID:17151019  	kvanwijk	2021-05-12
AT5G17010	locus:2148201	AT5G17010	enables	carbohydrate transmembrane transporter activity	GO:0015144	1797	F	transporter activity	ISS	Sequence similarity (homologue of/most closely related to)		Communication:1674994	TAIR	2003-03-29
AT5G17010	locus:2148201	AT5G17010	involved in	transmembrane transport	GO:0055085	28452	P	transport	IEA	none	InterPro:IPR003663|InterPro:IPR005828|InterPro:IPR005829	AnalysisReference:501756966		2023-06-11
AT5G17010	gene:2148200	AT5G17010.1	located in	plasma membrane	GO:0005886	570	C	plasma membrane	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G17020	locus:2148216	AT5G17020	acts upstream of or within	protein export from nucleus	GO:0006611	6919	P	transport	ISS	Sequence similarity (homologue of/most closely related to)	NONE	Publication:778|PMID:10652141  	TAIR	2003-05-20
AT5G17020	locus:2148216	AT5G17020	acts upstream of or within	pollen tube growth	GO:0009860	10927	P	pollination	IGI	double mutant analysis	AGI_LocusCode:AT3G03110	Publication:501728738|PMID:18791220  	TAIR	2008-10-13
AT5G17020	locus:2148216	AT5G17020	acts upstream of or within	pollen tube growth	GO:0009860	10927	P	other cellular processes	IGI	double mutant analysis	AGI_LocusCode:AT3G03110	Publication:501728738|PMID:18791220  	TAIR	2008-10-13
AT5G17020	locus:2148216	AT5G17020	acts upstream of or within	pollen development	GO:0009555	11329	P	multicellular organism development	IGI	double mutant analysis	AGI_LocusCode:AT3G03110	Publication:501728738|PMID:18791220  	TAIR	2008-10-13
AT5G17020	locus:2148216	AT5G17020	acts upstream of or within	pollen germination	GO:0009846	10881	P	reproduction	IGI	double mutant analysis	AGI_LocusCode:AT3G03110	Publication:501728738|PMID:18791220  	TAIR	2008-10-13
AT5G17020	locus:2148216	AT5G17020	acts upstream of or within	pollen germination	GO:0009846	10881	P	pollination	IGI	double mutant analysis	AGI_LocusCode:AT3G03110	Publication:501728738|PMID:18791220  	TAIR	2008-10-13
AT5G17020	locus:2148216	AT5G17020	acts upstream of or within	pollen tube growth	GO:0009860	10927	P	cell growth	IGI	double mutant analysis	AGI_LocusCode:AT3G03110	Publication:501728738|PMID:18791220  	TAIR	2008-10-13
AT5G17020	locus:2148216	AT5G17020	involved in	ribosomal small subunit export from nucleus	GO:0000056	7174	P	transport	IBA	none	PANTHER:PTN000131787|UniProtKB:O14980|FB:FBgn0020497	Communication:501741973		2023-01-14
AT5G17020	locus:2148216	AT5G17020	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:P41916	Publication:778|PMID:10652141  		2021-04-01
AT5G17020	locus:2148216	AT5G17020	is active in	nucleus	GO:0005634	537	C	nucleus	IBA	none	PANTHER:PTN000131749|SGD:S000002743|SGD:S000003450|MGI:MGI:2144013|PomBase:SPAC1805.17|TAIR:locus:2114784|FB:FBgn0020497|UniProtKB:C8VRM0|UniProtKB:Q9HAV4|UniProtKB:O14980	Communication:501741973		2023-06-07
AT5G17020	locus:2148216	AT5G17020	involved in	ribosomal large subunit export from nucleus	GO:0000055	7171	P	other cellular processes	IBA	none	PANTHER:PTN000131787|UniProtKB:O14980|SGD:S000003450	Communication:501741973		2023-01-14
AT5G17020	locus:2148216	AT5G17020	acts upstream of or within	pollen tube growth	GO:0009860	10927	P	cell differentiation	IGI	double mutant analysis	AGI_LocusCode:AT3G03110	Publication:501728738|PMID:18791220  	TAIR	2008-10-13
AT5G17020	locus:2148216	AT5G17020	enables	nuclear export signal receptor activity	GO:0005049	3373	F	other molecular functions	IBA	none	PANTHER:PTN000131749|PomBase:SPAC1805.17|MGI:MGI:2144013|CGD:CAL0000197974|TAIR:locus:2114784|SGD:S000002743|FB:FBgn0020497|MGI:MGI:1913789|UniProtKB:Q9HAV4|UniProtKB:O14980	Communication:501741973		2023-06-07
AT5G17020	locus:2148216	AT5G17020	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	yeast two-hybrid assay	AGI_LocusCode:AT5G20010	Publication:778|PMID:10652141  	TAIR	2010-09-01
AT5G17020	locus:2148216	AT5G17020	acts upstream of or within	protein export from nucleus	GO:0006611	6919	P	other cellular processes	ISS	Sequence similarity (homologue of/most closely related to)	NONE	Publication:778|PMID:10652141  	TAIR	2003-05-20
AT5G17020	locus:2148216	AT5G17020	involved in	response to heat	GO:0009408	5962	P	response to abiotic stimulus	IMP	none		Publication:501736321|PMID:20345641  		2021-03-31
AT5G17020	locus:2148216	AT5G17020	involved in	response to heat	GO:0009408	5962	P	response to stress	IMP	none		Publication:501736321|PMID:20345641  		2021-03-31
AT5G17020	locus:2148216	AT5G17020	involved in	ribosomal small subunit export from nucleus	GO:0000056	7174	P	other cellular processes	IBA	none	PANTHER:PTN000131787|UniProtKB:O14980|FB:FBgn0020497	Communication:501741973		2023-01-14
AT5G17020	locus:2148216	AT5G17020	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	Co-immunoprecipitation	AGI_LocusCode:AT1G15470	Publication:501785745|PMID:31295628  	TAIR	2019-07-30
AT5G17020	locus:2148216	AT5G17020	involved in	ribosomal large subunit export from nucleus	GO:0000055	7171	P	transport	IBA	none	PANTHER:PTN000131787|UniProtKB:O14980|SGD:S000003450	Communication:501741973		2023-01-14
AT5G17020	locus:2148216	AT5G17020	is active in	cytoplasm	GO:0005737	231	C	cytoplasm	IBA	none	PANTHER:PTN000131749|UniProtKB:Q9HAV4|SGD:S000002743|UniProtKB:O14980	Communication:501741973		2023-06-07
AT5G17020	locus:2148216	AT5G17020	enables	small GTPase binding	GO:0031267	20808	F	protein binding	IEA	none	InterPro:IPR001494	AnalysisReference:501756966		2023-06-11
AT5G17020	locus:2148216	AT5G17020	acts upstream of or within	pollen tube growth	GO:0009860	10927	P	growth	IGI	double mutant analysis	AGI_LocusCode:AT3G03110	Publication:501728738|PMID:18791220  	TAIR	2008-10-13
AT5G17020	locus:2148216	AT5G17020	acts upstream of or within	pollen tube growth	GO:0009860	10927	P	reproduction	IGI	double mutant analysis	AGI_LocusCode:AT3G03110	Publication:501728738|PMID:18791220  	TAIR	2008-10-13
AT5G17020	locus:2148216	AT5G17020	acts upstream of or within	embryo sac development	GO:0009553	11320	P	multicellular organism development	IGI	double mutant analysis	AGI_LocusCode:AT3G03110	Publication:501728738|PMID:18791220  	TAIR	2008-10-13
AT5G17020	locus:2148216	AT5G17020	acts upstream of or within	pollen tube growth	GO:0009860	10927	P	anatomical structure development	IGI	double mutant analysis	AGI_LocusCode:AT3G03110	Publication:501728738|PMID:18791220  	TAIR	2008-10-13
AT5G17020	locus:2148216	AT5G17020	located in	nucleus	GO:0005634	537	C	nucleus	TAS	inferred by author, from sequence similarity	yeast CRM1	Publication:778|PMID:10652141  	TAIR	2003-10-21
AT5G17030	gene:2148230	AT5G17030.1	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G17030	locus:2148231	AT5G17030	involved in	flavonol biosynthetic process	GO:0051555	21635	P	other metabolic processes	IBA	none	PANTHER:PTN002630556|TAIR:locus:2028190	Communication:501741973		2023-01-14
AT5G17030	locus:2148231	AT5G17030	enables	quercetin 3-O-glucosyltransferase activity	GO:0080043	31694	F	transferase activity	IDA	Enzyme assays		Publication:501713256|PMID:15352060  	TAIR	2009-04-22
AT5G17030	locus:2148231	AT5G17030	enables	flavonol 3-O-arabinosyltransferase activity	GO:0080059	31831	F	transferase activity	IMP	Functional complementation		Publication:501727419|PMID:18757557  	TAIR	2009-04-22
AT5G17030	locus:2148231	AT5G17030	enables	glycosyltransferase activity	GO:0016757	14916	F	transferase activity	ISS	Recognized domains		Communication:1674999	TAIR	2004-04-21
AT5G17030	locus:2148231	AT5G17030	enables	quercetin 3-O-glucosyltransferase activity	GO:0080043	31694	F	transferase activity	IBA	none	PANTHER:PTN002630556|TAIR:locus:2028190|TAIR:locus:2148126|TAIR:locus:2148231	Communication:501741973		2023-06-07
AT5G17030	locus:2148231	AT5G17030	involved in	flavonol biosynthetic process	GO:0051555	21635	P	biosynthetic process	IBA	none	PANTHER:PTN002630556|TAIR:locus:2028190	Communication:501741973		2023-01-14
AT5G17040	gene:2148240	AT5G17040.1	located in	chloroplast	GO:0009507	175	C	chloroplast	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G17040	locus:2148241	AT5G17040	involved in	flavonol biosynthetic process	GO:0051555	21635	P	biosynthetic process	IBA	none	PANTHER:PTN002630556|TAIR:locus:2028190	Communication:501741973		2023-01-14
AT5G17040	locus:2148241	AT5G17040	enables	quercetin 3-O-glucosyltransferase activity	GO:0080043	31694	F	transferase activity	IBA	none	PANTHER:PTN002630556|TAIR:locus:2028190|TAIR:locus:2148126|TAIR:locus:2148231	Communication:501741973		2023-06-07
AT5G17040	locus:2148241	AT5G17040	involved in	flavonol biosynthetic process	GO:0051555	21635	P	other metabolic processes	IBA	none	PANTHER:PTN002630556|TAIR:locus:2028190	Communication:501741973		2023-01-14
AT5G17040	locus:2148241	AT5G17040	enables	glycosyltransferase activity	GO:0016757	14916	F	transferase activity	ISS	Recognized domains		Communication:1674999	TAIR	2004-04-21
AT5G17040	locus:2148241	AT5G17040	enables	UDP-glycosyltransferase activity	GO:0008194	1295	F	transferase activity	IEA	none	InterPro:IPR002213	AnalysisReference:501756966		2023-06-11
AT5G17050	locus:2148126	AT5G17050	enables	anthocyanidin 3-O-glucosyltransferase activity	GO:0047213	15787	F	transferase activity	IDA	protein expression in heterologous system		Publication:501715000|PMID:15807784  	TAIR	2006-04-24
AT5G17050	locus:2148126	AT5G17050	involved in	coumarin biosynthetic process	GO:0009805	5455	P	other cellular processes	IDA	in vitro assay	NONE	Publication:501800223|PMID:35996753  	TAIR	2022-09-05
AT5G17050	locus:2148126	AT5G17050	enables	myricetin 3-O-glucosyltransferase activity	GO:0102425	52463	F	transferase activity	IEA	none	EC:2.4.1.91	AnalysisReference:501756967		2023-06-11
AT5G17050	gene:2148125	AT5G17050.1	located in	chloroplast	GO:0009507	175	C	chloroplast	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G17050	locus:2148126	AT5G17050	involved in	coumarin biosynthetic process	GO:0009805	5455	P	biosynthetic process	IDA	in vitro assay	NONE	Publication:501800223|PMID:35996753  	TAIR	2022-09-05
AT5G17050	locus:2148126	AT5G17050	enables	quercetin 3-O-glucosyltransferase activity	GO:0080043	31694	F	transferase activity	IDA	Enzyme assays		Publication:501713256|PMID:15352060  	TAIR	2009-04-22
AT5G17050	locus:2148126	AT5G17050	enables	quercetin 3-O-glucosyltransferase activity	GO:0080043	31694	F	transferase activity	IBA	none	PANTHER:PTN002630556|TAIR:locus:2028190|TAIR:locus:2148126|TAIR:locus:2148231	Communication:501741973		2023-06-07
AT5G17050	locus:2148126	AT5G17050	enables	daphnetin 3-O-glucosyltransferase activity	GO:0102360	51973	F	transferase activity	IEA	none	EC:2.4.1.91	AnalysisReference:501756967		2023-06-11
AT5G17050	locus:2148126	AT5G17050	involved in	flavonol biosynthetic process	GO:0051555	21635	P	biosynthetic process	IBA	none	PANTHER:PTN002630556|TAIR:locus:2028190	Communication:501741973		2023-01-14
AT5G17050	locus:2148126	AT5G17050	involved in	coumarin biosynthetic process	GO:0009805	5455	P	other metabolic processes	IDA	in vitro assay	NONE	Publication:501800223|PMID:35996753  	TAIR	2022-09-05
AT5G17050	locus:2148126	AT5G17050	involved in	flavonol biosynthetic process	GO:0051555	21635	P	other metabolic processes	IBA	none	PANTHER:PTN002630556|TAIR:locus:2028190	Communication:501741973		2023-01-14
AT5G17050	locus:2148126	AT5G17050	enables	UDP-glucosyltransferase activity	GO:0035251	19266	F	transferase activity	IDA	Enzyme assays		Publication:1545930|PMID:11641410  	TAIR	2009-05-26
AT5G17050	locus:2148126	AT5G17050	enables	glycosyltransferase activity	GO:0016757	14916	F	transferase activity	ISS	Recognized domains		Communication:1674999	TAIR	2006-04-24
AT5G17050	locus:2148126	AT5G17050	enables	flavonol 3-O-glucosyltransferase activity	GO:0047893	16829	F	transferase activity	IEA	none	EC:2.4.1.91	AnalysisReference:501756967		2023-06-11
AT5G17050	locus:2148126	AT5G17050	involved in	coumarin biosynthetic process	GO:0009805	5455	P	secondary metabolic process	IDA	in vitro assay	NONE	Publication:501800223|PMID:35996753  	TAIR	2022-09-05
AT5G17060	locus:2148141	AT5G17060	involved in	vesicle-mediated transport	GO:0016192	7567	P	other cellular processes	IBA	none	PANTHER:PTN000918181|UniProtKB:P18085|UniProtKB:P84085|UniProtKB:P51823|TAIR:locus:2065114|SGD:S000005620|SGD:S000002351|UniProtKB:P40616|RGD:621279|SGD:S000000368|RGD:621270|FB:FBgn0010348|FB:FBgn0035866|UniProtKB:Q5B360|UniProtKB:P56559|TAIR:locus:2204848|RGD:621326|MGI:MGI:99431|UniProtKB:P62330|UniProtKB:P61204	Communication:501741973		2023-06-07
AT5G17060	locus:2148141	AT5G17060	involved in	intracellular protein transport	GO:0006886	6091	P	other cellular processes	IBA	none	PANTHER:PTN000918181|SGD:S000000368|MGI:MGI:1927136|UniProtKB:P51823|CGD:CAL0000186453|UniProtKB:Q0IIM2|WB:WBGene00000182|SGD:S000005620	Communication:501741973		2023-06-07
AT5G17060	gene:6532554513	AT5G17060.2	located in	Golgi apparatus	GO:0005794	48	C	Golgi apparatus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G17060	locus:2148141	AT5G17060	involved in	intracellular protein transport	GO:0006886	6091	P	transport	IBA	none	PANTHER:PTN000918181|SGD:S000000368|MGI:MGI:1927136|UniProtKB:P51823|CGD:CAL0000186453|UniProtKB:Q0IIM2|WB:WBGene00000182|SGD:S000005620	Communication:501741973		2023-06-07
AT5G17060	locus:2148141	AT5G17060	is active in	cytoplasm	GO:0005737	231	C	cytoplasm	IBA	none	PANTHER:PTN000917851|SGD:S000002296|UniProtKB:Q0IIM2|UniProtKB:Q585W3|WB:WBGene00000184|CGD:CAL0000196747|PomBase:SPBC4F6.18c|PomBase:SPBC1539.08|UniProtKB:P36406|UniProtKB:P49703|RGD:621326|MGI:MGI:99595|UniProtKB:P18085|UniProtKB:Q57UH8|UniProtKB:P40617|FB:FBgn0000115|UniProtKB:P84080|WB:WBGene00000182|FB:FBgn0035866|UniProtKB:Q57VI8|SGD:S000000368|MGI:MGI:99435|RGD:621279|MGI:MGI:99434|TAIR:locus:2204848|FB:FBgn0013750|UniProtKB:A0A143ZY58|UniProtKB:P62330|MGI:MGI:99431|MGI:MGI:1927136|UniProtKB:P26990|MGI:MGI:1933161|UniProtKB:P56559|FB:FBgn0010348|UniProtKB:E2RHL4|MGI:MGI:99436|RGD:621270|UniProtKB:P40616|UniProtKB:G3N3N1|CGD:CAL0000186453|TAIR:locus:2827399|dictyBase:DDB_G0289173|MGI:MGI:99433|SGD:S000002351|TAIR:locus:2065114|MGI:MGI:99432|WB:WBGene00000193	Communication:501741973		2023-06-07
AT5G17060	locus:2148141	AT5G17060	enables	GTP binding	GO:0005525	1035	F	other binding	ISS	Sequence similarity (homologue of/most closely related to)	GB:P11076	Publication:501683546|PMID:12644670  	TAIR	2005-07-28
AT5G17060	locus:2148141	AT5G17060	has protein modification of type	N-terminal protein myristoylation	GO:0006499	6894	P	other metabolic processes	IDA	in vitro assay		Publication:501710157|PMID:12912986  	TAIR	2004-02-05
AT5G17060	locus:2148141	AT5G17060	has protein modification of type	N-terminal protein myristoylation	GO:0006499	6894	P	biosynthetic process	IDA	in vitro assay		Publication:501710157|PMID:12912986  	TAIR	2004-02-05
AT5G17060	locus:2148141	AT5G17060	enables	GTP binding	GO:0005525	1035	F	nucleotide binding	IBA	none	PANTHER:PTN000917851|UniProtKB:P18085|UniProtKB:Q57UH8|RGD:621587|UniProtKB:P40617|RGD:621270|UniProtKB:P40616|CGD:CAL0000196747|FB:FBgn0000115|UniProtKB:A0A143ZY58|UniProtKB:P62330|MGI:MGI:99431|UniProtKB:P36406|dictyBase:DDB_G0289173	Communication:501741973		2023-06-07
AT5G17060	locus:2148141	AT5G17060	has protein modification of type	N-terminal protein myristoylation	GO:0006499	6894	P	other cellular processes	IDA	in vitro assay		Publication:501710157|PMID:12912986  	TAIR	2004-02-05
AT5G17060	locus:2148141	AT5G17060	enables	GTP binding	GO:0005525	1035	F	other binding	IBA	none	PANTHER:PTN000917851|UniProtKB:P18085|UniProtKB:Q57UH8|RGD:621587|UniProtKB:P40617|RGD:621270|UniProtKB:P40616|CGD:CAL0000196747|FB:FBgn0000115|UniProtKB:A0A143ZY58|UniProtKB:P62330|MGI:MGI:99431|UniProtKB:P36406|dictyBase:DDB_G0289173	Communication:501741973		2023-06-07
AT5G17060	locus:2148141	AT5G17060	enables	GTP binding	GO:0005525	1035	F	nucleotide binding	ISS	Sequence similarity (homologue of/most closely related to)	GB:P11076	Publication:501683546|PMID:12644670  	TAIR	2005-07-28
AT5G17060	locus:2148141	AT5G17060	involved in	vesicle-mediated transport	GO:0016192	7567	P	transport	IBA	none	PANTHER:PTN000918181|UniProtKB:P18085|UniProtKB:P84085|UniProtKB:P51823|TAIR:locus:2065114|SGD:S000005620|SGD:S000002351|UniProtKB:P40616|RGD:621279|SGD:S000000368|RGD:621270|FB:FBgn0010348|FB:FBgn0035866|UniProtKB:Q5B360|UniProtKB:P56559|TAIR:locus:2204848|RGD:621326|MGI:MGI:99431|UniProtKB:P62330|UniProtKB:P61204	Communication:501741973		2023-06-07
AT5G17060	locus:2148141	AT5G17060	has protein modification of type	N-terminal protein myristoylation	GO:0006499	6894	P	protein metabolic process	IDA	in vitro assay		Publication:501710157|PMID:12912986  	TAIR	2004-02-05
AT5G17060	gene:2148140	AT5G17060.1	located in	Golgi apparatus	GO:0005794	48	C	Golgi apparatus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G17070	locus:2148161	AT5G17070	acts upstream of or within	RNA metabolic process	GO:0016070	4899	P	nucleobase-containing compound metabolic process	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT2G22430|AGI_LocusCode:AT4G27410|AGI_LocusCode:AT4G34000|AGI_LocusCode:AT5G46760|AGI_LocusCode:AT5G65310|AGI_LocusCode:AT5G67300	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G17070	locus:2148161	AT5G17070	located in	cytosol	GO:0005829	241	C	cytosol	IDA	localization of GFP/YFP fusion protein		Publication:501776130|PMID:28678890  	TAIR	2017-07-14
AT5G17070	locus:2148161	AT5G17070	is active in	cytoplasm	GO:0005737	231	C	cytoplasm	IBA	none	PANTHER:PTN000420658|SGD:S000000142|MGI:MGI:3027896|TAIR:locus:2148161	Communication:501741973		2023-06-07
AT5G17070	locus:2148161	AT5G17070	part of	protein phosphatase 4 complex	GO:0030289	7988	C	other cellular components	IBA	none	PANTHER:PTN000420658|UniProtKB:Q9NY27|dictyBase:DDB_G0272668|SGD:S000000142|MGI:MGI:3027896|FB:FBgn0030208	Communication:501741973		2023-06-07
AT5G17070	gene:2148160	AT5G17070.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G17070	locus:2148161	AT5G17070	acts upstream of or within	RNA metabolic process	GO:0016070	4899	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640|AGI_LocusCode:AT2G22430|AGI_LocusCode:AT4G27410|AGI_LocusCode:AT4G34000|AGI_LocusCode:AT5G46760|AGI_LocusCode:AT5G65310|AGI_LocusCode:AT5G67300	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G17070	locus:2148161	AT5G17070	acts upstream of or within	cellular component assembly	GO:0022607	25738	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G17070	locus:2148161	AT5G17070	acts upstream of or within	macromolecule methylation	GO:0043414	21460	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT3G20740|AGI_LocusCode:AT4G02020	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G17070	locus:2148161	AT5G17070	acts upstream of or within	macromolecule methylation	GO:0043414	21460	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT3G20740|AGI_LocusCode:AT4G02020	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G17070	locus:2148161	AT5G17070	enables	protein phosphatase regulator activity	GO:0019888	9558	F	enzyme regulator activity	IBA	none	PANTHER:PTN000420658|SGD:S000000142|UniProtKB:Q9NY27	Communication:501741973		2021-11-17
AT5G17070	locus:2148161	AT5G17070	located in	nucleus	GO:0005634	537	C	nucleus	IDA	localization of GFP/YFP fusion protein		Publication:501776130|PMID:28678890  	TAIR	2017-07-14
AT5G17070	locus:2148161	AT5G17070	is active in	nucleus	GO:0005634	537	C	nucleus	IBA	none	PANTHER:PTN000420658|SGD:S000000142|MGI:MGI:3027896|TAIR:locus:2148161|UniProtKB:Q9NY27	Communication:501741973		2023-06-07
AT5G17070	locus:2148161	AT5G17070	acts upstream of or within	cellular component assembly	GO:0022607	25738	P	cellular component organization	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G17080	locus:2148181	AT5G17080	enables	cysteine-type endopeptidase activity	GO:0004197	2042	F	catalytic activity	IBA	none	PANTHER:PTN000274640|UniProtKB:Q8IIL0|UniProtKB:P25774|TAIR:locus:2204873|UniProtKB:P09668|UniProtKB:P43235|FB:FBgn0013770|MGI:MGI:107285|TAIR:locus:505006093|TAIR:locus:2133402|UniProtKB:P07858|UniProtKB:Q7F3A8|FB:FBgn0030521|UniProtKB:Q01958|FB:FBgn0250848|TAIR:locus:2157712|RGD:621509|WB:WBGene00000784|MGI:MGI:88561|MGI:MGI:107341|RGD:2448|UniProtKB:P07711	Communication:501741973		2023-06-07
AT5G17080	locus:2148181	AT5G17080	is active in	extracellular space	GO:0005615	296	C	extracellular region	IBA	none	PANTHER:PTN000274640|MGI:MGI:107823|FB:FBgn0034229|MGI:MGI:88564|UniProtKB:P07711|RGD:708479|MGI:MGI:88561|RGD:2447|WB:WBGene00000776|FB:FBgn0034709|UniProtKB:Q9GLE3|RGD:621509|WB:WBGene00000788|FB:FBgn0250848|UniProtKB:P07858|UniProtKB:P09668|UniProtKB:P43235|RGD:61810|UniProtKB:Q9UBR2|UniProtKB:P25774	Communication:501741973		2023-06-07
AT5G17080	locus:2148181	AT5G17080	is active in	lysosome	GO:0005764	442	C	lysosome	IBA	none	PANTHER:PTN000274640|UniProtKB:Q9GLE3|RGD:2445|UniProtKB:Q8I6U4|WB:WBGene00000776|MGI:MGI:88561|WB:WBGene00000786|MGI:MGI:1860262|MGI:MGI:107341|UniProtKB:A0A144A2G5|MGI:MGI:107823|MGI:MGI:88564|UniProtKB:P07711|UniProtKB:P25774|UniProtKB:Q9UBR2|FB:FBgn0013770|UniProtKB:P09668|UniProtKB:P43235|UniProtKB:P07858	Communication:501741973		2023-06-07
AT5G17080	locus:2148181	AT5G17080	involved in	proteolysis involved in protein catabolic process	GO:0051603	21764	P	protein metabolic process	IBA	none	PANTHER:PTN000274640|UniProtKB:Q7F3A8|UniProtKB:P07711|MGI:MGI:88564|MGI:MGI:107823|MGI:MGI:107341|UniProtKB:P07858|UniProtKB:P43235|MGI:MGI:88561|WB:WBGene00000784|UniProtKB:P25774	Communication:501741973		2023-06-07
AT5G17080	locus:2148181	AT5G17080	enables	cysteine-type endopeptidase activity	GO:0004197	2042	F	hydrolase activity	IBA	none	PANTHER:PTN000274640|UniProtKB:Q8IIL0|UniProtKB:P25774|TAIR:locus:2204873|UniProtKB:P09668|UniProtKB:P43235|FB:FBgn0013770|MGI:MGI:107285|TAIR:locus:505006093|TAIR:locus:2133402|UniProtKB:P07858|UniProtKB:Q7F3A8|FB:FBgn0030521|UniProtKB:Q01958|FB:FBgn0250848|TAIR:locus:2157712|RGD:621509|WB:WBGene00000784|MGI:MGI:88561|MGI:MGI:107341|RGD:2448|UniProtKB:P07711	Communication:501741973		2023-06-07
AT5G17090	locus:2148196	AT5G17090	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	none	NONE	Communication:1345790	TIGR	2003-09-27
AT5G17090	gene:2148195	AT5G17090.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G17090	locus:2148196	AT5G17090	involved in	biological_process	GO:0008150	5239	P	unknown biological processes	ND	none	NONE	Communication:1345790	TIGR	2022-02-01
AT5G17100	locus:2148211	AT5G17100	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	none	NONE	Communication:1345790	TIGR	2003-09-27
AT5G17100	gene:2148210	AT5G17100.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G17100	locus:2148211	AT5G17100	involved in	biological_process	GO:0008150	5239	P	unknown biological processes	ND	none	NONE	Communication:1345790	TIGR	2022-02-01
AT5G17110	locus:2148226	AT5G17110	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	none	NONE	Communication:1345790	TIGR	2003-09-27
AT5G17110	locus:2148226	AT5G17110	is active in	cellular_component	GO:0005575	163	C	unknown cellular components	ND	none	NONE	Communication:1345790	TIGR	2022-02-01
AT5G17110	locus:2148226	AT5G17110	involved in	biological_process	GO:0008150	5239	P	unknown biological processes	ND	none	NONE	Communication:1345790	TIGR	2022-02-01
AT5G17120	locus:2148236	AT5G17120	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	none	NONE	Communication:1345790	TIGR	2003-09-27
AT5G17120	gene:2148235	AT5G17120.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G17120	locus:2148236	AT5G17120	involved in	biological_process	GO:0008150	5239	P	unknown biological processes	ND	none	NONE	Communication:1345790	TIGR	2022-02-01
AT5G17130	locus:2148246	AT5G17130	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	Unknown molecular function		Communication:1345790	TAIR	2021-10-28
AT5G17130	locus:2148246	AT5G17130	involved in	biological_process	GO:0008150	5239	P	unknown biological processes	ND	Unknown biological process	NONE	Communication:1345790	TAIR	2022-02-01
AT5G17130	gene:2148245	AT5G17130.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G17140	locus:2148251	AT5G17140	enables	cysteine-type peptidase activity	GO:0008234	2043	F	hydrolase activity	IEA	none	InterPro:IPR000668	AnalysisReference:501756966		2023-05-23
AT5G17140	gene:2148250	AT5G17140.1	located in	chloroplast	GO:0009507	175	C	chloroplast	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G17140	locus:2148251	AT5G17140	involved in	proteolysis	GO:0006508	6934	P	protein metabolic process	IEA	none	InterPro:IPR000668	AnalysisReference:501756966		2023-05-23
AT5G17140	locus:2148251	AT5G17140	enables	cysteine-type peptidase activity	GO:0008234	2043	F	catalytic activity	IEA	none	InterPro:IPR000668	AnalysisReference:501756966		2023-05-23
AT5G17150	locus:2148156	AT5G17150	involved in	biological_process	GO:0008150	5239	P	unknown biological processes	ND	none	NONE	Communication:1345790	TIGR	2022-02-01
AT5G17150	locus:2148156	AT5G17150	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	none	NONE	Communication:1345790	TIGR	2003-09-27
AT5G17150	gene:2148155	AT5G17150.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G17160	gene:2148175	AT5G17160.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G17160	locus:2148176	AT5G17160	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	none	NONE	Communication:1345790	TIGR	2003-10-04
AT5G17160	locus:2148176	AT5G17160	acts upstream of or within	RNA metabolic process	GO:0016070	4899	P	nucleobase-containing compound metabolic process	IEA	traceable computational prediction	AGI_LocusCode:AT2G45660	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G17160	locus:2148176	AT5G17160	acts upstream of or within	RNA metabolic process	GO:0016070	4899	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT2G45660	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G17165	locus:4010713961	AT5G17165	acts upstream of or within	response to light intensity	GO:0009642	7129	P	response to light stimulus	IEA	traceable computational prediction	AGI_LocusCode:AT2G43010	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G17165	gene:4010713137	AT5G17165.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G17165	locus:4010713961	AT5G17165	acts upstream of or within	regulation of biological quality	GO:0065008	25784	P	other biological processes	IEA	traceable computational prediction	AGI_LocusCode:AT2G43010	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G17165	locus:4010713961	AT5G17165	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	Unknown molecular function		Communication:1345790	TAIR	2009-05-08
AT5G17167	locus:6532567642	AT5G17167	involved in	biological_process	GO:0008150	5239	P	unknown biological processes	ND	Unknown biological process	NONE	Communication:1345790	TAIR	2022-02-01
AT5G17167	locus:6532567642	AT5G17167	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	Unknown molecular function		Communication:1345790	TAIR	2021-10-28
AT5G17167	gene:6532554292	AT5G17167.1	located in	mitochondrion	GO:0005739	486	C	mitochondrion	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G17170	locus:2167150	AT5G17170	enables	iron ion binding	GO:0005506	2918	F	other binding	IEA	none	InterPro:IPR024934	AnalysisReference:501756966		2023-05-23
AT5G17170	locus:2167150	AT5G17170	acts upstream of or within	regulation of cellular biosynthetic process	GO:0031326	20872	P	biosynthetic process	IEA	traceable computational prediction	AGI_LocusCode:AT4G35580|AGI_LocusCode:AT5G22290	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G17170	locus:2167150	AT5G17170	acts upstream of or within	organic cyclic compound biosynthetic process	GO:1901362	43684	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT4G35580|AGI_LocusCode:AT5G22290	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G17170	locus:2167150	AT5G17170	acts upstream of or within	organic cyclic compound biosynthetic process	GO:1901362	43684	P	biosynthetic process	IEA	traceable computational prediction	AGI_LocusCode:AT4G35580|AGI_LocusCode:AT5G22290	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G17170	gene:2167149	AT5G17170.1	located in	chloroplast thylakoid membrane	GO:0009535	706	C	chloroplast	HDA	none	NONE	Publication:501711788|PMID:14729914  	kvanwijk	2021-04-12
AT5G17170	gene:2167149	AT5G17170.1	located in	chloroplast envelope	GO:0009941	12482	C	chloroplast	HDA	none	NONE	Publication:501710686|PMID:12938931  	kvanwijk	2021-04-07
AT5G17170	locus:2167150	AT5G17170	acts upstream of or within	RNA metabolic process	GO:0016070	4899	P	nucleobase-containing compound metabolic process	IEA	traceable computational prediction	AGI_LocusCode:AT4G35580|AGI_LocusCode:AT5G22290	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G17170	gene:2167149	AT5G17170.1	located in	chloroplast	GO:0009507	175	C	chloroplast	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G17170	locus:2167150	AT5G17170	located in	chloroplast thylakoid membrane	GO:0009535	706	C	plastid	HDA	none	NONE	Publication:501713005|PMID:15322131  	TAIR	2021-05-12
AT5G17170	gene:2167149	AT5G17170.1	located in	chloroplast	GO:0009507	175	C	chloroplast	HDA	none	NONE	Publication:501724486|PMID:18431481  	kvanwijk	2021-04-07
AT5G17170	locus:2167150	AT5G17170	acts upstream of or within	RNA metabolic process	GO:0016070	4899	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT4G35580|AGI_LocusCode:AT5G22290	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G17170	locus:2167150	AT5G17170	acts upstream of or within	regulation of cellular biosynthetic process	GO:0031326	20872	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT4G35580|AGI_LocusCode:AT5G22290	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G17170	locus:2167150	AT5G17170	acts upstream of or within	cellular nitrogen compound biosynthetic process	GO:0044271	19681	P	biosynthetic process	IEA	traceable computational prediction	AGI_LocusCode:AT4G35580|AGI_LocusCode:AT5G22290	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G17170	locus:2167150	AT5G17170	acts upstream of or within	cellular nitrogen compound biosynthetic process	GO:0044271	19681	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT4G35580|AGI_LocusCode:AT5G22290	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G17170	locus:2167150	AT5G17170	located in	chloroplast thylakoid membrane	GO:0009535	706	C	other membranes	HDA	none	NONE	Publication:501713005|PMID:15322131  	TAIR	2021-05-12
AT5G17170	locus:2167150	AT5G17170	acts upstream of or within	regulation of nitrogen compound metabolic process	GO:0051171	19379	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT4G35580|AGI_LocusCode:AT5G22290	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G17170	locus:2167150	AT5G17170	located in	chloroplast thylakoid membrane	GO:0009535	706	C	thylakoid	HDA	none	NONE	Publication:501713005|PMID:15322131  	TAIR	2021-05-12
AT5G17170	gene:2167149	AT5G17170.1	located in	chloroplast thylakoid	GO:0009534	704	C	thylakoid	HDA	none	NONE	Publication:501735990|PMID:20061580  	nrolland	2021-04-07
AT5G17170	locus:2167150	AT5G17170	acts upstream of or within	aromatic compound biosynthetic process	GO:0019438	10128	P	biosynthetic process	IEA	traceable computational prediction	AGI_LocusCode:AT4G35580|AGI_LocusCode:AT5G22290	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G17170	gene:2167149	AT5G17170.1	located in	chloroplast thylakoid membrane	GO:0009535	706	C	other intracellular components	HDA	none	NONE	Publication:501711788|PMID:14729914  	kvanwijk	2021-04-12
AT5G17170	locus:2167150	AT5G17170	acts upstream of or within	cellular nitrogen compound biosynthetic process	GO:0044271	19681	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT4G35580|AGI_LocusCode:AT5G22290	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G17170	gene:2167149	AT5G17170.1	located in	chloroplast thylakoid	GO:0009534	704	C	other intracellular components	HDA	none	NONE	Publication:501735990|PMID:20061580  	nrolland	2021-04-07
AT5G17170	locus:2167150	AT5G17170	acts upstream of or within	heterocycle biosynthetic process	GO:0018130	8928	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT4G35580|AGI_LocusCode:AT5G22290	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G17170	gene:2167149	AT5G17170.1	located in	chloroplast envelope	GO:0009941	12482	C	other intracellular components	HDA	none	NONE	Publication:501710686|PMID:12938931  	kvanwijk	2021-04-07
AT5G17170	locus:2167150	AT5G17170	acts upstream of or within	aromatic compound biosynthetic process	GO:0019438	10128	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT4G35580|AGI_LocusCode:AT5G22290	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G17170	gene:2167149	AT5G17170.1	located in	chloroplast thylakoid	GO:0009534	704	C	plastid	HDA	none	NONE	Publication:501735990|PMID:20061580  	nrolland	2021-04-07
AT5G17170	gene:2167149	AT5G17170.1	located in	chloroplast thylakoid membrane	GO:0009535	706	C	thylakoid	HDA	none	NONE	Publication:501711788|PMID:14729914  	kvanwijk	2021-04-12
AT5G17170	locus:2167150	AT5G17170	acts upstream of or within	heterocycle biosynthetic process	GO:0018130	8928	P	biosynthetic process	IEA	traceable computational prediction	AGI_LocusCode:AT4G35580|AGI_LocusCode:AT5G22290	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G17170	gene:4010713138	AT5G17170.2	located in	chloroplast	GO:0009507	175	C	chloroplast	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G17170	gene:2167149	AT5G17170.1	located in	chloroplast thylakoid membrane	GO:0009535	706	C	plastid	HDA	none	NONE	Publication:501711788|PMID:14729914  	kvanwijk	2021-04-12
AT5G17170	locus:2167150	AT5G17170	acts upstream of or within	regulation of cellular biosynthetic process	GO:0031326	20872	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT4G35580|AGI_LocusCode:AT5G22290	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G17170	locus:2167150	AT5G17170	acts upstream of or within	heterocycle biosynthetic process	GO:0018130	8928	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT4G35580|AGI_LocusCode:AT5G22290	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G17170	locus:2167150	AT5G17170	located in	chloroplast thylakoid membrane	GO:0009535	706	C	other intracellular components	HDA	none	NONE	Publication:501713005|PMID:15322131  	TAIR	2021-05-12
AT5G17170	gene:2167149	AT5G17170.1	located in	chloroplast thylakoid	GO:0009534	704	C	chloroplast	HDA	none	NONE	Publication:501735990|PMID:20061580  	nrolland	2021-04-07
AT5G17170	locus:2167150	AT5G17170	located in	chloroplast thylakoid membrane	GO:0009535	706	C	chloroplast	HDA	none	NONE	Publication:501713005|PMID:15322131  	TAIR	2021-05-12
AT5G17170	locus:2167150	AT5G17170	enables	mRNA binding	GO:0003729	3091	F	RNA binding	IDA	mRNA interactome capture evidence	NONE	Publication:501789490|PMID:32344669  	MarcelBachPages	2020-05-16
AT5G17170	locus:2167150	AT5G17170	acts upstream of or within	macromolecule biosynthetic process	GO:0009059	6237	P	biosynthetic process	IEA	traceable computational prediction	AGI_LocusCode:AT4G35580|AGI_LocusCode:AT5G22290	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G17170	locus:2167150	AT5G17170	acts upstream of or within	aromatic compound biosynthetic process	GO:0019438	10128	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT4G35580|AGI_LocusCode:AT5G22290	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G17170	locus:2167150	AT5G17170	acts upstream of or within	macromolecule biosynthetic process	GO:0009059	6237	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT4G35580|AGI_LocusCode:AT5G22290	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G17170	gene:4010713138	AT5G17170.2	located in	chloroplast	GO:0009507	175	C	chloroplast	HDA	none	NONE	Publication:501724486|PMID:18431481  	kvanwijk	2021-04-07
AT5G17170	gene:2167149	AT5G17170.1	located in	chloroplast thylakoid membrane	GO:0009535	706	C	other membranes	HDA	none	NONE	Publication:501711788|PMID:14729914  	kvanwijk	2021-04-12
AT5G17170	gene:2167149	AT5G17170.1	located in	chloroplast envelope	GO:0009941	12482	C	plastid	HDA	none	NONE	Publication:501710686|PMID:12938931  	kvanwijk	2021-04-07
AT5G17170	locus:2167150	AT5G17170	located in	plastid	GO:0009536	576	C	plastid	HDA	none		Publication:501776792|PMID:28887381  	dszymanski	2019-04-04
AT5G17180	locus:2167160	AT5G17180	involved in	biological_process	GO:0008150	5239	P	unknown biological processes	ND	Unknown biological process	NONE	Communication:1345790	TAIR	2022-02-01
AT5G17180	locus:2167160	AT5G17180	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	Unknown molecular function		Communication:1345790	TAIR	2009-05-08
AT5G17180	gene:2167159	AT5G17180.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G17190	locus:2167170	AT5G17190	is active in	endoplasmic reticulum membrane	GO:0005789	271	C	other membranes	IBA	none	PANTHER:PTN000294723|RGD:1302944	Communication:501741973		2021-04-02
AT5G17190	locus:2167170	AT5G17190	involved in	endoplasmic reticulum to Golgi vesicle-mediated transport	GO:0006888	4768	P	transport	IBA	none	PANTHER:PTN000294723|MGI:MGI:101917|MGI:MGI:1350933	Communication:501741973		2023-06-07
AT5G17190	gene:2167169	AT5G17190.1	located in	mitochondrion	GO:0005739	486	C	mitochondrion	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G17190	locus:2167170	AT5G17190	is active in	endoplasmic reticulum membrane	GO:0005789	271	C	other intracellular components	IBA	none	PANTHER:PTN000294723|RGD:1302944	Communication:501741973		2021-04-02
AT5G17190	locus:2167170	AT5G17190	involved in	endoplasmic reticulum to Golgi vesicle-mediated transport	GO:0006888	4768	P	other cellular processes	IBA	none	PANTHER:PTN000294723|MGI:MGI:101917|MGI:MGI:1350933	Communication:501741973		2023-06-07
AT5G17190	locus:2167170	AT5G17190	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	none	NONE	Communication:1345790	TIGR	2003-10-04
AT5G17190	locus:2167170	AT5G17190	is active in	endoplasmic reticulum membrane	GO:0005789	271	C	endoplasmic reticulum	IBA	none	PANTHER:PTN000294723|RGD:1302944	Communication:501741973		2021-04-02
AT5G17200	locus:2167185	AT5G17200	enables	polygalacturonase activity	GO:0004650	3789	F	hydrolase activity	IEA	none	InterPro:IPR000743	AnalysisReference:501756966		2023-05-23
AT5G17200	gene:2167184	AT5G17200.1	located in	extracellular region	GO:0005576	294	C	extracellular region	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G17200	locus:2167185	AT5G17200	involved in	carbohydrate metabolic process	GO:0005975	5291	P	carbohydrate metabolic process	IEA	none	InterPro:IPR000743	AnalysisReference:501756966		2023-05-23
AT5G17210	locus:2167200	AT5G17210	acts upstream of or within	response to inorganic substance	GO:0010035	14767	P	response to chemical	IEA	traceable computational prediction	AGI_LocusCode:AT5G47640|AGI_LocusCode:AT2G23150	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G17210	locus:2167200	AT5G17210	acts upstream of or within	response to water deprivation	GO:0009414	5647	P	response to stress	IEA	traceable computational prediction	AGI_LocusCode:AT5G47640	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G17210	locus:2167200	AT5G17210	acts upstream of or within	cell wall organization or biogenesis	GO:0071554	34183	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G34350|AGI_LocusCode:AT5G45750	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G17210	gene:2167199	AT5G17210.1	located in	extracellular region	GO:0005576	294	C	extracellular region	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT5G17210	locus:2167200	AT5G17210	acts upstream of or within	response to water deprivation	GO:0009414	5647	P	response to chemical	IEA	traceable computational prediction	AGI_LocusCode:AT5G47640	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G17210	locus:2167200	AT5G17210	acts upstream of or within	response to water deprivation	GO:0009414	5647	P	response to abiotic stimulus	IEA	traceable computational prediction	AGI_LocusCode:AT5G47640	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT5G17210	locus:2167200	AT5G17210	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	none	NONE	Communication:1345790	TIGR	2003-10-04
AT5G17220	locus:2167215	AT5G17220	acts upstream of or within	toxin catabolic process	GO:0009407	7444	P	catabolic process	TAS	original experiments are traceable through an article	NONE	Publication:501681713|PMID:12090627  	TAIR	2004-02-26
AT5G17220	locus:2167215	AT5G17220	acts upstream of or within	toxin catabolic process	GO:0009407	7444	P	other cellular processes	TAS	original experiments are traceable through an article	NONE	Publication:501681713|PMID:12090627  	TAIR	2004-02-26
AT5G17220	locus:2167215	AT5G17220	acts upstream of or within	toxin catabolic process	GO:0009407	7444	P	response to chemical	TAS	original experiments are traceable through an article	NONE	Publication:501681713|PMID:12090627  	TAIR	2004-02-26
AT5G17220	locus:2167215	AT5G17220	acts upstream of or within	anthocyanin-containing compound metabolic process	GO:0046283	13001	P	other metabolic processes	IMP	biochemical/chemical analysis	NONE	Publication:501746059|PMID:22201047  	TAIR	2012-04-17
AT5G17220	locus:2167215	AT5G17220	located in	plant-type vacuole membrane	GO:0009705	11040	C	other membranes	IDA	localization of GFP/YFP fusion protein		Publication:501746059|PMID:22201047  	TAIR	2012-04-17
AT5G17220	locus:2167215	AT5G17220	located in	cytoplasm	GO:0005737	231	C	cytoplasm	IDA	localization of GFP/YFP fusion protein		Publication:501746059|PMID:22201047  	TAIR	2012-04-17
AT5G17220	locus:2167215	AT5G17220	acts upstream of or within	regulation of flavonol biosynthetic process	GO:1900384	40961	P	biosynthetic process	IMP	biochemical/chemical analysis	NONE	Publication:501746059|PMID:22201047  	TAIR	2012-04-17
AT5G17220	locus:2167215	AT5G17220	involved in	glutathione metabolic process	GO:0006749	5897	P	other metabolic processes	IBA	none	PANTHER:PTN000136307|SGD:S000003983	Communication:501741973		2022-11-23
AT5G17220	locus:2167215	AT5G17220	located in	cytoplasm	GO:0005737	231	C	cytoplasm	NAS	Statements in papers that a curator cant trace to another publication</li>
</ul>
<h2 id="germplasm-phenotype">Germplasm Phenotype:</h2>
<ul>
<li><a href=""



 


>toz</a>
 &mdash; Embryo proper is unable to execute regulated divisions in reference to the choice of division plane and the timeing of the division, particularly longitudinal divisions of the plane of the apical-basal axis.</li>
<li><a href=""



 


>toz</a>
 &mdash; Homozygotes are embryo lethal, due to the abortion of toz/toz seeds.  Transmission of the mutation is not affected through the pollen, only through the ovules.  Homozygote embryos also show abnormal expression of patterning genes such as STM, MP, FIL and ANT.</li>
</ul>
<h2 id="function-related-keywords">Function-related keywords:</h2>
<ul>
<li><a href="/tags/root/"



 


>root</a>
</li>
</ul>
<h2 id="literature">Literature:</h2>
<ul>
<li><a href="https://www.doi.org/10.1104/pp.104.045179"




 target="_blank"
 


>Identification of genes required for embryo development in Arabidopsis.</a>
  DOI:  <a href="https://www.doi.org/10.1104/pp.104.045179"




 target="_blank"
 


>10.1104/pp.104.045179</a>
;  PMID:  <a href="https://pubmed.ncbi.nlm.nih.gov/15266054/"




 target="_blank"
 


>15266054</a>
</li>
<li><a href="https://www.doi.org/10.1093/pcp/pci033"




 target="_blank"
 


>Combinatorial microarray analysis revealing arabidopsis genes implicated in  cytokinin responses through the His-&gt;Asp Phosphorelay circuitry.</a>
  DOI:  <a href="https://www.doi.org/10.1093/pcp/pci033"




 target="_blank"
 


>10.1093/pcp/pci033</a>
;  PMID:  <a href="https://pubmed.ncbi.nlm.nih.gov/15695462/"




 target="_blank"
 


>15695462</a>
</li>
<li><a href="https://www.doi.org/10.1111/nph.16071"




 target="_blank"
 


>Genome-wide identification of EMBRYO-DEFECTIVE (EMB) genes required for growth  and development in Arabidopsis.</a>
  DOI:  <a href="https://www.doi.org/10.1111/nph.16071"




 target="_blank"
 


>10.1111/nph.16071</a>
;  PMID:  <a href="https://pubmed.ncbi.nlm.nih.gov/31334862/"




 target="_blank"
 


>31334862</a>
</li>
</ul>
<h2 id="sequence">Sequence:</h2>
<p>





  



<div class="accordion ">
  <div class="accordion-header " data-accordion>
    cDNA Sequence
    <svg
      class="accordion-icon"
      x="0px"
      y="0px"
      viewBox="0 0 512 512"
      xmlspace="preserve">
      <path
        fill="currentColor"
        d="M505.755,123.592c-8.341-8.341-21.824-8.341-30.165,0L256.005,343.176L36.421,123.592c-8.341-8.341-21.824-8.341-30.165,0 s-8.341,21.824,0,30.165l234.667,234.667c4.16,4.16,9.621,6.251,15.083,6.251c5.462,0,10.923-2.091,15.083-6.251l234.667-234.667 C514.096,145.416,514.096,131.933,505.755,123.592z"></path>
    </svg>
</div>


  <div class="accordion-content ">
    <div class="content-wrapper">
    <button class="copy-btn" aria-label="Copy Content">Copy</button>
    </div>
  
      <div class="text-to-copy">
    <ul>
<li>&gt;AT5G16750.1
CTTAAACCCTAATTCTTTCTCCGGCTACCGTTTTTCTTCGCGCAAAGTCCTTCAACGTTCTCTTCACCGGACGGGAAAGAGAGTTCGCCGTCGTGATGGCTCCTCATTCGTTAAAGAAGAACTACAGGTGCTCACGATCTCTAAAGCAGTTCTACGGCGGTGGTCCTTTCATTGTTTCATCTGATGGTTCTTTCATCGCCTGCGCCTGTGGTGACGTCATTAACATCGTTGATTCTACGGATTCATCAGTGAAATCCACGATTGAAGGCGAGTCGGATACGCTCACTGCTTTAGCTCTTAGTCCTGATGATAAGCTCCTTTTCTCTGCTGGACATAGTAGACAAATCCGAGTTTGGGATTTGGAAACCTTGAAATGCATTCGCTCTTGGAAGGGACACGAAGGACCTGTGATGGGAATGGCTTGCCACGCATCTGGAGGATTGCTAGCTACTGCTGGAGCTGATAGGAAAGTTCTTGTTTGGGATGTTGATGGTGGTTTCTGCACTCATTATTTCAGAGGCCATAAAGGGGTTGTGTCAAGTATCCTCTTCCATCCTGATTCAAACAAAAACATTCTTATCTCGGGAAGTGATGATGCAACCGTTCGTGTCTGGGATCTTAATGCAAAGAATACTGAGAAGAAATGTCTTGCCATTATGGAGAAGCACTTTTCAGCTGTGACTTCAATTGCTCTATCAGAGGATGGATTGACTTTATTCAGTGCTGGAAGAGATAAAGTTGTGAACTTGTGGGACCTTCATGATTATAGCTGCAAGGCTACAGTTGCAACGTATGAGGTACTAGAAGCTGTGACAACAGTTTCTTCTGGGACGCCTTTCGCTTCATTTGTAGCTTCTTTGGATCAGAAGAAGAGTAAGAAGAAGGAATCTGATTCTCAAGCAACATATTTTATTACTGTTGGTGAACGTGGTGTTGTGCGCATCTGGAAGTCTGAAGGTTCAATTTGCCTCTATGAGCAAAAGTCGTCGGATATTACTGTCAGCTCGGATGATGAGGAATCTAAAAGAGGATTCACTGCAGCTGCTATGCTGCCTTCTGATCACGGACTGCTTTGTGTGACTGCTGATCAGCAGTTCTTTTTCTACTCCGTTGTGGAAAATGTGGAAGAAACAGAGTTAGTGCTAAGCAAGAGGCTGGTTGGATATAATGAAGAAATAGCTGATATGAAGTTCCTAGGTGATGAAGAACAGTTTCTCGCAGTAGCTACAAATCTCGAGGAGGTTCGTGTATATGATGTTGCAACAATGTCATGTTCCTACGTGTTAGCTGGCCATAAGGAAGTTGTTCTGTCCCTCGACACTTGCGTATCTAGTTCTGGGAATGTTCTAATCGTAACTGGAAGTAAAGACAAAACTGTAAGGCTATGGAACGCAACAAGCAAATCTTGTATCGGAGTTGGTACTGGCCATAATGGCGATATCTTAGCGGTTGCTTTTGCGAAGAAGTCCTTCAGTTTCTTTGTCAGTGGAAGTGGTGATCGTACACTGAAGGTCTGGAGCCTTGATGGTATTTCAGAGGACTCAGAGGAGCCCATTAATCTGAAAACTAGAAGTGTTGTCGCTGCCCATGATAAAGACATCAATTCCGTGGCTGTTGCTCGTAACGATAGCTTAGTTTGCACTGGTTCTGAGGATCGTACAGCTAGCATATGGAGACTACCAGACTTGGTACATGTTGTTACACTTAAAGGACACAAGAGGCGGATTTTTTCTGTTGAGTTCTCAACTGTTGATCAGTGCGTAATGACAGCTTCAGGAGATAAGACAGTAAAGATATGGGCTATATCTGATGGTTCCTGCCTCAAAACATTTGAAGGTCATACCTCAAGTGTACTTAGGGCCTCATTTATAACTGATGGAACCCAATTTGTCTCATGTGGTGCCGATGGTTTGTTGAAACTTTGGAATGTGAACACTAGCGAGTGCATTGCCACATATGACCAGCACGAGGACAAGGTATGGGCTTTAGCTGTTGGAAAGAAGACAGAAATGATTGCAACTGGTGGAGGTGATGCAGTTATTAATTTATGGCATGATTCAACCGCTTCAGATAAAGAAGACGATTTCCGAAAAGAGGAAGAAGCAATCTTGAGAGGTCAAGAGCTGGAAAACGCAGTGTTAGATGCGGAATACACTAAAGCTATAAGATTAGCATTTGAGCTTTGTAGGCCTCACAAAGTTTTTGAGCTCTTCAGTGGTCTCTGCAGGAAAAGAGATTCAGACGAACAGATAGTGAAAGCTCTTCAAGGACTTGAAAAAGAAGAGTTCCGTCTACTTTTTGAATATGTCCGAGAATGGAACACAAAACCAAAGCTATGCCACATTGCTCAGTTTGTTCTTTACAAAACCTTCAACATACTTCCTCCCACAGAAATTGTTCAGGTAAAAGGAATTGGGGAACTCCTGGAAGGGTTAATCCCATATAGTCAGAGACATTTCAGTAGAATTGACAGATTTGTAAGAAGCAGTTTCTTGTTAGACTACACACTTGGTGAGATGTCTGTGATCGATCCAGAGACTGTTGAGACAGAGTATCCTAAGGACGAGAAGAAAAAAGAAAAAGATGTCATTGCTGCAATGGAACAAGACACAGACGAGCTAAAGCAGGAGACTCCTTCGAGGAAACGAAAGTCTCAGAAATCCAAAGGTAAATCGAATAAGAAGAGACTCATTGCCGAAGCACAGGGAAGCGTAATTGCAGTTTGAGTCTTGAAATACTTCATCAATACCATGGCATGAATTATAAATTTGATTGAGATCTGAGGGGTGGAATCCCAATTTTTGGTTATTCTCGTTTTCTTGTGTAATTGCCTAAATTGGTTGTTGTAAGTGCTGTTAGTTTTTAAATTTTTGTTGAGATTTAAAAAAGTATTTCGTAGCTGAAAAAGATCTATTCACCTTTTTGTTGTTTCTAGATTTTTTTTTGGGTTTTTGGTTTTACGTCAGCGTATATTAGCGTCATGGGAACGAACATGATTAGTAACGTAAAGTCGACCAAATTAGTA</li>
</ul>

      </div>
  </div>
</div>








  



<div class="accordion ">
  <div class="accordion-header " data-accordion>
    CDS Sequence
    <svg
      class="accordion-icon"
      x="0px"
      y="0px"
      viewBox="0 0 512 512"
      xmlspace="preserve">
      <path
        fill="currentColor"
        d="M505.755,123.592c-8.341-8.341-21.824-8.341-30.165,0L256.005,343.176L36.421,123.592c-8.341-8.341-21.824-8.341-30.165,0 s-8.341,21.824,0,30.165l234.667,234.667c4.16,4.16,9.621,6.251,15.083,6.251c5.462,0,10.923-2.091,15.083-6.251l234.667-234.667 C514.096,145.416,514.096,131.933,505.755,123.592z"></path>
    </svg>
</div>


  <div class="accordion-content ">
    <div class="content-wrapper">
    <button class="copy-btn" aria-label="Copy Content">Copy</button>
    </div>
  
      <div class="text-to-copy">
    
      </div>
  </div>
</div>








  



<div class="accordion ">
  <div class="accordion-header " data-accordion>
    Protein Sequence
    <svg
      class="accordion-icon"
      x="0px"
      y="0px"
      viewBox="0 0 512 512"
      xmlspace="preserve">
      <path
        fill="currentColor"
        d="M505.755,123.592c-8.341-8.341-21.824-8.341-30.165,0L256.005,343.176L36.421,123.592c-8.341-8.341-21.824-8.341-30.165,0 s-8.341,21.824,0,30.165l234.667,234.667c4.16,4.16,9.621,6.251,15.083,6.251c5.462,0,10.923-2.091,15.083-6.251l234.667-234.667 C514.096,145.416,514.096,131.933,505.755,123.592z"></path>
    </svg>
</div>


  <div class="accordion-content ">
    <div class="content-wrapper">
    <button class="copy-btn" aria-label="Copy Content">Copy</button>
    </div>
  
      <div class="text-to-copy">
    
      </div>
  </div>
</div>

</p>

          </div>
        </div>
      </div>
    </div>
  </section>

    </main>
  
    
    
      <footer class="bg-theme-light dark:bg-darkmode-theme-light">
  <div class="container">
    <div class="row items-center py-0">
      <div class="lg:col-6 mb-8 text-center lg:mb-0">
        <ul>
          
        </ul>
      </div>
      <div class="lg:col-3 mb-8 text-center lg:mb-0 lg:mt-0 lg:text-right">
        <ul class="social-icons">
          
        </ul>
      </div>
    </div>
  </div>
  <div class="border-border dark:border-darkmode-border border-t py-7">
    <div class="text-light dark:text-darkmode-light container text-center">
      <p>
        Designed &amp; Developed by <a href="https://gohugo.io/"




 target="_blank"
 


>Hugo</a>
 and <a href="https://www.netlify.com/"




 target="_blank"
 


>netlify</a>
 | 2024 Jinbiao Cao (曹金彪)
      </p>
    </div>
  </div>
</footer>
      



  
    
      
    
  

  
    
      
    
  

  
    
      
    
  

  
    
      
    
  

  
    
      
    
  

  
    
      
    
  

  
    
      
    
  

  
    
      
    
  

  
    
      
    
  










  
  



<script
  crossorigin="anonymous"
  integrity="sha256-qB/nHiPX28YVBLSaTV&#43;c5NTxlhkrSM0wXi7NrqdMnrw="
  src="/js/script.min.a81fe71e23d7dbc61504b49a4d5f9ce4d4f196192b48cd305e2ecdaea74c9ebc.js"></script>


<script
  defer
  async
  crossorigin="anonymous"
  integrity="sha256-9e9iyznGCSfFZB4ElW&#43;WFonrJpHJr5pYxsaD7VVh4wc="
  src="/js/script-lazy.min.f5ef62cb39c60927c5641e04956f961689eb2691c9af9a58c6c683ed5561e307.js"></script>



<script>
  if ('serviceWorker' in navigator){navigator.serviceWorker.register("/service-worker.js");}
</script>






  








    
  </body>
  
</html>
