<!doctype html>
<html
  itemscope
  class=""
  lang="en-us"
  itemtype="http://schema.org/WebPage">
  <head>
    
    <meta charset="utf-8" />


<meta
  name="viewport"
  content="width=device-width, initial-scale=1, maximum-scale=5" />


<meta name="theme-name" content="hugoplate" />















<link rel="shortcut icon" href="/images/fpg_logo_hu8359c659081f4f0de026e43505d3fe2b_71406_96x0_resize_lanczos_3.png" type="image/x-icon">
<link rel="icon" href="/images/fpg_logo_hu8359c659081f4f0de026e43505d3fe2b_71406_96x0_resize_lanczos_3.png" type="image/x-icon">
<link rel="icon" type="image/png" sizes="48x48" href="/images/fpg_logo_hu8359c659081f4f0de026e43505d3fe2b_71406_48x0_resize_lanczos_3.png">
<link rel="icon" type="image/png" sizes="96x96" href="/images/fpg_logo_hu8359c659081f4f0de026e43505d3fe2b_71406_96x0_resize_lanczos_3.png">
<link rel="apple-touch-icon" sizes="144x144" href="/images/fpg_logo_hu8359c659081f4f0de026e43505d3fe2b_71406_144x0_resize_lanczos_3.png">








<link rel="manifest" href="/manifest.webmanifest" />
<meta
  name="msapplication-TileColor"
  content="#ddd" />
<meta
  name="theme-color"
  content="#ffffff" />





























  


<title>AT3G45760</title>















  <meta
    name="keywords"
    content="Boilerplate, Hugo" />




<meta
  name="description"
  content="AT3G45760 ; NTP5 ; Arabidopsis thaliana" />



  <meta name="author" content="zeon.studio" />






  
  












  

  
  
  


  
  


  





<meta property="og:title" content="AT3G45760" />
<meta property="og:description" content="AT3G45760 ; NTP5 ; Arabidopsis thaliana" />
<meta property="og:type" content="website" />
<meta property="og:url" content="/genes/arabidopsis_thaliana/at3g45760/" />


<meta name="twitter:title" content="AT3G45760" />
<meta name="twitter:description" content="AT3G45760 ; NTP5 ; Arabidopsis thaliana" />





















<script>
  let indexURL = "/searchindex.json";
  let includeSectionsInSearch = ["genes","pages","sections","submit"];
  let no_results_for = "No results for";
  let empty_search_results_placeholder = "Type something to search..";
</script>




















    
    
<meta http-equiv="x-dns-prefetch-control" content="on" />
<link rel="preconnect" href="https://use.fontawesome.com" crossorigin />
<link rel="preconnect" href="//cdnjs.cloudflare.com" />
<link rel="preconnect" href="//www.googletagmanager.com" />
<link rel="preconnect" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="https://use.fontawesome.com" />
<link rel="dns-prefetch" href="//ajax.googleapis.com" />
<link rel="dns-prefetch" href="//cdnjs.cloudflare.com" />
<link rel="dns-prefetch" href="//www.googletagmanager.com" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//connect.facebook.net" />
<link rel="dns-prefetch" href="//platform.linkedin.com" />
<link rel="dns-prefetch" href="//platform.twitter.com" />




<link rel="preconnect" href="https://fonts.googleapis.com" />
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
<script>
  (function () {
    const googleFont = document.createElement("link");
    googleFont.href = "https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Montserrat:wght@500;700&display=swap";
    googleFont.type = "text/css";
    googleFont.rel = "stylesheet";
    document.head.appendChild(googleFont);
  })();
</script>





  
    
      
    
  

  
    
      
    
  

  
    
      
    
  

  
    
      
    
  

  
    
      
    
  











  
  



<link
  href="/css/style.min.ad82ade437df81f5c5a48af63c5d6c5da442a19f83f854c3b4168220f83e4f5f.css"
  integrity="sha256-rYKt5DffgfXFpIr2PF1sXaRCoZ+D+FTDtBaCIPg+T18="
  rel="stylesheet" />


<link
  defer
  async
  rel="stylesheet"
  href="/css/style-lazy.min.901e5cbd6a6883c52a645fd5c91bb3fec340173bc306529fe6cb027c818eb6bf.css"
  integrity="sha256-kB5cvWpog8UqZF/VyRuz/sNAFzvDBlKf5ssCfIGOtr8="
  media="print"
  onload="this.media='all'; this.onload=null;" />

  </head>

  <body style="padding-top: 2px; padding-bottom: 2px;">
    
    
      



      

    
  
    
    <header
  class="header sticky top-0 z-30">
  <nav class="navbar container">
    
    <div class="order-0">
      
      <a class="navbar-brand block" href="/">
        






















  
  funPlantGenes


      </a>
    </div>
    
    <input id="nav-toggle" type="checkbox" class="hidden" />
    <label
      for="nav-toggle"
      class="order-3 cursor-pointer flex items-center lg:hidden text-dark dark:text-white lg:order-1">
      <svg id="show-button" class="h-6 fill-current block" viewBox="0 0 20 20">
        <title>Menu Open</title>
        <path d="M0 3h20v2H0V3z m0 6h20v2H0V9z m0 6h20v2H0V0z"></path>
      </svg>
      <svg id="hide-button" class="h-6 fill-current hidden" viewBox="0 0 20 20">
        <title>Menu Close</title>
        <polygon
          points="11 9 22 9 22 11 11 11 11 22 9 22 9 11 -2 11 -2 9 9 9 9 -2 11 -2"
          transform="rotate(45 10 10)"></polygon>
      </svg>
    </label>
    

    
    <ul
      id="nav-menu"
      class="navbar-nav order-3 hidden lg:flex w-full pb-6 lg:order-1 lg:w-auto lg:space-x-2 lg:pb-0 xl:space-x-8">
      
      
        
        
        
        
          <li class="nav-item">
            <a
              class="nav-link "
              
              href="/species/"
              >SPECIES</a
            >
          </li>
        
      
        
        
        
        
          <li class="nav-item">
            <a
              class="nav-link "
              
              href="/genes/"
              >GENES</a
            >
          </li>
        
      
        
        
        
        
          <li class="nav-item">
            <a
              class="nav-link "
              
              href="/tags/"
              >FUNCTION</a
            >
          </li>
        
      
        
        
        
        
          <li class="nav-item nav-dropdown group relative">
            <span
              class="nav-link 
                
                
                
              
                
                
                
               inline-flex items-center">
              HELP
              <svg class="h-4 w-4 fill-current" viewBox="0 0 20 20">
                <path
                  d="M9.293 12.95l.707.707L15.657 8l-1.414-1.414L10 10.828 5.757 6.586 4.343 8z" />
              </svg>
            </span>
            <ul
              class="nav-dropdown-list lg:group-hover:visible lg:group-hover:opacity-100">
              
                
                
                <li class="nav-dropdown-item">
                  <a
                    class="nav-dropdown-link "
                    
                    href="/link/">
                    LINKS
                  </a>
                </li>
              
                
                
                <li class="nav-dropdown-item">
                  <a
                    class="nav-dropdown-link "
                    
                    href="/submit/">
                    SUBMIT
                  </a>
                </li>
              
            </ul>
          </li>
        
      
      
        <li class="mt-4 inline-block lg:hidden">
          <a
            class="btn btn-outline-primary btn-sm"
            href="/submit">
            Submit
          </a>
        </li>
      
    </ul>

    
     

  <div class="order-1 ml-auto flex items-center md:order-2 lg:ml-0" style="margin-right: 20px;">

    <input
  id="search-modal-input"
  type="text"
  placeholder="Search"
  autocomplete="on"
  aria-label="Search"
  aria-describedby="Search"
  placeholder="Search Post ..."
  style="width: 120px; /* 设置宽度为800像素 */
         padding: 5px;
         border: 2px solid #ccc;
         border-radius: 20px;
         box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
         outline: none;
         transition: border-color 0.3s ease-in-out;
         margin-right: 20px; /* 添加右边距 */"/>
    <button
        id="search-button"
        aria-label="search"
        class="border-border text-dark hover:text-primary dark:border-darkmode-border mr-5 inline-block border-r pr-5 text-xl dark:text-white dark:hover:text-darkmode-primary"
        data-target="search-modal"
        onclick="search()">
        <i class="fa-solid fa-search"></i>
    </button>

   
   </div>

    <div class="order-1 ml-auto flex items-center md:order-2 lg:ml-0" style="margin-right: 20px;"></div>
          









          


  <div class="theme-switcher mr-5">
    <input id="theme-switcher" data-theme-switcher type="checkbox" />
    <label for="theme-switcher">
      <span class="sr-only">theme switcher</span>
      <span>
        
        <svg
          class="absolute left-[4px] top-[4px] z-10 opacity-100 dark:opacity-0"
          viewBox="0 0 56 56"
          fill="#fff"
          height="16"
          width="16">
          <path
            d="M30 4.6c0-1-.9-2-2-2a2 2 0 0 0-2 2v5c0 1 .9 2 2 2s2-1 2-2Zm9.6 9a2 2 0 0 0 0 2.8c.8.8 2 .8 2.9 0L46 13a2 2 0 0 0 0-2.9 2 2 0 0 0-3 0Zm-26 2.8c.7.8 2 .8 2.8 0 .8-.7.8-2 0-2.9L13 10c-.7-.7-2-.8-2.9 0-.7.8-.7 2.1 0 3ZM28 16a12 12 0 0 0-12 12 12 12 0 0 0 12 12 12 12 0 0 0 12-12 12 12 0 0 0-12-12Zm23.3 14c1.1 0 2-.9 2-2s-.9-2-2-2h-4.9a2 2 0 0 0-2 2c0 1.1 1 2 2 2ZM4.7 26a2 2 0 0 0-2 2c0 1.1.9 2 2 2h4.9c1 0 2-.9 2-2s-1-2-2-2Zm37.8 13.6a2 2 0 0 0-3 0 2 2 0 0 0 0 2.9l3.6 3.5a2 2 0 0 0 2.9 0c.8-.8.8-2.1 0-3ZM10 43.1a2 2 0 0 0 0 2.9c.8.7 2.1.8 3 0l3.4-3.5c.8-.8.8-2.1 0-2.9-.8-.8-2-.8-2.9 0Zm20 3.4c0-1.1-.9-2-2-2a2 2 0 0 0-2 2v4.9c0 1 .9 2 2 2s2-1 2-2Z" />
        </svg>
        
        <svg
          class="absolute left-[4px] top-[4px] z-10 opacity-0 dark:opacity-100"
          viewBox="0 0 24 24"
          fill="none"
          height="16"
          width="16">
          <path
            fill="#000"
            fill-rule="evenodd"
            clip-rule="evenodd"
            d="M8.2 2.2c1-.4 2 .6 1.6 1.5-1 3-.4 6.4 1.8 8.7a8.4 8.4 0 0 0 8.7 1.8c1-.3 2 .5 1.5 1.5v.1a10.3 10.3 0 0 1-9.4 6.2A10.3 10.3 0 0 1 3.2 6.7c1-2 2.9-3.5 4.9-4.4Z" />
        </svg>
      </span>
    </label>
  </div>

  
  <script>
    var darkMode = false;

    
    if (window.matchMedia("(prefers-color-scheme: dark)").matches){darkMode = true}
    

    if (localStorage.getItem("theme") === "dark"){darkMode = true}
    else if (localStorage.getItem("theme") === "light"){darkMode = false}
    if (darkMode){document.documentElement.classList.toggle("dark")}
    var themeSwitch = document.querySelectorAll("[data-theme-switcher]");

    document.addEventListener("DOMContentLoaded", () => {
      [].forEach.call(themeSwitch, function (ts) {
        ts.checked = darkMode ? true : false;
        ts.addEventListener("click", () => {
          document.documentElement.classList.toggle("dark");
          localStorage.setItem(
            "theme",
            document.documentElement.classList.contains("dark") ? "dark" : "light"
          );
        });
      });
    });
  </script>


    </div>

  </nav>
       
</header>

    




  
    <div
      class="search-modal "
      aria-hidden="true"
      style="--color-primary: #121212">
      <div data-target="close-search-modal" class="search-modal-overlay"></div>
      <div
        class="search-wrapper"
        #data-image="true"
        data-description="true"
        data-tags="true"
        data-categories="true">
        <div class="search-wrapper-header">
          <label for="search-modal-input" style="margin-top:-1px">
            <span class="sr-only">search icon</span>
          </label>
        </div>   
        <div class="search-wrapper-body">
          <div class="search-result" data-search-result></div>
          <span class="search-result-empty">
            Type something to search..
          </span>
        </div>
        <div class="search-wrapper-footer">
          <span>
            <kbd>
              <svg
                xmlns="http://www.w3.org/2000/svg"
                width="14"
                height="14"
                fill="currentColor"
                viewBox="0 0 16 16">
                <path
                  d="M3.204 11h9.592L8 5.519 3.204 11zm-.753-.659 4.796-5.48a1 1 0 0 1 1.506 0l4.796 5.48c.566.647.106 1.659-.753 1.659H3.204a1 1 0 0 1-.753-1.659z" />
              </svg>
            </kbd>
            <kbd>
              <svg
                xmlns="http://www.w3.org/2000/svg"
                width="14"
                height="14"
                fill="currentColor"
                style="margin-top:1px;"
                viewBox="0 0 16 16">
                <path
                  d="M3.204 5h9.592L8 10.481 3.204 5zm-.753.659 4.796 5.48a1 1 0 0 0 1.506 0l4.796-5.48c.566-.647.106-1.659-.753-1.659H3.204a1 1 0 0 0-.753 1.659z" />
              </svg>
            </kbd>
            to navigate
          </span>
          <span>
            <kbd>
              <svg
                xmlns="http://www.w3.org/2000/svg"
                width="12"
                height="12"
                fill="currentColor"
                style="display:inline-block;"
                viewBox="0 0 16 16">
                <path
                  fill-rule="evenodd"
                  d="M14.5 1.5a.5.5 0 0 1 .5.5v4.8a2.5 2.5 0 0 1-2.5 2.5H2.707l3.347 3.346a.5.5 0 0 1-.708.708l-4.2-4.2a.5.5 0 0 1 0-.708l4-4a.5.5 0 1 1 .708.708L2.707 8.3H12.5A1.5 1.5 0 0 0 14 6.8V2a.5.5 0 0 1 .5-.5z" />
              </svg>
            </kbd>
            to select
          </span>
          <span class="search-result-info"></span>
          <span data-target="close-search-modal">
            <kbd>ESC</kbd> to close
          </span>
        </div>
      </div>
    </div>
  


  
    <main>
      
  <section class="section-sm" style="padding-top: 15px; padding-bottom: 15px;">
    <div class="container">
      <div class="row justify-center">
        <div class="lg:col-11">
          <div class="content">
            <h2 id="gene-details">Gene Details:</h2>
<ul>
<li><strong>Gene ID:</strong> <a href="https://www.arabidopsis.org/locus?name=AT3G45760"




 target="_blank"
 


>AT3G45760</a>
</li>
<li><strong>Gene Symbol:</strong> <u>NTP5</u></li>
<li><strong>Gene Name:</strong></li>
<li><strong>Description:</strong>   Nucleotidyltransferase family protein;(source:Araport11)</li>
<li><strong>TAIR Accession:</strong> locus:2102714</li>
<li><strong>Genome:</strong> <a href="https://www.arabidopsis.org/download/list?dir=Genes%2FAraport11_genome_release"




 target="_blank"
 


>Araport11_genome_release</a>
</li>
<li><strong>Species:</strong> <em>Arabidopsis thaliana</em></li>
</ul>
<h2 id="transcripts">Transcripts:</h2>
<ul>
<li><a href="https://www.arabidopsis.org/gene?name=AT3G45760.1"




 target="_blank"
 


>AT3G45760.1</a>
    <a href="https://www.arabidopsis.org/gene?name=AT3G45760.2"




 target="_blank"
 


>AT3G45760.2</a>
</li>
</ul>
<h2 id="gene-ontology">Gene Ontology:</h2>
<ul>
<li><a href="https://amigo.geneontology.org/amigo/term/GO:0005737"




 target="_blank"
 


>GO:0005737</a>
 &mdash; located in &mdash; cytoplasm</li>
<li><a href="https://amigo.geneontology.org/amigo/term/GO:0071051"




 target="_blank"
 


>GO:0071051</a>
 &mdash; involved in &mdash; RNA 3-end processing	GO:0031123	19783	P	other metabolic processes	IBA	none	PANTHER:PTN000263330|UniProtKB:Q5TAX3|UniProtKB:Q9NVV4|MGI:MGI:2387179|UniProtKB:Q5VYS8|UniProtKB:Q9H6E5|TAIR:locus:2063937|UniProtKB:Q57UM5|FB:FBgn0037470|PomBase:SPAC19D5.03|WB:WBGene00014124|MGI:MGI:2445126|FB:FBgn0260780|TAIR:locus:2043669|WB:WBGene00019629|UniProtKB:F1QDP4	Communication:501741973		2023-06-07
AT3G45760	locus:2102714	AT3G45760	enables	nucleotidyltransferase activity	GO:0016779	3397	F	transferase activity	IBA	none	PANTHER:PTN000263330|TAIR:locus:2043669|MGI:MGI:2445126|FB:FBgn0038934|PomBase:SPAC821.04c|UniProtKB:F1QDP4|FB:FBgn0037470|UniProtKB:Q6PIY7|UniProtKB:Q5VYS8|FB:FBgn0024360|MGI:MGI:2387179|FB:FBgn0260780|WB:WBGene00019629|UniProtKB:Q57UM5|TAIR:locus:2063937|WB:WBGene00014124|PomBase:SPAC19D5.03|PomBase:SPAC17H9.01|WB:WBGene00001596|UniProtKB:Q9H6E5|WB:WBGene00019628|UniProtKB:Q9NVV4|UniProtKB:Q5TAX3	Communication:501741973		2023-06-07
AT3G45770	locus:2102664	AT3G45770	involved in	fatty acid biosynthetic process	GO:0006633	5751	P	other cellular processes	IGI	Functional complementation in heterologous system	SGD:S000000230	Publication:501789055|PMID:32245791  	TAIR	2021-08-06
AT3G45770	locus:2102664	AT3G45770	involved in	fatty acid metabolic process	GO:0006631	5754	P	lipid metabolic process	IBA	none	PANTHER:PTN000190628|UniProtKB:Q7YS70|TAIR:locus:2102664|SGD:S000000230|UniProtKB:Q9BV79	Communication:501741973		2023-06-07
AT3G45770	locus:2102664	AT3G45770	involved in	fatty acid biosynthetic process	GO:0006633	5751	P	other metabolic processes	IGI	Functional complementation in heterologous system	SGD:S000000230	Publication:501789055|PMID:32245791  	TAIR	2021-08-06
AT3G45770	gene:3698249	AT3G45770.1	located in	chloroplast	GO:0009507	175	C	chloroplast	HDA	none	NONE	Publication:501724486|PMID:18431481  	kvanwijk	2021-04-07
AT3G45770	gene:3698249	AT3G45770.1	located in	mitochondrion	GO:0005739	486	C	mitochondrion	HDA	none	NONE	Publication:501720283|PMID:17137349  	kvanwijk	2021-05-12
AT3G45770	locus:2102664	AT3G45770	located in	mitochondrion	GO:0005739	486	C	mitochondrion	HDA	none	NONE	Publication:501711651|PMID:14671022  	hmillar	2021-04-15
AT3G45770	locus:2102664	AT3G45770	located in	cytosol	GO:0005829	241	C	cytosol	HDA	none		Publication:501776792|PMID:28887381  	dszymanski	2019-04-04
AT3G45770	locus:2102664	AT3G45770	enables	trans-2-enoyl-CoA reductase (NADPH) activity	GO:0019166	8434	F	catalytic activity	IBA	none	PANTHER:PTN000190628|UniProtKB:Q7YS70|UniProtKB:Q9BV79	Communication:501741973		2023-06-07
AT3G45770	locus:2102664	AT3G45770	enables	copper ion binding	GO:0005507	1990	F	other binding	HDA	none	NONE	Publication:501735675|PMID:20018591  	TAIR	2021-05-10
AT3G45770	gene:3698249	AT3G45770.1	located in	nucleus	GO:0005634	537	C	nucleus	HDA	none	NONE	Publication:501711432|PMID:14617066  	kvanwijk	2021-04-22
AT3G45770	gene:3698249	AT3G45770.1	enables	ATP binding	GO:0005524	894	F	nucleotide binding	HDA	none	NONE	Publication:501720283|PMID:17137349  	kvanwijk	2021-05-12
AT3G45770	locus:2102664	AT3G45770	involved in	fatty acid metabolic process	GO:0006631	5754	P	other metabolic processes	IBA	none	PANTHER:PTN000190628|UniProtKB:Q7YS70|TAIR:locus:2102664|SGD:S000000230|UniProtKB:Q9BV79	Communication:501741973		2023-06-07
AT3G45770	locus:2102664	AT3G45770	involved in	fatty acid biosynthetic process	GO:0006633	5751	P	biosynthetic process	IGI	Functional complementation in heterologous system	SGD:S000000230	Publication:501789055|PMID:32245791  	TAIR	2021-08-06
AT3G45770	gene:1006228440	AT3G45770.2	enables	ATP binding	GO:0005524	894	F	nucleotide binding	HDA	none	NONE	Publication:501720283|PMID:17137349  	kvanwijk	2021-05-12
AT3G45770	gene:1006228440	AT3G45770.2	located in	mitochondrion	GO:0005739	486	C	mitochondrion	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G45770	gene:1006228440	AT3G45770.2	located in	mitochondrion	GO:0005739	486	C	mitochondrion	HDA	none	NONE	Publication:501720283|PMID:17137349  	kvanwijk	2021-05-12
AT3G45770	gene:1006228440	AT3G45770.2	located in	chloroplast	GO:0009507	175	C	chloroplast	HDA	none	NONE	Publication:501724486|PMID:18431481  	kvanwijk	2021-04-07
AT3G45770	gene:3698249	AT3G45770.1	located in	chloroplast	GO:0009507	175	C	chloroplast	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G45770	locus:2102664	AT3G45770	involved in	fatty acid biosynthetic process	GO:0006633	5751	P	lipid metabolic process	IGI	Functional complementation in heterologous system	SGD:S000000230	Publication:501789055|PMID:32245791  	TAIR	2021-08-06
AT3G45770	locus:2102664	AT3G45770	enables	enoyl-[acyl-carrier-protein] reductase activity	GO:0016631	2257	F	catalytic activity	IGI	Functional complementation in heterologous system	SGD:S000000230	Publication:501789055|PMID:32245791  	TAIR	2021-08-06
AT3G45770	locus:2102664	AT3G45770	enables	oxidoreductase activity	GO:0016491	3460	F	catalytic activity	IEA	none	InterPro:IPR020843	AnalysisReference:501756966		2023-06-11
AT3G45770	locus:2102664	AT3G45770	involved in	fatty acid metabolic process	GO:0006631	5754	P	other cellular processes	IBA	none	PANTHER:PTN000190628|UniProtKB:Q7YS70|TAIR:locus:2102664|SGD:S000000230|UniProtKB:Q9BV79	Communication:501741973		2023-06-07
AT3G45770	locus:2102664	AT3G45770	is active in	mitochondrion	GO:0005739	486	C	mitochondrion	IBA	none	PANTHER:PTN000190628|UniProtKB:Q9BV79|RGD:3208|SGD:S000000230|UniProtKB:Q57TX8	Communication:501741973		2023-06-07
AT3G45770	gene:3698249	AT3G45770.1	enables	ATP binding	GO:0005524	894	F	other binding	HDA	none	NONE	Publication:501720283|PMID:17137349  	kvanwijk	2021-05-12
AT3G45770	gene:1006228440	AT3G45770.2	enables	ATP binding	GO:0005524	894	F	other binding	HDA	none	NONE	Publication:501720283|PMID:17137349  	kvanwijk	2021-05-12
AT3G45780	locus:2102674	AT3G45780	acts upstream of or within	blue light signaling pathway	GO:0009785	11759	P	response to light stimulus	IDA	none		Publication:1999|PMID:9831559   		2021-04-01
AT3G45780	locus:2102674	AT3G45780	is active in	plasma membrane	GO:0005886	570	C	plasma membrane	IBA	none	PANTHER:PTN000682243|TAIR:locus:2008331|TAIR:locus:2178388|SGD:S000005330|TAIR:locus:2102674|TAIR:locus:2091443	Communication:501741973		2023-06-07
AT3G45780	locus:2102674	AT3G45780	enables	FMN binding	GO:0010181	17729	F	other binding	TAS	original experiments are traceable through an article	NONE	Publication:501717658|PMID:16150710  	TAIR	2006-10-04
AT3G45780	locus:2102674	AT3G45780	acts upstream of or within	protein autophosphorylation	GO:0046777	13750	P	other metabolic processes	TAS	original experiments are traceable through an article	NONE	Publication:501682623|PMID:12383086  	TAIR	2005-11-01
AT3G45780	locus:2102674	AT3G45780	acts upstream of or within	blue light signaling pathway	GO:0009785	11759	P	response to abiotic stimulus	IDA	none		Publication:1999|PMID:9831559   		2021-04-01
AT3G45780	locus:2102674	AT3G45780	acts upstream of or within	regulation of stomatal movement	GO:0010119	13863	P	other cellular processes	IMP	analysis of physiological response	NONE	Publication:1546234|PMID:11740564  	TAIR	2003-08-05
AT3G45780	locus:2102674	AT3G45780	is active in	cytoplasm	GO:0005737	231	C	cytoplasm	IBA	none	PANTHER:PTN000682243|TAIR:locus:2102674|TAIR:locus:2155821|TAIR:locus:2085191|SGD:S000005330|TAIR:locus:2008331	Communication:501741973		2023-06-07
AT3G45780	locus:2102674	AT3G45780	acts upstream of or within	regulation of proton transport	GO:0010155	17246	P	other cellular processes	IGI	double mutant analysis	AGI_LocusCode:AT5G58140	Publication:501716548|PMID:15821287  	TAIR	2008-05-24
AT3G45780	locus:2102674	AT3G45780	involved in	blue light signaling pathway	GO:0009785	11759	P	signal transduction	IMP	biochemical/chemical analysis		Publication:501720921|PMID:17319842  	TAIR	2023-02-21
AT3G45780	gene:3698263	AT3G45780.1	located in	plant-type vacuole	GO:0000325	11545	C	vacuole	HDA	none	NONE	Publication:501714874|PMID:15539469  	kvanwijk	2021-05-12
AT3G45780	locus:2102674	AT3G45780	enables	kinase activity	GO:0016301	2953	F	kinase activity	ISS	Recognized domains		Publication:501710530|PMID:13678909  	TAIR	2004-02-26
AT3G45780	locus:2102674	AT3G45780	located in	cytoplasm	GO:0005737	231	C	cytoplasm	IDA	localization of GFP/YFP fusion protein		Publication:501681826|PMID:12172018  	TAIR	2003-08-05
AT3G45780	gene:1009021722	AT3G45780.2	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G45780	locus:2102674	AT3G45780	acts upstream of or within	circadian rhythm	GO:0007623	5410	P	circadian rhythm	IMP	analysis of physiological response	NONE	Publication:501765448|PMID:26215041  	phototropin	2015-09-02
AT3G45780	locus:2102674	AT3G45780	enables	FMN binding	GO:0010181	17729	F	nucleotide binding	TAS	original experiments are traceable through an article	NONE	Publication:501717658|PMID:16150710  	TAIR	2006-10-04
AT3G45780	locus:2102674	AT3G45780	enables	protein kinase activity	GO:0004672	3889	F	transferase activity	IDA	Enzyme assays		Publication:501750464|PMID:22781128  	TAIR	2012-08-06
AT3G45780	locus:2102674	AT3G45780	acts upstream of or within	regulation of proton transport	GO:0010155	17246	P	transport	IGI	double mutant analysis	AGI_LocusCode:AT5G58140	Publication:501716548|PMID:15821287  	TAIR	2008-05-24
AT3G45780	locus:2102674	AT3G45780	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:Q9FMF5	Publication:501719400|PMID:16777956  		2023-05-23
AT3G45780	locus:2102674	AT3G45780	located in	cytoplasmic side of plasma membrane	GO:0009898	12030	C	other membranes	IDA	localization of GFP/YFP fusion protein		Publication:501681826|PMID:12172018  	TAIR	2003-08-05
AT3G45780	locus:2102674	AT3G45780	acts upstream of or within	protein autophosphorylation	GO:0046777	13750	P	other cellular processes	TAS	original experiments are traceable through an article	NONE	Publication:501682623|PMID:12383086  	TAIR	2005-11-01
AT3G45780	locus:2102674	AT3G45780	enables	blue light photoreceptor activity	GO:0009882	9792	F	signaling receptor activity	IDA	Enzyme assays		Publication:1999|PMID:9831559   	TAIR	2003-03-29
AT3G45780	locus:2102674	AT3G45780	acts upstream of or within	blue light signaling pathway	GO:0009785	11759	P	signal transduction	IDA	none		Publication:1999|PMID:9831559   		2021-04-01
AT3G45780	locus:2102674	AT3G45780	acts upstream of or within	phototropism	GO:0009638	6762	P	response to light stimulus	IMP	Functional complementation	NONE	Publication:2818|PMID:9405347   	TAIR	2006-10-02
AT3G45780	locus:2102674	AT3G45780	enables	protein kinase activity	GO:0004672	3889	F	kinase activity	IDA	Enzyme assays		Publication:501750464|PMID:22781128  	TAIR	2012-08-06
AT3G45780	locus:2102674	AT3G45780	enables	FMN binding	GO:0010181	17729	F	nucleotide binding	IDA	none		Publication:501725066|PMID:18585389  		2021-04-01
AT3G45780	locus:2102674	AT3G45780	involved in	blue light signaling pathway	GO:0009785	11759	P	other cellular processes	IMP	biochemical/chemical analysis		Publication:501720921|PMID:17319842  	TAIR	2023-02-21
AT3G45780	locus:2102674	AT3G45780	located in	cell surface	GO:0009986	14229	C	other cellular components	IDA	none		Publication:501735847|PMID:20031918  	TAIR	2011-10-07
AT3G45780	gene:1009021722	AT3G45780.2	located in	plant-type vacuole	GO:0000325	11545	C	vacuole	HDA	none	NONE	Publication:501714874|PMID:15539469  	kvanwijk	2021-05-12
AT3G45780	locus:2102674	AT3G45780	enables	protein serine/threonine kinase activity	GO:0004674	3907	F	kinase activity	IBA	none	PANTHER:PTN000682243|SGD:S000005330|TAIR:locus:2102674|UniProtKB:Q8LPD9|TAIR:locus:2062268|TAIR:locus:2032785|TAIR:locus:2155821	Communication:501741973		2023-06-07
AT3G45780	locus:2102674	AT3G45780	acts upstream of or within	chloroplast accumulation movement	GO:0009904	11939	P	cellular component organization	IMP	analysis of physiological response	NONE	Publication:501680704|PMID:11371609  	TAIR	2003-08-05
AT3G45780	locus:2102674	AT3G45780	enables	protein serine/threonine kinase activity	GO:0004674	3907	F	transferase activity	IBA	none	PANTHER:PTN000682243|SGD:S000005330|TAIR:locus:2102674|UniProtKB:Q8LPD9|TAIR:locus:2062268|TAIR:locus:2032785|TAIR:locus:2155821	Communication:501741973		2023-06-07
AT3G45780	locus:2102674	AT3G45780	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:Q9SWI1	Publication:501719400|PMID:16777956  		2023-05-23
AT3G45780	locus:2102674	AT3G45780	enables	identical protein binding	GO:0042802	15210	F	protein binding	IPI	none	UniProtKB:O48963	Publication:501725066|PMID:18585389  		2021-04-01
AT3G45780	locus:2102674	AT3G45780	enables	identical protein binding	GO:0042802	15210	F	protein binding	IPI	none	UniProtKB:O48963	Publication:501745443|PMID:22001205  		2023-05-23
AT3G45780	locus:2102674	AT3G45780	is active in	nucleus	GO:0005634	537	C	nucleus	IBA	none	PANTHER:PTN000682243|TAIR:locus:2085191|TAIR:locus:2008331	Communication:501741973		2023-04-13
AT3G45780	locus:2102674	AT3G45780	enables	protein serine/threonine kinase activity	GO:0004674	3907	F	catalytic activity	IDA	Enzyme assays		Publication:1999|PMID:9831559   	TAIR	2003-03-29
AT3G45780	locus:2102674	AT3G45780	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	fusion protein pull-down assay	AGI_LocusCode:AT2G02950	Publication:501719400|PMID:16777956  	TAIR	2008-08-22
AT3G45780	locus:2102674	AT3G45780	acts upstream of or within	protein autophosphorylation	GO:0046777	13750	P	other metabolic processes	IDA	immunoprecipitation	NONE	Publication:501724637|PMID:18378899  	TAIR	2008-05-24
AT3G45780	locus:2102674	AT3G45780	acts upstream of or within	chloroplast avoidance movement	GO:0009903	11940	P	other cellular processes	IMP	analysis of physiological response	NONE	Publication:501680704|PMID:11371609  	TAIR	2003-08-05
AT3G45780	locus:2102674	AT3G45780	enables	protein serine/threonine kinase activity	GO:0004674	3907	F	catalytic activity	IBA	none	PANTHER:PTN000682243|SGD:S000005330|TAIR:locus:2102674|UniProtKB:Q8LPD9|TAIR:locus:2062268|TAIR:locus:2032785|TAIR:locus:2155821	Communication:501741973		2023-06-07
AT3G45780	locus:2102674	AT3G45780	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:Q9FMF5	Publication:1052|PMID:10542152  		2023-05-23
AT3G45780	locus:2102674	AT3G45780	enables	protein kinase activity	GO:0004672	3889	F	catalytic activity	IDA	Enzyme assays		Publication:501750464|PMID:22781128  	TAIR	2012-08-06
AT3G45780	gene:3698263	AT3G45780.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G45780	locus:2102674	AT3G45780	enables	protein serine/threonine kinase activity	GO:0004674	3907	F	transferase activity	IDA	Enzyme assays		Publication:1999|PMID:9831559   	TAIR	2003-03-29
AT3G45780	locus:2102674	AT3G45780	involved in	blue light signaling pathway	GO:0009785	11759	P	response to light stimulus	IMP	biochemical/chemical analysis		Publication:501720921|PMID:17319842  	TAIR	2023-02-21
AT3G45780	locus:2102674	AT3G45780	enables	identical protein binding	GO:0042802	15210	F	protein binding	IPI	none	UniProtKB:O48963	Publication:501732937|PMID:19166850  		2023-05-23
AT3G45780	locus:2102674	AT3G45780	acts upstream of or within	chloroplast avoidance movement	GO:0009903	11940	P	cellular component organization	IMP	analysis of physiological response	NONE	Publication:501680704|PMID:11371609  	TAIR	2003-08-05
AT3G45780	locus:2102674	AT3G45780	acts upstream of or within	phototropism	GO:0009638	6762	P	tropism	IMP	Functional complementation	NONE	Publication:2818|PMID:9405347   	TAIR	2006-10-02
AT3G45780	locus:2102674	AT3G45780	involved in	protein phosphorylation	GO:0006468	6897	P	other cellular processes	IBA	none	PANTHER:PTN000682243|TAIR:locus:2178388|TAIR:locus:2008331|TAIR:locus:2090310|TAIR:locus:2102674|UniProtKB:Q8LPD9	Communication:501741973		2023-06-07
AT3G45780	locus:2102674	AT3G45780	enables	mRNA binding	GO:0003729	3091	F	RNA binding	HDA	none	NONE	Publication:501771908|PMID:27729395  	tmillar	2021-04-22
AT3G45780	locus:2102674	AT3G45780	acts upstream of or within	protein autophosphorylation	GO:0046777	13750	P	other cellular processes	IDA	immunoprecipitation	NONE	Publication:501724637|PMID:18378899  	TAIR	2008-05-24
AT3G45780	locus:2102674	AT3G45780	acts upstream of or within	chloroplast accumulation movement	GO:0009904	11939	P	other cellular processes	IMP	analysis of physiological response	NONE	Publication:501680704|PMID:11371609  	TAIR	2003-08-05
AT3G45780	locus:2102674	AT3G45780	located in	cytoplasm	GO:0005737	231	C	cytoplasm	IDA	none		Publication:501735847|PMID:20031918  	TAIR	2011-10-07
AT3G45780	locus:2102674	AT3G45780	acts upstream of or within	blue light signaling pathway	GO:0009785	11759	P	other cellular processes	IDA	none		Publication:1999|PMID:9831559   		2021-04-01
AT3G45780	locus:2102674	AT3G45780	involved in	blue light signaling pathway	GO:0009785	11759	P	response to abiotic stimulus	IMP	biochemical/chemical analysis		Publication:501720921|PMID:17319842  	TAIR	2023-02-21
AT3G45780	locus:2102674	AT3G45780	acts upstream of or within	response to blue light	GO:0009637	5255	P	response to light stimulus	IGI	double mutant analysis	AGI_LocusCode:AT5G58140	Publication:501714985|PMID:15749755  	TAIR	2011-03-25
AT3G45780	locus:2102674	AT3G45780	involved in	protein phosphorylation	GO:0006468	6897	P	other metabolic processes	IBA	none	PANTHER:PTN000682243|TAIR:locus:2178388|TAIR:locus:2008331|TAIR:locus:2090310|TAIR:locus:2102674|UniProtKB:Q8LPD9	Communication:501741973		2023-06-07
AT3G45780	locus:2102674	AT3G45780	enables	FMN binding	GO:0010181	17729	F	other binding	IDA	none		Publication:501725066|PMID:18585389  		2021-04-01
AT3G45780	locus:2102674	AT3G45780	located in	cytoplasmic side of plasma membrane	GO:0009898	12030	C	plasma membrane	IDA	localization of GFP/YFP fusion protein		Publication:501681826|PMID:12172018  	TAIR	2003-08-05
AT3G45780	locus:2102674	AT3G45780	enables	protein serine/threonine kinase activity	GO:0004674	3907	F	kinase activity	IDA	Enzyme assays		Publication:1999|PMID:9831559   	TAIR	2003-03-29
AT3G45780	locus:2102674	AT3G45780	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:P48349	Publication:501733339|PMID:19524572  		2023-05-23
AT3G45790	locus:2102684	AT3G45790	enables	protein kinase activity	GO:0004672	3889	F	catalytic activity	IBA	none	PANTHER:PTN001969686|TAIR:locus:2205523|TAIR:locus:2116417|TAIR:locus:2074835|TAIR:locus:2007337	Communication:501741973		2023-06-07
AT3G45790	locus:2102684	AT3G45790	enables	protein kinase activity	GO:0004672	3889	F	kinase activity	IBA	none	PANTHER:PTN001969686|TAIR:locus:2205523|TAIR:locus:2116417|TAIR:locus:2074835|TAIR:locus:2007337	Communication:501741973		2023-06-07
AT3G45790	locus:2102684	AT3G45790	involved in	protein phosphorylation	GO:0006468	6897	P	other metabolic processes	IBA	none	PANTHER:PTN001969686|TAIR:locus:2007337|TAIR:locus:2205523|TAIR:locus:2074835|TAIR:locus:2116417	Communication:501741973		2023-06-07
AT3G45790	locus:2102684	AT3G45790	enables	protein kinase activity	GO:0004672	3889	F	transferase activity	IBA	none	PANTHER:PTN001969686|TAIR:locus:2205523|TAIR:locus:2116417|TAIR:locus:2074835|TAIR:locus:2007337	Communication:501741973		2023-06-07
AT3G45790	locus:2102684	AT3G45790	involved in	signal transduction	GO:0007165	7243	P	signal transduction	IBA	none	PANTHER:PTN001969686|TAIR:locus:2205523|TAIR:locus:2074835|TAIR:locus:2060246	Communication:501741973		2023-04-13
AT3G45790	locus:2102684	AT3G45790	involved in	protein phosphorylation	GO:0006468	6897	P	other cellular processes	IBA	none	PANTHER:PTN001969686|TAIR:locus:2007337|TAIR:locus:2205523|TAIR:locus:2074835|TAIR:locus:2116417	Communication:501741973		2023-06-07
AT3G45790	locus:2102684	AT3G45790	acts upstream of or within	phosphorylation	GO:0016310	6748	P	other cellular processes	ISS	none	Pfam:PF00069	Communication:501714663		2021-04-01
AT3G45790	gene:3698245	AT3G45790.1	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G45790	locus:2102684	AT3G45790	acts upstream of or within	phosphorylation	GO:0016310	6748	P	other metabolic processes	ISS	none	Pfam:PF00069	Communication:501714663		2021-04-01
AT3G45800	locus:2077187	AT3G45800	is active in	cellular_component	GO:0005575	163	C	unknown cellular components	ND	none	NONE	Communication:1345790	TIGR	2022-02-01
AT3G45800	locus:2077187	AT3G45800	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	none	NONE	Communication:1345790	TIGR	2003-09-13
AT3G45800	locus:2077187	AT3G45800	involved in	biological_process	GO:0008150	5239	P	unknown biological processes	ND	none	NONE	Communication:1345790	TIGR	2022-02-01
AT3G45810	locus:2077192	AT3G45810	enables	calcium ion binding	GO:0005509	1755	F	other binding	IEA	none	InterPro:IPR002048	AnalysisReference:501756966		2023-06-11
AT3G45810	gene:2077191	AT3G45810.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G45810	locus:2077192	AT3G45810	is active in	plasma membrane	GO:0005886	570	C	plasma membrane	IBA	none	PANTHER:PTN001735877|TAIR:locus:2025351|UniProtKB:Q948U0|UniProtKB:P04839|MGI:MGI:2681162|UniProtKB:Q9NRD8|UniProtKB:Q948T9|UniProtKB:Q9Y5S8|TAIR:locus:2183309|RGD:620574|TAIR:locus:2157027|UniProtKB:Q9NRD9|WB:WBGene00000253|MGI:MGI:88574|MGI:MGI:2139422|RGD:620598|SGD:S000005512|RGD:620600	Communication:501741973		2023-06-07
AT3G45810	locus:2077192	AT3G45810	enables	NAD(P)H oxidase H2O2-forming activity	GO:0016174	1158	F	catalytic activity	IBA	none	PANTHER:PTN000944270|TAIR:locus:2036104|TAIR:locus:2160917|TAIR:locus:2024603|TAIR:locus:2157348	Communication:501741973		2023-06-07
AT3G45820	locus:2077202	AT3G45820	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	none	NONE	Communication:1345790	TIGR	2003-10-04
AT3G45820	gene:2077201	AT3G45820.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G45820	locus:2077202	AT3G45820	involved in	biological_process	GO:0008150	5239	P	unknown biological processes	ND	Unknown biological process	NONE	Communication:1345790	TAIR	2022-02-01
AT3G45830	locus:2077102	AT3G45830	part of	Ino80 complex	GO:0031011	19245	C	nucleus	IDA	protein separation and fragment identification		Publication:501795269|PMID:34270751  	fengchao	2021-08-20
AT3G45830	locus:2077102	AT3G45830	part of	Ino80 complex	GO:0031011	19245	C	other intracellular components	IDA	protein separation and fragment identification		Publication:501795269|PMID:34270751  	fengchao	2021-08-20
AT3G45830	locus:2077102	AT3G45830	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	Unknown molecular function		Communication:1345790	TAIR	2021-10-28
AT3G45830	locus:2077102	AT3G45830	involved in	biological_process	GO:0008150	5239	P	unknown biological processes	ND	Unknown biological process	NONE	Communication:1345790	TAIR	2022-02-01
AT3G45830	gene:2077101	AT3G45830.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G45840	locus:2077117	AT3G45840	acts upstream of or within	defense response to bacterium	GO:0042742	14972	P	response to external stimulus	IEA	traceable computational prediction	AGI_LocusCode:AT5G67300	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G45840	locus:2077117	AT3G45840	acts upstream of or within	defense response to bacterium	GO:0042742	14972	P	response to biotic stimulus	IEA	traceable computational prediction	AGI_LocusCode:AT5G67300	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G45840	locus:2077117	AT3G45840	acts upstream of or within	defense response to bacterium	GO:0042742	14972	P	response to stress	IEA	traceable computational prediction	AGI_LocusCode:AT5G67300	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G45840	locus:2077117	AT3G45840	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	Unknown molecular function		Communication:1345790	TAIR	2021-10-28
AT3G45840	gene:2077116	AT3G45840.1	located in	extracellular region	GO:0005576	294	C	extracellular region	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G45850	locus:2077132	AT3G45850	is active in	mitotic spindle	GO:0072686	38521	C	cytoskeleton	IBA	none	PANTHER:PTN000649234|CGD:CAL0000181692|FB:FBgn0004378|UniProtKB:P52732|PomBase:SPAC25G10.07c	Communication:501741973		2023-06-07
AT3G45850	locus:2077132	AT3G45850	involved in	spindle elongation	GO:0051231	19534	P	cell cycle	IBA	none	PANTHER:PTN000649234|SGD:S000000787|dictyBase:DDB_G0288361|PomBase:SPAC25G10.07c|SGD:S000000159|UniProtKB:P17120	Communication:501741973		2023-06-07
AT3G45850	locus:2077132	AT3G45850	involved in	spindle elongation	GO:0051231	19534	P	cellular component organization	IBA	none	PANTHER:PTN000649234|SGD:S000000787|dictyBase:DDB_G0288361|PomBase:SPAC25G10.07c|SGD:S000000159|UniProtKB:P17120	Communication:501741973		2023-06-07
AT3G45850	locus:2077132	AT3G45850	involved in	mitotic spindle assembly	GO:0090307	35141	P	other cellular processes	IBA	none	PANTHER:PTN000649234|SGD:S000000159|UniProtKB:P17120|PomBase:SPAC25G10.07c|UniProtKB:P52732|SGD:S000000787	Communication:501741973		2023-06-07
AT3G45850	locus:2077132	AT3G45850	involved in	spindle elongation	GO:0051231	19534	P	other cellular processes	IBA	none	PANTHER:PTN000649234|SGD:S000000787|dictyBase:DDB_G0288361|PomBase:SPAC25G10.07c|SGD:S000000159|UniProtKB:P17120	Communication:501741973		2023-06-07
AT3G45850	gene:2077131	AT3G45850.1	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G45850	locus:2077132	AT3G45850	involved in	mitotic spindle assembly	GO:0090307	35141	P	cellular component organization	IBA	none	PANTHER:PTN000649234|SGD:S000000159|UniProtKB:P17120|PomBase:SPAC25G10.07c|UniProtKB:P52732|SGD:S000000787	Communication:501741973		2023-06-07
AT3G45850	locus:2077132	AT3G45850	is active in	mitotic spindle	GO:0072686	38521	C	other intracellular components	IBA	none	PANTHER:PTN000649234|CGD:CAL0000181692|FB:FBgn0004378|UniProtKB:P52732|PomBase:SPAC25G10.07c	Communication:501741973		2023-06-07
AT3G45850	locus:2077132	AT3G45850	involved in	mitotic spindle assembly	GO:0090307	35141	P	cell cycle	IBA	none	PANTHER:PTN000649234|SGD:S000000159|UniProtKB:P17120|PomBase:SPAC25G10.07c|UniProtKB:P52732|SGD:S000000787	Communication:501741973		2023-06-07
AT3G45850	locus:2077132	AT3G45850	is active in	spindle microtubule	GO:0005876	682	C	cytoskeleton	IBA	none	PANTHER:PTN000649234|SGD:S000000787|CGD:CAL0000181692|PomBase:SPAC25G10.07c|SGD:S000000159|FB:FBgn0004378	Communication:501741973		2023-06-07
AT3G45850	locus:2077132	AT3G45850	enables	microtubule motor activity	GO:0003777	3215	F	motor activity	IEA	none	InterPro:IPR001752|InterPro:IPR019821	AnalysisReference:501756966		2023-06-11
AT3G45850	locus:2077132	AT3G45850	enables	microtubule binding	GO:0008017	3214	F	protein binding	IEA	none	InterPro:IPR001752	AnalysisReference:501756966		2023-06-11
AT3G45850	locus:2077132	AT3G45850	is active in	spindle microtubule	GO:0005876	682	C	other intracellular components	IBA	none	PANTHER:PTN000649234|SGD:S000000787|CGD:CAL0000181692|PomBase:SPAC25G10.07c|SGD:S000000159|FB:FBgn0004378	Communication:501741973		2023-06-07
AT3G45850	locus:2077132	AT3G45850	involved in	microtubule-based movement	GO:0007018	6359	P	other cellular processes	IEA	none	InterPro:IPR001752|InterPro:IPR019821	AnalysisReference:501756966		2023-06-11
AT3G45850	gene:6530297160	AT3G45850.2	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G45851	gene:4515101604	AT3G45851.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G45851	locus:4515103205	AT3G45851	involved in	biological_process	GO:0008150	5239	P	unknown biological processes	ND	Unknown biological process	NONE	Communication:1345790	TAIR	2022-02-01
AT3G45851	locus:4515103205	AT3G45851	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	Unknown molecular function		Communication:1345790	TAIR	2011-10-26
AT3G45860	locus:2077147	AT3G45860	acts upstream of or within	response to salicylic acid	GO:0009751	11421	P	response to chemical	IEP	Transcript levels (e.g. Northerns)	NONE	Publication:501730911|PMID:15604743  	TAIR	2010-03-01
AT3G45860	locus:2077147	AT3G45860	involved in	defense response to bacterium	GO:0042742	14972	P	response to external stimulus	IBA	none	PANTHER:PTN007811734|TAIR:locus:2121691|TAIR:locus:2077147|TAIR:locus:2121606|TAIR:locus:2121686|TAIR:locus:2121611|UniProtKB:O65479	Communication:501741973		2023-06-07
AT3G45860	locus:2077147	AT3G45860	acts upstream of or within	programmed cell death	GO:0012501	10841	P	cell death	IMP	Analysis of overexpression/ectopic expression phenotype	NONE	Publication:501730911|PMID:15604743  	TAIR	2010-03-01
AT3G45860	locus:2077147	AT3G45860	enables	protein serine/threonine kinase activity	GO:0004674	3907	F	catalytic activity	IBA	none	PANTHER:PTN002819812|TAIR:locus:2197729|TAIR:locus:2018546|TAIR:locus:2200151|UniProtKB:Q8GY82|TAIR:locus:2137134	Communication:501741973		2023-06-07
AT3G45860	locus:2077147	AT3G45860	is active in	plasma membrane	GO:0005886	570	C	plasma membrane	IBA	none	PANTHER:PTN002819812|TAIR:locus:2121621|TAIR:locus:2137134|TAIR:locus:2200151|TAIR:locus:2197729	Communication:501741973		2023-06-07
AT3G45860	locus:2077147	AT3G45860	acts upstream of or within	defense response to bacterium	GO:0042742	14972	P	response to biotic stimulus	IEP	Transcript levels (e.g. Northerns)	NONE	Publication:501730911|PMID:15604743  	TAIR	2010-03-01
AT3G45860	locus:2077147	AT3G45860	involved in	defense response to bacterium	GO:0042742	14972	P	response to stress	IBA	none	PANTHER:PTN007811734|TAIR:locus:2121691|TAIR:locus:2077147|TAIR:locus:2121606|TAIR:locus:2121686|TAIR:locus:2121611|UniProtKB:O65479	Communication:501741973		2023-06-07
AT3G45860	locus:2077147	AT3G45860	involved in	protein phosphorylation	GO:0006468	6897	P	other metabolic processes	IBA	none	PANTHER:PTN002819812|UniProtKB:Q8L710|TAIR:locus:2018546|TAIR:locus:2200151|TAIR:locus:2197729|TAIR:locus:2202054|TAIR:locus:2018506	Communication:501741973		2023-06-07
AT3G45860	locus:2077147	AT3G45860	acts upstream of or within	defense response	GO:0006952	5542	P	response to stress	TAS	none	NONE	Publication:1345932|PMID:11135117  	TIGR	2003-04-17
AT3G45860	locus:2077147	AT3G45860	acts upstream of or within	defense response to bacterium	GO:0042742	14972	P	response to external stimulus	IEP	Transcript levels (e.g. Northerns)	NONE	Publication:501730911|PMID:15604743  	TAIR	2010-03-01
AT3G45860	locus:2077147	AT3G45860	enables	kinase activity	GO:0016301	2953	F	kinase activity	ISS	Sequence similarity (homologue of/most closely related to)		Communication:1674994	TAIR	2003-03-29
AT3G45860	locus:2077147	AT3G45860	involved in	protein phosphorylation	GO:0006468	6897	P	other cellular processes	IBA	none	PANTHER:PTN002819812|UniProtKB:Q8L710|TAIR:locus:2018546|TAIR:locus:2200151|TAIR:locus:2197729|TAIR:locus:2202054|TAIR:locus:2018506	Communication:501741973		2023-06-07
AT3G45860	locus:2077147	AT3G45860	involved in	defense response to bacterium	GO:0042742	14972	P	response to biotic stimulus	IBA	none	PANTHER:PTN007811734|TAIR:locus:2121691|TAIR:locus:2077147|TAIR:locus:2121606|TAIR:locus:2121686|TAIR:locus:2121611|UniProtKB:O65479	Communication:501741973		2023-06-07
AT3G45860	locus:2077147	AT3G45860	enables	protein serine/threonine kinase activity	GO:0004674	3907	F	kinase activity	IBA	none	PANTHER:PTN002819812|TAIR:locus:2197729|TAIR:locus:2018546|TAIR:locus:2200151|UniProtKB:Q8GY82|TAIR:locus:2137134	Communication:501741973		2023-06-07
AT3G45860	locus:2077147	AT3G45860	enables	protein serine/threonine kinase activity	GO:0004674	3907	F	transferase activity	IBA	none	PANTHER:PTN002819812|TAIR:locus:2197729|TAIR:locus:2018546|TAIR:locus:2200151|UniProtKB:Q8GY82|TAIR:locus:2137134	Communication:501741973		2023-06-07
AT3G45860	locus:2077147	AT3G45860	acts upstream of or within	defense response to bacterium	GO:0042742	14972	P	response to stress	IEP	Transcript levels (e.g. Northerns)	NONE	Publication:501730911|PMID:15604743  	TAIR	2010-03-01
AT3G45870	locus:2077162	AT3G45870	acts upstream of or within	regulation of cellular biosynthetic process	GO:0031326	20872	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT5G52240	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G45870	locus:2077162	AT3G45870	acts upstream of or within	plant-type cell wall organization	GO:0009664	5349	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT3G26370	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G45870	locus:2077162	AT3G45870	acts upstream of or within	cell wall polysaccharide biosynthetic process	GO:0070592	32030	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT3G26370	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G45870	locus:2077162	AT3G45870	acts upstream of or within	regulation of cellular biosynthetic process	GO:0031326	20872	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT5G52240	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G45870	locus:2077162	AT3G45870	acts upstream of or within	lignin biosynthetic process	GO:0009809	6181	P	secondary metabolic process	IEA	traceable computational prediction	AGI_LocusCode:AT5G52240	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G45870	locus:2077162	AT3G45870	acts upstream of or within	cell wall polysaccharide biosynthetic process	GO:0070592	32030	P	biosynthetic process	IEA	traceable computational prediction	AGI_LocusCode:AT3G26370	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G45870	gene:6532558527	AT3G45870.5	located in	chloroplast	GO:0009507	175	C	chloroplast	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G45870	locus:2077162	AT3G45870	acts upstream of or within	cell wall polysaccharide biosynthetic process	GO:0070592	32030	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT3G26370	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G45870	locus:2077162	AT3G45870	acts upstream of or within	lignin biosynthetic process	GO:0009809	6181	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT5G52240	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G45870	locus:2077162	AT3G45870	acts upstream of or within	lignin biosynthetic process	GO:0009809	6181	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT5G52240	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G45870	locus:2077162	AT3G45870	acts upstream of or within	lignin biosynthetic process	GO:0009809	6181	P	biosynthetic process	IEA	traceable computational prediction	AGI_LocusCode:AT5G52240	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G45870	gene:6532550539	AT3G45870.7	located in	mitochondrion	GO:0005739	486	C	mitochondrion	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G45870	gene:6532550538	AT3G45870.6	located in	mitochondrion	GO:0005739	486	C	mitochondrion	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G45870	locus:2077162	AT3G45870	is active in	plasma membrane	GO:0005886	570	C	plasma membrane	IBA	none	PANTHER:PTN001261134|TAIR:locus:2200990|TAIR:locus:2058460|TAIR:locus:2125167|TAIR:locus:2194864|TAIR:locus:2039792	Communication:501741973		2023-06-07
AT3G45870	locus:2077162	AT3G45870	acts upstream of or within	regulation of cellular biosynthetic process	GO:0031326	20872	P	biosynthetic process	IEA	traceable computational prediction	AGI_LocusCode:AT5G52240	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G45870	locus:2077162	AT3G45870	acts upstream of or within	plant-type cell wall biogenesis	GO:0009832	10265	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT3G26370	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G45870	gene:6532558552	AT3G45870.3	located in	mitochondrion	GO:0005739	486	C	mitochondrion	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G45870	locus:2077162	AT3G45870	enables	transmembrane transporter activity	GO:0022857	27431	F	transporter activity	IEA	none	InterPro:IPR030184	AnalysisReference:501756966		2023-05-23
AT3G45870	gene:6532550533	AT3G45870.8	located in	mitochondrion	GO:0005739	486	C	mitochondrion	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G45870	locus:2077162	AT3G45870	acts upstream of or within	cell wall polysaccharide biosynthetic process	GO:0070592	32030	P	carbohydrate metabolic process	IEA	traceable computational prediction	AGI_LocusCode:AT3G26370	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G45870	locus:2077162	AT3G45870	acts upstream of or within	plant-type cell wall organization	GO:0009664	5349	P	cellular component organization	IEA	traceable computational prediction	AGI_LocusCode:AT3G26370	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G45880	gene:6532550440	AT3G45880.2	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G45880	locus:2077172	AT3G45880	enables	histone H3K27me2/H3K27me3 demethylase activity	GO:0071558	34020	F	catalytic activity	IMP	none		Publication:501784396|PMID:30859592  		2022-05-04
AT3G45880	locus:2077172	AT3G45880	acts upstream of or within	primary root development	GO:0080022	30849	P	anatomical structure development	IMP	none		Publication:501784811|PMID:30983495  		2022-03-30
AT3G45880	locus:2077172	AT3G45880	involved in	regulation of photoperiodism, flowering	GO:2000028	35576	P	response to light stimulus	IMP	none		Publication:501761846|PMID:25267112  		2022-03-30
AT3G45880	locus:2077172	AT3G45880	acts upstream of or within	regulation of seedling development	GO:1900140	40337	P	multicellular organism development	IMP	none		Publication:501784396|PMID:30859592  		2022-05-04
AT3G45880	locus:2077172	AT3G45880	involved in	regulation of abscisic acid-activated signaling pathway	GO:0009787	11755	P	response to chemical	IMP	none		Publication:501792377|PMID:33324437  		2022-05-04
AT3G45880	locus:2077172	AT3G45880	involved in	regulation of photoperiodism, flowering	GO:2000028	35576	P	multicellular organism development	IMP	none		Publication:501761846|PMID:25267112  		2022-03-30
AT3G45880	locus:2077172	AT3G45880	located in	cytoplasm	GO:0005737	231	C	cytoplasm	IDA	none		Publication:501761846|PMID:25267112  		2022-03-30
AT3G45880	locus:2077172	AT3G45880	acts upstream of or within	response to abscisic acid	GO:0009737	11395	P	response to chemical	IMP	none		Publication:501784396|PMID:30859592  		2022-05-04
AT3G45880	locus:2077172	AT3G45880	located in	nucleus	GO:0005634	537	C	nucleus	IDA	none		Publication:501761846|PMID:25267112  		2022-03-30
AT3G45880	locus:2077172	AT3G45880	enables	histone H3K27me2/H3K27me3 demethylase activity	GO:0071558	34020	F	catalytic activity	IDA	none		Publication:501761846|PMID:25267112  		2022-03-30
AT3G45880	locus:2077172	AT3G45880	enables	histone H3K27me2/H3K27me3 demethylase activity	GO:0071558	34020	F	catalytic activity	IMP	none		Publication:501761846|PMID:25267112  		2022-03-30
AT3G45880	locus:2077172	AT3G45880	involved in	regulation of brassinosteroid mediated signaling pathway	GO:1900457	41205	P	signal transduction	IMP	none		Publication:501792377|PMID:33324437  		2022-05-04
AT3G45880	locus:2077172	AT3G45880	involved in	negative regulation of gene expression, epigenetic	GO:0045814	12764	P	regulation of gene expression, epigenetic	IDA	none		Publication:501761846|PMID:25267112  		2022-09-19
AT3G45880	locus:2077172	AT3G45880	involved in	regulation of brassinosteroid mediated signaling pathway	GO:1900457	41205	P	cell communication	IMP	none		Publication:501792377|PMID:33324437  		2022-05-04
AT3G45880	locus:2077172	AT3G45880	involved in	regulation of photoperiodism, flowering	GO:2000028	35576	P	post-embryonic development	IMP	none		Publication:501761846|PMID:25267112  		2022-03-30
AT3G45880	locus:2077172	AT3G45880	involved in	regulation of abscisic acid-activated signaling pathway	GO:0009787	11755	P	signal transduction	IMP	none		Publication:501792377|PMID:33324437  		2022-05-04
AT3G45880	locus:2077172	AT3G45880	involved in	response to temperature stimulus	GO:0009266	7411	P	response to abiotic stimulus	IMP	none		Publication:501761846|PMID:25267112  		2022-03-30
AT3G45880	locus:2077172	AT3G45880	involved in	regulation of abscisic acid-activated signaling pathway	GO:0009787	11755	P	other cellular processes	IMP	none		Publication:501792377|PMID:33324437  		2022-05-04
AT3G45880	locus:2077172	AT3G45880	acts upstream of or within	primary root development	GO:0080022	30849	P	multicellular organism development	IMP	none		Publication:501784811|PMID:30983495  		2022-03-30
AT3G45880	locus:2077172	AT3G45880	acts upstream of or within	regulation of seedling development	GO:1900140	40337	P	post-embryonic development	IMP	none		Publication:501784396|PMID:30859592  		2022-05-04
AT3G45880	locus:2077172	AT3G45880	involved in	response to abscisic acid	GO:0009737	11395	P	response to endogenous stimulus	IMP	none		Publication:501784811|PMID:30983495  		2022-03-30
AT3G45880	locus:2077172	AT3G45880	located in	endoplasmic reticulum	GO:0005783	268	C	endoplasmic reticulum	IDA	none		Publication:501761846|PMID:25267112  		2022-03-30
AT3G45880	locus:2077172	AT3G45880	involved in	regulation of brassinosteroid mediated signaling pathway	GO:1900457	41205	P	other cellular processes	IMP	none		Publication:501792377|PMID:33324437  		2022-05-04
AT3G45880	locus:2077172	AT3G45880	involved in	regulation of brassinosteroid mediated signaling pathway	GO:1900457	41205	P	response to endogenous stimulus	IMP	none		Publication:501792377|PMID:33324437  		2022-05-04
AT3G45880	locus:2077172	AT3G45880	acts upstream of or within	response to abscisic acid	GO:0009737	11395	P	response to endogenous stimulus	IMP	none		Publication:501784396|PMID:30859592  		2022-05-04
AT3G45880	locus:2077172	AT3G45880	involved in	regulation of brassinosteroid mediated signaling pathway	GO:1900457	41205	P	response to chemical	IMP	none		Publication:501792377|PMID:33324437  		2022-05-04
AT3G45880	locus:2077172	AT3G45880	involved in	regulation of photoperiodism, flowering	GO:2000028	35576	P	anatomical structure development	IMP	none		Publication:501761846|PMID:25267112  		2022-03-30
AT3G45880	locus:2077172	AT3G45880	involved in	regulation of abscisic acid-activated signaling pathway	GO:0009787	11755	P	cell communication	IMP	none		Publication:501792377|PMID:33324437  		2022-05-04
AT3G45880	locus:2077172	AT3G45880	involved in	response to abscisic acid	GO:0009737	11395	P	response to chemical	IMP	none		Publication:501784811|PMID:30983495  		2022-03-30
AT3G45880	locus:2077172	AT3G45880	involved in	regulation of photoperiodism, flowering	GO:2000028	35576	P	reproduction	IMP	none		Publication:501761846|PMID:25267112  		2022-03-30
AT3G45880	locus:2077172	AT3G45880	involved in	negative regulation of gene expression, epigenetic	GO:0045814	12764	P	other metabolic processes	IDA	none		Publication:501761846|PMID:25267112  		2022-09-19
AT3G45880	gene:2077171	AT3G45880.1	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G45880	locus:2077172	AT3G45880	enables	2-oxoglutarate-dependent dioxygenase activity	GO:0016706	3496	F	catalytic activity	IBA	none	PANTHER:PTN004362614|UniProtKB:Q8N371|UniProtKB:Q9NWT6|TAIR:locus:2077172|UniProtKB:P0C870|UniProtKB:Q8IIE4|UniProtKB:A2RUC4|TAIR:locus:2091861	Communication:501741973		2023-06-07
AT3G45880	locus:2077172	AT3G45880	involved in	regulation of abscisic acid-activated signaling pathway	GO:0009787	11755	P	response to endogenous stimulus	IMP	none		Publication:501792377|PMID:33324437  		2022-05-04
AT3G45890	locus:2077197	AT3G45890	acts upstream of or within	response to UV-B	GO:0010224	18529	P	response to light stimulus	IMP	analysis of physiological response	NONE	Publication:501729351|PMID:19075229  	TAIR	2009-01-20
AT3G45890	gene:2077196	AT3G45890.1	located in	chloroplast envelope	GO:0009941	12482	C	other intracellular components	HDA	none	NONE	Publication:501735990|PMID:20061580  	nrolland	2021-04-07
AT3G45890	locus:2077197	AT3G45890	acts upstream of or within	developmental process	GO:0032502	25755	P	other biological processes	IMP	analysis of visible trait	NONE	Publication:501729351|PMID:19075229  	TAIR	2009-01-20
AT3G45890	locus:2077197	AT3G45890	involved in	response to UV-B	GO:0010224	18529	P	response to light stimulus	IBA	none	PANTHER:PTN000979455|TAIR:locus:2077197	Communication:501741973		2021-03-31
AT3G45890	gene:2077196	AT3G45890.1	located in	chloroplast envelope	GO:0009941	12482	C	chloroplast	HDA	none	NONE	Publication:501735990|PMID:20061580  	nrolland	2021-04-07
AT3G45890	locus:2077197	AT3G45890	located in	plastid	GO:0009536	576	C	plastid	HDA	none	NONE	Publication:501750648|PMID:22923678  	dupree	2021-04-07
AT3G45890	locus:2077197	AT3G45890	involved in	developmental process	GO:0032502	25755	P	other biological processes	IBA	none	PANTHER:PTN000979455|TAIR:locus:2077197	Communication:501741973		2021-03-31
AT3G45890	gene:2077196	AT3G45890.1	located in	chloroplast envelope	GO:0009941	12482	C	plastid	HDA	none	NONE	Publication:501735990|PMID:20061580  	nrolland	2021-04-07
AT3G45900	locus:2077207	AT3G45900	involved in	biological_process	GO:0008150	5239	P	unknown biological processes	ND	Unknown biological process	NONE	Communication:1345790	TAIR	2022-02-01
AT3G45900	gene:2077206	AT3G45900.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G45900	locus:2077207	AT3G45900	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	Unknown molecular function		Communication:1345790	TAIR	2021-10-28
AT3G45910	locus:2077097	AT3G45910	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	none	NONE	Communication:1345790	TIGR	2003-10-04
AT3G45910	gene:2077096	AT3G45910.1	located in	mitochondrion	GO:0005739	486	C	mitochondrion	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G45910	locus:2077097	AT3G45910	involved in	biological_process	GO:0008150	5239	P	unknown biological processes	ND	Unknown biological process	NONE	Communication:1345790	TAIR	2022-02-01
AT3G45920	locus:2077112	AT3G45920	enables	protein kinase activity	GO:0004672	3889	F	catalytic activity	IEA	none	InterPro:IPR000719|InterPro:IPR008271	AnalysisReference:501756966		2023-05-23
AT3G45920	locus:2077112	AT3G45920	involved in	protein phosphorylation	GO:0006468	6897	P	other metabolic processes	IEA	none	InterPro:IPR000719|InterPro:IPR008271	AnalysisReference:501756966		2023-05-23
AT3G45920	gene:2077111	AT3G45920.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G45920	locus:2077112	AT3G45920	involved in	protein phosphorylation	GO:0006468	6897	P	other cellular processes	IEA	none	InterPro:IPR000719|InterPro:IPR008271	AnalysisReference:501756966		2023-05-23
AT3G45920	locus:2077112	AT3G45920	enables	protein kinase activity	GO:0004672	3889	F	kinase activity	IEA	none	InterPro:IPR000719|InterPro:IPR008271	AnalysisReference:501756966		2023-05-23
AT3G45920	locus:2077112	AT3G45920	enables	ATP binding	GO:0005524	894	F	nucleotide binding	IEA	none	InterPro:IPR000719	AnalysisReference:501756966		2023-05-23
AT3G45920	locus:2077112	AT3G45920	enables	ATP binding	GO:0005524	894	F	other binding	IEA	none	InterPro:IPR000719	AnalysisReference:501756966		2023-05-23
AT3G45920	locus:2077112	AT3G45920	enables	protein kinase activity	GO:0004672	3889	F	transferase activity	IEA	none	InterPro:IPR000719|InterPro:IPR008271	AnalysisReference:501756966		2023-05-23
AT3G45930	gene:2077126	AT3G45930.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G45930	locus:2077127	AT3G45930	located in	cytosol	GO:0005829	241	C	cytosol	HDA	none		Publication:501776792|PMID:28887381  	dszymanski	2019-04-04
AT3G45930	locus:2077127	AT3G45930	acts upstream of or within	regulation of nucleobase-containing compound metabolic process	GO:0019219	10748	P	nucleobase-containing compound metabolic process	IEA	traceable computational prediction	AGI_LocusCode:AT2G36270	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G45930	locus:2077127	AT3G45930	acts upstream of or within	regulation of nitrogen compound metabolic process	GO:0051171	19379	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT3G11100|AGI_LocusCode:AT2G36270	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G45930	locus:2077127	AT3G45930	acts upstream of or within	regulation of nucleobase-containing compound metabolic process	GO:0019219	10748	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT2G36270	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G45930	locus:2077127	AT3G45930	acts upstream of or within	regulation of nucleobase-containing compound metabolic process	GO:0019219	10748	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT2G36270	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G45930	locus:2077127	AT3G45930	acts upstream of or within	regulation of primary metabolic process	GO:0080090	31866	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT3G11100|AGI_LocusCode:AT2G36270	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G45930	locus:2077127	AT3G45930	acts upstream of or within	macromolecule modification	GO:0043412	21458	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT3G11100	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G45935	locus:1005716593	AT3G45935	acts upstream of or within	translational elongation	GO:0006414	6905	P	other metabolic processes	TAS	text book or dictionary; -material has become common knowledge	NONE	Publication:501710108|PMID:8980477   	TAIR	2005-08-31
AT3G45935	locus:1005716593	AT3G45935	acts upstream of or within	translation	GO:0006412	6869	P	translation	TAS	none		Publication:501710108|PMID:8980477   		2021-04-01
AT3G45935	locus:1005716593	AT3G45935	acts upstream of or within	translational elongation	GO:0006414	6905	P	other cellular processes	TAS	text book or dictionary; -material has become common knowledge	NONE	Publication:501710108|PMID:8980477   	TAIR	2005-08-31
AT3G45935	locus:1005716593	AT3G45935	is active in	cellular_component	GO:0005575	163	C	unknown cellular components	ND	Unknown cellular component	NONE	Communication:1345790	TAIR	2022-02-01
AT3G45935	locus:1005716593	AT3G45935	acts upstream of or within	translational elongation	GO:0006414	6905	P	biosynthetic process	TAS	text book or dictionary; -material has become common knowledge	NONE	Publication:501710108|PMID:8980477   	TAIR	2005-08-31
AT3G45935	locus:1005716593	AT3G45935	enables	triplet codon-amino acid adaptor activity	GO:0030533	9889	F	RNA binding	TAS	text book or dictionary; -material has become common knowledge		Publication:501710108|PMID:8980477   	TAIR	2005-08-31
AT3G45935	locus:1005716593	AT3G45935	acts upstream of or within	translational elongation	GO:0006414	6905	P	translation	TAS	text book or dictionary; -material has become common knowledge	NONE	Publication:501710108|PMID:8980477   	TAIR	2005-08-31
AT3G45940	gene:2077141	AT3G45940.1	located in	plasma membrane	GO:0005886	570	C	plasma membrane	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G45940	locus:2077142	AT3G45940	enables	hydrolase activity, hydrolyzing O-glycosyl compounds	GO:0004553	2781	F	hydrolase activity	IBA	none	PANTHER:PTN000509739|PomBase:SPAPB24D3.10c|MGI:MGI:1097667|UniProtKB:Q5AWI5|WB:WBGene00009583|dictyBase:DDB_G0269154|MGI:MGI:95609|RGD:3675|UniProtKB:P14410|MGI:MGI:1923301|TAIR:locus:2163976|TAIR:locus:2026895|WB:WBGene00019895|PomBase:SPAC1002.03c|MGI:MGI:1203495|UniProtKB:O43451|SGD:S000000433|RGD:735227|UniProtKB:Q14697|WB:WBGene00017071|UniProtKB:P10253|UniProtKB:Q5BET9	Communication:501741973		2023-06-07
AT3G45940	locus:2077142	AT3G45940	enables	carbohydrate binding	GO:0030246	8584	F	carbohydrate binding	IEA	none	InterPro:IPR011013	AnalysisReference:501756966		2023-06-11
AT3G45940	locus:2077142	AT3G45940	enables	alpha-D-xyloside xylohydrolase	GO:0061634	46270	F	hydrolase activity	IEA	none	EC:3.2.1.177	AnalysisReference:501756967		2023-06-11
AT3G45940	gene:2077141	AT3G45940.1	located in	plasmodesma	GO:0009506	571	C	other cellular components	HDA	none	NONE	Publication:501742898|PMID:21533090  	maule	2021-04-07
AT3G45940	locus:2077142	AT3G45940	located in	cytosol	GO:0005829	241	C	cytosol	HDA	none		Publication:501776792|PMID:28887381  	dszymanski	2019-04-04
AT3G45950	locus:2077157	AT3G45950	involved in	RNA splicing	GO:0008380	4903	P	other metabolic processes	IBA	none	PANTHER:PTN000310059|UniProtKB:O95391|PomBase:SPBC365.05c	Communication:501741973		2023-06-07
AT3G45950	gene:2077156	AT3G45950.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G45950	locus:2077157	AT3G45950	involved in	RNA splicing	GO:0008380	4903	P	nucleobase-containing compound metabolic process	IBA	none	PANTHER:PTN000310059|UniProtKB:O95391|PomBase:SPBC365.05c	Communication:501741973		2023-06-07
AT3G45950	locus:2077157	AT3G45950	part of	spliceosomal complex	GO:0005681	684	C	nucleus	IBA	none	PANTHER:PTN000310059|PomBase:SPBC365.05c|UniProtKB:O95391	Communication:501741973		2022-11-23
AT3G45950	locus:2077157	AT3G45950	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	Unknown molecular function		Communication:1345790	TAIR	2021-10-28
AT3G45955	locus:1005716591	AT3G45955	acts upstream of or within	translation	GO:0006412	6869	P	translation	TAS	none		Publication:501710108|PMID:8980477   		2021-04-01
AT3G45955	locus:1005716591	AT3G45955	enables	triplet codon-amino acid adaptor activity	GO:0030533	9889	F	RNA binding	TAS	text book or dictionary; -material has become common knowledge		Publication:501710108|PMID:8980477   	TAIR	2005-08-31
AT3G45955	locus:1005716591	AT3G45955	acts upstream of or within	translational elongation	GO:0006414	6905	P	other metabolic processes	TAS	text book or dictionary; -material has become common knowledge	NONE	Publication:501710108|PMID:8980477   	TAIR	2005-08-31
AT3G45955	locus:1005716591	AT3G45955	acts upstream of or within	translational elongation	GO:0006414	6905	P	translation	TAS	text book or dictionary; -material has become common knowledge	NONE	Publication:501710108|PMID:8980477   	TAIR	2005-08-31
AT3G45955	locus:1005716591	AT3G45955	acts upstream of or within	translational elongation	GO:0006414	6905	P	other cellular processes	TAS	text book or dictionary; -material has become common knowledge	NONE	Publication:501710108|PMID:8980477   	TAIR	2005-08-31
AT3G45955	locus:1005716591	AT3G45955	is active in	cellular_component	GO:0005575	163	C	unknown cellular components	ND	Unknown cellular component	NONE	Communication:1345790	TAIR	2022-02-01
AT3G45955	locus:1005716591	AT3G45955	acts upstream of or within	translational elongation	GO:0006414	6905	P	biosynthetic process	TAS	text book or dictionary; -material has become common knowledge	NONE	Publication:501710108|PMID:8980477   	TAIR	2005-08-31
AT3G45960	gene:1009021765	AT3G45960.2	located in	plasmodesma	GO:0009506	571	C	other cellular components	HDA	none	NONE	Publication:501742898|PMID:21533090  	maule	2021-04-07
AT3G45960	locus:2077167	AT3G45960	acts upstream of or within	unidimensional cell growth	GO:0009826	10252	P	cell growth	NAS	none	NONE	Publication:1546271|PMID:11641069  	TIGR	2003-04-17
AT3G45960	gene:1009021765	AT3G45960.2	located in	extracellular region	GO:0005576	294	C	extracellular region	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G45960	locus:2077167	AT3G45960	acts upstream of or within	plant-type cell wall loosening	GO:0009828	10268	P	other cellular processes	NAS	none	NONE	Publication:1546271|PMID:11641069  	TIGR	2003-04-17
AT3G45960	locus:2077167	AT3G45960	acts upstream of or within	unidimensional cell growth	GO:0009826	10252	P	anatomical structure development	NAS	none	NONE	Publication:1546271|PMID:11641069  	TIGR	2003-04-17
AT3G45960	locus:2077167	AT3G45960	acts upstream of or within	plant-type cell wall loosening	GO:0009828	10268	P	cellular component organization	NAS	none	NONE	Publication:1546271|PMID:11641069  	TIGR	2003-04-17
AT3G45960	locus:2077167	AT3G45960	acts upstream of or within	unidimensional cell growth	GO:0009826	10252	P	growth	NAS	none	NONE	Publication:1546271|PMID:11641069  	TIGR	2003-04-17
AT3G45960	locus:2077167	AT3G45960	located in	plant-type cell wall	GO:0009505	156	C	cell wall	HDA	none	NONE	Publication:501718077|PMID:16287169  	TAIR	2021-05-12
AT3G45965	locus:1005716592	AT3G45965	acts upstream of or within	translation	GO:0006412	6869	P	translation	TAS	none		Publication:501710108|PMID:8980477   		2021-04-01
AT3G45965	locus:1005716592	AT3G45965	acts upstream of or within	translational elongation	GO:0006414	6905	P	other cellular processes	TAS	text book or dictionary; -material has become common knowledge	NONE	Publication:501710108|PMID:8980477   	TAIR	2005-08-31
AT3G45965	locus:1005716592	AT3G45965	acts upstream of or within	translational elongation	GO:0006414	6905	P	other metabolic processes	TAS	text book or dictionary; -material has become common knowledge	NONE	Publication:501710108|PMID:8980477   	TAIR	2005-08-31
AT3G45965	locus:1005716592	AT3G45965	acts upstream of or within	translational elongation	GO:0006414	6905	P	biosynthetic process	TAS	text book or dictionary; -material has become common knowledge	NONE	Publication:501710108|PMID:8980477   	TAIR	2005-08-31
AT3G45965	locus:1005716592	AT3G45965	acts upstream of or within	translational elongation	GO:0006414	6905	P	translation	TAS	text book or dictionary; -material has become common knowledge	NONE	Publication:501710108|PMID:8980477   	TAIR	2005-08-31
AT3G45965	locus:1005716592	AT3G45965	enables	triplet codon-amino acid adaptor activity	GO:0030533	9889	F	RNA binding	TAS	text book or dictionary; -material has become common knowledge		Publication:501710108|PMID:8980477   	TAIR	2005-08-31
AT3G45965	locus:1005716592	AT3G45965	is active in	cellular_component	GO:0005575	163	C	unknown cellular components	ND	Unknown cellular component	NONE	Communication:1345790	TAIR	2022-02-01
AT3G45970	gene:6532554812	AT3G45970.2	located in	extracellular region	GO:0005576	294	C	extracellular region	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G45970	locus:2077177	AT3G45970	acts upstream of or within	unidimensional cell growth	GO:0009826	10252	P	growth	NAS	none	NONE	Publication:1546271|PMID:11641069  	TIGR	2003-04-17
AT3G45970	locus:2077177	AT3G45970	located in	secretory vesicle	GO:0099503	51212	C	cytoplasm	HDA	none		Publication:501761766|PMID:25293756  	dszymanski	2019-04-04
AT3G45970	locus:2077177	AT3G45970	acts upstream of or within	plant-type cell wall loosening	GO:0009828	10268	P	other cellular processes	NAS	none	NONE	Publication:1546271|PMID:11641069  	TIGR	2003-04-17
AT3G45970	locus:2077177	AT3G45970	located in	plant-type cell wall	GO:0009505	156	C	cell wall	HDA	none	NONE	Publication:501718077|PMID:16287169  	TAIR	2021-05-12
AT3G45970	locus:2077177	AT3G45970	located in	secretory vesicle	GO:0099503	51212	C	other intracellular components	HDA	none		Publication:501761766|PMID:25293756  	dszymanski	2019-04-04
AT3G45970	gene:2077176	AT3G45970.1	located in	Golgi apparatus	GO:0005794	48	C	Golgi apparatus	HDA	none	NONE	Publication:501747810|PMID:22430844  	jheazlewoo	2021-05-12
AT3G45970	locus:2077177	AT3G45970	located in	extracellular region	GO:0005576	294	C	extracellular region	IEA	none	InterPro:IPR007118	AnalysisReference:501756966		2023-05-23
AT3G45970	locus:2077177	AT3G45970	acts upstream of or within	plant-type cell wall loosening	GO:0009828	10268	P	cellular component organization	NAS	none	NONE	Publication:1546271|PMID:11641069  	TIGR	2003-04-17
AT3G45970	gene:2077176	AT3G45970.1	located in	plasmodesma	GO:0009506	571	C	other cellular components	HDA	none	NONE	Publication:501742898|PMID:21533090  	maule	2021-04-07
AT3G45970	gene:2077176	AT3G45970.1	located in	extracellular region	GO:0005576	294	C	extracellular region	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G45970	locus:2077177	AT3G45970	acts upstream of or within	unidimensional cell growth	GO:0009826	10252	P	anatomical structure development	NAS	none	NONE	Publication:1546271|PMID:11641069  	TIGR	2003-04-17
AT3G45970	locus:2077177	AT3G45970	acts upstream of or within	unidimensional cell growth	GO:0009826	10252	P	cell growth	NAS	none	NONE	Publication:1546271|PMID:11641069  	TIGR	2003-04-17
AT3G45970	gene:2077176	AT3G45970.1	located in	plant-type cell wall	GO:0009505	156	C	cell wall	HDA	none	NONE	Publication:501711493|PMID:14595688  	kvanwijk	2022-02-08
AT3G45980	gene:2077181	AT3G45980.1	located in	chloroplast thylakoid	GO:0009534	704	C	plastid	HDA	none	NONE	Publication:501735990|PMID:20061580  	nrolland	2021-04-07
AT3G45980	gene:2077181	AT3G45980.1	located in	nucleolus	GO:0005730	527	C	nucleolus	HDA	none	NONE	Publication:501714224|PMID:15496452  	kvanwijk	2021-05-10
AT3G45980	locus:2077182	AT3G45980	acts upstream of or within	regulation of nucleobase-containing compound metabolic process	GO:0019219	10748	P	nucleobase-containing compound metabolic process	IEA	traceable computational prediction	AGI_LocusCode:AT1G21970|AGI_LocusCode:AT1G56170|AGI_LocusCode:AT3G19290|AGI_LocusCode:AT4G34000	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G45980	gene:2077181	AT3G45980.1	located in	chloroplast thylakoid	GO:0009534	704	C	thylakoid	HDA	none	NONE	Publication:501735990|PMID:20061580  	nrolland	2021-04-07
AT3G45980	locus:2077182	AT3G45980	acts upstream of or within	RNA metabolic process	GO:0016070	4899	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G21970|AGI_LocusCode:AT1G56170|AGI_LocusCode:AT3G19290|AGI_LocusCode:AT4G34000	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G45980	gene:2077181	AT3G45980.1	located in	chloroplast thylakoid	GO:0009534	704	C	chloroplast	HDA	none	NONE	Publication:501735990|PMID:20061580  	nrolland	2021-04-07
AT3G45980	locus:2077182	AT3G45980	enables	DNA binding	GO:0003677	961	F	DNA binding	IBA	none	PANTHER:PTN000603808|CGD:CAL0000199081|SGD:S000002632	Communication:501741973		2022-11-23
AT3G45980	locus:2077182	AT3G45980	acts upstream of or within	regulation of nucleobase-containing compound metabolic process	GO:0019219	10748	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G21970|AGI_LocusCode:AT1G56170|AGI_LocusCode:AT3G19290|AGI_LocusCode:AT4G34000	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G45980	locus:2077182	AT3G45980	acts upstream of or within	regulation of gene expression	GO:0010468	28541	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G21970|AGI_LocusCode:AT1G56170|AGI_LocusCode:AT3G11100|AGI_LocusCode:AT3G19290|AGI_LocusCode:AT4G34000	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G45980	gene:2077181	AT3G45980.1	located in	chloroplast stroma	GO:0009570	178	C	plastid	HDA	none	NONE	Publication:501735990|PMID:20061580  	nrolland	2021-04-07
AT3G45980	locus:2077182	AT3G45980	acts upstream of or within	regulation of nucleobase-containing compound metabolic process	GO:0019219	10748	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT1G21970|AGI_LocusCode:AT1G56170|AGI_LocusCode:AT3G19290|AGI_LocusCode:AT4G34000	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G45980	locus:2077182	AT3G45980	enables	protein heterodimerization activity	GO:0046982	15254	F	protein binding	IEA	none	InterPro:IPR009072	AnalysisReference:501756966		2023-05-23
AT3G45980	locus:2077182	AT3G45980	located in	cytosol	GO:0005829	241	C	cytosol	HDA	none		Publication:501776792|PMID:28887381  	dszymanski	2019-04-04
AT3G45980	gene:2077181	AT3G45980.1	located in	chloroplast thylakoid	GO:0009534	704	C	other intracellular components	HDA	none	NONE	Publication:501735990|PMID:20061580  	nrolland	2021-04-07
AT3G45980	gene:2077181	AT3G45980.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G45980	locus:2077182	AT3G45980	enables	structural constituent of chromatin	GO:0030527	9869	F	structural molecule activity	IEA	none	InterPro:IPR000558	AnalysisReference:501756966		2023-05-23
AT3G45980	locus:2077182	AT3G45980	acts upstream of or within	RNA metabolic process	GO:0016070	4899	P	nucleobase-containing compound metabolic process	IEA	traceable computational prediction	AGI_LocusCode:AT1G21970|AGI_LocusCode:AT1G56170|AGI_LocusCode:AT3G19290|AGI_LocusCode:AT4G34000	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G45980	gene:2077181	AT3G45980.1	located in	chloroplast stroma	GO:0009570	178	C	chloroplast	HDA	none	NONE	Publication:501735990|PMID:20061580  	nrolland	2021-04-07
AT3G45990	locus:2077092	AT3G45990	is active in	cytoplasm	GO:0005737	231	C	cytoplasm	IBA	none	PANTHER:PTN000227258|PomBase:SPAC20G4.06c|SGD:S000003973|WB:WBGene00006794|UniProtKB:P23528|UniProtKB:Q8I467|UniProtKB:C4LVG4|UniProtKB:Q9Y281|UniProtKB:Q580V7|dictyBase:DDB_G0277833|RGD:69285|MGI:MGI:101757|MGI:MGI:1929270	Communication:501741973		2023-06-07
AT3G45990	locus:2077092	AT3G45990	involved in	actin filament depolymerization	GO:0030042	8888	P	cellular component organization	IBA	none	PANTHER:PTN000227258|TAIR:locus:2127073|TAIR:locus:2077107|dictyBase:DDB_G0272568|TAIR:locus:2042546|UniProtKB:P60981|FB:FBgn0011726|SGD:S000003973|WB:WBGene00006794|UniProtKB:P23528|TAIR:locus:2168063|TAIR:locus:2168052|MGI:MGI:101763|dictyBase:DDB_G0277833|UniProtKB:Q39250|TAIR:locus:2156727|MGI:MGI:1929270|MGI:MGI:101757|UniProtKB:Q9Y281|TAIR:locus:2131879|TAIR:locus:2198175	Communication:501741973		2023-06-07
AT3G45990	locus:2077092	AT3G45990	is active in	actin cytoskeleton	GO:0015629	99	C	cytoskeleton	IBA	none	PANTHER:PTN000227258|UniProtKB:Q8I467|UniProtKB:Q8ID92|dictyBase:DDB_G0272568|PomBase:SPAC20G4.06c|SGD:S000003973|WB:WBGene00006794|dictyBase:DDB_G0277833|MGI:MGI:101757|MGI:MGI:1929270|TAIR:locus:2156727|RGD:69285|TAIR:locus:2131879	Communication:501741973		2023-06-07
AT3G45990	locus:2077092	AT3G45990	enables	actin filament binding	GO:0051015	18617	F	protein binding	IBA	none	PANTHER:PTN000227258|PomBase:SPAC20G4.06c|TAIR:locus:2059861|UniProtKB:P23528|SGD:S000003973|WB:WBGene00006794|UniProtKB:P60981|MGI:MGI:101763|UniProtKB:P10668|MGI:MGI:1929270|MGI:MGI:101757|RGD:69285	Communication:501741973		2023-06-07
AT3G45990	locus:2077092	AT3G45990	involved in	actin filament depolymerization	GO:0030042	8888	P	other cellular processes	IBA	none	PANTHER:PTN000227258|TAIR:locus:2127073|TAIR:locus:2077107|dictyBase:DDB_G0272568|TAIR:locus:2042546|UniProtKB:P60981|FB:FBgn0011726|SGD:S000003973|WB:WBGene00006794|UniProtKB:P23528|TAIR:locus:2168063|TAIR:locus:2168052|MGI:MGI:101763|dictyBase:DDB_G0277833|UniProtKB:Q39250|TAIR:locus:2156727|MGI:MGI:1929270|MGI:MGI:101757|UniProtKB:Q9Y281|TAIR:locus:2131879|TAIR:locus:2198175	Communication:501741973		2023-06-07
AT3G45990	locus:2077092	AT3G45990	enables	actin filament binding	GO:0051015	18617	F	other binding	IBA	none	PANTHER:PTN000227258|PomBase:SPAC20G4.06c|TAIR:locus:2059861|UniProtKB:P23528|SGD:S000003973|WB:WBGene00006794|UniProtKB:P60981|MGI:MGI:101763|UniProtKB:P10668|MGI:MGI:1929270|MGI:MGI:101757|RGD:69285	Communication:501741973		2023-06-07
AT3G45990	locus:2077092	AT3G45990	located in	cytosol	GO:0005829	241	C	cytosol	HDA	none		Publication:501761766|PMID:25293756  	dszymanski	2019-04-04
AT3G46000	gene:2077106	AT3G46000.1	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G46000	locus:2077107	AT3G46000	involved in	actin filament depolymerization	GO:0030042	8888	P	cellular component organization	IEA	none	InterPro:IPR017904	AnalysisReference:501756966		2023-06-11
AT3G46000	locus:2077107	AT3G46000	acts upstream of or within	actin filament depolymerization	GO:0030042	8888	P	cellular component organization	IDA	in vitro assay	NONE	Publication:501774113|PMID:28123105  	xiangyun	2017-04-17
AT3G46000	gene:6532545377	AT3G46000.2	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G46000	locus:2077107	AT3G46000	enables	actin filament binding	GO:0051015	18617	F	protein binding	IBA	none	PANTHER:PTN000227258|PomBase:SPAC20G4.06c|TAIR:locus:2059861|UniProtKB:P23528|SGD:S000003973|WB:WBGene00006794|UniProtKB:P60981|MGI:MGI:101763|UniProtKB:P10668|MGI:MGI:1929270|MGI:MGI:101757|RGD:69285	Communication:501741973		2023-06-07
AT3G46000	locus:2077107	AT3G46000	located in	cytosol	GO:0005829	241	C	cytosol	HDA	none		Publication:501761766|PMID:25293756  	dszymanski	2019-04-04
AT3G46000	locus:2077107	AT3G46000	is active in	cytoplasm	GO:0005737	231	C	cytoplasm	IBA	none	PANTHER:PTN000227258|PomBase:SPAC20G4.06c|SGD:S000003973|WB:WBGene00006794|UniProtKB:P23528|UniProtKB:Q8I467|UniProtKB:C4LVG4|UniProtKB:Q9Y281|UniProtKB:Q580V7|dictyBase:DDB_G0277833|RGD:69285|MGI:MGI:101757|MGI:MGI:1929270	Communication:501741973		2023-06-07
AT3G46000	locus:2077107	AT3G46000	located in	cytoplasm	GO:0005737	231	C	cytoplasm	HDA	none	NONE	Publication:501714452|PMID:15610358  	TAIR	2021-04-07
AT3G46000	locus:2077107	AT3G46000	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:Q9ZU34	Publication:501715803|PMID:12417710  		2021-04-01
AT3G46000	gene:6532545378	AT3G46000.3	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G46000	locus:2077107	AT3G46000	located in	nucleus	GO:0005634	537	C	nucleus	HDA	none	NONE	Publication:501714452|PMID:15610358  	TAIR	2021-04-07
AT3G46000	locus:2077107	AT3G46000	located in	actin cytoskeleton	GO:0015629	99	C	cytoskeleton	IEA	none	InterPro:IPR017904	AnalysisReference:501756966		2023-06-11
AT3G46000	locus:2077107	AT3G46000	enables	actin filament binding	GO:0051015	18617	F	other binding	IBA	none	PANTHER:PTN000227258|PomBase:SPAC20G4.06c|TAIR:locus:2059861|UniProtKB:P23528|SGD:S000003973|WB:WBGene00006794|UniProtKB:P60981|MGI:MGI:101763|UniProtKB:P10668|MGI:MGI:1929270|MGI:MGI:101757|RGD:69285	Communication:501741973		2023-06-07
AT3G46000	locus:2077107	AT3G46000	involved in	actin filament depolymerization	GO:0030042	8888	P	other cellular processes	IBA	none	PANTHER:PTN000227258|TAIR:locus:2127073|TAIR:locus:2077107|dictyBase:DDB_G0272568|TAIR:locus:2042546|UniProtKB:P60981|FB:FBgn0011726|SGD:S000003973|WB:WBGene00006794|UniProtKB:P23528|TAIR:locus:2168063|TAIR:locus:2168052|MGI:MGI:101763|dictyBase:DDB_G0277833|UniProtKB:Q39250|TAIR:locus:2156727|MGI:MGI:1929270|MGI:MGI:101757|UniProtKB:Q9Y281|TAIR:locus:2131879|TAIR:locus:2198175	Communication:501741973		2023-06-07
AT3G46000	locus:2077107	AT3G46000	located in	cytosol	GO:0005829	241	C	cytosol	HDA	none		Publication:501776792|PMID:28887381  	dszymanski	2019-04-04
AT3G46000	locus:2077107	AT3G46000	involved in	actin filament depolymerization	GO:0030042	8888	P	cellular component organization	IBA	none	PANTHER:PTN000227258|TAIR:locus:2127073|TAIR:locus:2077107|dictyBase:DDB_G0272568|TAIR:locus:2042546|UniProtKB:P60981|FB:FBgn0011726|SGD:S000003973|WB:WBGene00006794|UniProtKB:P23528|TAIR:locus:2168063|TAIR:locus:2168052|MGI:MGI:101763|dictyBase:DDB_G0277833|UniProtKB:Q39250|TAIR:locus:2156727|MGI:MGI:1929270|MGI:MGI:101757|UniProtKB:Q9Y281|TAIR:locus:2131879|TAIR:locus:2198175	Communication:501741973		2023-06-07
AT3G46000	locus:2077107	AT3G46000	is active in	actin cytoskeleton	GO:0015629	99	C	cytoskeleton	IBA	none	PANTHER:PTN000227258|UniProtKB:Q8I467|UniProtKB:Q8ID92|dictyBase:DDB_G0272568|PomBase:SPAC20G4.06c|SGD:S000003973|WB:WBGene00006794|dictyBase:DDB_G0277833|MGI:MGI:101757|MGI:MGI:1929270|TAIR:locus:2156727|RGD:69285|TAIR:locus:2131879	Communication:501741973		2023-06-07
AT3G46000	locus:2077107	AT3G46000	involved in	actin filament depolymerization	GO:0030042	8888	P	other cellular processes	IEA	none	InterPro:IPR017904	AnalysisReference:501756966		2023-06-11
AT3G46000	locus:2077107	AT3G46000	acts upstream of or within	actin filament depolymerization	GO:0030042	8888	P	other cellular processes	IDA	in vitro assay	NONE	Publication:501774113|PMID:28123105  	xiangyun	2017-04-17
AT3G46010	locus:2077122	AT3G46010	acts upstream of or within	actin filament depolymerization	GO:0030042	8888	P	other cellular processes	IDA	in vitro assay	NONE	Publication:501742755|PMID:21570971  	TAIR	2011-06-30
AT3G46010	locus:2077122	AT3G46010	acts upstream of or within	actin filament depolymerization	GO:0030042	8888	P	cellular component organization	IDA	in vitro assay	NONE	Publication:501774113|PMID:28123105  	xiangyun	2017-04-17
AT3G46010	locus:2077122	AT3G46010	located in	cytosol	GO:0005829	241	C	cytosol	HDA	none		Publication:501776792|PMID:28887381  	dszymanski	2019-04-04
AT3G46010	locus:2077122	AT3G46010	located in	cytosol	GO:0005829	241	C	cytosol	HDA	none		Publication:501761766|PMID:25293756  	dszymanski	2019-04-04
AT3G46010	locus:2077122	AT3G46010	acts upstream of or within	actin filament organization	GO:0007015	4992	P	cellular component organization	IMP	analysis of visible trait	NONE	Publication:501680725|PMID:11402164  	TAIR	2003-09-29
AT3G46010	gene:2077121	AT3G46010.1	located in	cytoplasm	GO:0005737	231	C	cytoplasm	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G46010	locus:2077122	AT3G46010	is active in	actin cytoskeleton	GO:0015629	99	C	cytoskeleton	IBA	none	PANTHER:PTN000227258|UniProtKB:Q8I467|UniProtKB:Q8ID92|dictyBase:DDB_G0272568|PomBase:SPAC20G4.06c|SGD:S000003973|WB:WBGene00006794|dictyBase:DDB_G0277833|MGI:MGI:101757|MGI:MGI:1929270|TAIR:locus:2156727|RGD:69285|TAIR:locus:2131879	Communication:501741973		2023-06-07
AT3G46010	locus:2077122	AT3G46010	acts upstream of or within	actin filament organization	GO:0007015	4992	P	other cellular processes	IMP	analysis of visible trait	NONE	Publication:501680725|PMID:11402164  	TAIR	2003-09-29
AT3G46010	gene:4010712685	AT3G46010.2	located in	chloroplast	GO:0009507	175	C	chloroplast	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G46010	locus:2077122	AT3G46010	enables	actin filament binding	GO:0051015	18617	F	protein binding	IBA	none	PANTHER:PTN000227258|PomBase:SPAC20G4.06c|TAIR:locus:2059861|UniProtKB:P23528|SGD:S000003973|WB:WBGene00006794|UniProtKB:P60981|MGI:MGI:101763|UniProtKB:P10668|MGI:MGI:1929270|MGI:MGI:101757|RGD:69285	Communication:501741973		2023-06-07
AT3G46010	locus:2077122	AT3G46010	involved in	actin filament depolymerization	GO:0030042	8888	P	cellular component organization	IBA	none	PANTHER:PTN000227258|TAIR:locus:2127073|TAIR:locus:2077107|dictyBase:DDB_G0272568|TAIR:locus:2042546|UniProtKB:P60981|FB:FBgn0011726|SGD:S000003973|WB:WBGene00006794|UniProtKB:P23528|TAIR:locus:2168063|TAIR:locus:2168052|MGI:MGI:101763|dictyBase:DDB_G0277833|UniProtKB:Q39250|TAIR:locus:2156727|MGI:MGI:1929270|MGI:MGI:101757|UniProtKB:Q9Y281|TAIR:locus:2131879|TAIR:locus:2198175	Communication:501741973		2023-06-07
AT3G46010	locus:2077122	AT3G46010	acts upstream of or within	actin filament depolymerization	GO:0030042	8888	P	other cellular processes	IDA	in vitro assay	NONE	Publication:501774113|PMID:28123105  	xiangyun	2017-04-17
AT3G46010	locus:2077122	AT3G46010	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	yeast two-hybrid assay	AGI_LocusCode:At3G53310	Publication:501789671|PMID:32391591  	TAIR	2020-08-04
AT3G46010	locus:2077122	AT3G46010	acts upstream of or within	actin filament depolymerization	GO:0030042	8888	P	cellular component organization	IDA	in vitro assay	NONE	Publication:501742755|PMID:21570971  	TAIR	2011-06-30
AT3G46010	locus:2077122	AT3G46010	enables	actin filament binding	GO:0051015	18617	F	other binding	IBA	none	PANTHER:PTN000227258|PomBase:SPAC20G4.06c|TAIR:locus:2059861|UniProtKB:P23528|SGD:S000003973|WB:WBGene00006794|UniProtKB:P60981|MGI:MGI:101763|UniProtKB:P10668|MGI:MGI:1929270|MGI:MGI:101757|RGD:69285	Communication:501741973		2023-06-07
AT3G46010	locus:2077122	AT3G46010	involved in	actin filament depolymerization	GO:0030042	8888	P	other cellular processes	IBA	none	PANTHER:PTN000227258|TAIR:locus:2127073|TAIR:locus:2077107|dictyBase:DDB_G0272568|TAIR:locus:2042546|UniProtKB:P60981|FB:FBgn0011726|SGD:S000003973|WB:WBGene00006794|UniProtKB:P23528|TAIR:locus:2168063|TAIR:locus:2168052|MGI:MGI:101763|dictyBase:DDB_G0277833|UniProtKB:Q39250|TAIR:locus:2156727|MGI:MGI:1929270|MGI:MGI:101757|UniProtKB:Q9Y281|TAIR:locus:2131879|TAIR:locus:2198175	Communication:501741973		2023-06-07
AT3G46010	locus:2077122	AT3G46010	is active in	cytoplasm	GO:0005737	231	C	cytoplasm	IBA	none	PANTHER:PTN000227258|PomBase:SPAC20G4.06c|SGD:S000003973|WB:WBGene00006794|UniProtKB:P23528|UniProtKB:Q8I467|UniProtKB:C4LVG4|UniProtKB:Q9Y281|UniProtKB:Q580V7|dictyBase:DDB_G0277833|RGD:69285|MGI:MGI:101757|MGI:MGI:1929270	Communication:501741973		2023-06-07
AT3G46010	locus:2077122	AT3G46010	enables	actin binding	GO:0003779	1353	F	protein binding	TAS	inferred by author, from structural similarity	NONE	Publication:5924|PMID:11025548  	TAIR	2003-09-29
AT3G46010	locus:2077122	AT3G46010	located in	actin cytoskeleton	GO:0015629	99	C	cytoskeleton	IEA	none	InterPro:IPR017904	AnalysisReference:501756966		2023-06-11
AT3G46020	locus:2077137	AT3G46020	involved in	positive regulation of mRNA splicing, via spliceosome	GO:0048026	14130	P	nucleobase-containing compound metabolic process	IBA	none	PANTHER:PTN000391532|UniProtKB:P62995|MGI:MGI:1933972|RGD:1306751|MGI:MGI:106016	Communication:501741973		2023-06-07
AT3G46020	locus:2077137	AT3G46020	involved in	positive regulation of mRNA splicing, via spliceosome	GO:0048026	14130	P	other metabolic processes	IBA	none	PANTHER:PTN000391532|UniProtKB:P62995|MGI:MGI:1933972|RGD:1306751|MGI:MGI:106016	Communication:501741973		2023-06-07
AT3G46020	locus:2077137	AT3G46020	part of	spliceosomal complex	GO:0005681	684	C	nucleus	IBA	none	PANTHER:PTN000391532|RGD:1306751|UniProtKB:P62995|UniProtKB:P38159	Communication:501741973		2023-06-07
AT3G46020	locus:2077137	AT3G46020	enables	RNA binding	GO:0003723	1217	F	RNA binding	IBA	none	PANTHER:PTN000391532|FB:FBgn0003742|MGI:MGI:97286|UniProtKB:P19338|MGI:MGI:893588|UniProtKB:Q494N5|RGD:1565256|UniProtKB:P0DJD3|RGD:3153|RGD:1306751|UniProtKB:Q14011|UniProtKB:P0DJD4|UniProtKB:P62995|UniProtKB:P38159	Communication:501741973		2023-06-07
AT3G46020	locus:2077137	AT3G46020	involved in	positive regulation of mRNA splicing, via spliceosome	GO:0048026	14130	P	other cellular processes	IBA	none	PANTHER:PTN000391532|UniProtKB:P62995|MGI:MGI:1933972|RGD:1306751|MGI:MGI:106016	Communication:501741973		2023-06-07
AT3G46030	gene:2077151	AT3G46030.1	located in	plasmodesma	GO:0009506	571	C	other cellular components	HDA	none	NONE	Publication:501742898|PMID:21533090  	maule	2021-04-07
AT3G46030	locus:2077152	AT3G46030	enables	DNA binding	GO:0003677	961	F	DNA binding	IBA	none	PANTHER:PTN000603808|CGD:CAL0000199081|SGD:S000002632	Communication:501741973		2022-11-23
AT3G46030	locus:2077152	AT3G46030	enables	structural constituent of chromatin	GO:0030527	9869	F	structural molecule activity	IEA	none	InterPro:IPR000558	AnalysisReference:501756966		2023-05-23
AT3G46030	locus:2077152	AT3G46030	enables	protein heterodimerization activity	GO:0046982	15254	F	protein binding	IEA	none	InterPro:IPR009072	AnalysisReference:501756966		2023-05-23
AT3G46030	gene:2077151	AT3G46030.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G46030	locus:2077152	AT3G46030	located in	cytosol	GO:0005829	241	C	cytosol	HDA	none		Publication:501776792|PMID:28887381  	dszymanski	2019-04-04
AT3G46040	locus:2075231	AT3G46040	enables	structural constituent of ribosome	GO:0003735	4271	F	structural molecule activity	IBA	none	PANTHER:PTN000204723|MGI:MGI:2389091|SGD:S000003726|SGD:S000004359|FB:FBgn0010198|SGD:S000004767|UniProtKB:P0A7W7|UniProtKB:P62244	Communication:501741973		2023-06-07
AT3G46040	gene:2075230	AT3G46040.1	located in	cytosolic ribosome	GO:0022626	29018	C	ribosome	HDA	none	NONE	Publication:501714926|PMID:15734919  	kvanwijk	2021-04-12
AT3G46040	locus:2075231	AT3G46040	involved in	translation	GO:0006412	6869	P	translation	IEA	none	InterPro:IPR000630|InterPro:IPR035987	AnalysisReference:501756966		2023-05-23
AT3G46040	gene:2075230	AT3G46040.1	located in	cytosolic ribosome	GO:0022626	29018	C	cytosol	HDA	none	NONE	Publication:501714926|PMID:15734919  	kvanwijk	2021-04-12
AT3G46040	locus:2075231	AT3G46040	located in	cytosol	GO:0005829	241	C	cytosol	HDA	none		Publication:501776792|PMID:28887381  	dszymanski	2019-04-04
AT3G46040	locus:2075231	AT3G46040	located in	cytosolic small ribosomal subunit	GO:0022627	29019	C	cytosol	HDA	none	NONE	Publication:501723363|PMID:17934214  	TAIR	2021-04-12
AT3G46040	locus:2075231	AT3G46040	part of	cytosolic small ribosomal subunit	GO:0022627	29019	C	ribosome	IBA	none	PANTHER:PTN000204723|UniProtKB:P62244|RGD:619939|UniProtKB:P0A7W7|SGD:S000003726|MGI:MGI:2389091|SGD:S000004359	Communication:501741973		2023-06-07
AT3G46040	locus:2075231	AT3G46040	part of	cytosolic small ribosomal subunit	GO:0022627	29019	C	cytosol	IBA	none	PANTHER:PTN000204723|UniProtKB:P62244|RGD:619939|UniProtKB:P0A7W7|SGD:S000003726|MGI:MGI:2389091|SGD:S000004359	Communication:501741973		2023-06-07
AT3G46040	gene:2075230	AT3G46040.1	located in	chloroplast	GO:0009507	175	C	chloroplast	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G46040	locus:2075231	AT3G46040	acts upstream of or within	macromolecule modification	GO:0043412	21458	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT3G11100	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G46040	locus:2075231	AT3G46040	located in	cytosolic small ribosomal subunit	GO:0022627	29019	C	ribosome	HDA	none	NONE	Publication:501723363|PMID:17934214  	TAIR	2021-04-12
AT3G46050	locus:2075241	AT3G46050	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	none	NONE	Communication:1345790	TIGR	2004-02-06
AT3G46050	gene:2075240	AT3G46050.1	located in	chloroplast	GO:0009507	175	C	chloroplast	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G46050	locus:2075241	AT3G46050	involved in	biological_process	GO:0008150	5239	P	unknown biological processes	ND	none	NONE	Communication:1345790	TIGR	2022-02-01
AT3G46060	locus:2075251	AT3G46060	acts upstream of or within	abscisic acid-activated signaling pathway involved in stomatal movement	GO:1901527	44084	P	other cellular processes	IMP	analysis of visible trait		Publication:501793839|PMID:33775364  	TAIR	2021-05-06
AT3G46060	locus:2075251	AT3G46060	acts upstream of or within	abscisic acid-activated signaling pathway involved in stomatal movement	GO:1901527	44084	P	signal transduction	IMP	analysis of visible trait		Publication:501793839|PMID:33775364  	TAIR	2021-05-06
AT3G46060	locus:2075251	AT3G46060	involved in	protein secretion	GO:0009306	6901	P	other cellular processes	IBA	none	PANTHER:PTN000635132|CGD:CAL0000186445|WB:WBGene00004272|MGI:MGI:97843|FB:FBgn0052670	Communication:501741973		2023-06-07
AT3G46060	locus:2075251	AT3G46060	involved in	protein secretion	GO:0009306	6901	P	transport	IBA	none	PANTHER:PTN000635132|CGD:CAL0000186445|WB:WBGene00004272|MGI:MGI:97843|FB:FBgn0052670	Communication:501741973		2023-06-07
AT3G46060	locus:2075251	AT3G46060	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:Q8LA96	Publication:501743366|PMID:21798944  		2023-05-23
AT3G46060	locus:2075251	AT3G46060	involved in	vesicle docking involved in exocytosis	GO:0006904	6502	P	other cellular processes	IBA	none	PANTHER:PTN000635132|UniProtKB:P61006	Communication:501741973		2021-03-31
AT3G46060	gene:2075250	AT3G46060.1	located in	plasma membrane	GO:0005886	570	C	plasma membrane	HDA	none	NONE	Publication:501712339|PMID:15060130  	kvanwijk	2021-05-10
AT3G46060	locus:2075251	AT3G46060	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	Bimolecular fluorescence complementation (BiFC)	AGI_LocusCode:AT2G38310	Publication:501793839|PMID:33775364  	TAIR	2021-08-27
AT3G46060	gene:2075250	AT3G46060.1	located in	plant-type vacuole	GO:0000325	11545	C	vacuole	HDA	none	NONE	Publication:501720626|PMID:17151019  	kvanwijk	2021-05-12
AT3G46060	locus:2075251	AT3G46060	involved in	vesicle docking involved in exocytosis	GO:0006904	6502	P	transport	IBA	none	PANTHER:PTN000635132|UniProtKB:P61006	Communication:501741973		2021-03-31
AT3G46060	locus:2075251	AT3G46060	located in	cytosol	GO:0005829	241	C	cytosol	HDA	none		Publication:501776792|PMID:28887381  	dszymanski	2019-04-04
AT3G46060	locus:2075251	AT3G46060	involved in	regulation of exocytosis	GO:0017157	9390	P	other cellular processes	IBA	none	PANTHER:PTN000635132|MGI:MGI:1914545|MGI:MGI:97844|UniProtKB:O95716|MGI:MGI:97843|MGI:MGI:1917158|RGD:3528|UniProtKB:P20336	Communication:501741973		2023-06-07
AT3G46060	locus:2075251	AT3G46060	enables	GTPase activity	GO:0003924	1041	F	hydrolase activity	IBA	none	PANTHER:PTN000635132|CGD:CAL0000186445|UniProtKB:P61006|SGD:S000001889|UniProtKB:P20336|UniProtKB:Q96E17|FB:FBgn0030391|UniProtKB:P20337	Communication:501741973		2023-06-07
AT3G46060	locus:2075251	AT3G46060	acts upstream of or within	ethylene-activated signaling pathway	GO:0009873	8852	P	signal transduction	IEP	Correlation of expression with a physiological assay	NONE	Publication:501705855|PMID:12692329  	TAIR	2003-10-30
AT3G46060	locus:2075251	AT3G46060	acts upstream of or within	ethylene-activated signaling pathway	GO:0009873	8852	P	response to chemical	IEP	Correlation of expression with a physiological assay	NONE	Publication:501705855|PMID:12692329  	TAIR	2003-10-30
AT3G46060	gene:2075250	AT3G46060.1	located in	plasma membrane	GO:0005886	570	C	plasma membrane	HDA	none	NONE	Publication:501722708|PMID:17644812  	kvanwijk	2021-04-07
AT3G46060	locus:2075251	AT3G46060	is active in	intracellular membrane-bounded organelle	GO:0043231	19388	C	other intracellular components	IBA	none	PANTHER:PTN000635132|RGD:620880|FB:FBgn0005586|UniProtKB:D6XLJ6|UniProtKB:O95716|MGI:MGI:1927232|MGI:MGI:96960|UniProtKB:P20337|UniProtKB:Q92930|SGD:S000001889|FB:FBgn0030391|FB:FBgn0052670|TAIR:locus:2142684|MGI:MGI:1917158|MGI:MGI:97843|dictyBase:DDB_G0280043|UniProtKB:P0C0E4|UniProtKB:Q8WXH6|RGD:620922|WB:WBGene00004272|RGD:620923|MGI:MGI:894284|MGI:MGI:97844|UniProtKB:Q96S21|UniProtKB:P61007|WB:WBGene00004267|UniProtKB:P61006|UniProtKB:Q12829|UniProtKB:P51153|UniProtKB:F1PTE3|RGD:3528|MGI:MGI:1914545|RGD:621144|FB:FBgn0262518|RGD:3527	Communication:501741973		2023-06-07
AT3G46060	gene:4010712686	AT3G46060.2	located in	plant-type vacuole	GO:0000325	11545	C	vacuole	HDA	none	NONE	Publication:501720626|PMID:17151019  	kvanwijk	2021-05-12
AT3G46060	gene:4515101605	AT3G46060.3	located in	plasma membrane	GO:0005886	570	C	plasma membrane	HDA	none	NONE	Publication:501722708|PMID:17644812  	kvanwijk	2021-04-07
AT3G46060	locus:2075251	AT3G46060	acts upstream of or within	ethylene-activated signaling pathway	GO:0009873	8852	P	response to endogenous stimulus	IEP	Correlation of expression with a physiological assay	NONE	Publication:501705855|PMID:12692329  	TAIR	2003-10-30
AT3G46060	locus:2075251	AT3G46060	acts upstream of or within	abscisic acid-activated signaling pathway involved in stomatal movement	GO:1901527	44084	P	response to endogenous stimulus	IMP	analysis of visible trait		Publication:501793839|PMID:33775364  	TAIR	2021-05-06
AT3G46060	gene:4515101605	AT3G46060.3	located in	plant-type vacuole	GO:0000325	11545	C	vacuole	HDA	none	NONE	Publication:501720626|PMID:17151019  	kvanwijk	2021-05-12
AT3G46060	locus:2075251	AT3G46060	involved in	regulation of exocytosis	GO:0017157	9390	P	transport	IBA	none	PANTHER:PTN000635132|MGI:MGI:1914545|MGI:MGI:97844|UniProtKB:O95716|MGI:MGI:97843|MGI:MGI:1917158|RGD:3528|UniProtKB:P20336	Communication:501741973		2023-06-07
AT3G46060	gene:2075250	AT3G46060.1	located in	plant-type vacuole	GO:0000325	11545	C	vacuole	HDA	none	NONE	Publication:501714874|PMID:15539469  	kvanwijk	2021-05-12
AT3G46060	locus:2075251	AT3G46060	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:Q8L796	Publication:501735470|PMID:19903693  		2021-04-01
AT3G46060	locus:2075251	AT3G46060	acts upstream of or within	ethylene-activated signaling pathway	GO:0009873	8852	P	other cellular processes	IEP	Correlation of expression with a physiological assay	NONE	Publication:501705855|PMID:12692329  	TAIR	2003-10-30
AT3G46060	locus:2075251	AT3G46060	enables	GTP binding	GO:0005525	1035	F	nucleotide binding	ISS	Sequence similarity (homologue of/most closely related to)	NONE	Publication:5473|PMID:1748311   	TAIR	2006-10-04
AT3G46060	gene:4010712686	AT3G46060.2	located in	plasma membrane	GO:0005886	570	C	plasma membrane	HDA	none	NONE	Publication:501722708|PMID:17644812  	kvanwijk	2021-04-07
AT3G46060	locus:2075251	AT3G46060	acts upstream of or within	abscisic acid-activated signaling pathway involved in stomatal movement	GO:1901527	44084	P	response to chemical	IMP	analysis of visible trait		Publication:501793839|PMID:33775364  	TAIR	2021-05-06
AT3G46060	locus:2075251	AT3G46060	enables	GTP binding	GO:0005525	1035	F	other binding	ISS	Sequence similarity (homologue of/most closely related to)	NONE	Publication:5473|PMID:1748311   	TAIR	2006-10-04
AT3G46060	locus:2075251	AT3G46060	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:Q8LA96	Publication:501790246|PMID:32612234  		2023-05-23
AT3G46070	locus:2075261	AT3G46070	acts upstream of or within	regulation of DNA-templated transcription	GO:0006355	7461	P	other cellular processes	TAS	inferred by author, from sequence similarity	NONE	Publication:1345963|PMID:11118137  	TAIR	2011-03-18
AT3G46070	locus:2075261	AT3G46070	involved in	response to chitin	GO:0010200	17780	P	response to chemical	IBA	none	PANTHER:PTN001980936|TAIR:locus:2168073	Communication:501741973		2021-09-10
AT3G46070	locus:2075261	AT3G46070	acts upstream of or within	regulation of DNA-templated transcription	GO:0006355	7461	P	other metabolic processes	TAS	inferred by author, from sequence similarity	NONE	Publication:1345963|PMID:11118137  	TAIR	2011-03-18
AT3G46070	locus:2075261	AT3G46070	involved in	response to stress	GO:0006950	7330	P	response to stress	IBA	none	PANTHER:PTN001980941|TAIR:locus:2168073|TAIR:locus:2084046|TAIR:locus:2075291	Communication:501741973		2023-06-07
AT3G46070	locus:2075261	AT3G46070	enables	DNA-binding transcription factor activity	GO:0003700	4449	F	DNA-binding transcription factor activity	ISS	Recognized domains		Publication:1345963|PMID:11118137  	TAIR	2003-06-06
AT3G46070	locus:2075261	AT3G46070	acts upstream of or within	regulation of DNA-templated transcription	GO:0006355	7461	P	biosynthetic process	TAS	inferred by author, from sequence similarity	NONE	Publication:1345963|PMID:11118137  	TAIR	2011-03-18
AT3G46070	locus:2075261	AT3G46070	acts upstream of or within	regulation of DNA-templated transcription	GO:0006355	7461	P	nucleobase-containing compound metabolic process	TAS	inferred by author, from sequence similarity	NONE	Publication:1345963|PMID:11118137  	TAIR	2011-03-18
AT3G46080	locus:2075276	AT3G46080	acts upstream of or within	cellular response to hypoxia	GO:0071456	33992	P	response to abiotic stimulus	HEP	none	NONE	Publication:501786480|PMID:31519798  	baileyserr	2019-09-19
AT3G46080	locus:2075276	AT3G46080	acts upstream of or within	regulation of DNA-templated transcription	GO:0006355	7461	P	other cellular processes	TAS	inferred by author, from sequence similarity	NONE	Publication:1345963|PMID:11118137  	TAIR	2011-03-18
AT3G46080	locus:2075276	AT3G46080	involved in	response to chitin	GO:0010200	17780	P	response to chemical	IBA	none	PANTHER:PTN001980936|TAIR:locus:2168073	Communication:501741973		2021-09-10
AT3G46080	locus:2075276	AT3G46080	acts upstream of or within	regulation of DNA-templated transcription	GO:0006355	7461	P	other metabolic processes	TAS	inferred by author, from sequence similarity	NONE	Publication:1345963|PMID:11118137  	TAIR	2011-03-18
AT3G46080	locus:2075276	AT3G46080	acts upstream of or within	regulation of DNA-templated transcription	GO:0006355	7461	P	nucleobase-containing compound metabolic process	TAS	inferred by author, from sequence similarity	NONE	Publication:1345963|PMID:11118137  	TAIR	2011-03-18
AT3G46080	gene:2075275	AT3G46080.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G46080	locus:2075276	AT3G46080	acts upstream of or within	cellular response to hypoxia	GO:0071456	33992	P	response to chemical	HEP	none	NONE	Publication:501786480|PMID:31519798  	baileyserr	2019-09-19
AT3G46080	locus:2075276	AT3G46080	enables	DNA-binding transcription factor activity	GO:0003700	4449	F	DNA-binding transcription factor activity	ISS	Recognized domains		Publication:1345963|PMID:11118137  	TAIR	2003-06-06
AT3G46080	locus:2075276	AT3G46080	acts upstream of or within	cellular response to hypoxia	GO:0071456	33992	P	response to stress	HEP	none	NONE	Publication:501786480|PMID:31519798  	baileyserr	2019-09-19
AT3G46080	locus:2075276	AT3G46080	involved in	response to stress	GO:0006950	7330	P	response to stress	IBA	none	PANTHER:PTN001980941|TAIR:locus:2168073|TAIR:locus:2084046|TAIR:locus:2075291	Communication:501741973		2023-06-07
AT3G46080	locus:2075276	AT3G46080	acts upstream of or within	cellular response to hypoxia	GO:0071456	33992	P	other cellular processes	HEP	none	NONE	Publication:501786480|PMID:31519798  	baileyserr	2019-09-19
AT3G46080	locus:2075276	AT3G46080	acts upstream of or within	regulation of DNA-templated transcription	GO:0006355	7461	P	biosynthetic process	TAS	inferred by author, from sequence similarity	NONE	Publication:1345963|PMID:11118137  	TAIR	2011-03-18
AT3G46086	locus:4515103206	AT3G46086	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	Unknown molecular function		Communication:1345790	TAIR	2011-10-26
AT3G46086	locus:4515103206	AT3G46086	involved in	biological_process	GO:0008150	5239	P	unknown biological processes	ND	Unknown biological process	NONE	Communication:1345790	TAIR	2022-02-01
AT3G46086	locus:4515103206	AT3G46086	is active in	cellular_component	GO:0005575	163	C	unknown cellular components	ND	Unknown cellular component	NONE	Communication:1345790	TAIR	2022-02-01
AT3G46090	gene:2075290	AT3G46090.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G46090	locus:2075291	AT3G46090	acts upstream of or within	cellular response to hypoxia	GO:0071456	33992	P	response to abiotic stimulus	HEP	none	NONE	Publication:501786480|PMID:31519798  	baileyserr	2019-09-19
AT3G46090	locus:2075291	AT3G46090	acts upstream of or within	cellular response to hypoxia	GO:0071456	33992	P	response to chemical	HEP	none	NONE	Publication:501786480|PMID:31519798  	baileyserr	2019-09-19
AT3G46090	locus:2075291	AT3G46090	involved in	negative regulation of growth	GO:0045926	12749	P	growth	IMP	none		Publication:501711809|PMID:14722088  		2021-03-31
AT3G46090	locus:2075291	AT3G46090	acts upstream of or within	cellular response to hypoxia	GO:0071456	33992	P	other cellular processes	HEP	none	NONE	Publication:501786480|PMID:31519798  	baileyserr	2019-09-19
AT3G46090	locus:2075291	AT3G46090	acts upstream of or within	regulation of DNA-templated transcription	GO:0006355	7461	P	other metabolic processes	TAS	inferred by author, from sequence similarity	NONE	Publication:1345963|PMID:11118137  	TAIR	2011-03-18
AT3G46090	locus:2075291	AT3G46090	involved in	response to stress	GO:0006950	7330	P	response to stress	IBA	none	PANTHER:PTN001980941|TAIR:locus:2168073|TAIR:locus:2084046|TAIR:locus:2075291	Communication:501741973		2023-06-07
AT3G46090	locus:2075291	AT3G46090	enables	DNA-binding transcription factor activity	GO:0003700	4449	F	DNA-binding transcription factor activity	ISS	Recognized domains		Publication:1345963|PMID:11118137  	TAIR	2003-06-06
AT3G46090	locus:2075291	AT3G46090	acts upstream of or within	cellular response to hypoxia	GO:0071456	33992	P	response to stress	HEP	none	NONE	Publication:501786480|PMID:31519798  	baileyserr	2019-09-19
AT3G46090	locus:2075291	AT3G46090	acts upstream of or within	regulation of DNA-templated transcription	GO:0006355	7461	P	other cellular processes	TAS	inferred by author, from sequence similarity	NONE	Publication:1345963|PMID:11118137  	TAIR	2011-03-18
AT3G46090	locus:2075291	AT3G46090	acts upstream of or within	regulation of DNA-templated transcription	GO:0006355	7461	P	biosynthetic process	TAS	inferred by author, from sequence similarity	NONE	Publication:1345963|PMID:11118137  	TAIR	2011-03-18
AT3G46090	locus:2075291	AT3G46090	acts upstream of or within	regulation of DNA-templated transcription	GO:0006355	7461	P	nucleobase-containing compound metabolic process	TAS	inferred by author, from sequence similarity	NONE	Publication:1345963|PMID:11118137  	TAIR	2011-03-18
AT3G46090	locus:2075291	AT3G46090	involved in	response to oxidative stress	GO:0006979	6625	P	response to stress	IMP	none		Publication:501711809|PMID:14722088  		2021-03-31
AT3G46090	locus:2075291	AT3G46090	involved in	response to chitin	GO:0010200	17780	P	response to chemical	IBA	none	PANTHER:PTN001980936|TAIR:locus:2168073	Communication:501741973		2021-09-10
AT3G46100	locus:2075306	AT3G46100	acts upstream of or within	DNA-templated transcription	GO:0006351	7470	P	biosynthetic process	IEA	traceable computational prediction	AGI_LocusCode:AT5G03150	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G46100	locus:2075306	AT3G46100	acts upstream of or within	regulation of gene expression	GO:0010468	28541	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT5G03150	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G46100	locus:2075306	AT3G46100	acts upstream of or within	histidyl-tRNA aminoacylation	GO:0006427	6000	P	nucleobase-containing compound metabolic process	ISS	Sequence similarity (homologue of/most closely related to)	NONE	Publication:2300|PMID:9684861   	TAIR	2004-05-12
AT3G46100	locus:2075306	AT3G46100	involved in	histidyl-tRNA aminoacylation	GO:0006427	6000	P	other metabolic processes	IBA	none	PANTHER:PTN000159104|UniProtKB:P60906	Communication:501741973		2021-03-31
AT3G46100	locus:2075306	AT3G46100	enables	histidine-tRNA ligase activity	GO:0004821	2718	F	catalytic activity	ISS	Sequence similarity (homologue of/most closely related to)		Publication:2300|PMID:9684861   	TAIR	2004-05-12
AT3G46100	locus:2075306	AT3G46100	involved in	histidyl-tRNA aminoacylation	GO:0006427	6000	P	nucleobase-containing compound metabolic process	IBA	none	PANTHER:PTN000159104|UniProtKB:P60906	Communication:501741973		2021-03-31
AT3G46100	locus:2075306	AT3G46100	acts upstream of or within	histidyl-tRNA aminoacylation	GO:0006427	6000	P	other metabolic processes	ISS	Sequence similarity (homologue of/most closely related to)	NONE	Publication:2300|PMID:9684861   	TAIR	2004-05-12
AT3G46100	locus:2075306	AT3G46100	acts upstream of or within	DNA-templated transcription	GO:0006351	7470	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT5G03150	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G46100	gene:2075305	AT3G46100.1	located in	chloroplast	GO:0009507	175	C	chloroplast	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G46100	locus:2075306	AT3G46100	located in	mitochondrion	GO:0005739	486	C	mitochondrion	IDA	localization of GFP/YFP fusion protein		Publication:501717830|PMID:16251277  	TAIR	2005-11-17
AT3G46100	locus:2075306	AT3G46100	acts upstream of or within	histidyl-tRNA aminoacylation	GO:0006427	6000	P	other cellular processes	ISS	Sequence similarity (homologue of/most closely related to)	NONE	Publication:2300|PMID:9684861   	TAIR	2004-05-12
AT3G46100	locus:2075306	AT3G46100	acts upstream of or within	DNA-templated transcription	GO:0006351	7470	P	nucleobase-containing compound metabolic process	IEA	traceable computational prediction	AGI_LocusCode:AT5G03150	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G46100	locus:2075306	AT3G46100	located in	chloroplast	GO:0009507	175	C	chloroplast	IDA	localization of GFP/YFP fusion protein		Publication:501717830|PMID:16251277  	TAIR	2005-11-17
AT3G46100	locus:2075306	AT3G46100	located in	mitochondrion	GO:0005739	486	C	mitochondrion	IDA	localization of GFP/YFP fusion protein		Publication:2300|PMID:9684861   	TAIR	2004-05-12
AT3G46100	locus:2075306	AT3G46100	acts upstream of or within	histidyl-tRNA aminoacylation	GO:0006427	6000	P	translation	ISS	Sequence similarity (homologue of/most closely related to)	NONE	Publication:2300|PMID:9684861   	TAIR	2004-05-12
AT3G46100	gene:2075305	AT3G46100.1	located in	chloroplast	GO:0009507	175	C	chloroplast	HDA	none	NONE	Publication:501724486|PMID:18431481  	kvanwijk	2021-04-07
AT3G46100	locus:2075306	AT3G46100	enables	histidine-tRNA ligase activity	GO:0004821	2718	F	catalytic activity	IBA	none	PANTHER:PTN000159104|UniProtKB:P60906	Communication:501741973		2021-04-01
AT3G46100	locus:2075306	AT3G46100	located in	chloroplast	GO:0009507	175	C	chloroplast	IDA	localization of GFP/YFP fusion protein		Publication:2300|PMID:9684861   	TAIR	2004-05-12
AT3G46100	locus:2075306	AT3G46100	involved in	histidyl-tRNA aminoacylation	GO:0006427	6000	P	other cellular processes	IBA	none	PANTHER:PTN000159104|UniProtKB:P60906	Communication:501741973		2021-03-31
AT3G46100	locus:2075306	AT3G46100	acts upstream of or within	DNA-templated transcription	GO:0006351	7470	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT5G03150	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G46100	locus:2075306	AT3G46100	involved in	histidyl-tRNA aminoacylation	GO:0006427	6000	P	translation	IBA	none	PANTHER:PTN000159104|UniProtKB:P60906	Communication:501741973		2021-03-31
AT3G46110	locus:2075331	AT3G46110	involved in	response to osmotic stress	GO:0006970	6618	P	response to abiotic stimulus	IMP	none		Publication:501785578|PMID:31207460  		2021-03-31
AT3G46110	locus:2075331	AT3G46110	involved in	regulation of auxin biosynthetic process	GO:0010600	29507	P	other cellular processes	IMP	none		Publication:501785578|PMID:31207460  		2021-03-31
AT3G46110	locus:2075331	AT3G46110	involved in	inflorescence development	GO:0010229	19795	P	multicellular organism development	IMP	none		Publication:501785578|PMID:31207460  		2021-03-31
AT3G46110	locus:2075331	AT3G46110	involved in	regulation of auxin mediated signaling pathway	GO:0010928	31699	P	response to endogenous stimulus	IMP	none		Publication:501785578|PMID:31207460  		2021-03-31
AT3G46110	locus:2075331	AT3G46110	involved in	regulation of cell division	GO:0051302	20665	P	other cellular processes	IMP	none		Publication:501783868|PMID:30737509  		2021-03-31
AT3G46110	locus:2075331	AT3G46110	involved in	regulation of auxin mediated signaling pathway	GO:0010928	31699	P	other cellular processes	IMP	none		Publication:501785578|PMID:31207460  		2021-03-31
AT3G46110	locus:2075331	AT3G46110	involved in	regulation of root morphogenesis	GO:2000067	35736	P	multicellular organism development	IMP	none		Publication:501783868|PMID:30737509  		2021-03-31
AT3G46110	locus:2075331	AT3G46110	involved in	specification of plant organ axis polarity	GO:0090708	52939	P	anatomical structure development	IMP	none		Publication:501783868|PMID:30737509  		2021-03-31
AT3G46110	locus:2075331	AT3G46110	involved in	regulation of auxin mediated signaling pathway	GO:0010928	31699	P	cell communication	IMP	none		Publication:501785578|PMID:31207460  		2021-03-31
AT3G46110	locus:2075331	AT3G46110	involved in	regulation of auxin biosynthetic process	GO:0010600	29507	P	other metabolic processes	IMP	none		Publication:501785578|PMID:31207460  		2021-03-31
AT3G46110	gene:2075330	AT3G46110.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G46110	locus:2075331	AT3G46110	involved in	regulation of leaf development	GO:2000024	35572	P	anatomical structure development	IMP	none		Publication:501785578|PMID:31207460  		2021-03-31
AT3G46110	locus:2075331	AT3G46110	involved in	plant organ morphogenesis	GO:1905392	52613	P	anatomical structure development	IMP	none		Publication:501783868|PMID:30737509  		2021-03-31
AT3G46110	locus:2075331	AT3G46110	involved in	response to osmotic stress	GO:0006970	6618	P	response to stress	IMP	none		Publication:501785578|PMID:31207460  		2021-03-31
AT3G46110	locus:2075331	AT3G46110	involved in	inflorescence development	GO:0010229	19795	P	anatomical structure development	IMP	none		Publication:501785578|PMID:31207460  		2021-03-31
AT3G46110	locus:2075331	AT3G46110	enables	protein binding	GO:0005515	3877	F	protein binding	IPI	none	UniProtKB:O23702	Publication:501788223|PMID:32004461  		2021-04-01
AT3G46110	locus:2075331	AT3G46110	involved in	plant organ morphogenesis	GO:1905392	52613	P	multicellular organism development	IMP	none		Publication:501783868|PMID:30737509  		2021-03-31
AT3G46110	locus:2075331	AT3G46110	involved in	response to salt	GO:1902074	45327	P	response to chemical	IMP	none		Publication:501785578|PMID:31207460  		2021-03-31
AT3G46110	locus:2075331	AT3G46110	involved in	regulation of auxin mediated signaling pathway	GO:0010928	31699	P	response to chemical	IMP	none		Publication:501785578|PMID:31207460  		2021-03-31
AT3G46110	gene:6532557607	AT3G46110.3	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G46110	locus:2075331	AT3G46110	involved in	regulation of leaf development	GO:2000024	35572	P	multicellular organism development	IMP	none		Publication:501785578|PMID:31207460  		2021-03-31
AT3G46110	gene:1006228427	AT3G46110.2	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G46110	locus:2075331	AT3G46110	involved in	regulation of root morphogenesis	GO:2000067	35736	P	anatomical structure development	IMP	none		Publication:501783868|PMID:30737509  		2021-03-31
AT3G46110	locus:2075331	AT3G46110	involved in	regulation of auxin biosynthetic process	GO:0010600	29507	P	biosynthetic process	IMP	none		Publication:501785578|PMID:31207460  		2021-03-31
AT3G46110	locus:2075331	AT3G46110	involved in	inflorescence development	GO:0010229	19795	P	reproduction	IMP	none		Publication:501785578|PMID:31207460  		2021-03-31
AT3G46110	locus:2075331	AT3G46110	involved in	regulation of auxin mediated signaling pathway	GO:0010928	31699	P	signal transduction	IMP	none		Publication:501785578|PMID:31207460  		2021-03-31
AT3G46110	locus:2075331	AT3G46110	involved in	inflorescence development	GO:0010229	19795	P	post-embryonic development	IMP	none		Publication:501785578|PMID:31207460  		2021-03-31
AT3G46110	locus:2075331	AT3G46110	involved in	specification of plant organ axis polarity	GO:0090708	52939	P	multicellular organism development	IMP	none		Publication:501783868|PMID:30737509  		2021-03-31
AT3G46110	locus:2075331	AT3G46110	enables	protein homodimerization activity	GO:0042803	15211	F	protein binding	ISS	none	UniProtKB:Q9LX14	Communication:501789215		2021-04-01
AT3G46120	locus:2075341	AT3G46120	acts upstream of or within	dephosphorylation	GO:0016311	5574	P	other cellular processes	ISS	none		Publication:501717843|PMID:16244908  		2021-04-01
AT3G46120	locus:2075341	AT3G46120	acts upstream of or within	dephosphorylation	GO:0016311	5574	P	other metabolic processes	ISS	none		Publication:501717843|PMID:16244908  		2021-04-01
AT3G46120	gene:2075340	AT3G46120.1	located in	extracellular region	GO:0005576	294	C	extracellular region	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G46120	locus:2075341	AT3G46120	enables	acid phosphatase activity	GO:0003993	1343	F	hydrolase activity	ISS	Sequence similarity (homologue of/most closely related to)		Publication:501717843|PMID:16244908  	TAIR	2005-12-07
AT3G46130	locus:2075236	AT3G46130	enables	transcription cis-regulatory region binding	GO:0000976	35767	F	DNA binding	IPI	yeast one-hybrid assay	AGI_LocusCode:AT2G39840	Publication:501786139|PMID:30356219  	sibrady	2020-12-18
AT3G46130	locus:2075236	AT3G46130	acts upstream of or within	regulation of DNA-templated transcription	GO:0006355	7461	P	other cellular processes	TAS	inferred by author, from sequence similarity	NONE	Publication:1345963|PMID:11118137  	TAIR	2011-03-18
AT3G46130	locus:2075236	AT3G46130	enables	sequence-specific DNA binding	GO:0043565	22416	F	DNA binding	IEA	none	InterPro:IPR044676	AnalysisReference:501756966		2023-06-11
AT3G46130	locus:2075236	AT3G46130	acts upstream of or within	regulation of DNA-templated transcription	GO:0006355	7461	P	nucleobase-containing compound metabolic process	TAS	inferred by author, from sequence similarity	NONE	Publication:1345963|PMID:11118137  	TAIR	2011-03-18
AT3G46130	locus:2075236	AT3G46130	involved in	regulation of DNA-templated transcription	GO:0006355	7461	P	nucleobase-containing compound metabolic process	IBA	none	PANTHER:PTN002903678|TAIR:locus:2065226|TAIR:locus:2199357|TAIR:locus:2089812|TAIR:locus:2140847|TAIR:locus:2087183|UniProtKB:Q10MB4	Communication:501741973		2023-06-07
AT3G46130	locus:2075236	AT3G46130	enables	sequence-specific DNA binding	GO:0043565	22416	F	DNA binding	IBA	none	PANTHER:PTN002903678|TAIR:locus:2065226|TAIR:locus:2075236|TAIR:locus:2157844|TAIR:locus:2202633|TAIR:locus:2084168|TAIR:locus:2174324|TAIR:locus:2087183|TAIR:locus:2084269	Communication:501741973		2023-06-07
AT3G46130	locus:2075236	AT3G46130	acts upstream of or within	regulation of DNA-templated transcription	GO:0006355	7461	P	other metabolic processes	TAS	inferred by author, from sequence similarity	NONE	Publication:1345963|PMID:11118137  	TAIR	2011-03-18
AT3G46130	locus:2075236	AT3G46130	involved in	regulation of DNA-templated transcription	GO:0006355	7461	P	other metabolic processes	IEA	none	InterPro:IPR044676	AnalysisReference:501756966		2023-06-11
AT3G46130	locus:2075236	AT3G46130	involved in	regulation of DNA-templated transcription	GO:0006355	7461	P	other cellular processes	IBA	none	PANTHER:PTN002903678|TAIR:locus:2065226|TAIR:locus:2199357|TAIR:locus:2089812|TAIR:locus:2140847|TAIR:locus:2087183|UniProtKB:Q10MB4	Communication:501741973		2023-06-07
AT3G46130	locus:2075236	AT3G46130	acts upstream of or within	regulation of DNA-templated transcription	GO:0006355	7461	P	biosynthetic process	TAS	inferred by author, from sequence similarity	NONE	Publication:1345963|PMID:11118137  	TAIR	2011-03-18
AT3G46130	locus:2075236	AT3G46130	involved in	regulation of DNA-templated transcription	GO:0006355	7461	P	other cellular processes	IEA	none	InterPro:IPR044676	AnalysisReference:501756966		2023-06-11
AT3G46130	locus:2075236	AT3G46130	involved in	regulation of DNA-templated transcription	GO:0006355	7461	P	other metabolic processes	IBA	none	PANTHER:PTN002903678|TAIR:locus:2065226|TAIR:locus:2199357|TAIR:locus:2089812|TAIR:locus:2140847|TAIR:locus:2087183|UniProtKB:Q10MB4	Communication:501741973		2023-06-07
AT3G46130	locus:2075236	AT3G46130	involved in	regulation of DNA-templated transcription	GO:0006355	7461	P	biosynthetic process	IEA	none	InterPro:IPR044676	AnalysisReference:501756966		2023-06-11
AT3G46130	locus:2075236	AT3G46130	located in	cytosol	GO:0005829	241	C	cytosol	HDA	none		Publication:501761766|PMID:25293756  	dszymanski	2019-04-04
AT3G46130	gene:2075235	AT3G46130.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G46130	locus:2075236	AT3G46130	is active in	nucleus	GO:0005634	537	C	nucleus	IBA	none	PANTHER:PTN002903678|TAIR:locus:2174324|TAIR:locus:2199357|TAIR:locus:2075236|UniProtKB:Q10MB4|TAIR:locus:2084269|TAIR:locus:2087183	Communication:501741973		2023-06-07
AT3G46130	locus:2075236	AT3G46130	involved in	regulation of DNA-templated transcription	GO:0006355	7461	P	nucleobase-containing compound metabolic process	IEA	none	InterPro:IPR044676	AnalysisReference:501756966		2023-06-11
AT3G46130	locus:2075236	AT3G46130	located in	nucleus	GO:0005634	537	C	nucleus	IDA	localization of GFP/YFP fusion protein		Publication:501718746|PMID:16531467  	TAIR	2009-03-18
AT3G46130	locus:2075236	AT3G46130	enables	transcription cis-regulatory region binding	GO:0000976	35767	F	nucleic acid binding	IPI	yeast one-hybrid assay	AGI_LocusCode:AT2G39840	Publication:501786139|PMID:30356219  	sibrady	2020-12-18
AT3G46130	gene:6530297161	AT3G46130.4	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G46130	gene:4010712687	AT3G46130.3	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G46130	locus:2075236	AT3G46130	enables	DNA-binding transcription factor activity	GO:0003700	4449	F	DNA-binding transcription factor activity	IEA	none	InterPro:IPR044676	AnalysisReference:501756966		2023-06-11
AT3G46130	locus:2075236	AT3G46130	enables	DNA-binding transcription factor activity	GO:0003700	4449	F	DNA-binding transcription factor activity	ISS	Recognized domains		Publication:1345963|PMID:11118137  	TAIR	2003-06-06
AT3G46130	locus:2075236	AT3G46130	involved in	regulation of DNA-templated transcription	GO:0006355	7461	P	biosynthetic process	IBA	none	PANTHER:PTN002903678|TAIR:locus:2065226|TAIR:locus:2199357|TAIR:locus:2089812|TAIR:locus:2140847|TAIR:locus:2087183|UniProtKB:Q10MB4	Communication:501741973		2023-06-07
AT3G46130	gene:1009021761	AT3G46130.2	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G46140	locus:2075246	AT3G46140	acts upstream of or within	phosphorylation	GO:0016310	6748	P	other cellular processes	ISS	none	Pfam:PF00069	Communication:501714663		2021-04-01
AT3G46140	gene:2075245	AT3G46140.1	located in	mitochondrion	GO:0005739	486	C	mitochondrion	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G46140	locus:2075246	AT3G46140	enables	protein kinase activity	GO:0004672	3889	F	transferase activity	IBA	none	PANTHER:PTN001969686|TAIR:locus:2205523|TAIR:locus:2116417|TAIR:locus:2074835|TAIR:locus:2007337	Communication:501741973		2023-06-07
AT3G46140	locus:2075246	AT3G46140	enables	protein kinase activity	GO:0004672	3889	F	kinase activity	IBA	none	PANTHER:PTN001969686|TAIR:locus:2205523|TAIR:locus:2116417|TAIR:locus:2074835|TAIR:locus:2007337	Communication:501741973		2023-06-07
AT3G46140	locus:2075246	AT3G46140	enables	protein kinase activity	GO:0004672	3889	F	catalytic activity	IBA	none	PANTHER:PTN001969686|TAIR:locus:2205523|TAIR:locus:2116417|TAIR:locus:2074835|TAIR:locus:2007337	Communication:501741973		2023-06-07
AT3G46140	locus:2075246	AT3G46140	involved in	signal transduction	GO:0007165	7243	P	signal transduction	IBA	none	PANTHER:PTN001969686|TAIR:locus:2205523|TAIR:locus:2074835|TAIR:locus:2060246	Communication:501741973		2023-04-13
AT3G46140	locus:2075246	AT3G46140	involved in	protein phosphorylation	GO:0006468	6897	P	other cellular processes	IBA	none	PANTHER:PTN001969686|TAIR:locus:2007337|TAIR:locus:2205523|TAIR:locus:2074835|TAIR:locus:2116417	Communication:501741973		2023-06-07
AT3G46140	locus:2075246	AT3G46140	involved in	protein phosphorylation	GO:0006468	6897	P	other metabolic processes	IBA	none	PANTHER:PTN001969686|TAIR:locus:2007337|TAIR:locus:2205523|TAIR:locus:2074835|TAIR:locus:2116417	Communication:501741973		2023-06-07
AT3G46140	locus:2075246	AT3G46140	acts upstream of or within	phosphorylation	GO:0016310	6748	P	other metabolic processes	ISS	none	Pfam:PF00069	Communication:501714663		2021-04-01
AT3G46150	locus:2075266	AT3G46150	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	none	NONE	Communication:1345790	TIGR	2004-02-06
AT3G46150	locus:2075266	AT3G46150	involved in	biological_process	GO:0008150	5239	P	unknown biological processes	ND	Unknown biological process	NONE	Communication:1345790	TAIR	2022-02-01
AT3G46150	locus:2075266	AT3G46150	is active in	cellular_component	GO:0005575	163	C	unknown cellular components	ND	Unknown cellular component	NONE	Communication:1345790	TAIR	2022-02-01
AT3G46160	locus:2075281	AT3G46160	enables	protein kinase activity	GO:0004672	3889	F	transferase activity	IBA	none	PANTHER:PTN001969686|TAIR:locus:2205523|TAIR:locus:2116417|TAIR:locus:2074835|TAIR:locus:2007337	Communication:501741973		2023-06-07
AT3G46160	locus:2075281	AT3G46160	enables	protein kinase activity	GO:0004672	3889	F	kinase activity	IBA	none	PANTHER:PTN001969686|TAIR:locus:2205523|TAIR:locus:2116417|TAIR:locus:2074835|TAIR:locus:2007337	Communication:501741973		2023-06-07
AT3G46160	locus:2075281	AT3G46160	enables	ATP binding	GO:0005524	894	F	other binding	IEA	none	InterPro:IPR000719	AnalysisReference:501756966		2023-06-11
AT3G46160	locus:2075281	AT3G46160	involved in	signal transduction	GO:0007165	7243	P	signal transduction	IBA	none	PANTHER:PTN001969686|TAIR:locus:2205523|TAIR:locus:2074835|TAIR:locus:2060246	Communication:501741973		2023-04-13
AT3G46160	locus:2075281	AT3G46160	enables	protein kinase activity	GO:0004672	3889	F	catalytic activity	IBA	none	PANTHER:PTN001969686|TAIR:locus:2205523|TAIR:locus:2116417|TAIR:locus:2074835|TAIR:locus:2007337	Communication:501741973		2023-06-07
AT3G46160	locus:2075281	AT3G46160	acts upstream of or within	phosphorylation	GO:0016310	6748	P	other metabolic processes	ISS	none	Pfam:PF00069	Communication:501714663		2021-04-01
AT3G46160	locus:2075281	AT3G46160	involved in	protein phosphorylation	GO:0006468	6897	P	other cellular processes	IBA	none	PANTHER:PTN001969686|TAIR:locus:2007337|TAIR:locus:2205523|TAIR:locus:2074835|TAIR:locus:2116417	Communication:501741973		2023-06-07
AT3G46160	locus:2075281	AT3G46160	involved in	protein phosphorylation	GO:0006468	6897	P	other metabolic processes	IBA	none	PANTHER:PTN001969686|TAIR:locus:2007337|TAIR:locus:2205523|TAIR:locus:2074835|TAIR:locus:2116417	Communication:501741973		2023-06-07
AT3G46160	locus:2075281	AT3G46160	acts upstream of or within	phosphorylation	GO:0016310	6748	P	other cellular processes	ISS	none	Pfam:PF00069	Communication:501714663		2021-04-01
AT3G46160	locus:2075281	AT3G46160	enables	ATP binding	GO:0005524	894	F	nucleotide binding	IEA	none	InterPro:IPR000719	AnalysisReference:501756966		2023-06-11
AT3G46160	gene:2075280	AT3G46160.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G46170	locus:2075296	AT3G46170	is active in	cellular_component	GO:0005575	163	C	unknown cellular components	ND	Unknown cellular component	NONE	Communication:1345790	TAIR	2022-02-01
AT3G46170	locus:2075296	AT3G46170	enables	oxidoreductase activity	GO:0016491	3460	F	catalytic activity	IEA	none	InterPro:IPR020904	AnalysisReference:501756966		2023-05-23
AT3G46170	locus:2075296	AT3G46170	acts upstream of or within	defense response to other organism	GO:0098542	46569	P	response to external stimulus	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G46170	locus:2075296	AT3G46170	acts upstream of or within	defense response to other organism	GO:0098542	46569	P	response to biotic stimulus	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G46170	locus:2075296	AT3G46170	acts upstream of or within	defense response to other organism	GO:0098542	46569	P	response to stress	IEA	traceable computational prediction	AGI_LocusCode:AT1G32640	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G46180	locus:2075311	AT3G46180	enables	3-phosphoadenosine 5-phosphosulfate transmembrane transporter activity	GO:0046964	15191	F	transporter activity	IBA	none	PANTHER:PTN000081360|UniProtKB:Q9VEI3|UniProtKB:Q8TB61|WB:WBGene00004206|MGI:MGI:1921086	Communication:501741973		2023-06-07
AT3G46180	locus:2075311	AT3G46180	involved in	transmembrane transport	GO:0055085	28452	P	transport	IEA	none	InterPro:IPR013657	AnalysisReference:501756966		2023-05-23
AT3G46180	locus:2075311	AT3G46180	involved in	transmembrane transport	GO:0055085	28452	P	other cellular processes	IEA	none	InterPro:IPR013657	AnalysisReference:501756966		2023-05-23
AT3G46180	gene:6532554905	AT3G46180.2	located in	plasma membrane	GO:0005886	570	C	plasma membrane	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G46180	locus:2075311	AT3G46180	enables	galactose transmembrane transporter activity	GO:0005354	2417	F	transporter activity	ISS	Sequence similarity (homologue of/most closely related to)		Communication:1674994	TAIR	2003-03-29
AT3G46190	locus:2075321	AT3G46190	acts upstream of or within	organic cyclic compound biosynthetic process	GO:1901362	43684	P	biosynthetic process	IEA	traceable computational prediction	AGI_LocusCode:AT4G27410	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G46190	locus:2075321	AT3G46190	acts upstream of or within	organic cyclic compound biosynthetic process	GO:1901362	43684	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT4G27410	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G46190	locus:2075321	AT3G46190	acts upstream of or within	aromatic compound biosynthetic process	GO:0019438	10128	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT4G27410	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G46190	locus:2075321	AT3G46190	acts upstream of or within	response to lipid	GO:0033993	28865	P	response to chemical	IEA	traceable computational prediction	AGI_LocusCode:AT4G27410	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G46190	gene:6532551618	AT3G46190.2	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G46190	locus:2075321	AT3G46190	acts upstream of or within	aromatic compound biosynthetic process	GO:0019438	10128	P	biosynthetic process	IEA	traceable computational prediction	AGI_LocusCode:AT4G27410	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G46190	locus:2075321	AT3G46190	acts upstream of or within	secondary metabolic process	GO:0019748	10327	P	secondary metabolic process	IEA	traceable computational prediction	AGI_LocusCode:AT5G63790	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G46190	gene:2075320	AT3G46190.1	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G46190	locus:2075321	AT3G46190	enables	molecular_function	GO:0003674	3226	F	unknown molecular functions	ND	none	NONE	Communication:1345790	TIGR	2003-09-06
AT3G46190	locus:2075321	AT3G46190	acts upstream of or within	oxoacid metabolic process	GO:0043436	21524	P	other metabolic processes	IEA	traceable computational prediction	AGI_LocusCode:AT5G63790	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G46190	locus:2075321	AT3G46190	acts upstream of or within	oxoacid metabolic process	GO:0043436	21524	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT5G63790	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G46190	locus:2075321	AT3G46190	acts upstream of or within	aromatic compound biosynthetic process	GO:0019438	10128	P	other cellular processes	IEA	traceable computational prediction	AGI_LocusCode:AT4G27410	Publication:501796011|PMID:34562334  	klaasvdp	2022-11-14
AT3G46200	gene:6532552774	AT3G46200.6	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G46200	gene:6532552776	AT3G46200.5	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G46200	gene:6532552775	AT3G46200.4	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G46200	locus:2075336	AT3G46200	acts upstream of or within	cellular response to ammonium ion	GO:0071242	33670	P	response to chemical	IMP	biochemical/chemical analysis	NONE	Publication:501764619|PMID:26049160  	TAIR	2015-07-17
AT3G46200	locus:2075336	AT3G46200	enables	GDP-mannose hydrolase activity	GO:0052751	39013	F	hydrolase activity	IBA	none	PANTHER:PTN001274688|TAIR:locus:2075336	Communication:501741973		2021-04-02
AT3G46200	locus:2075336	AT3G46200	acts upstream of or within	cellular response to ammonium ion	GO:0071242	33670	P	other cellular processes	IMP	biochemical/chemical analysis	NONE	Publication:501764619|PMID:26049160  	TAIR	2015-07-17
AT3G46200	gene:6532552771	AT3G46200.2	located in	nucleus	GO:0005634	537	C	nucleus	ISM	predicted protein features		AnalysisReference:501780126	rkaundal	2018-08-31
AT3G46200	locus:2075336	AT3G46200	enables	GDP-mannose hydrolase activity	GO:0052751	39013	F	hydrolase activity	IDA	Enzyme assays		Publication:501764619|PMID:26049160  	TAIR	2015-07-17
AT3G46200	gene:2075335	AT3G46200.1	located in	cytosol	GO:0005829	241	C	cytosol	ISM	predicted protein features		Publication:501716296|PMID:15878881  	TAIR	2018-03-22
AT3G46210	locus:2075351	AT3G46210	involved in	polyadenylation-dependent snoRNA 3-end processing</li>
</ul>
<h2 id="literature">Literature:</h2>
<ul>
<li><a href="https://www.doi.org/10.1038/s41477-019-0562-1"




 target="_blank"
 


>Prevalent cytidylation and uridylation of precursor miRNAs in Arabidopsis.</a>
  DOI:  <a href="https://www.doi.org/10.1038/s41477-019-0562-1"




 target="_blank"
 


>10.1038/s41477-019-0562-1</a>
;  PMID:  <a href="https://pubmed.ncbi.nlm.nih.gov/31792392/"




 target="_blank"
 


>31792392</a>
</li>
</ul>
<h2 id="sequence">Sequence:</h2>
<p>





  



<div class="accordion ">
  <div class="accordion-header " data-accordion>
    cDNA Sequence
    <svg
      class="accordion-icon"
      x="0px"
      y="0px"
      viewBox="0 0 512 512"
      xmlspace="preserve">
      <path
        fill="currentColor"
        d="M505.755,123.592c-8.341-8.341-21.824-8.341-30.165,0L256.005,343.176L36.421,123.592c-8.341-8.341-21.824-8.341-30.165,0 s-8.341,21.824,0,30.165l234.667,234.667c4.16,4.16,9.621,6.251,15.083,6.251c5.462,0,10.923-2.091,15.083-6.251l234.667-234.667 C514.096,145.416,514.096,131.933,505.755,123.592z"></path>
    </svg>
</div>


  <div class="accordion-content ">
    <div class="content-wrapper">
    <button class="copy-btn" aria-label="Copy Content">Copy</button>
    </div>
  
      <div class="text-to-copy">
    <ul>
<li>&gt;AT3G45760.2
CTCGGTTCTCCATATCCCAATATTGTTTGATCTCATCCTAAGTACTCTATAACTCCCTTAGATCTTAGTTTGATAAAAAAAAAAAATAGGTTTAATAGCTCTGCTTGAGAGGTACTATGGACTTTCGAAAAAATCAAGCTGGGGAAAAGAATGTTTCGAGTAAAGGGATTCAAAAGAAAGTGAAGAACACTATGTCCATAGCATCTAAGAGATATAACATTGATAGTTATATCTTACTAGACCTTGATAAGGTATTAGATGATGCCTACAGCTCTTTTCGACCGGTTTCTGCTGATTACAATACTAGGAAGGAATTGGTCAAAAACCTCAATGCTATGGCTATAGATATTTTTGGTAAATCAGAAGAGAGCAGCCCGGTCCTTGAGGCGTATGGCTCTTTTGCGATGAATACGTTTTCCTCACAAAAGGATTTGGATGTATCTATTAACTTTAGTAGCGGAACATCTGAATTTTATCGTGAAAAGAAGCTAGAAATTCTTACAAGATTTGCAACGAAGCTCCGCTCTTTGGAGGGGCAAGGGTTTGTCAGGAATGTTGTACCTATCTTAAGTGCTAGGGTTCCAATTGTCAGATTCTGTGATCAAGGGACTGGTATCGAATGTGATTTGACAGTTGAAAGCAAAGACGGCATTTTAACTTCACAGATCATTCGGATTATATCTCAAATTGATGATAGGTTTCAGAAACTTTGTTTGTTGCACTGTCAATTGTTTAGTACTAATGTGAATATTGTCATATGTCAGATTAAGCATTGGGCCAGAGCACATGGGGTTAACAATGCGTCACACAATACCCTGAATTCGATATCTATAACAATGCTAGTCGCTCATCATTTACAGACTCAAAGTCCTCCAATTTTACCTCCATTCTCTACACTATTTAAAGATGGAATCGATCCTCCTATCGTGGAGAAAAGAACACAAAAGTTCTTAAACTGGGGACAACGTAATCAAGAATCTCTTGGTAGACTGTTTGCAACTTTTTTTATTAAGGTTGAGGATTTTACAGACGTGGCTAGAAATTTTGCAAGGGTTTTTAGTGACTACGGAGCCGAAAAGATATACAGCTCCATAAACAGAACAGTAGATGATATATTGGAGTTCCTCAATGGCAAGGTTGCTGGAGAACATCTCACACAAAAGTTGTTTAGCCAACAAACTGTTGTGGAGCCTATTCCTCCTGTGTCGCCACAACAACCTCATAAGAAAAGAGTCTGCTTGGAAAAGGGATACAGAGCTGTAGGAGGGATCAGTCACGGAAGAGAGGAGACTCATGAAACTTCTAGAGGAAAACGAAAAAGGTCTGGTGGAAACTGGGGCTGTAGGTTTAGTGGTGGTGAAGAACCAATGCCTGTGGGTCTGTGGAATGAGTACAGTCGAAGTCTTGATATGCCAACACCACCACCACATTATGATAAACTTCTTTATAAGAAGTATTAGAGGTCAGATGTTAATGATTCGATTAATACAAAAGATTGATTTTGAGTTTTGACTATCAAAGTAACAGAAAAGATGCTTGAGTGAGAAATTTACCTTGGAGATTTTTCGTAGGGAGAGAGGGCAGAGTTCCTCTCAAGACTGTTGAACTCATGTGTGGACATTGTAGTGGTGCAAACCGGCACCATCCTCACATTGGTTTAACCAGTCAGAAACCAGAACCACCTTTGATGACTCGCGGTCTACATTCTCAGGTTT</li>
<li>&gt;AT3G45760.1
CTCGGTTCTCCATATCCCAATATTGTTTGATCTCATCCTAAGTACTCTATAACTCCCTTAGATCTTAGTTTGATAAAAAAAAAAAATAGGTTTAATAGCTCTGCTTGAGAGGTACTATGGACTTTCGAAAAAATCAAGCTGGGGAAAAGAATGTTTCGAGTAAAGGGATTCAAAAGAAAGTGAAGAACACTATGTCCATAGCATCTAAGAGATATAACATTGATAGTTATATCTTACTAGACCTTGATAAGGTATTAGATGATGCCTACAGCTCTTTTCGACCGGTTTCTGCTGATTACAATACTAGGAAGGAATTGGTCAAAAACCTCAATGCTATGGCTATAGATATTTTTGGTAAATCAGAAGAGAGCAGCCCGGTCCTTGAGGCGTATGGCTCTTTTGCGATGAATACGTTTTCCTCACAAAAGGATTTGGATGTATCTATTAACTTTAGTAGCGGAACATCTGAATTTTATCGTGAAAAGAAGCTAGAAATTCTTACAAGATTTGCAACGAAGCTCCGCTCTTTGGAGGGGCAAGGGTTTGTCAGGAATGTTGTACCTATCTTAAGTGCTAGGGTTCCAATTGTCAGATTCTGTGATCAAGGGACTGGTATCGAATGTGATTTGACAGTTGAAAGCAAAGACGGCATTTTAACTTCACAGATCATTCGGATTATATCTCAAATTGATGATAGGTTTCAGAAACTTTGTTTGTTGCACTGTCAATTGTTTAGTACTAATGTGAATATTGTCATATGTCAGATTAAGCATTGGGCCAGAGCACATGGGGTTAACAATGCGTCACACAATACCCTGAATTCGATATCTATAACAATGCTAGTCGCTCATCATTTACAGACTCAAAGTCCTCCAATTTTACCTCCATTCTCTACACTATTTAAAGATGGAATCGATCCTCCTATCGTGGAGAAAAGAACACAAAAGTTCTTAAACTGGGGACAACGTAATCAAGAATCTCTTGGTAGACTGTTTGCAACTTTTTTTATTAAGGTTGAGGATTTTACAGACGTGGCTAGAAATTTTGCAAGGGTTTTTAGTGACTACGGAGCCGAAAAGATATACAGCTCCATAAACAGAACAGTAGATGATATATTGGAGTTCCTCAATGGCAAGGTTGCTGGAGAACATCTCACACAAAAGTTGTTTAGCCAACAAACTGTTGTGGAGCCTATTCCTCCTGTGTCGCCACAACAACCTCATAAGAAAAGAGTCTGCTTGGAAAAGGGATACAGAGCTGTAGGAGGGATCAGTCACGGAAGAGAGGAGACTCATGAAACTTCTAGAGGAAAACGAAAAAGGTCTGGTGGAAACTGGGGCTGTAGGTTTAGTGGTGGTGAAGAACCAATGCCTGTGGGTCTGTGGAATGAGTACAGTCGAAGTCTTGATATGCCAACACCACCACCACATTATGATAAACTTCTTTATAAGAAGTATTAGAGGTCAGATGTTAATGATTCGATTAATACAAAAGATTGATTTTGAGTTTTGACTATCAAAGTAACAGAAAAGATGCTTGAGTGAGAAATTTACCTTGGAGATTTTTCGTAGGGAGAGAGGGCAGAGTTCCTCTCAAGACTGTTGAACTCATGTGTGGACATTGTAGTGGTGCAAACCGGCACCATCCTCACATTGGTTTAACCAGTCAGAAACCAGAACCACCTTTGATGACTCGCGGTCTACATTCTCAGGTTT</li>
</ul>

      </div>
  </div>
</div>








  



<div class="accordion ">
  <div class="accordion-header " data-accordion>
    CDS Sequence
    <svg
      class="accordion-icon"
      x="0px"
      y="0px"
      viewBox="0 0 512 512"
      xmlspace="preserve">
      <path
        fill="currentColor"
        d="M505.755,123.592c-8.341-8.341-21.824-8.341-30.165,0L256.005,343.176L36.421,123.592c-8.341-8.341-21.824-8.341-30.165,0 s-8.341,21.824,0,30.165l234.667,234.667c4.16,4.16,9.621,6.251,15.083,6.251c5.462,0,10.923-2.091,15.083-6.251l234.667-234.667 C514.096,145.416,514.096,131.933,505.755,123.592z"></path>
    </svg>
</div>


  <div class="accordion-content ">
    <div class="content-wrapper">
    <button class="copy-btn" aria-label="Copy Content">Copy</button>
    </div>
  
      <div class="text-to-copy">
    <ul>
<li>&gt;AT3G45760.2
ATGGACTTTCGAAAAAATCAAGCTGGGGAAAAGAATGTTTCGAGTAAAGGGATTCAAAAGAAAGTGAAGAACACTATGTCCATAGCATCTAAGAGATATAACATTGATAGTTATATCTTACTAGACCTTGATAAGGTATTAGATGATGCCTACAGCTCTTTTCGACCGGTTTCTGCTGATTACAATACTAGGAAGGAATTGGTCAAAAACCTCAATGCTATGGCTATAGATATTTTTGGTAAATCAGAAGAGAGCAGCCCGGTCCTTGAGGCGTATGGCTCTTTTGCGATGAATACGTTTTCCTCACAAAAGGATTTGGATGTATCTATTAACTTTAGTAGCGGAACATCTGAATTTTATCGTGAAAAGAAGCTAGAAATTCTTACAAGATTTGCAACGAAGCTCCGCTCTTTGGAGGGGCAAGGGTTTGTCAGGAATGTTGTACCTATCTTAAGTGCTAGGGTTCCAATTGTCAGATTCTGTGATCAAGGGACTGGTATCGAATGTGATTTGACAGTTGAAAGCAAAGACGGCATTTTAACTTCACAGATCATTCGGATTATATCTCAAATTGATGATAGGTTTCAGAAACTTTGTTTGTTGCACTGTCAATTGTTTAGTACTAATGTGAATATTGTCATATGTCAGATTAAGCATTGGGCCAGAGCACATGGGGTTAACAATGCGTCACACAATACCCTGAATTCGATATCTATAACAATGCTAGTCGCTCATCATTTACAGACTCAAAGTCCTCCAATTTTACCTCCATTCTCTACACTATTTAAAGATGGAATCGATCCTCCTATCGTGGAGAAAAGAACACAAAAGTTCTTAAACTGGGGACAACGTAATCAAGAATCTCTTGGTAGACTGTTTGCAACTTTTTTTATTAAGGTTGAGGATTTTACAGACGTGGCTAGAAATTTTGCAAGGGTTTTTAGTGACTACGGAGCCGAAAAGATATACAGCTCCATAAACAGAACAGTAGATGATATATTGGAGTTCCTCAATGGCAAGGTTGCTGGAGAACATCTCACACAAAAGTTGTTTAGCCAACAAACTGTTGTGGAGCCTATTCCTCCTGTGTCGCCACAACAACCTCATAAGAAAAGAGTCTGCTTGGAAAAGGGATACAGAGCTGTAGGAGGGATCAGTCACGGAAGAGAGGAGACTCATGAAACTTCTAGAGGAAAACGAAAAAGGTCTGGTGGAAACTGGGGCTGTAGGTTTAGTGGTGGTGAAGAACCAATGCCTGTGGGTCTGTGGAATGAGTACAGTCGAAGTCTTGATATGCCAACACCACCACCACATTATGATAAACTTCTTTATAAGAAGTATTAG</li>
<li>&gt;AT3G45760.1
ATGGACTTTCGAAAAAATCAAGCTGGGGAAAAGAATGTTTCGAGTAAAGGGATTCAAAAGAAAGTGAAGAACACTATGTCCATAGCATCTAAGAGATATAACATTGATAGTTATATCTTACTAGACCTTGATAAGGTATTAGATGATGCCTACAGCTCTTTTCGACCGGTTTCTGCTGATTACAATACTAGGAAGGAATTGGTCAAAAACCTCAATGCTATGGCTATAGATATTTTTGGTAAATCAGAAGAGAGCAGCCCGGTCCTTGAGGCGTATGGCTCTTTTGCGATGAATACGTTTTCCTCACAAAAGGATTTGGATGTATCTATTAACTTTAGTAGCGGAACATCTGAATTTTATCGTGAAAAGAAGCTAGAAATTCTTACAAGATTTGCAACGAAGCTCCGCTCTTTGGAGGGGCAAGGGTTTGTCAGGAATGTTGTACCTATCTTAAGTGCTAGGGTTCCAATTGTCAGATTCTGTGATCAAGGGACTGGTATCGAATGTGATTTGACAGTTGAAAGCAAAGACGGCATTTTAACTTCACAGATCATTCGGATTATATCTCAAATTGATGATAGGTTTCAGAAACTTTGTTTGTTGCACTGTCAATTGTTTAGTACTAATGTGAATATTGTCATATGTCAGATTAAGCATTGGGCCAGAGCACATGGGGTTAACAATGCGTCACACAATACCCTGAATTCGATATCTATAACAATGCTAGTCGCTCATCATTTACAGACTCAAAGTCCTCCAATTTTACCTCCATTCTCTACACTATTTAAAGATGGAATCGATCCTCCTATCGTGGAGAAAAGAACACAAAAGTTCTTAAACTGGGGACAACGTAATCAAGAATCTCTTGGTAGACTGTTTGCAACTTTTTTTATTAAGGTTGAGGATTTTACAGACGTGGCTAGAAATTTTGCAAGGGTTTTTAGTGACTACGGAGCCGAAAAGATATACAGCTCCATAAACAGAACAGTAGATGATATATTGGAGTTCCTCAATGGCAAGGTTGCTGGAGAACATCTCACACAAAAGTTGTTTAGCCAACAAACTGTTGTGGAGCCTATTCCTCCTGTGTCGCCACAACAACCTCATAAGAAAAGAGTCTGCTTGGAAAAGGGATACAGAGCTGTAGGAGGGATCAGTCACGGAAGAGAGGAGACTCATGAAACTTCTAGAGGAAAACGAAAAAGGTCTGGTGGAAACTGGGGCTGTAGGTTTAGTGGTGGTGAAGAACCAATGCCTGTGGGTCTGTGGAATGAGTACAGTCGAAGTCTTGATATGCCAACACCACCACCACATTATGATAAACTTCTTTATAAGAAGTATTAG</li>
</ul>

      </div>
  </div>
</div>








  



<div class="accordion ">
  <div class="accordion-header " data-accordion>
    Protein Sequence
    <svg
      class="accordion-icon"
      x="0px"
      y="0px"
      viewBox="0 0 512 512"
      xmlspace="preserve">
      <path
        fill="currentColor"
        d="M505.755,123.592c-8.341-8.341-21.824-8.341-30.165,0L256.005,343.176L36.421,123.592c-8.341-8.341-21.824-8.341-30.165,0 s-8.341,21.824,0,30.165l234.667,234.667c4.16,4.16,9.621,6.251,15.083,6.251c5.462,0,10.923-2.091,15.083-6.251l234.667-234.667 C514.096,145.416,514.096,131.933,505.755,123.592z"></path>
    </svg>
</div>


  <div class="accordion-content ">
    <div class="content-wrapper">
    <button class="copy-btn" aria-label="Copy Content">Copy</button>
    </div>
  
      <div class="text-to-copy">
    <ul>
<li>&gt;AT3G45760.2
MDFRKNQAGEKNVSSKGIQKKVKNTMSIASKRYNIDSYILLDLDKVLDDAYSSFRPVSADYNTRKELVKNLNAMAIDIFGKSEESSPVLEAYGSFAMNTFSSQKDLDVSINFSSGTSEFYREKKLEILTRFATKLRSLEGQGFVRNVVPILSARVPIVRFCDQGTGIECDLTVESKDGILTSQIIRIISQIDDRFQKLCLLHCQLFSTNVNIVICQIKHWARAHGVNNASHNTLNSISITMLVAHHLQTQSPPILPPFSTLFKDGIDPPIVEKRTQKFLNWGQRNQESLGRLFATFFIKVEDFTDVARNFARVFSDYGAEKIYSSINRTVDDILEFLNGKVAGEHLTQKLFSQQTVVEPIPPVSPQQPHKKRVCLEKGYRAVGGISHGREETHETSRGKRKRSGGNWGCRFSGGEEPMPVGLWNEYSRSLDMPTPPPHYDKLLYKKY</li>
<li>&gt;AT3G45760.1
MDFRKNQAGEKNVSSKGIQKKVKNTMSIASKRYNIDSYILLDLDKVLDDAYSSFRPVSADYNTRKELVKNLNAMAIDIFGKSEESSPVLEAYGSFAMNTFSSQKDLDVSINFSSGTSEFYREKKLEILTRFATKLRSLEGQGFVRNVVPILSARVPIVRFCDQGTGIECDLTVESKDGILTSQIIRIISQIDDRFQKLCLLHCQLFSTNVNIVICQIKHWARAHGVNNASHNTLNSISITMLVAHHLQTQSPPILPPFSTLFKDGIDPPIVEKRTQKFLNWGQRNQESLGRLFATFFIKVEDFTDVARNFARVFSDYGAEKIYSSINRTVDDILEFLNGKVAGEHLTQKLFSQQTVVEPIPPVSPQQPHKKRVCLEKGYRAVGGISHGREETHETSRGKRKRSGGNWGCRFSGGEEPMPVGLWNEYSRSLDMPTPPPHYDKLLYKKY</li>
</ul>

      </div>
  </div>
</div>

</p>

          </div>
        </div>
      </div>
    </div>
  </section>

    </main>
  
    
    
      <footer class="bg-theme-light dark:bg-darkmode-theme-light">
  <div class="container">
    <div class="row items-center py-0">
      <div class="lg:col-6 mb-8 text-center lg:mb-0">
        <ul>
          
        </ul>
      </div>
      <div class="lg:col-3 mb-8 text-center lg:mb-0 lg:mt-0 lg:text-right">
        <ul class="social-icons">
          
        </ul>
      </div>
    </div>
  </div>
  <div class="border-border dark:border-darkmode-border border-t py-7">
    <div class="text-light dark:text-darkmode-light container text-center">
      <p>
        Designed &amp; Developed by <a href="https://gohugo.io/"




 target="_blank"
 


>Hugo</a>
 and <a href="https://www.netlify.com/"




 target="_blank"
 


>netlify</a>
 | 2024 Jinbiao Cao (曹金彪)
      </p>
    </div>
  </div>
</footer>
      



  
    
      
    
  

  
    
      
    
  

  
    
      
    
  

  
    
      
    
  

  
    
      
    
  

  
    
      
    
  

  
    
      
    
  

  
    
      
    
  

  
    
      
    
  










  
  



<script
  crossorigin="anonymous"
  integrity="sha256-qB/nHiPX28YVBLSaTV&#43;c5NTxlhkrSM0wXi7NrqdMnrw="
  src="/js/script.min.a81fe71e23d7dbc61504b49a4d5f9ce4d4f196192b48cd305e2ecdaea74c9ebc.js"></script>


<script
  defer
  async
  crossorigin="anonymous"
  integrity="sha256-9e9iyznGCSfFZB4ElW&#43;WFonrJpHJr5pYxsaD7VVh4wc="
  src="/js/script-lazy.min.f5ef62cb39c60927c5641e04956f961689eb2691c9af9a58c6c683ed5561e307.js"></script>



<script>
  if ('serviceWorker' in navigator){navigator.serviceWorker.register("/service-worker.js");}
</script>






  








    
  </body>
  
</html>
